{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# SARSCov2 NSP4 Structure Prediction\n",
    "\n",
    "We will perform comparative homology modelling via the satisfaction of spatial restraints using the `modeller` package."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false,
    "jupyter": {
     "outputs_hidden": false
    }
   },
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "655d671cfd5a488091623461aa729741",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": []
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "                         MODELLER 9.25, 2020/09/03, r11894\n",
      "\n",
      "     PROTEIN STRUCTURE MODELLING BY SATISFACTION OF SPATIAL RESTRAINTS\n",
      "\n",
      "\n",
      "                     Copyright(c) 1989-2020 Andrej Sali\n",
      "                            All Rights Reserved\n",
      "\n",
      "                             Written by A. Sali\n",
      "                               with help from\n",
      "              B. Webb, M.S. Madhusudhan, M-Y. Shen, G.Q. Dong,\n",
      "          M.A. Marti-Renom, N. Eswar, F. Alber, M. Topf, B. Oliva,\n",
      "             A. Fiser, R. Sanchez, B. Yerkovich, A. Badretdinov,\n",
      "                     F. Melo, J.P. Overington, E. Feyfant\n",
      "                 University of California, San Francisco, USA\n",
      "                    Rockefeller University, New York, USA\n",
      "                      Harvard University, Cambridge, USA\n",
      "                   Imperial Cancer Research Fund, London, UK\n",
      "              Birkbeck College, University of London, London, UK\n",
      "\n",
      "\n",
      "Kind, OS, HostName, Kernel, Processor: 4, Darwin Skynet.local 19.6.0 x86_64\n",
      "Date and time of compilation         : 2020/09/03 08:25:47\n",
      "MODELLER executable type             : mac10v4-intel64\n",
      "Job starting time (YY/MM/DD HH:MM:SS): 2020/12/21 14:58:34\n",
      "\n"
     ]
    }
   ],
   "source": [
    "from modeller import *\n",
    "from Bio.PDB import *\n",
    "import numpy as np\n",
    "import homology_modelling as hg\n",
    "import seaborn as sns; sns.set()\n",
    "import nglview as ngv\n",
    "\n",
    "env = environ()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### NSP4 amino acid sequence\n",
    "Observe the amino acid sequence of the region of the NSP4 protein specified in the `.ali` file."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false,
    "jupyter": {
     "outputs_hidden": false
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">P1;NSP4\n",
      "sequence:NSP4:::::::0.00: 0.00\n",
      "ITREIGFIVPGLPGTVLRAINGDFLHFLPRVFSAVGNICYTPSKLIEYSDFATSACVLAAECTIFKDAMGKPVPY\n",
      "CYDTNLLEGSISYSELRPDTRYVLMDGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCERSEVGICLSTSGRWVLN\n",
      "NEHYR*\n"
     ]
    }
   ],
   "source": [
    "!cat NSP4.ali"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1. PSI-BLAST\n",
    "Read the `.ali` file and modify the string into the required NCBI format `.faa` for PSI-BLAST."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "hg.ali_to_faa(\"NSP4.ali\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Call `psiblast` from the command line. Run PSI-BLAST over the `pdbaa` database which contains non-redundant sequences of all proteins with structures in the Protein Data Bank.\n",
    "\n",
    "*Note*: In this case we will run PSI-BLAST to convergence. To speed up the process we can run it in the local directory."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PSIBLAST 2.11.0+\n",
      "\n",
      "\n",
      "Reference: Stephen F. Altschul, Thomas L. Madden, Alejandro A.\n",
      "Schaffer, Jinghui Zhang, Zheng Zhang, Webb Miller, and David J.\n",
      "Lipman (1997), \"Gapped BLAST and PSI-BLAST: a new generation of\n",
      "protein database search programs\", Nucleic Acids Res. 25:3389-3402.\n",
      "\n",
      "\n",
      "Reference for compositional score matrix adjustment: Stephen F.\n",
      "Altschul, John C. Wootton, E. Michael Gertz, Richa Agarwala,\n",
      "Aleksandr Morgulis, Alejandro A. Schaffer, and Yi-Kuo Yu (2005)\n",
      "\"Protein database searches using compositionally adjusted\n",
      "substitution matrices\", FEBS J. 272:5101-5109.\n",
      "\n",
      "\n",
      "Reference for composition-based statistics starting in round 2:\n",
      "Alejandro A. Schaffer, L. Aravind, Thomas L. Madden, Sergei\n",
      "Shavirin, John L. Spouge, Yuri I. Wolf, Eugene V. Koonin, and\n",
      "Stephen F. Altschul (2001), \"Improving the accuracy of PSI-BLAST\n",
      "protein database searches with composition-based statistics and\n",
      "other refinements\", Nucleic Acids Res. 29:2994-3005.\n",
      "\n",
      "\n",
      "\n",
      "Database: PDB protein database\n",
      "           120,434 sequences; 31,663,421 total letters\n",
      "\n",
      "Results from round 1\n",
      "\n",
      "\n",
      "Query= P1;NSP4\n",
      "\n",
      "Length=156\n",
      "                                                                      Score     E\n",
      "Sequences producing significant alignments:                          (Bits)  Value\n",
      "\n",
      "1T70_A Crystal structure of a novel phosphatase from Deinococcus ...  42.0    1e-04\n",
      "\n",
      "\n",
      ">1T70_A Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_B Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_C Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_D Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_E Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_F Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_G Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_H Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      "Length=255\n",
      "\n",
      " Score = 42.0 bits (97),  Expect = 1e-04, Method: Compositional matrix adjust.\n",
      " Identities = 54/177 (31%), Positives = 74/177 (42%), Gaps = 31/177 (18%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTVLRA--INGDFL---HFLPRVF-SAVGNICYTPSKLIEYSDFATS  54\n",
      "            I R + +  PG PG   R   +NG+ L   + L RVF  AV N   T   L+E  D  T \n",
      "Sbjct  84   IVRPLNYADPGTPGVGWRTFDVNGEKLTVVNLLGRVFMEAVDNPFRTMDALLERDDLGTV  143\n",
      "\n",
      "Query  55   ACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYVLM  100\n",
      "                 AE T  K+AMG                VP   DT +L+G  +Y   + D  +   \n",
      "Sbjct  144  FVDFHAEATSEKEAMGWHLAGRVAAVIGTHTHVPTA-DTRILKGGTAY---QTDAGFTGP  199\n",
      "\n",
      "Query  101  DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCE-RSEV-GICLSTSGRWVLNNEHYR  155\n",
      "              SII    + +EG ++   T      R+G  E R+E+ G+ L   G      E YR\n",
      "Sbjct  200  HDSII---GSAIEGPLQRFLT--ERPHRYGVAEGRAELNGVALHFEGGKATAAERYR  251\n",
      "\n",
      "\n",
      "\n",
      "Lambda      K        H        a         alpha\n",
      "   0.327    0.143    0.454    0.792     4.96 \n",
      "\n",
      "Gapped\n",
      "Lambda      K        H        a         alpha    sigma\n",
      "   0.267   0.0410    0.140     1.90     42.6     43.6 \n",
      "\n",
      "Effective search space used: 1206105420\n",
      "Warning: [psiblast] Query_1 P1;NSP4: Composition-based score adjustment conditioned on sequence properties and unconditional composition-based score adjustment is not supported with PSSMs, resetting to default value of standard composition-based statistics \n",
      "Results from round 2\n",
      "\n",
      "\n",
      "Query= P1;NSP4\n",
      "\n",
      "Length=156\n",
      "                                                                      Score     E\n",
      "Sequences producing significant alignments:                          (Bits)  Value\n",
      "Sequences used in model and found again:\n",
      "\n",
      "1T70_A Crystal structure of a novel phosphatase from Deinococcus ...  194     6e-63\n",
      "\n",
      "Sequences not found previously or not previously below threshold:\n",
      "\n",
      "2Z06_A Crystal structure of uncharacterized conserved protein fro...  74.7    2e-16\n",
      "2CV9_A Crystal structure of a hypothetical protein from Thermus t...  73.1    7e-16\n",
      "4B2O_A Crystal structure of Bacillus subtilis YmdB, a global regu...  59.3    1e-10\n",
      "\n",
      "\n",
      ">1T70_A Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_B Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_C Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_D Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_E Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_F Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_G Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_H Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      "Length=255\n",
      "\n",
      " Score = 194 bits (494),  Expect = 6e-63, Method: Composition-based stats.\n",
      " Identities = 54/177 (31%), Positives = 74/177 (42%), Gaps = 31/177 (18%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTVLRA--INGDFL---HFLPRVF-SAVGNICYTPSKLIEYSDFATS  54\n",
      "            I R + +  PG PG   R   +NG+ L   + L RVF  AV N   T   L+E  D  T \n",
      "Sbjct  84   IVRPLNYADPGTPGVGWRTFDVNGEKLTVVNLLGRVFMEAVDNPFRTMDALLERDDLGTV  143\n",
      "\n",
      "Query  55   ACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYVLM  100\n",
      "                 AE T  K+AMG                VP   DT +L+G  +Y   + D  +   \n",
      "Sbjct  144  FVDFHAEATSEKEAMGWHLAGRVAAVIGTHTHVPTA-DTRILKGGTAY---QTDAGFTGP  199\n",
      "\n",
      "Query  101  DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCE-RSEV-GICLSTSGRWVLNNEHYR  155\n",
      "              SII    + +EG ++   T      R+G  E R+E+ G+ L   G      E YR\n",
      "Sbjct  200  HDSII---GSAIEGPLQRFLT--ERPHRYGVAEGRAELNGVALHFEGGKATAAERYR  251\n",
      "\n",
      "\n",
      ">2Z06_A Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      " 2Z06_B Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      " 2Z06_C Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      " 2Z06_D Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      "Length=252\n",
      "\n",
      " Score = 74.7 bits (182),  Expect = 2e-16, Method: Composition-based stats.\n",
      " Identities = 32/174 (18%), Positives = 55/174 (32%), Gaps = 28/174 (16%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTV-LR-AINGD---FLHFLPRVF-SAVGNICYTPSKLIEYSDFATS  54\n",
      "            + R + +  PG PG    R  + G+   F+  + R+F   + +      +L+E       \n",
      "Sbjct  82   VVRPLNYP-PGTPGKGFWRLEVGGESLLFVQVMGRIFMDPLDDPFRALDRLLEEEKADYV  140\n",
      "\n",
      "Query  55   ACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYVLM  100\n",
      "               + AE T  K A+                 VP    T L +G++     + D      \n",
      "Sbjct  141  LVEVHAEATSEKMALAHYLDGRASAVLGTHTHVPTLDATRLPKGTL----YQTDVGMTGT  196\n",
      "\n",
      "Query  101  DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCERSEVGICLSTSGRWVLNNEHY  154\n",
      "              SII      +E  +    T   +  R    +       L   G   +    Y\n",
      "Sbjct  197  YHSII---GGEVETFLARFLTGRPQPFRAAQGKARFHATELVFEGGRPVAISPY  247\n",
      "\n",
      "\n",
      ">2CV9_A Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      " 2CV9_B Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      " 2CV9_C Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      " 2CV9_D Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      "Length=252\n",
      "\n",
      " Score = 73.1 bits (178),  Expect = 7e-16, Method: Composition-based stats.\n",
      " Identities = 33/174 (19%), Positives = 55/174 (32%), Gaps = 28/174 (16%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTV-LR-AINGDFLHFL---PRVF-SAVGNICYTPSKLIEYSDFATS  54\n",
      "            + R + +  PG PG    R  + G+ L F+    R+F   + +      +L+E       \n",
      "Sbjct  82   VVRPLNYP-PGTPGKGFWRLEVGGESLLFVQVXGRIFXDPLDDPFRALDRLLEEEKADYV  140\n",
      "\n",
      "Query  55   ACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYVLM  100\n",
      "               + AE T  K A+                 VP    T L +G++     + D      \n",
      "Sbjct  141  LVEVHAEATSEKXALAHYLDGRASAVLGTHTHVPTLDATRLPKGTL----YQTDVGXTGT  196\n",
      "\n",
      "Query  101  DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCERSEVGICLSTSGRWVLNNEHY  154\n",
      "              SII      +E  +    T   +  R    +       L   G   +    Y\n",
      "Sbjct  197  YHSII---GGEVETFLARFLTGRPQPFRAAQGKARFHATELVFEGGRPVAISPY  247\n",
      "\n",
      "\n",
      ">4B2O_A Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      " 4B2O_B Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      " 4B2O_C Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      " 4B2O_D Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      "Length=286\n",
      "\n",
      " Score = 59.3 bits (142),  Expect = 1e-10, Method: Composition-based stats.\n",
      " Identities = 28/155 (18%), Positives = 43/155 (28%), Gaps = 32/155 (21%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTVLRAI--NGDFL---HFLPRVF-SAVGNICYTPSKLIEYSDFAT-  53\n",
      "            + R   F   G PG  +  +  NG  L   +   R F   + +      +LI  +   T \n",
      "Sbjct  85   LVRPANFPE-GTPGKGITYVKANGKELAVINLQGRTFLPPLDDPFLKADELIAEAAKRTP  143\n",
      "\n",
      "Query  54   -SACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYV  98\n",
      "                   AE T  K A+G                V    D  +L    +Y     D    \n",
      "Sbjct  144  YIFIDFHAEATSEKLALGWYTDGRASAVVGTHTHVQTA-DNRILPKGTAYI---TDVGMT  199\n",
      "\n",
      "Query  99   LMDGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCE  133\n",
      "                 I+      +   ++   T      R    E\n",
      "Sbjct  200  GPYDGILGMDRETI---IKRFKTNL--PVRFTVAE  229\n",
      "\n",
      "\n",
      "\n",
      "Lambda      K        H        a         alpha\n",
      "   0.318    0.131    0.357    0.792     4.96 \n",
      "\n",
      "Gapped\n",
      "Lambda      K        H        a         alpha    sigma\n",
      "   0.267   0.0401    0.140     1.90     42.6     43.6 \n",
      "\n",
      "Effective search space used: 1206105420\n",
      "Results from round 3\n",
      "\n",
      "\n",
      "Query= P1;NSP4\n",
      "\n",
      "Length=156\n",
      "                                                                      Score     E\n",
      "Sequences producing significant alignments:                          (Bits)  Value\n",
      "Sequences used in model and found again:\n",
      "\n",
      "1T70_A Crystal structure of a novel phosphatase from Deinococcus ...  175     2e-55\n",
      "2Z06_A Crystal structure of uncharacterized conserved protein fro...  171     6e-54\n",
      "2CV9_A Crystal structure of a hypothetical protein from Thermus t...  165     2e-51\n",
      "4B2O_A Crystal structure of Bacillus subtilis YmdB, a global regu...  132     3e-38\n",
      "\n",
      "Sequences not found previously or not previously below threshold:\n",
      "\n",
      "1T71_A Crystal structure of a novel phosphatase Mycoplasma pneumo...  57.4    5e-10\n",
      "\n",
      "\n",
      ">1T70_A Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_B Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_C Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_D Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_E Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_F Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_G Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_H Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      "Length=255\n",
      "\n",
      " Score = 175 bits (444),  Expect = 2e-55, Method: Composition-based stats.\n",
      " Identities = 54/177 (31%), Positives = 74/177 (42%), Gaps = 31/177 (18%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTVLRA--INGDFL---HFLPRVF-SAVGNICYTPSKLIEYSDFATS  54\n",
      "            I R + +  PG PG   R   +NG+ L   + L RVF  AV N   T   L+E  D  T \n",
      "Sbjct  84   IVRPLNYADPGTPGVGWRTFDVNGEKLTVVNLLGRVFMEAVDNPFRTMDALLERDDLGTV  143\n",
      "\n",
      "Query  55   ACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYVLM  100\n",
      "                 AE T  K+AMG                VP   DT +L+G  +Y   + D  +   \n",
      "Sbjct  144  FVDFHAEATSEKEAMGWHLAGRVAAVIGTHTHVPTA-DTRILKGGTAY---QTDAGFTGP  199\n",
      "\n",
      "Query  101  DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCE-RSEV-GICLSTSGRWVLNNEHYR  155\n",
      "              SII    + +EG ++   T      R+G  E R+E+ G+ L   G      E YR\n",
      "Sbjct  200  HDSII---GSAIEGPLQRFLT--ERPHRYGVAEGRAELNGVALHFEGGKATAAERYR  251\n",
      "\n",
      "\n",
      ">2Z06_A Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      " 2Z06_B Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      " 2Z06_C Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      " 2Z06_D Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      "Length=252\n",
      "\n",
      " Score = 171 bits (434),  Expect = 6e-54, Method: Composition-based stats.\n",
      " Identities = 32/174 (18%), Positives = 55/174 (32%), Gaps = 28/174 (16%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTV-LR-AINGD---FLHFLPRVF-SAVGNICYTPSKLIEYSDFATS  54\n",
      "            + R + +  PG PG    R  + G+   F+  + R+F   + +      +L+E       \n",
      "Sbjct  82   VVRPLNYP-PGTPGKGFWRLEVGGESLLFVQVMGRIFMDPLDDPFRALDRLLEEEKADYV  140\n",
      "\n",
      "Query  55   ACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYVLM  100\n",
      "               + AE T  K A+                 VP    T L +G++     + D      \n",
      "Sbjct  141  LVEVHAEATSEKMALAHYLDGRASAVLGTHTHVPTLDATRLPKGTL----YQTDVGMTGT  196\n",
      "\n",
      "Query  101  DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCERSEVGICLSTSGRWVLNNEHY  154\n",
      "              SII      +E  +    T   +  R    +       L   G   +    Y\n",
      "Sbjct  197  YHSII---GGEVETFLARFLTGRPQPFRAAQGKARFHATELVFEGGRPVAISPY  247\n",
      "\n",
      "\n",
      ">2CV9_A Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      " 2CV9_B Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      " 2CV9_C Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      " 2CV9_D Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      "Length=252\n",
      "\n",
      " Score = 165 bits (417),  Expect = 2e-51, Method: Composition-based stats.\n",
      " Identities = 32/174 (18%), Positives = 54/174 (31%), Gaps = 28/174 (16%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTV-LR-AINGD---FLHFLPRVF-SAVGNICYTPSKLIEYSDFATS  54\n",
      "            + R + +  PG PG    R  + G+   F+    R+F   + +      +L+E       \n",
      "Sbjct  82   VVRPLNYP-PGTPGKGFWRLEVGGESLLFVQVXGRIFXDPLDDPFRALDRLLEEEKADYV  140\n",
      "\n",
      "Query  55   ACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYVLM  100\n",
      "               + AE T  K A+                 VP    T L +G++     + D      \n",
      "Sbjct  141  LVEVHAEATSEKXALAHYLDGRASAVLGTHTHVPTLDATRLPKGTL----YQTDVGXTGT  196\n",
      "\n",
      "Query  101  DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCERSEVGICLSTSGRWVLNNEHY  154\n",
      "              SII      +E  +    T   +  R    +       L   G   +    Y\n",
      "Sbjct  197  YHSII---GGEVETFLARFLTGRPQPFRAAQGKARFHATELVFEGGRPVAISPY  247\n",
      "\n",
      "\n",
      ">4B2O_A Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      " 4B2O_B Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      " 4B2O_C Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      " 4B2O_D Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      "Length=286\n",
      "\n",
      " Score = 132 bits (332),  Expect = 3e-38, Method: Composition-based stats.\n",
      " Identities = 28/155 (18%), Positives = 43/155 (28%), Gaps = 32/155 (21%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTVLRAI--NGDFL---HFLPRVF-SAVGNICYTPSKLIEYSDFAT-  53\n",
      "            + R   F   G PG  +  +  NG  L   +   R F   + +      +LI  +   T \n",
      "Sbjct  85   LVRPANFPE-GTPGKGITYVKANGKELAVINLQGRTFLPPLDDPFLKADELIAEAAKRTP  143\n",
      "\n",
      "Query  54   -SACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYV  98\n",
      "                   AE T  K A+G                V    D  +L    +Y     D    \n",
      "Sbjct  144  YIFIDFHAEATSEKLALGWYTDGRASAVVGTHTHVQTA-DNRILPKGTAYI---TDVGMT  199\n",
      "\n",
      "Query  99   LMDGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCE  133\n",
      "                 I+      +   ++   T      R    E\n",
      "Sbjct  200  GPYDGILGMDRETI---IKRFKTNL--PVRFTVAE  229\n",
      "\n",
      "\n",
      ">1T71_A Crystal structure of a novel phosphatase Mycoplasma pneumoniaefrom \n",
      "[Mycoplasma pneumoniae M129]\n",
      "Length=281\n",
      "\n",
      " Score = 57.4 bits (137),  Expect = 5e-10, Method: Composition-based stats.\n",
      " Identities = 23/118 (19%), Positives = 32/118 (27%), Gaps = 22/118 (19%)\n",
      "\n",
      "Query  37   NICYTPSKLIEYSDFATSACVLAAECTIFKDAMG--------------KPVPYCYDTNLL  82\n",
      "            N      +LI   D         AE T  K+A                  VP   D  + \n",
      "Sbjct  136  NPFKVLKELILKRDCDLHIVDFHAETTSEKNAFCMAFDGYVTTIFGTHTHVPSA-DLRIT  194\n",
      "\n",
      "Query  83   EGSISYSELRPDTRYVLM-DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCERSEVGI  139\n",
      "                +Y     D        GS+I       E S+R+      E+     C     G+\n",
      "Sbjct  195  PKGSAYI---TDVGMCGPGFGSVI---GANPEQSIRLFCAGSREHFEVSKCGAQLNGV  246\n",
      "\n",
      "\n",
      "\n",
      "Lambda      K        H        a         alpha\n",
      "   0.317    0.129    0.330    0.792     4.96 \n",
      "\n",
      "Gapped\n",
      "Lambda      K        H        a         alpha    sigma\n",
      "   0.267   0.0395    0.140     1.90     42.6     43.6 \n",
      "\n",
      "Effective search space used: 1206105420\n",
      "Results from round 4\n",
      "\n",
      "\n",
      "Query= P1;NSP4\n",
      "\n",
      "Length=156\n",
      "                                                                      Score     E\n",
      "Sequences producing significant alignments:                          (Bits)  Value\n",
      "Sequences used in model and found again:\n",
      "\n",
      "1T70_A Crystal structure of a novel phosphatase from Deinococcus ...  174     8e-55\n",
      "2Z06_A Crystal structure of uncharacterized conserved protein fro...  168     1e-52\n",
      "2CV9_A Crystal structure of a hypothetical protein from Thermus t...  161     5e-50\n",
      "4B2O_A Crystal structure of Bacillus subtilis YmdB, a global regu...  131     8e-38\n",
      "1T71_A Crystal structure of a novel phosphatase Mycoplasma pneumo...  113     7e-31\n",
      "\n",
      "Sequences not found previously or not previously below threshold:\n",
      "\n",
      "\n",
      "\n",
      ">1T70_A Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_B Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_C Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_D Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_E Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_F Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_G Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_H Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      "Length=255\n",
      "\n",
      " Score = 174 bits (440),  Expect = 8e-55, Method: Composition-based stats.\n",
      " Identities = 52/177 (29%), Positives = 70/177 (40%), Gaps = 31/177 (18%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTVLRA--INGDFL---HFLPRVF-SAVGNICYTPSKLIEYSDFATS  54\n",
      "            I R + +  PG PG   R   +NG+ L   + L RVF  AV N   T   L+E  D  T \n",
      "Sbjct  84   IVRPLNYADPGTPGVGWRTFDVNGEKLTVVNLLGRVFMEAVDNPFRTMDALLERDDLGTV  143\n",
      "\n",
      "Query  55   ACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYVLM  100\n",
      "                 AE T  K+AMG                VP   DT +L+G  +Y   + D  +   \n",
      "Sbjct  144  FVDFHAEATSEKEAMGWHLAGRVAAVIGTHTHVPTA-DTRILKGGTAY---QTDAGFTGP  199\n",
      "\n",
      "Query  101  DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCE--RSEVGICLSTSGRWVLNNEHYR  155\n",
      "              SII    + +EG ++   T      R+G  E      G+ L   G      E YR\n",
      "Sbjct  200  HDSII---GSAIEGPLQRFLT--ERPHRYGVAEGRAELNGVALHFEGGKATAAERYR  251\n",
      "\n",
      "\n",
      ">2Z06_A Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      " 2Z06_B Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      " 2Z06_C Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      " 2Z06_D Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      "Length=252\n",
      "\n",
      " Score = 168 bits (425),  Expect = 1e-52, Method: Composition-based stats.\n",
      " Identities = 32/174 (18%), Positives = 55/174 (32%), Gaps = 28/174 (16%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTV-LR-AINGD---FLHFLPRVF-SAVGNICYTPSKLIEYSDFATS  54\n",
      "            + R + +  PG PG    R  + G+   F+  + R+F   + +      +L+E       \n",
      "Sbjct  82   VVRPLNYP-PGTPGKGFWRLEVGGESLLFVQVMGRIFMDPLDDPFRALDRLLEEEKADYV  140\n",
      "\n",
      "Query  55   ACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYVLM  100\n",
      "               + AE T  K A+                 VP    T L +G++     + D      \n",
      "Sbjct  141  LVEVHAEATSEKMALAHYLDGRASAVLGTHTHVPTLDATRLPKGTL----YQTDVGMTGT  196\n",
      "\n",
      "Query  101  DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCERSEVGICLSTSGRWVLNNEHY  154\n",
      "              SII      +E  +    T   +  R    +       L   G   +    Y\n",
      "Sbjct  197  YHSII---GGEVETFLARFLTGRPQPFRAAQGKARFHATELVFEGGRPVAISPY  247\n",
      "\n",
      "\n",
      ">2CV9_A Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      " 2CV9_B Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      " 2CV9_C Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      " 2CV9_D Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      "Length=252\n",
      "\n",
      " Score = 161 bits (408),  Expect = 5e-50, Method: Composition-based stats.\n",
      " Identities = 32/174 (18%), Positives = 54/174 (31%), Gaps = 28/174 (16%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTV-LR-AINGD---FLHFLPRVF-SAVGNICYTPSKLIEYSDFATS  54\n",
      "            + R + +  PG PG    R  + G+   F+    R+F   + +      +L+E       \n",
      "Sbjct  82   VVRPLNYP-PGTPGKGFWRLEVGGESLLFVQVXGRIFXDPLDDPFRALDRLLEEEKADYV  140\n",
      "\n",
      "Query  55   ACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYVLM  100\n",
      "               + AE T  K A+                 VP    T L +G++     + D      \n",
      "Sbjct  141  LVEVHAEATSEKXALAHYLDGRASAVLGTHTHVPTLDATRLPKGTL----YQTDVGXTGT  196\n",
      "\n",
      "Query  101  DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCERSEVGICLSTSGRWVLNNEHY  154\n",
      "              SII      +E  +    T   +  R    +       L   G   +    Y\n",
      "Sbjct  197  YHSII---GGEVETFLARFLTGRPQPFRAAQGKARFHATELVFEGGRPVAISPY  247\n",
      "\n",
      "\n",
      ">4B2O_A Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      " 4B2O_B Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      " 4B2O_C Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      " 4B2O_D Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      "Length=286\n",
      "\n",
      " Score = 131 bits (329),  Expect = 8e-38, Method: Composition-based stats.\n",
      " Identities = 28/177 (16%), Positives = 49/177 (28%), Gaps = 32/177 (18%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTVLRAI--NGDFL---HFLPRVF-SAVGNICYTPSKLIEYSDFAT-  53\n",
      "            + R   F   G PG  +  +  NG  L   +   R F   + +      +LI  +   T \n",
      "Sbjct  85   LVRPANFPE-GTPGKGITYVKANGKELAVINLQGRTFLPPLDDPFLKADELIAEAAKRTP  143\n",
      "\n",
      "Query  54   -SACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYV  98\n",
      "                   AE T  K A+G                V    D  +L    +Y     D    \n",
      "Sbjct  144  YIFIDFHAEATSEKLALGWYTDGRASAVVGTHTHVQTA-DNRILPKGTAYI---TDVGMT  199\n",
      "\n",
      "Query  99   LMDGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCERSEVGICLST--SGRWVLNNEH  153\n",
      "                 I+      +   ++   T           + +  G+ +      +  +  E \n",
      "Sbjct  200  GPYDGILGMDRETI---IKRFKTNLPVRFTVAEGKTTLSGVVIDIDDQTKKAVKIER  253\n",
      "\n",
      "\n",
      ">1T71_A Crystal structure of a novel phosphatase Mycoplasma pneumoniaefrom \n",
      "[Mycoplasma pneumoniae M129]\n",
      "Length=281\n",
      "\n",
      " Score = 113 bits (282),  Expect = 7e-31, Method: Composition-based stats.\n",
      " Identities = 23/118 (19%), Positives = 32/118 (27%), Gaps = 22/118 (19%)\n",
      "\n",
      "Query  37   NICYTPSKLIEYSDFATSACVLAAECTIFKDAMG--------------KPVPYCYDTNLL  82\n",
      "            N      +LI   D         AE T  K+A                  VP   D  + \n",
      "Sbjct  136  NPFKVLKELILKRDCDLHIVDFHAETTSEKNAFCMAFDGYVTTIFGTHTHVPSA-DLRIT  194\n",
      "\n",
      "Query  83   EGSISYSELRPDTRYVLM-DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCERSEVGI  139\n",
      "                +Y     D        GS+I       E S+R+      E+     C     G+\n",
      "Sbjct  195  PKGSAYI---TDVGMCGPGFGSVI---GANPEQSIRLFCAGSREHFEVSKCGAQLNGV  246\n",
      "\n",
      "\n",
      "\n",
      "Lambda      K        H        a         alpha\n",
      "   0.318    0.130    0.335    0.792     4.96 \n",
      "\n",
      "Gapped\n",
      "Lambda      K        H        a         alpha    sigma\n",
      "   0.267   0.0398    0.140     1.90     42.6     43.6 \n",
      "\n",
      "Effective search space used: 1206105420\n",
      "\n",
      "Search has CONVERGED!\n",
      "\n",
      "\n",
      "  Database: PDB protein database\n",
      "    Posted date:  Dec 10, 2020  3:46 AM\n",
      "  Number of letters in database: 31,663,421\n",
      "  Number of sequences in database:  120,434\n",
      "\n",
      "\n",
      "\n",
      "Matrix: BLOSUM62\n",
      "Gap Penalties: Existence: 11, Extension: 1\n",
      "Neighboring words threshold: 11\n",
      "Window for multiple hits: 40\n"
     ]
    }
   ],
   "source": [
    "#!update_blastdb.pl --decompress pdbaa #to download the pdbaa database locally\n",
    "!psiblast -db pdbaa -query \"NSP4.faa\" -inclusion_ethresh 1e-3 -evalue 1e-3 -num_iterations 0"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Template Selection\n",
    "\n",
    "Protein **1T70 (chain 'A')** which is a ***Deinococcus* phosphatase** is the hit with lowest **E-value =** $8 \\times 10^{-55}$  and the highest **sequence identity =** ~$30\\%$ (in round 4). The rest of the hits have sequence identity $<20\\%$. \n",
    "We will use 1T70A as a template for our model."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2. Refining Target-Template Alignment\n",
    "Get the `.pdb` file for the protein 1T70."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false,
    "jupyter": {
     "outputs_hidden": false
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Structure exists: '/Users/shivamchitnis/Documents/GitHub/Modeller/t7/pdb1t70.ent' \n"
     ]
    }
   ],
   "source": [
    "pdb_file = PDBList().retrieve_pdb_file('1T70',file_format='pdb')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "jupyter": {
     "outputs_hidden": false
    }
   },
   "source": [
    "Align the template and the target sequence precisely."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkapsa__637W> No residue topology library is in memory.\n",
      "              Better radii would be used if topology.read() is called first.\n",
      "iup2crm_280W> No topology library in memory or assigning a BLK residue.\n",
      "              Default CHARMM atom type assigned:  N -->  N\n",
      "              This message is written only for the first such atom.\n",
      "\n",
      "Pairwise dynamic programming alignment (ALIGN2D):\n",
      "  Residue-residue metric   : $(LIB)/as1.sim.mat\n",
      "  Diagonal                 :          100\n",
      "  Overhang                 :            0\n",
      "  Maximal gap length       :       999999\n",
      "  Local alignment          :            F\n",
      "  MATRIX_OFFSET (local aln):       0.0000\n",
      "  FIX_OFFSETS              :      0.0    -1.0    -2.0    -3.0    -4.0\n",
      "  N_SUBOPT                 :            0\n",
      "  SUBOPT_OFFSET            :       0.0000\n",
      "  Alignment block          :            1\n",
      "  Gap introduction penalty :    -100.0000\n",
      "  Gap extension penalty    :       0.0000\n",
      "  Gap diagonal penalty     :       0.0000\n",
      "  Structure gap penalties  :    3.500   3.500   3.500   0.200   4.000   6.500   2.000   0.000\n",
      "  Break-break bonus        :   10000.0000\n",
      "  Length of alignment      :          259\n",
      "  Score                    :   84183.5000\n"
     ]
    }
   ],
   "source": [
    "hg.precise_align(env,targ='NSP4',tmp='1t70',tmp_pdb=pdb_file,model_segment=('FIRST:A','LAST:A'))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Observe the alignment of the template 17T0A and the target NSP4."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": false,
    "jupyter": {
     "outputs_hidden": false
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " _aln.pos         10        20        30        40        50        60\n",
      "1t70A     MRVLFIGDVFGQPGRRVLQNHLPTIRPQFDFVIVNMENSAGGFGMHRDAARGALEAGAGCLTLGNHAW \n",
      "NSP4      -------------------------------------------------------------------- \n",
      " _consrvd\n",
      "\n",
      " _aln.p   70        80        90       100       110       120       130\n",
      "1t70A     HHKDIYPMLSEDTYPIVRPLNYADPGTPGVGWRTFDVNGEKLTVVNLLGRVFMEAVDNPFRTMDALLE \n",
      "NSP4      ---------------ITREIGFIVPGLPGTVLRAIN-----GDFLHFLPRVFS-AVGNICYTPSKLIE \n",
      " _consrvd                * *      ** **   *              * ***  ** *   *   * *\n",
      "\n",
      " _aln.pos  140       150       160       170       180       190       200\n",
      "1t70A     RDDLGTVFVDFHAEATSEKEAMGWHLAG--RVAAVIGTHTHVPT-ADTRILK-GGTAYQTDAGFTGPH \n",
      "NSP4      YSDFATSACVLAAECTIFKDAMGKPVPYCYDTNLLEGSISYSELRPDTRYVLMDGSIIQFPNTYLE-- \n",
      " _consrvd   *  *      ** *  * ***             *         ***     *   *\n",
      "\n",
      " _aln.pos    210       220       230       240       250\n",
      "1t70A     DSIIGSAIEGPLQRFLTERPHRYGVAEGRAELNGVALHFEGGKATAAERYRFIED \n",
      "NSP4      ----GSV--RVVTTFDAE-YCRHGTCE-RSEV-GICLSTSGRWVLNNEHYR---- \n",
      " _consrvd     **        *  *   * *  * * *  *  *   *      * **\n"
     ]
    }
   ],
   "source": [
    "!cat NSP4-1t70A.pap"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3. Constructing Model \n",
    "### by Satisying Spatial Restraints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": false,
    "jupyter": {
     "outputs_hidden": false
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "The following 1 residues contain 6-membered rings with poor geometries\n",
      "after transfer from templates. Rebuilding rings from internal coordinates:\n",
      "   <Residue 88 (type TYR)>\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    12801    11716\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ARG:OXT  ARG\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      155\n",
      "Number of all, selected real atoms                :     1215    1215\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    11716   11716\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   192249\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -12802.9893\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -12802.989258\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : /usr/local/Cellar/modeller/9.25_1/modlib/surf5.de\n",
      "Pair library             : /usr/local/Cellar/modeller/9.25_1/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    30.323000\n",
      "Sequence length          :          155\n",
      "Compactness              :     0.254398\n",
      "Native energy (pair)     :    30.361987\n",
      "Native energy (surface)  :    13.738875\n",
      "Native energy (combined) :     4.539245\n",
      "Z score (pair)           :    -2.069588\n",
      "Z score (surface)        :     0.080161\n",
      "Z score (combined)       :    -1.389705\n",
      "GA341 score              :     0.099237\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      155\n",
      "Number of all, selected real atoms                :     1215    1215\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    11716   11716\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     2493\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1182.2100\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    1244       0      0   0.005   0.005      9.4306       1.000\n",
      " 2 Bond angle potential               :    1693       0      2   2.035   2.035      132.79       1.000\n",
      " 3 Stereochemical cosine torsion poten:     835       0     29  48.352  48.352      306.55       1.000\n",
      " 4 Stereochemical improper torsion pot:     532       0      0   1.257   1.257      17.867       1.000\n",
      " 5 Soft-sphere overlap restraints     :    2493       0      0   0.005   0.005      6.4783       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    2568       0      3   0.647   0.647      140.66       1.000\n",
      "10 Distance restraints 2 (N-O)        :    2785       0      0   0.698   0.698      168.65       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     154       0      0   4.102   4.102      30.560       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     135       0      2  87.438  87.438      49.628       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      92       0      1  85.198  85.198      45.026       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      34       0      0  61.536  61.536      24.906       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      13       0      0  96.272  96.272      8.7994       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    1284       0      0   0.691   0.691      37.931       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     153      24     15  37.287  67.819      179.62       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     194       0      1   1.391   1.391      23.305       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: NSP4.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   18545.1836\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4311   7F   8I C   N      57   59  -90.94 -120.60   67.73    2.89  -63.40  116.32   18.21\n",
      "    1          8I   8I N   CA     59   60   69.41  130.30                  -43.60\n",
      "    2   4314  10P  11G C   N      79   81  167.48   82.20   98.71    7.70  -62.40  163.75   23.96\n",
      "    2         11G  11G N   CA     81   82   58.21    8.50                  -41.20\n",
      "    3   4325  21N  22G C   N     156  158   96.30   78.70   43.66    0.66   82.20  146.12    7.80\n",
      "    3         22G  22G N   CA    158  159  153.94 -166.10                    8.50\n",
      "    4   4335  31V  32F C   N     241  243   58.77   58.10    3.52    0.23  -63.20  142.53   25.68\n",
      "    4         32F  32F N   CA    243  244   29.45   32.90                  -44.30\n",
      "    5   4337  33S  34A C   N     258  260 -143.13 -134.00   26.90    1.65  -68.20   78.57    7.36\n",
      "    5         34A  34A N   CA    260  261  121.69  147.00                  145.30\n",
      "    6   4353  49S  50D C   N     380  382 -100.81  -96.50   81.71    3.43  -63.30   81.72    8.67\n",
      "    6         50D  50D N   CA    382  383   32.61  114.20                  -40.00\n",
      "    7   4355  51F  52A C   N     399  401   70.14  -68.20  151.40   14.52  -62.50  173.79   27.39\n",
      "    7         52A  52A N   CA    401  402 -153.19  145.30                  -40.90\n",
      "    8   4378  74P  75Y C   N     558  560 -149.99 -124.30   37.50    1.62  -63.50  176.53   32.17\n",
      "    8         75Y  75Y N   CA    560  561  162.71  135.40                  -43.40\n",
      "    9   4379  75Y  76C C   N     570  572  -61.68  -63.00    1.53    0.20 -117.90 -174.25    8.00\n",
      "    9         76C  76C N   CA    572  573  -41.86  -41.10                  141.10\n",
      "   10   4380  76C  77Y C   N     576  578 -110.24  -98.40   41.41    4.19  -63.50  140.15   20.10\n",
      "   10         77Y  77Y N   CA    578  579   88.72  128.40                  -43.40\n",
      "   11   4381  77Y  78D C   N     588  590 -155.10  -63.30  103.09   17.60  -63.30  103.09   17.60\n",
      "   11         78D  78D N   CA    590  591  -86.91  -40.00                  -40.00\n",
      "   12   4389  85S  86I C   N     646  648  129.11  -63.40  167.50   29.89 -120.60 -154.84   15.53\n",
      "   12         86I  86I N   CA    648  649  -42.69  -43.60                  130.30\n",
      "   13   4391  87S  88Y C   N     660  662   61.54   55.90   26.66    1.48 -124.30 -147.38   17.59\n",
      "   13         88Y  88Y N   CA    662  663   13.44   39.50                  135.40\n",
      "   14   4403  99L 100M C   N     766  768   77.45  -63.40  141.13   25.57  -63.40  141.13   25.57\n",
      "   14        100M 100M N   CA    768  769  -49.40  -40.50                  -40.50\n",
      "   15   4404 100M 101D C   N     774  776  -60.97  -63.30    2.54    0.43   54.50  140.41   18.10\n",
      "   15        101D 101D N   CA    776  777  -38.99  -40.00                   40.90\n",
      "   16   4411 107F 108P C   N     828  830 -124.76  -64.50   60.86    6.50  -64.50   60.86    6.50\n",
      "   16        108P 108P N   CA    830  831  138.72  147.20                  147.20\n",
      "   17   4415 111Y 112L C   N     862  864  -71.98  -63.50   11.27    1.48 -108.50  170.24    8.77\n",
      "   17        112L 112L N   CA    864  865  -33.78  -41.20                  132.50\n",
      "   18   4416 112L 113E C   N     870  872  -57.56  -63.60   25.18    3.07 -117.80  169.52    9.25\n",
      "   18        113E 113E N   CA    872  873  -64.75  -40.30                  136.80\n",
      "   19   4418 114G 115S C   N     883  885  -92.18  -72.40   23.25    1.24  -64.10  162.82   13.66\n",
      "   19        115S 115S N   CA    885  886  164.62  152.40                  -35.00\n",
      "   20   4420 116V 117R C   N     896  898 -111.76 -125.20   24.01    0.81  -63.00  168.99   20.03\n",
      "   20        117R 117R N   CA    898  899  120.71  140.60                  -41.10\n",
      "   21   4421 117R 118V C   N     907  909  -79.92  -73.50   25.22    2.05  -62.40  155.00   20.67\n",
      "   21        118V 118V N   CA    909  910  163.59  139.20                  -42.40\n",
      "   22   4426 122F 123D C   N     946  948  -65.65  -96.50   79.76    3.31  -63.30  132.27   16.47\n",
      "   22        123D 123D N   CA    948  949 -172.24  114.20                  -40.00\n",
      "   23   4428 124A 125E C   N     959  961 -119.33 -117.80   18.44    0.90  -63.60  173.70   26.31\n",
      "   23        125E 125E N   CA    961  962  155.18  136.80                  -40.30\n",
      "   24   4441 137V 138G C   N    1066 1068  -69.12  -62.40    7.92    1.50 -167.20  170.95    6.78\n",
      "   24        138G 138G N   CA   1068 1069  -45.39  -41.20                  174.60\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    8   19   78   89  122  133  131  161  179  187\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ARG:OXT  ARG\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      155\n",
      "Number of all, selected real atoms                :     1215    1215\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    11716   11716\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   187888\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -12878.1895\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -12878.189453\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : /usr/local/Cellar/modeller/9.25_1/modlib/surf5.de\n",
      "Pair library             : /usr/local/Cellar/modeller/9.25_1/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    30.323000\n",
      "Sequence length          :          155\n",
      "Compactness              :     0.136425\n",
      "Native energy (pair)     :    34.800232\n",
      "Native energy (surface)  :    10.799020\n",
      "Native energy (combined) :     3.895143\n",
      "Z score (pair)           :    -1.381103\n",
      "Z score (surface)        :    -0.820283\n",
      "Z score (combined)       :    -1.367832\n",
      "GA341 score              :     0.077455\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      155\n",
      "Number of all, selected real atoms                :     1215    1215\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    11716   11716\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     2456\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1124.6552\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    1244       0      0   0.005   0.005      8.6199       1.000\n",
      " 2 Bond angle potential               :    1693       0      2   2.133   2.133      146.60       1.000\n",
      " 3 Stereochemical cosine torsion poten:     835       0     25  49.066  49.066      321.04       1.000\n",
      " 4 Stereochemical improper torsion pot:     532       0      0   1.317   1.317      19.398       1.000\n",
      " 5 Soft-sphere overlap restraints     :    2456       0      0   0.004   0.004      4.5783       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    2568       0      3   0.585   0.585      111.82       1.000\n",
      "10 Distance restraints 2 (N-O)        :    2785       0      1   0.679   0.679      157.44       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     154       0      1   4.371   4.371      34.698       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     135       0      0  86.755  86.755      45.301       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      92       0      2  73.695  73.695      43.312       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      34       0      0  61.481  61.481      19.576       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      13       0      0 102.038 102.038      6.5032       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    1284       0      0   0.568   0.568      30.227       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     153      19     18  36.563  64.448      157.88       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     194       0      0   1.521   1.521      17.661       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: NSP4.V99990002\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   17383.8242\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4325  21N  22G C   N     156  158   96.11   78.70   41.81    0.63   82.20  148.04    7.88\n",
      "    1         22G  22G N   CA    158  159  155.89 -166.10                    8.50\n",
      "    2   4335  31V  32F C   N     241  243   58.82   58.10    0.86    0.13  -63.20  144.64   26.09\n",
      "    2         32F  32F N   CA    243  244   33.38   32.90                  -44.30\n",
      "    3   4337  33S  34A C   N     258  260 -145.03 -134.00   22.90    1.41  -68.20   79.00    7.26\n",
      "    3         34A  34A N   CA    260  261  126.94  147.00                  145.30\n",
      "    4   4353  49S  50D C   N     380  382 -104.51  -96.50   71.66    3.00  -63.30   92.65    9.84\n",
      "    4         50D  50D N   CA    382  383   42.99  114.20                  -40.00\n",
      "    5   4355  51F  52A C   N     399  401   71.95  -68.20  153.40   14.72  -62.50  174.62   27.59\n",
      "    5         52A  52A N   CA    401  402 -152.32  145.30                  -40.90\n",
      "    6   4378  74P  75Y C   N     558  560  173.71 -124.30   62.57    4.42  -63.50 -148.14   39.28\n",
      "    6         75Y  75Y N   CA    560  561  143.94  135.40                  -43.40\n",
      "    7   4379  75Y  76C C   N     570  572  -60.46  -63.00    5.84    0.61 -117.90 -178.15    7.86\n",
      "    7         76C  76C N   CA    572  573  -46.36  -41.10                  141.10\n",
      "    8   4389  85S  86I C   N     646  648  129.92  -63.40  166.83   29.57 -120.60 -160.44   15.16\n",
      "    8         86I  86I N   CA    648  649  -36.55  -43.60                  130.30\n",
      "    9   4391  87S  88Y C   N     660  662   74.18   55.90   47.16    2.28 -124.30 -146.66   17.43\n",
      "    9         88Y  88Y N   CA    662  663   -3.97   39.50                  135.40\n",
      "   10   4403  99L 100M C   N     766  768  -17.75  -73.00   84.55    5.31  -63.40  127.92   22.72\n",
      "   10        100M 100M N   CA    768  769   78.99  143.00                  -40.50\n",
      "   11   4404 100M 101D C   N     774  776 -158.26  -63.30   96.75   14.36   54.50  159.92   14.59\n",
      "   11        101D 101D N   CA    776  777  -21.51  -40.00                   40.90\n",
      "   12   4418 114G 115S C   N     883  885 -122.18 -136.60   23.55    0.82  -64.10  177.36   10.92\n",
      "   12        115S 115S N   CA    885  886  132.58  151.20                  -35.00\n",
      "   13   4421 117R 118V C   N     907  909 -113.26 -125.40   17.68    0.50  -62.40 -179.82   20.37\n",
      "   13        118V 118V N   CA    909  910  130.45  143.30                  -42.40\n",
      "   14   4422 118V 119V C   N     914  916 -150.75  -62.40   91.18   15.29  -62.40   91.18   15.29\n",
      "   14        119V 119V N   CA    916  917  -64.91  -42.40                  -42.40\n",
      "   15   4426 122F 123D C   N     946  948  -58.66  -96.50   61.98    2.55  -63.30  156.78   19.08\n",
      "   15        123D 123D N   CA    948  949  163.29  114.20                  -40.00\n",
      "   16   4428 124A 125E C   N     959  961  -74.77  -69.30   14.81    0.93  -63.60  163.81   22.46\n",
      "   16        125E 125E N   CA    961  962  156.27  142.50                  -40.30\n",
      "   17   4438 134R 135S C   N    1044 1046  -73.35  -64.10   14.54    1.09 -136.60 -173.94    7.90\n",
      "   17        135S 135S N   CA   1046 1047  -23.78  -35.00                  151.20\n",
      "   18   4448 144S 145G C   N    1111 1113   63.40   78.70   22.60    0.47   82.20  159.09    8.66\n",
      "   18        145G 145G N   CA   1113 1114 -149.47 -166.10                    8.50\n",
      "   19   4449 145G 146R C   N    1115 1117 -107.70 -125.20   96.05    4.23   57.30  165.20   18.86\n",
      "   19        146R 146R N   CA   1117 1118   46.16  140.60                   38.00\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    7   23  101   85  119  119  147  126  178  184\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ARG:OXT  ARG\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      155\n",
      "Number of all, selected real atoms                :     1215    1215\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    11716   11716\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   191934\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -12980.0156\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -12980.015625\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : /usr/local/Cellar/modeller/9.25_1/modlib/surf5.de\n",
      "Pair library             : /usr/local/Cellar/modeller/9.25_1/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    30.323000\n",
      "Sequence length          :          155\n",
      "Compactness              :     0.228891\n",
      "Native energy (pair)     :    24.688493\n",
      "Native energy (surface)  :     8.726888\n",
      "Native energy (combined) :     3.167280\n",
      "Z score (pair)           :    -1.968432\n",
      "Z score (surface)        :    -1.105710\n",
      "Z score (combined)       :    -1.899608\n",
      "GA341 score              :     0.153011\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      155\n",
      "Number of all, selected real atoms                :     1215    1215\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    11716   11716\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     2514\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1194.4651\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    1244       0      0   0.006   0.006      11.566       1.000\n",
      " 2 Bond angle potential               :    1693       0      2   2.227   2.227      157.80       1.000\n",
      " 3 Stereochemical cosine torsion poten:     835       0     36  49.353  49.353      319.32       1.000\n",
      " 4 Stereochemical improper torsion pot:     532       0      0   1.365   1.365      21.209       1.000\n",
      " 5 Soft-sphere overlap restraints     :    2514       0      0   0.005   0.005      7.6494       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    2568       0      3   0.541   0.541      116.24       1.000\n",
      "10 Distance restraints 2 (N-O)        :    2785       0      4   0.661   0.661      181.80       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     154       0      0   4.391   4.391      35.018       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     135       0      1  80.584  80.584      42.442       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      92       0      1  75.833  75.833      45.619       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      34       0      0  80.354  80.354      29.953       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      13       0      0 125.387 125.387      8.8319       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    1284       0      1   0.617   0.617      37.665       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     153      21     23  36.065  67.347      158.18       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     194       0      2   1.111   1.111      21.180       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: NSP4.V99990003\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   18077.3750\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4316  12L  13P C   N      91   93  -56.93  -58.70   27.73    2.15  -64.50  154.82   11.96\n",
      "    1         13P  13P N   CA     93   94  -58.17  -30.50                  147.20\n",
      "    2   4317  13P  14G C   N      98  100  -66.58  -80.20   18.83    1.34   78.70  145.44    7.50\n",
      "    2         14G  14G N   CA    100  101 -172.90  174.10                 -166.10\n",
      "    3   4325  21N  22G C   N     156  158   91.09   78.70   31.81    0.49   82.20  156.35    8.04\n",
      "    3         22G  22G N   CA    158  159  164.60 -166.10                    8.50\n",
      "    4   4335  31V  32F C   N     241  243   60.11   58.10    5.29    0.28  -63.20  142.94   25.74\n",
      "    4         32F  32F N   CA    243  244   28.00   32.90                  -44.30\n",
      "    5   4337  33S  34A C   N     258  260 -150.93 -134.00   21.03    1.16  -68.20   83.43    7.42\n",
      "    5         34A  34A N   CA    260  261  134.53  147.00                  145.30\n",
      "    6   4353  49S  50D C   N     380  382 -110.63  -96.50   65.87    2.75  -63.30  101.56   10.77\n",
      "    6         50D  50D N   CA    382  383   49.87  114.20                  -40.00\n",
      "    7   4355  51F  52A C   N     399  401   77.82  -68.20  160.95   15.48  -62.50  175.91   28.13\n",
      "    7         52A  52A N   CA    401  402 -146.99  145.30                  -40.90\n",
      "    8   4378  74P  75Y C   N     558  560  168.37 -124.30   67.34    5.03  -63.50 -139.15   30.47\n",
      "    8         75Y  75Y N   CA    560  561  136.48  135.40                  -43.40\n",
      "    9   4379  75Y  76C C   N     570  572  -61.62  -63.00    2.44    0.26 -117.90 -175.43    7.96\n",
      "    9         76C  76C N   CA    572  573  -43.11  -41.10                  141.10\n",
      "   10   4389  85S  86I C   N     646  648  118.49  -63.40  179.24   31.34 -120.60 -164.33   15.18\n",
      "   10         86I  86I N   CA    648  649  -23.54  -43.60                  130.30\n",
      "   11   4391  87S  88Y C   N     660  662   72.12   55.90   34.57    1.65 -124.30 -153.25   17.02\n",
      "   11         88Y  88Y N   CA    662  663    8.97   39.50                  135.40\n",
      "   12   4402  98V  99L C   N     758  760  -61.00  -63.50    2.51    0.43  -70.70  177.34   13.81\n",
      "   12         99L  99L N   CA    760  761  -41.33  -41.20                  141.60\n",
      "   13   4403  99L 100M C   N     766  768  -66.39  -73.00   37.86    2.46  -63.40  146.25   21.60\n",
      "   13        100M 100M N   CA    768  769  105.72  143.00                  -40.50\n",
      "   14   4411 107F 108P C   N     828  830 -125.10  -64.50   61.48    6.62  -64.50   61.48    6.62\n",
      "   14        108P 108P N   CA    830  831  136.89  147.20                  147.20\n",
      "   15   4415 111Y 112L C   N     862  864  -65.10  -63.50   12.46    1.83 -108.50  179.27   10.06\n",
      "   15        112L 112L N   CA    864  865  -53.56  -41.20                  132.50\n",
      "   16   4417 113E 114G C   N     879  881  152.31 -167.20   41.07    0.64   82.20 -173.34   12.33\n",
      "   16        114G 114G N   CA    881  882 -178.51  174.60                    8.50\n",
      "   17   4418 114G 115S C   N     883  885  -86.93  -72.40   53.76    2.70  -64.10  122.98   10.43\n",
      "   17        115S 115S N   CA    885  886 -155.84  152.40                  -35.00\n",
      "   18   4420 116V 117R C   N     896  898  -71.10 -125.20   60.73    1.84  -63.00  154.32   20.31\n",
      "   18        117R 117R N   CA    898  899  113.01  140.60                  -41.10\n",
      "   19   4428 124A 125E C   N     959  961   73.08   54.60   41.02    2.09  -63.60  144.24   24.48\n",
      "   19        125E 125E N   CA    961  962    5.78   42.40                  -40.30\n",
      "   20   4448 144S 145G C   N    1111 1113   62.05   78.70   30.23    0.49   82.20  150.73    8.32\n",
      "   20        145G 145G N   CA   1113 1114 -140.87 -166.10                    8.50\n",
      "   21   4449 145G 146R C   N    1115 1117 -112.90  -63.00   89.91   10.19   57.30  170.26   19.97\n",
      "   21        146R 146R N   CA   1117 1118   33.69  -41.10                   38.00\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    8   19   76   91  121  135  123  163  170  203\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ARG:OXT  ARG\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      155\n",
      "Number of all, selected real atoms                :     1215    1215\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    11716   11716\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   188014\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -12621.9756\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -12621.975586\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : /usr/local/Cellar/modeller/9.25_1/modlib/surf5.de\n",
      "Pair library             : /usr/local/Cellar/modeller/9.25_1/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    30.323000\n",
      "Sequence length          :          155\n",
      "Compactness              :     0.159532\n",
      "Native energy (pair)     :    28.502182\n",
      "Native energy (surface)  :    11.255480\n",
      "Native energy (combined) :     3.956695\n",
      "Z score (pair)           :    -1.540629\n",
      "Z score (surface)        :    -0.576348\n",
      "Z score (combined)       :    -1.316537\n",
      "GA341 score              :     0.077450\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      155\n",
      "Number of all, selected real atoms                :     1215    1215\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    11716   11716\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     2527\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1176.8257\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    1244       0      0   0.005   0.005      10.082       1.000\n",
      " 2 Bond angle potential               :    1693       0      0   2.054   2.054      134.26       1.000\n",
      " 3 Stereochemical cosine torsion poten:     835       0     32  48.563  48.563      308.20       1.000\n",
      " 4 Stereochemical improper torsion pot:     532       0      0   1.250   1.250      17.995       1.000\n",
      " 5 Soft-sphere overlap restraints     :    2527       0      0   0.005   0.005      7.4069       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    2568       0      4   0.538   0.538      115.25       1.000\n",
      "10 Distance restraints 2 (N-O)        :    2785       0      4   0.694   0.694      181.57       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     154       0      1   4.203   4.203      32.079       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     135       0      3  88.193  88.193      49.516       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      92       0      0  81.953  81.953      38.392       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      34       0      1  78.828  78.828      31.274       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      13       0      0 108.103 108.103      8.2430       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    1284       0      1   0.754   0.754      41.469       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     153      25     22  38.133  71.171      180.84       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     194       0      0   1.445   1.445      20.247       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: NSP4.V99990004\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   18897.1172\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4311   7F   8I C   N      57   59  -97.14 -120.60   75.73    3.48  -63.40  107.34   16.60\n",
      "    1          8I   8I N   CA     59   60   58.30  130.30                  -43.60\n",
      "    2   4325  21N  22G C   N     156  158   97.84   78.70   43.38    0.64   82.20  147.30    7.93\n",
      "    2         22G  22G N   CA    158  159  154.97 -166.10                    8.50\n",
      "    3   4335  31V  32F C   N     241  243   62.35   58.10    8.64    0.44  -63.20  143.59   25.83\n",
      "    3         32F  32F N   CA    243  244   25.38   32.90                  -44.30\n",
      "    4   4337  33S  34A C   N     258  260 -141.11 -134.00   26.97    1.64  -68.20   76.86    7.23\n",
      "    4         34A  34A N   CA    260  261  120.98  147.00                  145.30\n",
      "    5   4353  49S  50D C   N     380  382 -104.13  -96.50   79.65    3.34  -63.30   85.32    9.05\n",
      "    5         50D  50D N   CA    382  383   34.92  114.20                  -40.00\n",
      "    6   4355  51F  52A C   N     399  401   73.96  -68.20  155.63   14.93  -62.50  175.58   27.83\n",
      "    6         52A  52A N   CA    401  402 -151.38  145.30                  -40.90\n",
      "    7   4378  74P  75Y C   N     558  560 -163.87 -124.30   49.59    2.45  -63.50 -178.42   33.51\n",
      "    7         75Y  75Y N   CA    560  561  165.28  135.40                  -43.40\n",
      "    8   4379  75Y  76C C   N     570  572  -59.28  -63.00   11.54    1.22 -117.90  176.88    7.67\n",
      "    8         76C  76C N   CA    572  573  -52.02  -41.10                  141.10\n",
      "    9   4380  76C  77Y C   N     576  578 -113.09  -98.40   52.62    5.34  -63.50  131.03   18.59\n",
      "    9         77Y  77Y N   CA    578  579   77.88  128.40                  -43.40\n",
      "   10   4381  77Y  78D C   N     588  590 -134.31  -63.30   79.31   13.54  -63.30   79.31   13.54\n",
      "   10         78D  78D N   CA    590  591  -75.31  -40.00                  -40.00\n",
      "   11   4389  85S  86I C   N     646  648  120.72  -63.40  176.64   32.05 -120.60 -152.83    8.52\n",
      "   11         86I  86I N   CA    648  649  -59.89  -43.60                  130.30\n",
      "   12   4390  86I  87S C   N     654  656  -62.02  -72.40   11.11    0.66  -64.10  176.60   12.67\n",
      "   12         87S  87S N   CA    656  657  148.41  152.40                  -35.00\n",
      "   13   4391  87S  88Y C   N     660  662  -72.81  -63.50    9.39    1.56 -124.30 -175.07    7.86\n",
      "   13         88Y  88Y N   CA    662  663  -42.22  -43.40                  135.40\n",
      "   14   4403  99L 100M C   N     766  768  -30.50  -73.00   71.44    4.46  -63.40  130.30   22.08\n",
      "   14        100M 100M N   CA    768  769   85.58  143.00                  -40.50\n",
      "   15   4404 100M 101D C   N     774  776 -141.51  -63.30   80.85   11.68   54.50  174.77   16.58\n",
      "   15        101D 101D N   CA    776  777  -19.52  -40.00                   40.90\n",
      "   16   4411 107F 108P C   N     828  830 -126.50  -64.50   63.09    6.82  -64.50   63.09    6.82\n",
      "   16        108P 108P N   CA    830  831  135.51  147.20                  147.20\n",
      "   17   4417 113E 114G C   N     879  881 -173.51 -167.20   19.17    0.79   82.20 -178.95   13.28\n",
      "   17        114G 114G N   CA    881  882  156.50  174.60                    8.50\n",
      "   18   4418 114G 115S C   N     883  885  -81.22  -72.40    9.21    0.72  -64.10  176.08   13.85\n",
      "   18        115S 115S N   CA    885  886  149.75  152.40                  -35.00\n",
      "   19   4419 115S 116V C   N     889  891  -70.40  -62.40    8.04    1.24 -125.40 -176.46   10.20\n",
      "   19        116V 116V N   CA    891  892  -41.59  -42.40                  143.30\n",
      "   20   4420 116V 117R C   N     896  898  -57.04 -125.20   71.06    2.36  -63.00  161.70   22.25\n",
      "   20        117R 117R N   CA    898  899  120.49  140.60                  -41.10\n",
      "   21   4421 117R 118V C   N     907  909 -148.52 -125.40   78.52    3.24  -62.40  131.42   21.67\n",
      "   21        118V 118V N   CA    909  910 -141.66  143.30                  -42.40\n",
      "   22   4424 120T 121T C   N     928  930  -72.41  -78.10   10.80    0.65  -63.20  159.18   21.02\n",
      "   22        121T 121T N   CA    930  931  158.98  149.80                  -42.10\n",
      "   23   4425 121T 122F C   N     935  937   63.80   58.10   15.26    0.80  -63.20  141.79   25.41\n",
      "   23        122F 122F N   CA    937  938   18.75   32.90                  -44.30\n",
      "   24   4426 122F 123D C   N     946  948 -134.21  -96.50  107.54    4.53  -63.30  126.78   20.15\n",
      "   24        123D 123D N   CA    948  949 -145.09  114.20                  -40.00\n",
      "   25   4428 124A 125E C   N     959  961  -74.88  -69.30   21.67    1.45  -63.60  156.66   21.51\n",
      "   25        125E 125E N   CA    961  962  163.44  142.50                  -40.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   13   17   80   95  138  144  145  143  166  201\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ARG:OXT  ARG\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      155\n",
      "Number of all, selected real atoms                :     1215    1215\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    11716   11716\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   188772\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -12692.2764\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -12692.276367\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : /usr/local/Cellar/modeller/9.25_1/modlib/surf5.de\n",
      "Pair library             : /usr/local/Cellar/modeller/9.25_1/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    30.323000\n",
      "Sequence length          :          155\n",
      "Compactness              :     0.259114\n",
      "Native energy (pair)     :    36.159579\n",
      "Native energy (surface)  :     9.469758\n",
      "Native energy (combined) :     3.726784\n",
      "Z score (pair)           :    -2.079117\n",
      "Z score (surface)        :    -0.910826\n",
      "Z score (combined)       :    -1.842813\n",
      "GA341 score              :     0.152793\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      155\n",
      "Number of all, selected real atoms                :     1215    1215\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    11716   11716\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     2461\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1093.8330\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    1244       0      0   0.005   0.005      8.4293       1.000\n",
      " 2 Bond angle potential               :    1693       0      3   2.028   2.028      132.68       1.000\n",
      " 3 Stereochemical cosine torsion poten:     835       0     35  49.304  49.304      319.15       1.000\n",
      " 4 Stereochemical improper torsion pot:     532       0      0   1.220   1.220      16.707       1.000\n",
      " 5 Soft-sphere overlap restraints     :    2461       0      0   0.004   0.004      5.2847       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    2568       0      3   0.582   0.582      109.42       1.000\n",
      "10 Distance restraints 2 (N-O)        :    2785       0      0   0.632   0.632      136.07       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     154       0      0   4.208   4.208      32.164       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     135       0      1  81.603  81.603      44.399       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      92       0      2  72.132  72.132      36.215       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      34       0      0  67.653  67.653      24.098       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      13       0      0 117.872 117.872      9.0883       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    1284       0      0   0.659   0.659      30.081       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     153      20     19  38.291  64.889      163.32       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     194       0      1   2.002   2.002      26.742       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: NSP4.V99990005\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   16698.1777\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4325  21N  22G C   N     156  158   93.73   78.70   39.04    0.60   82.20  149.81    7.85\n",
      "    1         22G  22G N   CA    158  159  157.87 -166.10                    8.50\n",
      "    2   4335  31V  32F C   N     241  243   64.29   58.10   12.05    0.62  -63.20  143.95   25.85\n",
      "    2         32F  32F N   CA    243  244   22.55   32.90                  -44.30\n",
      "    3   4337  33S  34A C   N     258  260 -145.49 -134.00   27.47    1.70  -68.20   80.72    7.53\n",
      "    3         34A  34A N   CA    260  261  122.04  147.00                  145.30\n",
      "    4   4353  49S  50D C   N     380  382 -103.62  -96.50   76.53    3.21  -63.30   87.81    9.32\n",
      "    4         50D  50D N   CA    382  383   38.00  114.20                  -40.00\n",
      "    5   4355  51F  52A C   N     399  401   73.48  -68.20  155.26   14.90  -62.50  175.09   27.75\n",
      "    5         52A  52A N   CA    401  402 -151.20  145.30                  -40.90\n",
      "    6   4378  74P  75Y C   N     558  560  176.37 -124.30   59.44    4.36  -63.50 -145.49   39.64\n",
      "    6         75Y  75Y N   CA    560  561  138.88  135.40                  -43.40\n",
      "    7   4379  75Y  76C C   N     570  572  -57.79  -63.00    7.67    0.86 -117.90 -177.64    7.90\n",
      "    7         76C  76C N   CA    572  573  -46.73  -41.10                  141.10\n",
      "    8   4389  85S  86I C   N     646  648  129.69  -63.40  167.04   30.04 -120.60 -149.42    8.71\n",
      "    8         86I  86I N   CA    648  649  -49.95  -43.60                  130.30\n",
      "    9   4391  87S  88Y C   N     660  662   72.56   55.90   36.84    1.76 -124.30 -152.17   17.09\n",
      "    9         88Y  88Y N   CA    662  663    6.65   39.50                  135.40\n",
      "   10   4396  92R  93P C   N     706  708  -62.99  -64.50   35.55    2.57  -58.70  142.24   11.41\n",
      "   10         93P  93P N   CA    708  709  111.68  147.20                  -30.50\n",
      "   11   4397  93P  94D C   N     713  715  156.52  -63.30  142.20   21.30  -63.30  142.20   21.30\n",
      "   11         94D  94D N   CA    715  716  -16.12  -40.00                  -40.00\n",
      "   12   4403  99L 100M C   N     766  768    4.37  -73.00  112.18    7.09  -63.40  122.69   23.48\n",
      "   12        100M 100M N   CA    768  769   61.77  143.00                  -40.50\n",
      "   13   4404 100M 101D C   N     774  776 -151.56  -63.30   88.66   13.64   54.50  170.16   14.99\n",
      "   13        101D 101D N   CA    776  777  -31.62  -40.00                   40.90\n",
      "   14   4411 107F 108P C   N     828  830 -126.81  -64.50   62.74    6.66  -64.50   62.74    6.66\n",
      "   14        108P 108P N   CA    830  831  139.86  147.20                  147.20\n",
      "   15   4418 114G 115S C   N     883  885  -74.40  -72.40    4.04    0.19  -64.10  169.40   12.91\n",
      "   15        115S 115S N   CA    885  886  155.91  152.40                  -35.00\n",
      "   16   4420 116V 117R C   N     896  898 -133.63 -125.20   89.80    4.51  -63.00  116.22   13.28\n",
      "   16        117R 117R N   CA    898  899   51.20  140.60                  -41.10\n",
      "   17   4426 122F 123D C   N     946  948  -72.14  -96.50   49.77    2.06  -63.30  162.63   20.62\n",
      "   17        123D 123D N   CA    948  949  157.61  114.20                  -40.00\n",
      "   18   4428 124A 125E C   N     959  961  -68.71  -69.30    9.54    0.75  -63.60  167.75   22.65\n",
      "   18        125E 125E N   CA    961  962  152.02  142.50                  -40.30\n",
      "   19   4434 130G 131T C   N    1011 1013  -60.88  -63.20    3.75    0.40 -124.80 -177.01    9.67\n",
      "   19        131T 131T N   CA   1013 1014  -45.04  -42.10                  143.50\n",
      "   20   4436 132C 133E C   N    1024 1026  -68.52  -63.60    8.86    1.45 -117.80 -177.68    9.83\n",
      "   20        133E 133E N   CA   1026 1027  -47.67  -40.30                  136.80\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    9   25  102   85  117  140  133  147  171  205\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf     DOPE score    GA341 score\n",
      "----------------------------------------------------------------------\n",
      "NSP4.B99990001.pdb            1182.20996   -12802.98926        0.09924\n",
      "NSP4.B99990002.pdb            1124.65515   -12878.18945        0.07746\n",
      "NSP4.B99990003.pdb            1194.46509   -12980.01562        0.15301\n",
      "NSP4.B99990004.pdb            1176.82568   -12621.97559        0.07745\n",
      "NSP4.B99990005.pdb            1093.83301   -12692.27637        0.15279\n",
      "\n"
     ]
    }
   ],
   "source": [
    "hg.model_maker(env,tmp='1t70A',targ='NSP4')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 4. Evaluating Model and Template\n",
    "\n",
    "Evaluate the model and the template. To do this, construct the DOPE profile. \n",
    "\n",
    "### Model Selection\n",
    "We select `NSP4.B9999003.pdb` since it has the lowest overall DOPE score."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": false,
    "jupyter": {
     "outputs_hidden": false
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "read_to_681_> topology.submodel read from topology file:        3\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ARG:OXT  ARG\n",
      "preppdf_453W> No fixed restraints selected; there may be some dynamic ones.\n",
      "preppdf_454W> Restraints file was probably not read; use restraints.append().\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      155\n",
      "Number of all, selected real atoms                :     1215    1215\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :        0       0\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   191935\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -12980.2217\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 2 Bond angle potential               :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 3 Stereochemical cosine torsion poten:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 4 Stereochemical improper torsion pot:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "13 Mainchain Omega dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "25 Phi/Psi pair of dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "31 Non-bonded restraints              :  191689       0    352   6.710   6.710     -12980.       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "do_prof_492W> Are you sure you want to calculate a normalized energy profile!\n",
      "\n",
      "\n",
      "\n",
      "# Energy of each residue is written to: NSP4.profile\n",
      "# The profile IS normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:   13\n",
      "# The sum of all numbers in the file:      -5.3855\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -12980.221680\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ASP:OXT  ASP\n",
      "preppdf_453W> No fixed restraints selected; there may be some dynamic ones.\n",
      "preppdf_454W> Restraints file was probably not read; use restraints.append().\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      255\n",
      "Number of all, selected real atoms                :     1972    1972\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :        0       0\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   397191\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -30381.8125\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 2 Bond angle potential               :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 3 Stereochemical cosine torsion poten:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 4 Stereochemical improper torsion pot:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "13 Mainchain Omega dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "25 Phi/Psi pair of dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "31 Non-bonded restraints              :  397025       0    205   6.843   6.843     -30382.       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "do_prof_492W> Are you sure you want to calculate a normalized energy profile!\n",
      "\n",
      "\n",
      "\n",
      "# Energy of each residue is written to: 1t70A.profile\n",
      "# The profile IS normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:   13\n",
      "# The sum of all numbers in the file:      -9.6286\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -30381.812500\n"
     ]
    }
   ],
   "source": [
    "hg.dope_evaluate(env,'NSP4.B99990003.pdb', 'NSP4.profile')\n",
    "hg.dope_evaluate(env, pdb_file, '1t70A.profile', model_segment=('FIRST:A', 'LAST:A'))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 5. Visualizing Protein Structure\n",
    "### Model - `NSP4.B99990003.pdb`"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "29c8848788294b8f9c8c3b8edeeab294",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "NGLWidget()"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "mdl_v = ngv.show_structure_file('NSP4.B99990003.pdb')\n",
    "mdl_v.display()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "#mdl_v.render_image();"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "#mdl_v._display_image()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Template - 1T70 Chain A"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/usr/local/lib/python3.9/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain A is discontinuous at line 16535.\n",
      "  warnings.warn(\n",
      "/usr/local/lib/python3.9/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain B is discontinuous at line 16593.\n",
      "  warnings.warn(\n",
      "/usr/local/lib/python3.9/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain C is discontinuous at line 16649.\n",
      "  warnings.warn(\n",
      "/usr/local/lib/python3.9/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain D is discontinuous at line 16704.\n",
      "  warnings.warn(\n",
      "/usr/local/lib/python3.9/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain E is discontinuous at line 16743.\n",
      "  warnings.warn(\n",
      "/usr/local/lib/python3.9/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain F is discontinuous at line 16769.\n",
      "  warnings.warn(\n",
      "/usr/local/lib/python3.9/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain G is discontinuous at line 16811.\n",
      "  warnings.warn(\n",
      "/usr/local/lib/python3.9/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain H is discontinuous at line 16827.\n",
      "  warnings.warn(\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "68adf8c3ebd649cca0c0b81626f28a36",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "NGLWidget()"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "parser = PDBParser()\n",
    "structure = parser.get_structure(\"1t70\", pdb_file)\n",
    "tmp_v = ngv.show_biopython(structure[0][\"A\"])\n",
    "tmp_v"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "#tmp_v.render_image();"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "#tmp_v._display_image()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 6. Visualizing DOPE Profile"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAcgAAAEJCAYAAAAQFbf7AAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjMuMywgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/Il7ecAAAACXBIWXMAAAsTAAALEwEAmpwYAAB4BUlEQVR4nO2dd3gc5bX/PzOzVb1LtmzL3QZXjMF0B0IzxhRfQg2EQG4a4QZ+BGIgoYRODASSwCX3kkAMmJJLbEhsExNwaHaMce9dliWrd+1q28zvj9lZaa1dbVV/P8/Dg7U7O/Puq9Gc95z3nO+RNE3TEAgEAoFAEITc1wMQCAQCgaA/IgykQCAQCAQhEAZSIBAIBIIQCAMpEAgEAkEIhIEUCAQCgSAEwkAKBAKBQBACYSAFAoFAIAiBqbcvWFFRwd13301dXR1jxoxh8eLFpKamBh3jdru5//772b59OzabjcWLFzNu3Dja2tq47777OHjwIAA//OEPmT9/fkzXb2hoQ1XjK/3MzU2jrq41rs8OVsScdEXMSVfEnAQj5qMrfTEnsiyRnZ0a9v1eN5APP/ww119/PfPnz+f3v/89L774InfffXfQMUuWLMFut7Ny5Uq++uorFi1axLvvvssf/vAHhg8fzvPPP09dXR2XX345c+bMIS8vL+rrq6oWt4E0Pi8IRsxJV8ScdEXMSTBiPrrS3+akV0OsHo+Hr776iosuugiAhQsXsmrVqi7HrVmzhssuuwyAU045hYaGBioqKjj11FO58cYbAcjNzSUrK4va2tre+wICgUAgGDL0qgfZ0NBAWloaJpN+2fz8fKqqqrocV11dTX5+fuDn/Px8KisrOfPMMwOvrVixArfbzfjx43t+4AKBQCAYcvSYgVy5ciVPPPFE0GujR4/ucpwkSVGdT5Y7nN2VK1fy+OOP87//+78BYxstublpMR1/PPn56Ql9fjAi5qQrYk66IuYkGDEfXelvc9JjBnLevHnMmzcv6DWPx8OcOXPw+XwoikJNTQ0FBQVdPltQUEBNTQ0lJSUAQcctWbKEV155hVdeeYVJkybFPK66uta449z5+enU1LTE9dnBipiTrog56YqYk2DEfHSlL+ZElqVunaZe3YM0m83Mnj2bFStWALBs2TLOOeecLsfNnTuX5cuXA7BhwwasVivDhw/no48+4tVXX2Xp0qVxGUeBQCAQCKJF6u12V+Xl5SxatIi6ujqGDRvGs88+S2ZmJkuXLqW6upqf/vSnuFwuHnjgAbZv347FYuHRRx9lypQpXHbZZdTX15Obmxs436OPPsq0adOivr7wIJOLmJOuiDnpipiTYMR8dCXcnDS0uPD5VPKy7Em/ZiQPstcNZF8jDGRyEXPSFTEnXRFzEoyYj66EmhOvT+Xn/70Wt8fHb+/oGm1MlH4VYhUIBAKBIFr2H22iocVFeoqlT64vDKRAIBAI+iVbDuh17jPG50Y4smcQBlIgEAgE/ZLN++sAmDk+erW0ZNLrUnMCgUAgEIRC0zTe+/QgZdV6rkhVvYNUm4nxIzL7ZDzCQAoEAoGgX1Ba1cLf15YGvXbSxHwUuW+CncJACgQCgaBfsMUfUj1pQh7nzBiOIktMGJHVZ+MRBlIgEAgE/YLN+/WknHNmDGdGH+07dkYk6QgEAoGgz6lrclJa2YLFJHNCSXZfDwcQBlIgEAgE/YANu/TOTieOzsFiVvp4NDrCQAoEAoGgz1m/QzeQfVXzGAphIAUCgUDQp7g9PjbvqwFg+ri+33s0EAZSIBAIBH3KztIG3B4fJUXpZKdb+3o4AUQWq0AgGFL4VJWPN5bT4vAwuiidWRPze/yaLo+PjzcexenyAZCaYqHN4QagpDCdkyf1/Bj6M1v92aszxvWf8CoIAykQCIYY63dVs/SjfQBIwK9/fAY5GbYeveYnG8t595MDId8zKTIv/r9zMClDN6C3p6wR6F/hVRAGUiAQDDE27dO9FUWW8KkaW/bXcu6sET18TX1/7cxpReRn2UlNtdLW5gJgVGH6kDaOAPUt+lwU5SS/52MiCAMpEAiGDF6fyo5DulrLJaeV8MGXh9lyoK5HDWSr08P+8iYUWeL68ydit5pEP8hOOF1eXG4fFrOC3dq/TNLQXrYIBIIhxd6yRpwuH8V5qZx38ggkYOfhBlxuX49dc9uBOjQNJo7M6ncGoD/Q1KbvxeZkWJEkqY9HE4z4bQkA8FUfpP2zV7Gd/R2UgnF9PRyBIGkcOtbMS8u24/L4cHtUAGaMzyMz1cKY4RkcrGjmZy9+gSxLFGTZ+dm1J2G1JK9Q3ZBPi7dlU4vDzfN/2coZU4s4r4dDwX1Boz+82tP7wPEgPEgBAO6tK1HrjuDZ+2VfD0UgSCqfbCqntqmdFocHl8eH2SRz2pRCAM6ePgyAtnYvLQ4PByqaOVDRlLRre30q2/0h3XgL4Jvb3BysaOZvXx5G07Skja2/0Ojfi83uhwZSeJACNJ8Xb9l2ANSmyj4ejUCQPFRNY+sB3UAtumEWRTkpWC0KVr+U2dyZxZwyuQCvT+Ptj/exdkcVlfUOThydk5Tr7/OHdIflplCQnRLXOYbnpZKVZqGx1c2RqlZKitKTMrb+QmOLHmLN7YcGUniQAnyVe8HjBISBFAwuDh9robnNTW6GlQkjMslItQSMo0GKzUxGqoUR+WkAVNY7knb9zf72TfGGVwEkSQp0tjDCtYOJpn7sQQoDKcBbuinwb621Ds3r7sPRCATJ4XBlM//8+iig7zlGSgApytE9vKp6Z1Kur2l6CYlx/UQYzAaysdVI0ul/BlKEWIc4mqbhLd2s/yAroPpQm6pQckf26bgEgkQor2nlkVc3YOzYRWOgCv0GsrK+LSljqGlqp7rRSarNxLjijITOdWJJNhaTTGllCw0trn4lx5YoRpKOCLEK+h1q4zG0lhokWzpK8RT9NRFmFQxwvt5bg4buFV506kimRLGnmJ9lR5Kgtqkdj1dNeAwVNbqhHV2UjiIn9qi1mJVAj8StBwaXF9noL/PIzuh/Rl8YyCGO4T0qI6cjZw8HhIEUDHy2+Pf+vnXuOK45bwKyHLm+zmySycu0oWlQ3Zh4mNXYyyzKSU34XAAzJuhesPHdBguNrf4yj8z+paIDwkAOeXxHNgNgKpmJnFkEgNooDKRg4NLU6uLQsWbMJjnmbFTDmFUlIVGnqkE/R2GS5NNm+HVKdx6ux+3pOWGD3sRQ0TGbZFJt/W/Hr/+NSNBraO2t+Kr2gaxgGjEVX+1hQHiQgoGHo93D6//YS4vTQ6vDA8AJJdldMlYjUZhjZ9vBJBnIgAcZX3nH8WSnWykpTKe0qoVdpQ0JJ/70BwwVnaw0S0wqOmprPa6v/oJl5qUo/shXTyAM5BDGV70fNA2laDySxY6cpRdNCwMpGGh8vq2SdTurgl47ZXJBzOcpztM9yAMVzQmP6ViSDSTAnBMLKa1qweHyJu2cfUlZdSsAeTGGV91bVuDd9yVy9ghhIAU9g9qoP1Dk7GIAJHsmmG3gakNtb0G2Da6CZMHgxSinuOzM0YwvzsRmMTE2jszRqWN0tZsdh+rxeFXMpvh2oZwuL02tbkyKnNTyhQtPGcmJo7MZWZCWtHP2JcbvberY6EPhmqbhPbIFANOwiT0yLgOxBzmEMTxFY+9RkiTkTF2CSxP7kIIBgqPdy96yRmRJ4vzZI5k6NpfxIzKR4xC+zs20MSI/DZfHx54jDXGPqbpBT/IpzLZHlSAULbIsMaowvd+JeseDqnaoHM2IoQ+k2lARyLyX88f21PAAYSCHNAEDmVUUeE3OFGFWwcBi+6E6fKrG+BGZpNnNCZ9v5gTdi0ykKL8yTHhV87pRW2rwNFajttR0+U9TYy8vcQ3QhJ2DFc20Oj0UZNkZlht9GNp7RBc2UUZNR0qwfCYSIsQ6hDneg9T/XRj0nkDQ3zHKHuIVAz+eGePz+NuXpWzZX8cNF2hxeWvH6vQayMJOBlLzeWl79z60llrCSREoo2aQcvGdUV/nQHkTT76xkW+dO54LTxlY4h5bD3aoDMUyx75Sf3h11MyeGFYQwoMcomiedrS2BpBNSGkd4Q3DmxSlHoKBgKpqbDuYuN5pZ8YMyyAjxUxdczvlNfGp6uw4XB84l4Hv2G60llpQzJgy85HScoP+Q5LwHdmK6ow+Qai+xYVP1dhX1hjXOPuSsio9QWfCiMyoP6O2t+jJhf7M+55GeJBDFLXJn6CTWRAUpugIsVaF/JxA0J/YX96kh+my7UnLFpUlienj8vh82zG2HKhlRIwJMc0ONwfLmzEpElPGZAdeNxJLLNMuYsT871JT0xL0OcfKZ/CVbcN3ZAvypLOjulZmqgXoKLYfSATC0DGEV31HtuqZ98MnI1l6XlggKg+ysrKSf/3rX/h8Po4dO9bTYxL0AqHCq/rP/hBrc2Vc+yECQW+yxS+7NmNcbGG6SCQiDr7tQB0aMHlUNjaL7oN01jw2lcwM+TkjZBjQRo6CLL8mqyH4PVDw+lRqGtuRgIKs6A2dt5OwSW8Q0YNcs2YNDz30ELIs89Zbb3HJJZfw61//mvPPP783xifoIcIZSMliR7Jnojmb0NrqkNLz+2J4qM3VeCt2YZ50NpIkdgLCoXnduLevBrcDpWgSplHT+3pIvcqWQDup2PcfveU78ZXvALMNy5TzgzySKWOyMSkSB8ubaW5zk+H31KJh8/5a0iQn81K24lq/DwDN44qYeWkaNQPXF0vwlu9A83mQlMgJR1n+cTW1udC0+PZL+4LapnZUTSMv0wr7/oWrtY76FAsuF5hPPBfZ3rVER+9buw3onf1HiMJA/u53v+Odd97h+9//PgUFBbz55pv8/Oc/FwZygKM2VQMg+T3GzshZRficTahN1ch9ZCDb//UKvmN7kMx2zONO7ZMxDAQ8u9bgXv+u/oPyD9K++xKSPDR2TqobnVTUtmG3KkwYmRXTZzWvG+eHz4PXH5rUNKyzLgu8b7OYmDwqm+2H6tl2sI4zpw2L7ryaxu7SBi6xb2HEsb24jwu4mUpmhs28lNPzkHNGotaX4Tu2J6o9NotZIcVqwuHy0tbuTUoWb29QWaeHV2dl1OH6/D0ADB9YczZjO+vGLp/R+9a2I2cXI2f0znMp4tJcVVUKCjoUKU444YSEVikVFRXccMMNXHzxxfzoRz+ira3rJrjb7ebuu+9m3rx5XHnllRw4cACAtrY2br/9dhYsWMAVV1zBl19+Gfc4hjqaoxEAOTW7y3tSSnbQMb2N2t6i/zEA3tKNfTKGgULnXp74PGjNNX03mF4mUGQ+JheTEluUwVexUzeOsi5FFzSPfuIJs7Y4PLS1e5hm0ftQmqdfjOWUq7CcchXW067Fcuq3uv28EToMNZ5wZKb59yFbBs4+pLH/eIJcCujNEjJP0xco3tJNaJrW5TPGnPRWeBWiMJB2u52KioqAUdywYQNWa/xtSR5++GGuv/56Vq1axdSpU3nxxRe7HLNkyRLsdjsrV67kvvvuY9GiRQD86U9/oqSkhA8++IBnnnmGe+65J+5xDHU0h14EbRjDzkipWQCojvgLpRPB2IgH8JZtQ1MHZp1XT6O52vAd2wuSHAjbDaXyHMNAxpO96i3tSJhBMaPWHEI9bkFolI1s96vqRENlvYNipZ5M2YGUkoV1zjVYT7oU60mXYpl+ccjQYWdMo2b4x7c5pJEIRVaasQ85cAykLuSuMdylOz/WWZeRc96N+vZOWz1qfVnQ8Z33cJVeCq9CFAbyrrvu4pZbbuHIkSNcc8013HbbbfzsZz+L62Iej4evvvqKiy66CICFCxeyatWqLsetWbOGyy7TVxOnnHIKDQ0NVFRU8JOf/IQ77rgDgKNHj5KZGX16sCAY1dEEgJTSdQ7llCwANP8xvY2xEQ+Aqw1f1f4+GUd/x3t0O2g+lKIJKAVDy0A6XV72HGlEkmDauNj2H4OkysaeilJ8ItCRZWqQl2lnRH4qLrePvVGWUVTVO5hq1r1H06iZMUfb5IKxSPYMtNY61IbyqD7TYSAHTqJOZZ2DYUojNnejvi9bMBZJkjGVdCwQOhPUt7ZgXK+NM+JmxahRo3jnnXfYtGkTqqoyY8YMcnJiayFj0NDQQFpaGiaTftn8/HyqqrqWE1RXV5Of3xFjzs/Pp7KykuHDh2Mymbj11ltZu3Ytv/rVr+IaR1/gayin/aOXsJ76H5hKTurTsWheN7jaQFaQbF1T2CXDQLb1vgepb8RvB8A0ehbewxtxrngGZfhk7BffIRJ20H9/jr8/jVqrh6dMo2aCP6FjqNSv7jhUj0/VmBCjeo57979wr/8LWnsLUkoWct4oTKNm4juyRY9cTJ4bdPyM8XkcrWlj8/5apoyJ/NyrbHAw1R9eNR72sSBJMsrIGXj3foa3dDNKzoiIn8lKG3ilHmc1/51JGf7w6qgZgb9rZdRMPLs/xb1xOe5tH3Z8wKeLsysje149pzMRDeS3v/1tVq1axdy5cyMdGsTKlSt54okngl4bPXp0l+OiXWHJnSbllVdeoby8nGuvvZaTTjqJceOiX1Hk5iYm8pufH5+Ad+3Xn6E2HEXd8SH5s89JaAyJ4mmopBVQ0rIpKOjqQTodwzkGKJ7WqL5vvHMSCuehrbR6nJjzRlBw/g1UvLodzevGV7aVDE8VtuKeFSdOFsmck+Np2/1vWv1etWxLpfCUc3HXlVP5JSiO2h69diIkc1y7P9KzQ8+cURzTectXfInWrtcfZs66gJyCTHypczm6aTkpufnkHXeu02cU8/e1pZTXtkV1HWdTPaNMdWiymcIZc5DN4bejwp2v5YRZ1Oz9DFNzWVTXHFGkh23dPq3f/u47U7ZvP9MVPbSKbCL/1Aux+8ddNPM0jq5/C29jtb6I74xsIv/UCwLH9gYRDWRxcTEbN25k5syZQUYqEvPmzWPevHlBr3k8HubMmYPP50NRFGpqaoISgAwKCgqoqamhpKQEIHDc+vXrGT16NAUFBRQXF3PSSSexb9++mAxkXV0rqhpdbP948vPTuxT3RoOmabTt+QoAV/keqsoq+rRThreyQh+XLSPk91Hd+orU3VQX8fvGOyfhaN+6FgCpeDrNci6pN/0O19o38exaQ+2WL7Faossm7EuSPSfH075dnyPLrMuxnHQpDW4zKvpCx1VztEevHS/JnBNV1Vi/Q/eUxw2L7bzuWv3eT736CXxZw/yflbFf/xwqdDmXXdEX8EerW6O6jq16BwCe/EnUNbrpyM0Mprv58Ml6XkB7dXlU11TQn2fHaqIbY1+z9/OPGQOU2iYx5fr/R6vJSmtNC/n56dQ1urFd9QS4nV0/aDIHjk0Wsix16zRFtHgHDhzg+uuvZ/r06cyaNYuTTjqJWbNmxTUYs9nM7NmzWbFiBQDLli3jnHO6elNz585l+fLlQEdS0PDhw1mzZg1/+MMfAD0Mu337dqZNmxbXWHoTtf4oWqter4Wm6aGcPsRI0DH2Go/H2JfUHI1RJwokA31vaDMAij8MLZksmMbMBmIroB6saJraaf9sdqBWTkrLAcWk16+GergMIgyR6/wsG8NjUGHR2lt179FkQTq+/leSQkazstIsWM0KrU4PrU5Pt+dXVY2R7oMA2MbGv40Sq1hHYA+ybWCEWC1V+haKXDILydTVw5b8Wz9d/gtxbE8T0YN84403knrBBx98kEWLFvHSSy8xbNgwnn32WQCWLl1KdXU1P/3pT7nxxht54IEHmD9/PhaLhaeffhqAH//4x9x///0sWLAARVG47777KC4uTur4eoJA0onZBp52PHs/R0rNRhmeWMlMvGiBBJ2skO9LFntgrLgdYE3tlXGpTcfQmquRrGlBG/HKsElgsqLWl6G21iGnJUeUeiCi1hxCczYjpeUiZ3fsT0mSjJxRpIfxm6pQ8kf33SB7kIMVzXz0tZ7hGKvIdWdxjHCf0zQNX9U+lLzRSCa9y31hjp0jVa1U1jsYX9x1S0JtrgZZoa7dxASTXvhoHxufEwGxi3UE9iBbkp+kozZW4qs/YowMZfjkhKJfbc1NFHkr8CExauac5AyyB4kqxLpixQo+++wzPB4PZ511FldccUXcFywuLmbJkiVdXr/uuusC/7ZarTz11FNdjklLS+P555+P+9p9he+ovmKynnwlrnVL8VXswlmxC9uFt2MefXKvj8dIvglnII33tKZKVEcjSi8ZSJ8/Oef4NjaSYsY0Yirew1/jPbody+TY9sMHEwG5shAZknJmoW4g68sGpYGsrHfw6J83BH6eEWN5R4f+cFHYY7yHNtD+0e+xzJyP1V+zWJSTwpGqVqpCGEjN1Ubb/z2IZLZSP+4KhkteauR80kPUF8dCh1hHVUSxjsxOZR6qpsXVBzMUmtdN2/JHgvYCleIppMy/O+5zHt26geGSRrlczOTMrCSMsmeJGGJ95ZVXePnll5k0aRJTpkzhT3/6Ey+99FJvjG3QoDbo+x6mcadiPf26QM2a9+CG7j7Wc+MxRAK6MZCBUo+2xh4fj4ExT0oIKS7Zn80XCFUPUbrTolSGTdKP8ctxDTZKK/W9p9wMK5ecVsIJJbEZoVD9T4/HWDR6DqwPbC8YIuhGcXtnvGXbwONEczSSuvvvALTknBDTuEJhGHG1MbL2tdWskJlmwadq1De3J3xtA1/FLnC1Idkz9W0OScFXsQutvTXuc7pq9QxfV+aoZA2zR4loIJctW8Ybb7zBzTffzHe/+11ef/113n///d4Y26CgY9/DipSShWXaRdjP/U8AvGVb+6QIPlKItfN7vamm090DrPO+6FBFba1DrSsDkzVgDDsTUGE5ug1N9fby6Hoew0CdemIhV31jXMyekmFsuvMglYJxSLZ0tJaawPGF3RnITrWTmR5dxSjnhMSlEQNt56Ksay3KDj/GeDG+m/mEb2C/4CcowyaCpuI9msACrEX34k3dLFL6E1GlpaaldWT5pKenB+oYBZEJte8hZw3TNVD7qAg+oKLjV8wJRYeaTmPPD8hPx1yF0If1K/6ovejR9jcC4dURU5FMXcWz5YwC5Ozh4HbqCjuDjCqjPVKcba2iCbFKsowyUhd8N+bbuF7VccZHU314y4IT7pq1FIZPSqIHGWXbucLAGJOToBWq+0iHDN7muM9rb9cjQCkFkes7+wMRDWRxcTGvvfYaHo8Hj8fDq6++yvDhw3tjbIOCcF5RPK1tkoUahQcp97IHqbn1MBWKSW8eexwdHmTi6j6Odi9/XLGLnf6mtgOFQPZqN1qUfXlfhUJtqsL54fMce+Mh3Dv+mdC5KhMwkJqmdjKQXRdgnTEK/D3bVuH4+9PkbXiZyeZyqhqc+FQf7V+8jmf/On1x62pDyijAo9gAqEubgOLXd02EWEKs0H0YOB7U+jK0tnq/mEIJbo+PDw7r9ZZ65Cv2CIWqqmSq+uI8p7gkKePsaSIayIcffpiPPvqImTNnMnPmTP7xj3/wwAMP9MbYBgWGssnxq1bTSL08xVe1r1fHo/m8+qa7JIdU0TGQUnXVkN7a81Ob/Q+vjKKQajlSqiGgnri6T0VtG59vPcY7nwwsCTu1Wi8hMKTRQqEYq/wj0Wt59iTuravwlm7CeXgbrrVL0Tzx7ZFpmhZ4+BfGYyDbGsDnRrJnIEVIOjONmAYWO5qzGV/5Tji2gytSN+HxqjQe2o1nx0e4d3ykZ3kD5vGns1MbD4Blwmkxjy0UUkY+KBZdci4KRatwXm68+Cr155IyYgqSJFPX3M7fdzip1zLA7QzkC8RCa0MDdslNu2YmPU41tt4mYqy0sLCQxx9/nLy8PHw+H0eOHBkQpRX9hXBhQzlbn0MtyhBKstBc+ga7ZE3tVrYtUIvVS9qeHQuJ0Kt7yZ4BSGjOFjTVh5TAKr2kKA2LWeZIVSsNLS6y03u/vipWNJ9H/91JcmCxEAqlYDxYU9Gaq1GbjqFk9V20p7PmqWxPR3W24D26A/OY2DO3m9vctLt9pNpMpMfR0incQjUUksVO6lWP+j+j4fzHbxnmrSdbbqWl0kMuIKflYho1g5RvPUarks2fP/qCPPMEHpyZnNZskmxCKT4B35EteMu2RszcLszRe1kmzYP0/90r/udUYU4KGSlmyj0Z5FiaUZsqUXJjS7SpO1pKHtAoZ5Pfi3JxiRBxlEuWLOHHP/4xdrudhoYGbr/9dt59993eGNugIGxj4pQsMFnR2lsSygqLFeNa3XmP0NlAVkdVrJwokTIMJVlBsqcDGpqzOaFrmU0KU0brK9gtcXSM7wu0TuLy3S1sJFnG5N9D8/VxmFWtOxII02XOWaCPqbMQfQx09h7jqR0O93cYDjktF9OIKZhGTA30ZTzRXI6rrjzoPEp2MVsPNeJFoWDUaKyWxMOrBoZmczS/x/wsO7IkUdfUjsebeOKfMV+GoIIsSUwfl0e1T9/qiEfz11GtZ7C2WwdOHXNEA/n222+zdOlSAEaOHMmyZcv485//3OMDGwwE7Xsc9+CXJKljn6G597xIQ4dSsnZvICWzTfdUVC9aa88bkWgeYMnMrDVq6L7eU01TW//vgmB85+72jQ2MB+vx3Sl6m0BJyqgZpE7wqyEd2YKmxb7gqmrQk08Ks+NN0Al+4MeC0YJqqvkoNHf9ezb6RcZalxntdb3lO/QGA90dq8jkZ9nQgOqGxBN1DAOopRcGDO6M8XlUq/o+pKOmnKoGR0yynd5O5xwoRDSQPp+vSxZrX6i/DET0fQ+Pvu9h6fqHHfDSerEDQ7QeJHTOpOv58RkLie4eYFISazOn+1sk7TjcwJ2//bzfe5LR1K4amEZO1WvWKveiHS/43It4/ZKKppKZmPNHIaXlojmbUWsOx3wuowN9kT+UGCvR1ECGQxk1Aw2JCeZjpLYHL+Q8Xh87/MleM2JsuxUJOTUbOa8EvG58x3ZHPL67cpRY0HwefVEsSTzyf4d55LWvAZgyJlvfgwSOHjjAvS+v4/m/RC+bqbRWA2DJ6f96ygYRDeTYsWNZvHgxZWVllJWV8fzzz4fsyiHoitqq/+FIaaFXlrHWOiUD44EZnYE0wqw97+EayUByevhVuJzE0pOsNCsXnjIy0Cpp95G+aQ4dLcaiIBoPUrKkIOeOAE2LOguyJzD21+X8MUiS1KlMYFPM5wpksObGp+oUyx7k8cgpmag5JZgllUy1Keg8u4804vaojCpIIyfDFtfYukMp0rvX+OqORjw2N1O/fn1LYpqsanM1aBpeey7l9S7cfg/SZjExY5Yebi5UminIsjF+RHQ9eb0+Fbm9EYC8EQOjxAOizGItLS3liiuu4KqrruLw4cM89NBDvTC0gU9AFDxMvWFvemgGgf3OCCFWiD3VPF401YfmbAEkfzJOaJItXnDtNydw00V6wX2y6sd6ikCItZva1c7Imfoqva8aKAclFfl/p4ESlDj2IfUO9FCYHbsH2dkjkjO6l20Lh3V0J/Fxe6auV0xHeHV6ksOrBsbfoBbF7zGrk+RcIhiLiQYpC4AZ4zq+28Vzp4LZjl1y8/jN01hwxuiozrm3rJF0SV+c5xQODJEAiCKLNS8vj9/+9reAPyvN68Vsjj2LbCgSSbGmwwD1pgcZQ4g14OH2rAepJ91oSPbMbrNTOwxk8rw9Iz3+WBIVSHoCNYY9SOib8H1nQiUVKcMng9mGWheb6LxPVQP7avHsQRoekZRREOh+EiuWMSfh3fhXALwpupHVNI2tfgM5s4cNZDQLHUO0vKk1sT1141qlbfpczxjf8XuSJAk5qwi15hBqYyVyUXTC5Vv21TBP0stiJHt0Xmd/IKIHuWHDBl588UXcbjcLFy4Malcl6J5IouCdDVA8iQtxjakf7kFGI57e+X01CWIBBgXZdiSgttGJ19c7v4N40GLYg4S+Cd93JlRSkaSYMRVPAWJLIKptasenauRkWOPKEo01gzUUcs5IWiXdGLRacjl0rJnXV++lrtlFRqqF0cN6pr9rLL/HZHmQhrd6sM2O3aowcWRW8JiO82o9XpXVG8qobQwdhdE0jf0HjqJIGqolFUkZOEpsEQ3kr3/9a2bOnMlHH31EXl4ef//73/njH//YG2Mb8BgP8rB9Fy0pevjJ5w4YiZ4mYCCjCLFK6Xkgm/Ri5QRLK7pDjTJ82BMC6hazQk6GDZ+qUduUPKHnZBNLFiv0fYg1XFJRPHJlRvF73BmsEWpso0GSJGoyJgNwTM3hDx/s5JONesnHzPF5Seug0eW6qdm6YICzOWLClWEgE/UgfbWlAFT5Mpk6JheTEmwmjl847z7SwNKP9vHmR6FFT47VOfC06PkYSoJdTnqbqLJYzzjjDL788kvOP/98RowYgdoLdXGDgWgear0dZo0lxCrJJj0sBvh6sENEh3fUfeilpwTUi5JcZN0TxJKkA8EJVr0VnehMuHtfGTUDkPBV7ETzROfpdGSwxmcgtUAGa2LZk9JJV/Ln1rN4+0gRVfUOUm0mrjt/Av8xt2v3mWQhSXLUyXKZRl/IBDxIta0Bte4IHkwc9uYHhVcNjg/fjyzQnyU7Dtfj8nStwdxyoJYMWfcuw+Vj9FciGkhVVdm6dStr1qzhzDPPZO/evXg83XfWFuhEk1jR24k6sYRYobO2Z+yZh9HS8TDtfnUppRhqOs1J7YJSmGSZrmQTnPASXSjPaLqLz4PW2vuas+EMumzPQC4YCz4v3vIdUZ2r0r//2JMi5dEweWwh27TxNLTp9970cblcMHsk6SldheOTSbRh1jS7GUWWaGv3xi0WYHj2uz3D8EkK08aGMJDHjScrzcroonQ8XpVdpV0jYVv21ZIp639bkf7G+xsRDeQPf/hD7rrrLq666ipGjBjBD3/4Q+64445eGNrAJ5rEil7fK4ohxAodws3eo9t1HdceINrwoSSbkqam05lk1Y/1FNGq6BxPX+5Ddhc2Nwrgo1X6qUpAgxU6t7lKrEC9swITJF8YIBzRLqJlSerkRcYXZjUyjLe7RzCuODOk8e8scGKobBlJSluPqydudXrYV95ElqxvX0SKEvU3Iu6WXnjhhVx44YWBn1evXo2iJE9OabCied1+UXCle1HwXvQgNU1Fcxt1kNHVk8np+cjZI1AbjuI7tgfTiClJH5fRwiqaBBQpJUv3IB2NkKT9jEAnhLr4DaR71xpc698F1UerJKFpGpLZhu382zAVTUhofLHuPxrImUX4ju3RQ2F+ubRYcX70e1ylW2n2mPhT+4WcfOoMLo0itd8Y83//o4zdK/6FJEmcMrmAm+dNxlRyEu4N7/lVdbSwwiNVDQ6eeWszdf694XhEAjRXm78fq6VbDdtomTE+j037alFkialjekcyLZZtmKw0K/XNLhpbXeRnxTZfmteti7MDO93FXBhmASCZbfrfoaMRrbUOKSOfGePzWPb5If61pYJ1OztCwaqmoWkwMsMLnujLlPoLMSvGCuMYHdGu+nt1D9LlAE0Dsx1Jjj6TTBmmFyurDZGLleMhlhq/ntiHNB4kdQl0Y3dvXaUviDztaG6n/n9HI56dHyc8vlhUdDqTaPhea2/Fe/ArFJ+LbLmNcZTy4foj+KLIQTB+PzUuK+1uH06Xl8+2VNDc5kbOGaE3JXY2dft7/GLbMWqb2tHQ97nyMmM3kJ1bXMXifYdj1sR88jJtnDG1iBRb72RjSv7aTTWKzjqZqfGXeqgNFeDzUEs2zVoKJ5SEX1Acf2+NKkxj7PAMNA3a3b7Af26PiixJjErXQ76xLvL6moGTbzvAiHbVL2cUgCShtdai+Txx12lFNaYYEnQ609FqKnnlFZ2JxUOSU7LwkdzGycZDpbHV3a1HEw61sVJPBLGmknbt0+TlZ1B98CCO9x70985LrPtI3B5kgiHW4z83ytbKx41eDpQ3d0n9Px5fWyMS4DKl8bvbz+GPK3ezcU81Ww7Ucvb04bp3296C2ngMOYxnt2W/bhBuXziNGRPiyxTtCK8mpzg9zW7m6R+dkZRzRUssvVmz/J1pGuJI1FGb9Lmq8OjCDt3t+cpZRfiO7dbvkZHTkCSJ+288mXZ3171PRZbw/O1TVGJf5PU1A6PnyAAksOqP4BVJigkpPV+XBWuu7tExxZqgY2Dc1NH0pYuVaFV0DAwvM5kepN1qwmZR8PpUHK7Y91kDotwjpyFZU5GtKSh5JUiZheBq0xvrJkCsGawGiXqQhvfVouoSZiUperLM5gi6tZrPi+RqxadJpGflkGIzMWeqPhbD6EUy3nVN7ZRVt2I1K0wdmxt3GUUyaiD7ms5Rk0g9PrMS8SD9UawqbzqZaRbs1vD+U6h7S5Ik7FZTl/8sZiXqWuf+RlQGctWqVTz33HM4nU7+9re/9fSYBgUxeUW9VZAfp4EM/IE6k+9BdqjoZETlZSU7xKp5XXgOfc3paWVkSA4a49CxNDL/jIxfg44M4M0JjTHaOtHjkdLzQZLRWuoidoMIeV2/97XDo2tnGt3gIwm7G/dJi2anMFe/12afoCfI7DhUj8erdtp7D126sPWAfo0pY3Iwm+Jfxycrg7UvkUwWsKSA6gtEgcKRiFiAMVfVvgyKItScxrI1pKlqIKlOGmBJOhHvvD/84Q8sXbqUVatW0d7ezu9+9zt+//vf98bYBjSxrJg66op6WNLNFVsGq0HAa0tiWNNAa/MLuke5suxQ00nOWNxfL6d99W+5go/4XvonNMbY+kpzteGr3AuSjGnktKD3jKJ4X4Jtp6KtEz0eSTH596801OaamK/r9RvIfZ4iNMWC4m4lx+rjWJ0joI0acrz+spJm1R7IPC3ITmFkQRouj4/95U0RdX63H0pOhwwjbBhPF4/+hBxl5MQIsTYlEGKtVjMiZgzHEr7XnE2gqUjWtB7dQuoJIhrIv//97/zP//wPdrud7Oxs3nnnHeFFRkFg5ZpREPFYo4BZa+phUfC4Q6z6HlGyjFJnOuYpOhHpWPZjIqFpGp5DGwI/l5jqaKuNbZHiLdsGmopSNBHJGpwZrBRNAIsdtbEiofB5tHWioeiITsR+bzlrK/znGIaSpS/i5ozU3zNCpaEwvmutLz1oH2u0X7ezorYtos5vRa2ebT1mWOSweziC+rEOYA8Som/11nk/PRY0TQv2ICMYSCk9DyQFrbU+YnTCuB+kKJ6F/Y2IBtJkMmGxdNTCZGRkYDKJ3J5IxPKHKUcINyWLWGTmgrCmgmwCtwPNm5jO4/HE+gBLZk9ItekYWnM1kjWNylQ9U9dcuT2mcwT2H/31op2RZBOmEbpXmYjQQueM6FiJN3yvaSpKm+51Fo8bE5Cum5qri4R0F2Y1rlWtBj9oi3I7BBn0haOE1lLTpb7W61OpaWxHQtfKjRetrRG8biRbepfFy0Aj2q0Fw4OMNcSqOZvA0067ZMOh2SIbSLlzdKL751bHPvDAaZRsENFADhs2jDVr1iBJEm63m5deeoni4uLeGNuARVNV1Obob4oOKake3oOMN4tVkgIP52RnsqoxyoAF1HTaE1fTMQrVlVHTacnR6zvTGyI3pjXQVJ/uQQKmUSeFPCagPRpnmFXzefU6PklGssXuTXWEMmNbfKmt9Zg0L82qjamTRgTu0RG2NmRJYm9ZI4720AlNPn/YtNqXQWGn2kVjX6uywYGkmHUvRFNRW4K965pGJ6qmkZNhw2KOP/t3MCToGMhRbi3Eq6Zj7CXWqvo9VhhFzWm0+5CBTOIEpf76gogG8pe//CV/+tOf2LNnDzNnzuTTTz/ll7/8ZW+MbcCitdWBz4uUkhXoG9cdUmo2mKITJE5oXHGGWCH5e38GsQpJB9R0tMTVdDon1/iGTUHVIM95CMeKxfiqD0b8vK9qP7jakDKLwu5xmUZOR0PCU74bt6Ml5DHltW385t0tPPXGRp56YyP/88EO3H5NSyPhRUrJRJJjT1bpCGXGFmKtOnwIgHqyKClKDzwMTW01jB+RiU/V2H4odJjVU69fy2nNxWbpiDZ19iA7j007zngbvTmN4+PFMJDSIDCQ0UZO4lXTMeaqwp2OLElRiQxE2w5PG8Bh7oh/cYWFhbz22mt8/fXXrF+/nqVLlwoPMgKdi5OjIRZB4kSIO8RKz3TS0Pc9Yl/lB+oyoyicDnvt9lZ8VftAUjCNnEpGTi77vEUoqPiObse96YOI5+gwsF3Dq4HrWFMpVQuRUWk9ENqL3He0ka0H6thT1sieskbW7qhi0z49hJloeryxB67FmKRz7JC+QNDSC5AlCTlHz2T1Ve1nxjhdbi3UPqSmqeD3CK05wR5DfpYdSdLbV3m8atjwryH5FymTMhKBxdcAT9CB2LK3M1Nj7+qhtej3R40vnfwsW5cOHqGIdn97IIdYI24m/ulPfwr5+ne/+92kD2awEE9xspxZpDeSbapEKeiZ7gDxhlihZ+oPNUcjeF16dlsMY5IzClFrS1GbqlAKx8d1bW/ZVtA0lOJJSJYUMtPgNy3nMiurnmvlD/GW70DzuvUU+zD4AvuPM8MeU1rZwpb24YxOqcResxM4q8sxc2cMp6QwHZfbx6Z9tazeUMaW/bXMObEwYsu0SARC4+3NaKoatRfaVqO3ckov0rNy5NyRSPZMtLZ6ZhW4eRfYdrAOVdWQ5Y4aRa2tAVn10KLamDRueNA5TYpMfqad6kYn1Y1O8gOLwuAHbGVAezX+/Uf9vIMnxGr8/UUTwcmKo6uHIbzR1CnzOBLR5E5oqi+QpDMQfw8R/1r27t0b+G/79u289tpr7N4d/T7NUKRjXy02Awnh096TQTJCrMk0kIEQWIwrfGMvI5G5Ot77y0y14MLMhtZhyHkl4HXjqwh/n6tNVfr1LSl6tmoYtuyvZbtbNzK+sq0BcefOSJLEmGEZTC7J5txZenRm28E6fKqK5jA8yPjqx/SQdIYekm6PLiTd1OrC5tQ92KKSMf4xyoG5ymrcTUGWnVanhwMVwXvS3gY987XKl8GMCV21PIMSdQI9K48PsTqCjo2XwWQgY1LTiaMW0jhvs5YSddeUjhB5+D1IraUWVB9Sag6S2Rr1ePoLET3IJ554Iujn+vp67rnnnh4b0GAgntTyns5k1TQt7jpIiD5JIBbiTcFPJKlJ01S05mq8R/3JNSV6co3dasJqUXC5fajDp0NtKd4jmzGNmh7yPEHqOd3o2m7eX0u1moHHnovZWYev+kC34uVFOSkU5qRQVe/g6z01TG0xDGT8QttSSqa+v93WCFF4olsO1DFS0Y2pJa/DC1RKZuDZ8yneI5uZMf46Vm8oY/P+WiaM6DhnVelhsoBWc07IBsf6a3UcKG9i+PgMUum60KlsSDzEqvm8/rChhJw58MoLjqdzklwkOcSsOPYgDQPZpNo5JUoDKdkzwWxDc7WitbeGXHgP5PAqxCE1l5OTQ3l5eU+MZdAQV4i1p1sTed3g84Jihm7ChuHoSBJIntyc2mDU2cXpQcYxV661S2l7exG4nchZw4PqVI2Vd2vuCYDuZYaT9gp4oN2EV+ub2zlS1YrFrGAbOwsAXxTlHjP9TWr/e/kONm7VZeoSUSAxjGu03v+O/VXkyK1oSMjpHfNjKp4Cigm1+hCzRulzdfw+ZGO53o3elhscXjUwOnKs/PcR7n9jL25N0Y23WzeKjnYPTa1uTIpMToYt+i95HGpLNWgaUnregCtOD4VksoLFDqpXF8Xvhsy02MUC1ICBTIk6xCpJUsQyoliz1PsbMe1BaprG9u3byc3tnTYvAxHN69aVRCQ5oMIfDZ1vNE1Tk9J5IGhcnfYfYxXjBr3tFYDaErsiSzh85XrNYax7rp0TmmKZK01V8e5fp58jaxiWWZcFvZ+VaqGq3kG9qYD0lCy0tnrU+jKU3FHB53E78B3zq+eMCFbP6czWA7rxmDI6B8voApw7VuM9shnrnKu7Hec3Tipmb1kjR2vaSPU0gLlj/uNBTsnUBd6jNJCO2gpkCXypuUhKxyNCMtuQc0tQqw8wOqUNu1WhoraN6kYnBf6sx/SmfQDkjTsh5LlnTshn7Y4qWpwe2t1eanwZFJsa9P3k/DHsOKwvwMYMSw/a24yVWLOjBwKSPQPN7URtb0bpZpsk1hCr0ZrPp0k4NGtMjanlzCLU2sN67kSIfICB/nuIaQ9y3759DBs2jMWLF/fG2AYk+oa0hpSRH1NLKcmaimRLB6+7ZyTd2lv814k9vAogZeQlpOt5PGpzte5BWuyBdlpRj8WSou+r+TwxebS+6gNo7S1I6fmkfOtxzONPC3o/INPV5gmEVkPpqHrLtoPmQyma0O1+riHqPWN8HkrRRDDbURsiq+oUZqfwy++cwjnTh1PgD3UmkokZi/fv9amY/AIB5hCrfmMhJ7VUBfohGqIB1aWHyKUJh2Zh5NSZIc+fnW7lvhtP5onvn8ai62dR7a+7M/YujXPNTLAZsTbAPZdQGH+7Wnv3HmQgxBqlbKJR29ys2rGYTYHPR0OHTGYEDzJzYP4eYt6DFHRPIu11Ai2AmiqR03IifyAGEknQgQ7lDK2pCrW5GsWf9h8vgRDliO738MIhZw3D52zW2yWlRRfR6Jx1GsqL7izTZRp1Ep7dn+It3Yz1OE/TUMU5Xpy8My6Pj12lukGaMU73xEwjp+E9uB7vkS1Ypl4QcbwzR6eQcbgdD6aEmv12JFhFFnmoaXSSJ+tGWckOYSA7bQXMGD+Jr3ZXs3V/LRfMHknVtnWMBqrt4yg0RQ5rFuaksMmSA5RSV1ZK0fgzAl73jAQN5EDf+wpF4G+3PUrB8iiF9wP7j1oKhTn2mCJMkbY7AnkGA7TUJuyT6cYbb+x2ov785z/3yIAGOoloP0qZRVC1T7/Zik9M6rg6aiDjl9ySM4vwNVXp4ZREDaRfWaa7Pbzux1KI79gefa5GTO32WM3twL31Qzz71vqvGVr1pnNoShlxIihm1JqDqI6mgFC4rp6zFdCTVsKx63ADHq/KmGHpgT0h06gZuoEs3RyVgRyf7sINVHvT0Zrd1DY5aWx1M+fE2B76HSUCkT3IynoHBYq/tCTEQ62zesr0GXlIEuw+0sjSj/YxrUJPfOpuXo4nrWAEVG+ipeoojopmWp0e8rNsDEs0gzUQ2huYD+ZQdHiQoQUnDNJSOqvpqBG7oRih92bVHlN4FbqXMtQ8Lr0ZgawgRbmI7W+ENZDf/va3AVi9ejWtra38x3/8B4qisHz5cjIy4hcQHuwkkloeWJ1H0UImVhKpgTSQM4vwsSXh8WmqD9+xPYCuNBPvWCC6rF/31g9xb1yu/2BN1cOdIchK7/AgJZMVZfgJ+Mq24juyBXnyOfr16o/q6jnp+d2GjTqHVw2MMgnfsT1RNVGWW/3i0WoGZXtreP+LQ7S1exmRn0pxfgx1ozF4kFX1Tgpk/V4JJbTf+YGYZjczaWQWu4808tGGI1ycXQkSjJp5WpfPhWP4mLFQDVJrTdCcxbNP3pmBnhwSCuNvN1LLK1mSyEi10NDioqnVRV4EVRwtIQMZPh8g8DvIKEyoYXhfEnZpcdFFF3HRRRdx5MgRXnzxRS644ALOO+88nnvuuYTqICsqKrjhhhu4+OKL+dGPfkRbW9d4utvt5u6772bevHlceeWVHDhwIOh9r9fLNddcw3vvvRf3OHqKeGogDXqyL6SxbyHZ0uM+R7LGp9dGefXaqDgNdqCGLopaSCOca55yPimX/Cwo8aQzWanB2X8dOqqbA8cY11NyR4V9iGuaxhZ/P8MZ4zoMpGRLQ0rNAdUblQqQYfxrfBn8fe1h2vzap5EaFh9PLDWslfUO0iT9+4dqYG08ELXmGjTVyy3zT+C6b07ghrOHYZJUVHMqaZlZUY9t5Bg9EmFXW/l8q74PmfD+o9uhyxAq5oRC0/2NgIGMEGKFztGQyPuQWhwZrIExWez6/eXzBNqcGcSqKNYfiZik09DQgMvVEctua2ujqSl+weqHH36Y66+/nlWrVjF16lRefPHFLscsWbIEu93OypUrue+++1i0aFHQ+7///e85fPhw3GPoSbQEQjs9aiATqIE0SFYpSiKLCAMpKzppPrWtAbWuFEwWrHOuRskfE/bY4zshGB6f9+iOQGJSNH/0B4420dTqJjvdyqjC4PmOlNQQNHa/Ma7xZdDs8ARe767VVCgkuz887GwKKVTQmcp6Bymy30CGuFckk0UPl2k+tJZa8jLtXHDKSOZO0kP3pvTYDJLJb8DSpXZaHC7sVoWJI7NiOsfxdM6cTHY2eF8SCLFG8CAhNjUdQ0UnHg8SwtclGwpJA1kLN+Ldc+mll3L11Vfzwgsv8Pzzz3P11Vdz9dXdp6mHw+Px8NVXX3HRRRcBsHDhQlatWtXluDVr1nDZZXpixCmnnEJDQwMVFfrq8uuvv2bPnj2ce+65cY2hJ9HaW/Wb12SNSztTL2gO3QIoGWODxEOs0L1yRjQkQ+FETi/wZ9XWdptVG0gGKp7SrWwcdCTpNLW60TQNOS0XOXcUeF34ju32jz1yEtb6nfr3CxUqjKU/o2GMzcdpmh4ob6LZEX0msaSY9MhBFGo6VfVtpEnt+ufC3Cuhujh09KzMinpcxti8plQUSSNNcjFlTG5UOqDdMZgUdDrT4UFGbmgQS6mH2jlJJw5xhnBdPQaDFm7E9MGf/vSnTJ06lbVr9QSHRYsWMXfu3Lgu1tDQQFpaWqCfZH5+PlVVXT2A6upq8vM76r7y8/OprKwkIyODJ598kpdeeqlflpp0/sOMZw/FaAGktdSgNlehZCdPFD5Q5mGLP0lHSskCk7Vb5YxoSEboRVJMSOn5aM1VqM01KDmh58oIjypRJAN1VtNxuryk2MyYSmbirjuCt3QzppHTA2OXutnb+sowkOO6JiZE3QGhk5B78dixUFlJqs3EqMJ0dpU28LPff4Hsv8dMiszN8ybjUzX++ulBfvqt6QzLDf49SylZaO0tuiELY8Qc7V7a29pQrBqYLGEXFHJmEb7yHUEegxqngQS/19nQRobsCIgkJMKgN5BReJBGR4+mKEo9vC0NyIDPmkGKLZ6M8tCRpcHwewg7GwcOHGDcuHHs2LGDoqIirrzyysB7O3bsYMqUKd2eeOXKlV1KREaPHt3luGgNiSzLPPzww/zwhz8kLy/+PYrc3Pg9KID8/PB7eK3VbTgAW15Rt8d1h2/4WBx7akhpOUzGxMlxjrIr5T4nPiC7qBBbnGMDcOcOx111iAy5BVu+biRi/a4Vjho8QPaoMaQkMBZvfjHO5irStSZS80PPVWn9EQAKps7GnB35WrkZNr3jvcVMfn467dPPoGLj+2hHt5KXl0arvzlswdhxKKldz+f1qRyq0L20s2aNxGYN/hNzjBpD5VowOWq6nTefo5lWrwvJmsJF507jo+2NXDinhNHDMtj7+ga8Pg3QVX68qobZaqamtpXqRidf7a3l1suCH0rerDyc9WWkm1ykhrnuv7cfI9UfXlVSMsKOr2lECXU7weKqDxzTsNuBC0jLKyAnzOfCnc+bnYez4ShjszW+edoY0uyJKd9U1OhiBVljJ5GWwP3V08T6d+NSCygHZE9bxM+OHKaH1Z0eNeKxTS79fs3Iy4vrudU2cgxV68DkrA18Xm1vo6XuCEgyBRMmo9ije+7G+9zsKcIayKeffpqXX36Z22+/vct7kiTxz3/+s9sTz5s3j3nz5gW95vF4mDNnDj6fD0VRqKmpoaCga6ZcQUEBNTU1lJSUAFBTU0N+fj5r165l7969vPDCCxw7dox169ZhMpkC4dhoqKtrRVVDy4dFIj8/nZqa8CnW7kp9xeQxpXV7XHeohVNgz79p2PFvXKPOjOscIcfWqv8RNDklWuIcG4CaVgBVh6g/fACzZVjEOQmFy98pokXKoi2BsXjt+kKpoewgjtyuZTGa24mvtQEUEw1uO1IU10r3P5wPHqnHJoNmLkCyZ+BtrqVy8zo0lwMsKdS1SUgh+jtW1TvwqRq5GVZamp0cf4SK/uBqrynvdt589UcBff9Q8vlY/OMzAu+9dNdcfJ3uYVmSMJtkbP5EwbXbjnHZ6SVB5/Oa9QdU47FjOLJCX/fTjWWk+hN0NHNK2PF5lSwAHJVHAse01+jiB05Cf667+8Rj0h+KN5xViLO1HWdre8jjokFrb6X96G6QFBwZ43AmcH/1JPH83ahOPfTsbWuO+FlF0/eaq+rauj1W01Qkv3RdSnpmXM8tVTLu6aOBz3sOrAfVhzJsEvWtGrRGPm88c5Iosix16zSFNZAvv/wyAB9//HHSBmM2m5k9ezYrVqxgwYIFLFu2jHPOOafLcXPnzmX58uXMnj2bDRs2YLVaKS4u5vPPPw8cs2jRIk499dSYjGNPowY6L8SfOacYpQAVO9E8rqQp4HfsQSa2Qou2i3jYcSSxNqqjLCZ0uDIQys0ojLrNUyA05c/+M7pYePZ8hnvrqsB1w0U+jgVaNYXey5HS80BW0Nrqu/39GmpKoUKWJkXGFCJrfvyITFJtJqrqHVTWO4ISLiJlsqqaxpYDdQwzEnS6CZ93dOEIsQeZ2nW8kQjUmCZBQer4NmaDic57kJEFy6Pcg3Q7kdBo18wU5sX3bJAy8oNUtiSTJSoxjYFAxKfGgQMHePfdd9E0jdtuu41vfvObrFu3Lu4LPvjgg7zzzjtccsklbNiwgTvuuAOApUuX8vzzzwO6SIHb7Wb+/Pk89thjPP3003Ffrzcx6szkOB4SBnJKJnL+WPB58ZbvCHucr+4I6nFp1WHHpXrB4wRJ0gWPEyCRThqdPydnFCRcGxVIGooklBzDHkioB4uxf+k7uj3i+QKtmsIZSFkJ1Beqzd300YtjT0+RZaaN1RcdK9aV8uX2Y4H/DjXqD9NwRqi0soWmVjcFdt3z6C7bWUrLBdmE5mhE8+jenrEHGU/fymT2Go2mifVARVLMYLKC5gNP9152VkCwvPs9SGPh3KZa4+6eYqhsgYbaXKNrHkchpjEQiLgj++CDD3L11VfzySefUF9fz+OPP86zzz7L22+/HdcFi4uLWbJkSZfXr7vuusC/rVYrTz31VLfnefLJJ+O6fk8Sbybf8ZhKZuKuOYjvyGbMo2d1eV9trsbx14eRMwtJueqxiPu4gRpIa1rCae+JlqKoDeVB50loLP5EGV9DechmwPGUkxgPloZOMl2m4qmgmPRuKJ2uG4qqCB4k+AWeG4+h1nUVQg+MPc57acb4PNbtrOLzrcf4fGtHpux0czO3pkNLfQ2hemQYGqjj803Q0L0HKckycmaBrivbVImSNzqhez9ZvUY11duljdlgQ7KlobW69CS5bha7hppOq9PTrZqOkfDTplkpiKPEw6BDZesYmrtNF9PIKBywGqwGEZ+WLpeLyy67jC+++IJ58+YxZ84cPB5PpI8NSZJpIAG8pVvQtK51a97Dm0D16Q+oxorI4wrUQMafwWrQOQsz1Ngi4T3iX1kOm5TwWKSULKT0fHA78FUf6PJ+PHJjBdn6Q6eqwdlxHbMV29zvYRp/OuYTz8M8qeu2gEFlBA8SOr67t2xb2GOMeynWaMTJk/K5eM4oTp9SGPSfPUv3LNubQtdQGrWVo/yXi5Sh3CHUUImmqoHoSTxtuZLVa9RXuS9kG7PBRLRyc4aaDnTf9kp16udp0yzkZsS/ndN54ezr1AouUUWkviaiB+l2u6mtrWXNmjW8/PLL1NbWBgkHCDowCm7jCTN1Rs4ZiZSao7dbqjncpR1UZ2UXb+mWiOUgAeWNBGogDYxOGnoD3gYg+gdiZx3TeDVYg8YiSZhKZuLZvhrfkc1dGhHHE2I1PD/DEzQwjz+tS/ePUERjIE0lM3Gtewtv2dawknPd7UF2h0mRufrcrm2H9u+xw7/+D8XVtQ6yocVFaVULFrNMns2Hj8iCEkESY+0toKl6hCKO3ovJ8iCN8KoyCMOrBrGUemSl6XJzjW3usHJzLn/yXjt2bJbYSzwMOnITqlBr9MVqMv7G+5qIHuQ111zDueeey8knn8z48eO56qqr+M53vtMbYxtQaB6Xvs8nmyBBT8148EOHqHfgOq42vRehH18nYxl2bO2Jq+h0puOPIXKxe2f0Fb4DObMoabVRRhLA8W2pOtcRSjGEWAuy7EhATZMTry82D9np8uqdQBSZ3G6a/cqZRbq6iNuhz0kIVKfhkWXFNIZwjB6rS7qlak4q64IfrkZ49cSSHCS3IUkYyUB2CB4kGjkxvE7N0RRXVMLA26lby2AlLrm5brp6OJv1+8xnTiyhyYgs+cq2drSxO27BOhCJuGS4/vrrufbaa5H9+zt//etfyc4ePPqGyaJzFl8ywgqmUTPx7PxYb7c0W69BdW9fjWf3p6D5kPNGo9aX4avaF7Fov0OoPDk1RnJWEb7KvTHvQ8ZStB8tyrBJ/j6L5ajNNcj+JtWas0lPZLCmxrQwMJtkcjNt1Da1U93gZHhe9Iudan9YdlheasRmv6aSmXi2rsJ7ZDOm4V1rOI3ejYlGIwLXM1tol+zYcPLK/60PCoUa4545IQ/taHSKS4ZQgrd0C2qdvyQlzuQ0STEjWdN0AQpnS0xh2vZ1byGnZmMaOR2tqUoXow/RuHew0CE3F1lNx+gi051YgKu1CRugWRJb1AcS5vwLu3jb2PU3InqQbW1tPProo3znO9+hsbGR5557LqTA+FAnESWRUCjDJ4PJglpXitpaj+Zpx7XubdT6MgDMk89BGTYZNC0QtgxHR4lH4nuQAHK27o34qrru+3VHYG8iianfep9Fvd1V59Czr2o/AEp2ccwLlqLc0GHWSOwv1x8OJUWRFyLGHPhCNGTWNK3Tnl5WTGPoDsOAuZrq2H+0KfBfc5sbi1lmxvi8qDV7lZxiMNvA40Rt0A1kuISjqMbmX9iotaVRf0ZTVTzbVuNa9zaefV8CeneYgdo5Ihpi8SCz/drCdU3hM169jsQaqQfGlZIVVLZlGj8nofP1FyKa+EcffZSCggLq6uqwWq20trbywAMP8Mwzz/TG+AYMWgJp7qGQTBZMxVPwlm7Ce2SL3llB9SLnlmCbewty7khQffjKd+At3Yx5whlhz5UMHdbOmEZOw7WWwB5aNKhNlbrHaUlJeujFNGom3oNfBTUi9pbqoWkljnZaRdkpbKc+sJ8YLUao8uTJkSX0lKLxYEkJzEtQyNnVBqoXLPak1cECpGTl4ms9xq3nDcdVEKyElZdpIzPVQmuU94pkSSH16idQW2r0n2UTcl5Jt5/pDtPIabhrDuke9ajofmeSLKMUTcB3bHegRnUwh1chNgNp7IN3dx+r/vMoKYlFlyRJInXhw/gaK5As9sAieqAT0YPctWsXd955JyaTCbvdzuLFi9m1a1dvjG1A0bEPE3sWXziUTu2WAvVdY2ej5JX4C9j975dt617cPAmdPDojZw1DyiwEVxvtR/dE9ZnA+Htgha+Mmg5I+Cp2o7mdaJqKryz+hsyFUTxYjqfd7WX3kQYkYPYJkQ2kJJsCvTANY26QSE1ht9f0C1gMS/EwcWRW0H85/j1TI3QXzWJKTs3GVDQRU9FElIKxCf1eO+8la1r0SleB36/PA5KCKULz7IGOkYkeKYsVoDCQkd3Nfex/NlhSE+/xK9nS9HshZ+SAz141iGgg5eNqy3w+X5fXBB0ZrImo6BxPoMFu+U68pRv11zo98OWMfOTsYvA48VXuDXUKIPkepD42fRyOfRuiOt5INuqJFb5sS0cuHAeqF8++L/Ae+hrN2YyUlqvPT4zEE2LdcagBr09j7PCMQNusSJj8RdTe4xKtklUudDxyp2SYUGg+j75vKylgTkxQIlbk/NFI9ky0tnp8R7aE7dCiuR268IWfzveTMmxiUkqZ+jNGHkE0e5DGQq+6wYkvTJszxavf47aMxA3kYCSipTvllFP49a9/TXt7O5999hm33347c+YMjvhyMukIsSbPg5RTspDzx+irY1eb/4EfHLoI9CwMsZcVGFuSs1ih48EUjYEMZN5KMqaR05I2hlDjcX3xOu0f/T7wWjwrWUNRJBYPcttBf4PkGJr9mkZOB0nGd2wvmruj7tJoPJtsA9mhWNMQ8v3Oe9W97QEYkn4Azg9/g+P9x7p4kmpLDa2v30n7p68GXuucET3QZc2iIZYQq9WskJNhxadq1IbZhzT79PsuLUMkXoYiooH82c9+RkpKCunp6Tz33HNMmjSJe+65pzfGNqBQm3WxZik9P8KRsWGddRly7kjknBFYZy/s8uBS/Ioh3iPhQ1PJzmIF9H1Eix1PXXnE1k3esm2g+VCKJvTYCt888SyUYZOQs4fr/xWMxTzlm3GdKzvdiiRBs8MTdanH0Rp9RT9hRPQLJMmaipw9HDRfUMmMt2InAEoCe3ohr+ePboTzIAP7iQlq5MaLeeoFusyiYkKtLe3SM9N7cAN4XXDcvrfl1KtQRs3EPDF54v79lViaJgOB/o7hoiE21W8gc4SBDEXEJJ0XXniBu+66i9tuu603xjNgiUfWLBpMJSd1K5ulFIzTU+Sbq1GbjqFkDe9yTLKzWKFjD8174N94j2zGMu2isMf2ZHjVQE7JImXBvck5lyyRmWqhsdVNU6ub3MzwNY2gZ51W1kUWCAh5rcwi1PqjumRbwVi/mIIhlzYzrvGHvZY/uhFOsUaLQ3komSi5I0m98gGcH7+Md/9afKWbg+7ncHWO5jGzMY+Z3Ysj7Tti8SBB3y7YVdpAZZ2D6eOC31M97ZglH15NJiOjf7WZ6i9E9CDXrFnTC8MY2KjtLXrmodmGZE9eiDUaJFn2J6l0UzLg6tBiTSZRhXc7q+eMGjj6mJkxdGRvdXpwuLzYLEpA3itajte29VXtB1dbUsUUDKRUvwfZFjrE2l8a3BoLws73ldbeqosqSEqPhekHBGabvkfsdYXdp+1MYLugk3SigSES4NCspNgS68E5WInoQY4YMYJbbrmFWbNmkZra4YF897vf7dGBDSQ6r7z7InvLVDIT774v9TKHGZcEv+l2gKbqxltJbuFu8B6aI2R7IeOBL2UWJt277kmy06yU0hKVgewsLxfr798QPjd0Y402QckUUzAwFm+aszmMuLu/PVgf/55MI6eCpOCr2kf72qWYJ89FrSsFTUUZfsKga2MVC5IkIdlS9d+hqw3J1P2CrLuEs+aGBtKAdrl3E7IGEhGfmFlZWQCUl5f39FgGLH298jaN8D9QKruq6gQky5K4/2gg2dKwjZxM+5GdeMu2Yx53apdjvD0gDtAbZPn7QjZGaBcE0emvhqOzpin0jJiCgaSYkGzpaO0taO3NXZKAjD2/vvYgJUsKSvEJ+I5ux7PtQ9T6o4H7d6DdRz2BZEvXDWR7K6R2v3fYXclSQ20daYBXEQYyHBEN5BNPPBH4944dO5gyZUo3Rw9Nemr/MVokSwrK8El6OUjZ1iDRAM3wCjIj1+bFQ8r4k3UDWboppIH0DVB9zFhCrFX1eviquxZX4egcYg0WU+gZuTQpJVM3kI4m6GQgNVVFbar2j6ln7pVYsJ1zC97DX+Na+ya+Y7vB7ykNtPuoJ4hFsDwvw4ZJkWhoceFy+7BaOmpVj5ZVMRIwJ6EGcrASU0HjL37xi54ax4AmnrZKySZQaH2cuHlPe7cpE/TkCF1VJzjjU22q0rMzB6BwseFBRmo4C5GbJHeHZEvTvSOvC/euNYAhptAzOpYdnTOC9yG11jpQvUip2Ujm7pOSegM5LQfL1AtQCifoWatuJ3LWsH5hvPuajpZXkQ2kLEsUGJmsnQQDVE2jplovTcoUGaxhERX/SSCwd9OXBtJQ3SnbGlRI3dPG25xbjJShq+r4qvcHvRfIOuzBB35PEYsHmUiIFdBViQDP1g+BDgGBnkAK9F4MLvXoL+HV41E6hVQHcxurWDCy0aPNZDUUdTqHWQ8da0bx6D+nZgkDGY6YDGQsElBDBU1VUZsNI9R3q1s5owA5azi4nUEtlHo6/CtJUofij3//THU20/rWPbjWvgUMzLBYdsBAdu9BqqoWaK5cmBPfXk5H13XNL6YQu35s1NcyPEi/8pOB2sOh+HjpfO90V+40lIi1FtJYuG3cW8PdL37B+l1VbDtQR6qsL/7kJCpsDTYiGsjOouTLli3rybEMSHzVB8DnRcooQLL07Wa3MvwEANTaw4HXesO7DXivfgPpPfgVWnM1oCGl5wUM6ECiI0mnew/yQEUTXp9KQXb8DWdNo2fqfUQB04TTe1QuLVxz4o5IwzD6E3LWMJRRM5ALxw/qNlaxEHMtpN9Art9VTV2zi4raNvIy7WSa9MVfssu/BhMR/6LXrFnDXXfd1RtjGZAEklD6gREwvES1UTeKmtupPwgVE1JaTo9dVxk2Ue/J2FiB2lwdCK1az/ku5klnI0kDL5KfnmJBkvQaR69PxaSE/g6b/R08ZoyLXmLueMyjT8Z0y8ugqUhKz9ajdcjNNQa93hFp6F8epCRJpFx8Z18Po18R0GONNsR6XOh/6thcxhdn4ihPxVeWXI3mwYaog0yQQBlDPwj/BGrq/A+7gPeYUdSjRkpX1ZmG9+B6PAf+ja98FyBhGjVzQBpHiF5NZ+v+OgBmjk9Mnk3vhNHzfQzlwB5kY9DrfV2qJIieeEOsAOkpZsYO07NWe0KjebAh6iBjQPW48Bz4N3jdejhT01AbysFsRyma2NfD61RTVxn0/94oPzGVzMR7cD3uTR/ofSsLxiZVuL0vyEyz0tjqprHNFdJA1jQ6Ka9tw25VmDAyq/cHGAehQqya161nsUpK0rWEBT1AjCHW9BQzKVYTDpeX6WNzkWVdzKJDo1kYyHBEXQfZ3NxMxhBvidK88R+0//NVAOSckZgnnwPoyh/JVqmJByk1FxQTmqMRze1Erdc7vfeGV2Co6uCXvxoMBd2Gmk5Dswu6StwGwqtTx+SGDcH2N/R+pRKaownN40IyW1GbjUhDftJ7dQqSjxyjgZQkieF5qewvbwrqNmP0lOwJEZHBQsS/6kOHDjF//nzmz59PVVUV8+bN48CBA70xtn5H6uQ5mE/8JphtqPVleHZ+DPQfYyDJMnKG34tsrsLrbxqsDJvU89e2pWGbeyumiWdhnnoBljg7afQn8rP0pKvqxq46lgBb/QZyZgwtrvoaSTHrLdQ0FV+53jXESNCRRHh1QCDZ/SFSZ+iuLKG47vwJfOvcccyapEcINH9tKUgwhKX7IhHRQD7yyCPcd9995ObmUlhYyLe//W0eeOCB3hhbv8OcWYDtrBsDCTlq4zGQpIBYeH/A8Ba9R3ei1pWByYoyfHKvXNs88Uzs3/getjNuGBSNaw0dy1AyXU6Xl91HGpEkmDaub9pDxUsg6/iIrvva10pQghixpIBiAU97UB/R7hgzLIN5c0qQJSO86m+4bE3poskr6CDizDQ2NnLmmR191m644QZaW6Nz7QcrnTNWlYLxyP0oRGE85DzbVgG6TmtPZ0YOVopCFFgb7DhUj0/VGF+cSZp9YM1vR1nOFjRNFQk6AwxJkvyh8vC9PSPR0QJP7D92R1RLB5fLFehSUFNTg6pG10R2sBLYb6Nnui4kgvGQ05zNwMAs0u8vGOnxoTohbDHKOwZQeNVAzhmJlJqD5mzyNybueyUoQWzIfpFy1RG6dVkkAgk6IoO1WyIayOuvv55bb72Vuro6nnnmGa655hquu+663hhbv0WypemGR7FgHntKXw8nCLlwHOBvuWS2CXmuBMhKt2Ixy7Q4PLS1e4Le21+ur9ynjO65+tKeQpKkwMLJc2A9am0pAHJ2iEwkQb+kIxtZeJA9ScTUy6uuuoqSkhLWrFmD1+vlV7/6FWeddVZvjK1fYzvvh2huZ78rZVCyhpN63a/RHI1IabnI9qGdeZwIsiRRlJ3CkepWKusdjBuu/669PpWaxnYkYFjuwExwMI2agWfnx3h2rAafFzl/jLhXBhAdIdb4PEiEgYyKqGoTxo8fT2NjI7IsM2OG8EgAJJMlYrPSvkJOz4P0gRf6648U5ugGsqqTgaxpdKJqGrkZNizmgVkWoQw/QW8hZZTliFD8gEJKMUKs8XmQqhAJiIqIIdbVq1dz4YUX8tprr/G///u/XHDBBaxbt643xiYQ9DmhGs4a/R+LBqj3CPoCz1Tc0du1v5QqCaLDiFwdLzofNUIkICoiepDPPfccr7/+OpMm6bV0O3bs4Be/+AV//etfe3xwAkFfM8xvICtqOwxkoL1V9sA1kKAnmHlLNyGlZiPnjurr4QhiQPIn6cQbYhUyc9ER0UDabLaAcQSYMmVKIKNVIBjsjC3W9+V2lzYERMuNxrPxtrfqL5jHnorv6HZMo2eJv+kBhrEHGW+IVcjMRUfEEOucOXP4wx/+gMPhwOVy8fbbbzNhwgSamppobGzshSEKBH1HYXYKRTkpOFxe9h/VH0aVdX4PcgCHWAEkix37+bdhHn96Xw9FECNymLZl0SKyWKMjogf55z//GZ/Px7PPPhv0+vLly5EkiV27dvXY4ASC/sDM8XmsWn+ELQdqmVySTWXD4AixCgYwlhRQzAE1nVh70QoPMjoiGsgdO3b0xjgEgn7LjPG5rFp/hM3767jszDE0tboxKTI5YVpgCQQ9ja6mk4XWUoPmaIrdQIo9yKgQInwCQQTGj8gk1Waiqt7BV7urAb3Hniz27QR9SKC3Zwyi5QCapqG161qswoPsnl7v0VRRUcHdd99NXV0dY8aMYfHixUGNmAHcbjf3338/27dvx2azsXjxYsaNG4fH42HOnDmMHDkycOx7772HogzMWjTBwECRZaaNzWXdzir+skbvZDN1zMBT0BEMLqQY214F8DhB84HJKnSaI9DrHuTDDz/M9ddfz6pVq5g6dSovvvhil2OWLFmC3W5n5cqV3HfffSxatAiAPXv2cNJJJ7F8+fLAf8I4CnoDQ3O11enx/zywOngIBiFWw0C2xPQxkaATPWENpNfrDfuh3bt3x3Uxj8fDV199xUUXXQTAwoULWbVqVZfj1qxZw2WXXQbAKaecQkNDAxUVFWzbto36+nquvvpqrr76atavXx/XOASCWJk6NicQUk2xmhg/on9JDAqGHgEP0hWbByn2H6MnrIH81re+Ffj3I488EvTevffeG9fFGhoaSEtLw2TSI7v5+flUVVV1Oa66upr8/PzAz/n5+VRWViJJEt/85jd5++23eeihh7jzzjupr6+PaywCQSyk2sxMHKkbxWnjclFEDz1BHxNviFVksEZP2D1ITdMC/964cWPY98KxcuVKnnjiiaDXRo8e3eW4aAuUZVnm2muvDfx84oknMn36dDZu3Mj5558f1TkAcnMTuyny8/tP78f+wlCZk//45kR+9+5mrjx3QsTvPFTmJBbEnAST6Hw05+VRC1hxx3SuliofTsCWmdXvfif9bTxhDWR3hisaozZv3jzmzZsX9JqRZOPz+VAUhZqaGgoKCrp8tqCggJqaGkpKSgACxy1btoxZs2YxapQui6VpGmZzbJvMdXWtqGpkAx+K/Px0ampii/cPdobSnIwrTOO5n+idbLr7zkNpTqJFzEkwyZgPj0d/fLc3NcR0Lnd1jf5/bP3qd9IX94gsS906TWHjRNF4ibFiNpuZPXs2K1asAGDZsmWcc845XY6bO3cuy5cvB2DDhg1YrVaGDx/Onj17+OMf/wjAwYMH2bVrFyeffHLSxykQCAT9HRFi7XnCGsie0mZ88MEHeeedd7jkkkvYsGEDd9xxBwBLly7l+eefB+DGG2/E7XYzf/58HnvsMZ5++mkAbrvtNurr67n00kv56U9/ylNPPUVamvglCwSCoYckslh7nLAh1j179jBr1iwA2tvbA//WNA232x33BYuLi1myZEmX16+77rrAv61WK0899VSXY9LS0njhhRfivrZAIBAMFiSbXj+uudpi+lxHFmtqhCMFYQ3k6tWre3McAoFAIIiBzmUemqYiSdFlVhsGVbL1r4SY/khYA1lcXMzevXs5fPgwM2bMoLCwsDfHJRAIBIJukGQTmO26Mo7bCVF6hCLEGj1hlxz/93//x7e//W3+53/+h8suu4zPP/+8N8clEAgEggjEk6hj7FkKoYDIhPUglyxZwgcffEBhYSGbNm3iueee46yzzurNsQkEAoGgGyRbmt7Rw9UKRBflE1ms0dNt0NoIq5500kk0NDT0yoAEAoFAEB1Gok20HqTmdYPXDZICZtGuLRJRl3kIUXCBQCDoX8QaYu1I0EntsVK+wUTUgpJiMgUCgaB/YWSiRm0gAwk6IoM1GqKqg4SOWkhN05AkqYs+q0AgEAh6l4BYQJQdPcT+Y2yIOkiBQCAYoATEAmL1IEUGa1R0WwcJsH37dr7++mskSWLWrFlMnTq11wYnEAgEgvDE7EEaJR42oaITDWENpKqq3H333axbt46TTz4Zt9vNyy+/zKmnnsozzzyDLPrhCQQCQZ8Sc5KO8CBjIqyBfPXVV/H5fHzyySdYLBYA2trauPfee/njH//I9773vV4bpEAgEAi6En8WqzCQ0RDWDfzb3/7GI488EjCOAKmpqTzyyCO8//77vTI4gUAgEIQn9hCryGKNhbAGUlVV0tO7TmJmZmaP9IoUCAQCQWzEXObhEiHWWAhrIB0OB6qqdnldVVU8Hk+PDkogEAgEUWCygGICnwfN64p4eMCQihBrVIQ1kHPmzOG1117r8vorr7zCGWec0aODEggEAkFkJEnq1Dg5shfZEWIVWazREDZJ58477+S6665j27ZtzJ49G6/Xy7///W8OHjzI22+/3ZtjFAgEAkEYJFsamqNRN35pud0eK0KssRHWg8zJyeEvf/kLkydP5tNPP+XLL79kxowZvPvuu2RkZPTmGAUCgUAQhmg9SE1VweXwf0Z4kNEQ1oMESE9P5/vf/35vjUUgEAgEMRIo9YiQyaq52wANrKlIsmg+EQ3dGsjVq1fzyiuvsGfPHux2OxMnTuSWW27hnHPO6a3xCQQCgaAbot6DFCIBMRPWQC5btowXX3yR//qv/2Ly5MlIksTWrVt59NFH+dnPfsaFF17Ym+MUCAQCQQii9iBFgk7MhDWQS5Ys4dVXX2X48OGB18aNG8fMmTO57777hIEUCASCfkC0ajoiQSd2wibpeDyeIONoMGbMGFyuyPU2AoFAIOh5ojaQAQ9SGMhoCWsgFSX8Jq5Q0hEIBIL+QbRyc0KoPHZESw6BQCAYwMQcYhUeZNSE3YPcs2cPs2bN6vK6pmm43e4eHZRAIBAIoiPqOkgRYo2ZsAZy9erVvTkOgUAgEMRB7FmswkBGS1gDWVxc3JvjEAgEAkE8WFMACdxONFVFCtPMXmSxxo7YgxQIBIIBjCTJYLbpP3icYY8THmTsdKukMxTQNI2Ghhrc7nag++zc6mo5ZAuwoUznOVEUE2lpWdjtohBZIOhNJGsKmseJ5naE1VkVWayxM+QNZGtrE5IkUVg4Ql+JdYPJJOP1CgPZGWNONE3D43HT2FgDIIykQNCLSBY7GqC5HNC1zz2apoks1jgY8iFWp7OV9PSsiMZR0D2SJGGxWMnKyqe1tbGvhyMQDCkkSwoAmjtMiNXTDqoPTBYkk6UXRzawGfJWQVV9KMqQd6SThtlswefz9vUwBIKhhcWu/z+MgRQJOvEx5A0k6N6PIDmIuRQIep8OD9IR8n2tvU0/ToRXY0IYyH7GsWMVnHXWbJ5++rGg1/ft28NZZ81mxYoPojrPxo0b+MlPuu/l+dhjD0V9PoFA0H+R/B5kWAMp9h/jQhjIfkhmZib//vdafD5f4LV//nM1WVnZfTgqgUDQX4m0B6m1t+jHiRBrTIjNt36I3Z7ChAkT2bJlE7NmzQZg/fp1zJ59KgBffPEZ//M/L6FpKsOHF3P33feRk5PL+vXreOGFZ7FYLJSUjA6c7+jRMhYvfoLm5iasVht33nk3EydO7ouvJhAIeoKIIVbhQcZDrxvIiooK7r77burq6hgzZgyLFy8mNTW4JMDtdnP//fezfft2bDYbixcvZty4cWiaxosvvsjq1atxOp386Ec/4oorrkjq+H7z7ha2HqhL6jkNpo/L5Y5vzYjq2HPPvYBPPvkns2bNZteuHYwfP8Ffs1nPH/7wIi+99ArDhg3nzTf/zLPPPs0DDzzCY489yPPP/zejR4/hyScfCZzrscce5M4772HixMkcOnSQ++77GUuXvtcj31EgEPQ+klU3kAgDmVR6PcT68MMPc/3117Nq1SqmTp3Kiy++2OWYJUuWYLfbWblyJffddx+LFi0C4P333+fLL7/knXfe4fXXX+fpp5+mubm5t79Cr3DWWWezbt2XqKrKP/+5mvPOuwAAm83GCSdMYdgwvVfnZZct5Ouvv+Lgwf3k5uYzevQYAObNuxQAh8PBrl07efzxX3Hzzdfz8MO/wOl00tTU2CffSyAQJJ+OPUiRxZpMetWD9Hg8fPXVV/z+978HYOHChXz729/m7rvvDjpuzZo1/PSnPwXglFNOoaGhgYqKClauXMktt9yCxWIhPz+fN998E5vNltQxdufh9aZQQEpKKuPHT2Dr1s1s3PgVP/zhT/jnP/+Bqgar/Wia5t+rlNC0jrEZ/TxVVcVisfLqq28G3quuriIjI7NXvodAIOh5AgbSJbJYk0mvepANDQ2kpaVhMul2OT8/n6qqqi7HVVdXk5+fH/g5Pz+fyspKSktLOXDgANdccw1XXnklO3fuxGIZvEWv5513Pv/9379j0qQTA3PmcrWzc+c2jh2rAOD9999j1qyTGT9+Ag0NDezbtxeAjz76EIC0tDRGjBjJhx+uAOCrr9Zx223dZ7cKBIKBRcQkHeFBxkWPeZArV67kiSeeCHpt9OjRXY6Ltm5OlmV8Ph979uzh9ddfp7a2luuuu44TTzwx5HnDkZsbfINUV8uYTNGvE2I5Nh4URQ5cZ+7cb/Dkk4/ygx/8CJNJRpIk8vLyWLToF9x//914PB6KioZx//0PYLNZeOSRx3n00QdQFBOTJk1GkiRMJplf/eoxnnrqcd5888+YzWYee+xJzGYFSZKQZSnh73T852VZJj8/hN7VEGKof/9QiDkJJpnz4ZbycQCK2h7yvEfdrfiAnOFFWPvx76G/3SM9ZiDnzZvHvHnzgl7zeDzMmTMHn8+HoijU1NRQUFDQ5bMFBQXU1NRQUlICEDguLy+Piy++GLPZzLBhw5gxYwY7d+6MyUDW1bUGhSlVVY06bNobIdaCgiL+8pcP8HpVLBYb//zn5wB4vSr33fdg4LjTTz876HNer8q0aSfx2mtvdXl9xIgSfvvbl7u8bpwvke8Uak5UVaWmpiXucw508vPTh/T3D4WYk2CSPR9qm/436HW0hjyvp1lPPGx0W5D76e+hL+4RWZa6OE1B7/fiWDCbzcyePZsVK/Rw37JlyzjnnHO6HDd37lyWL18OwIYNG7BarQwfPpxzzz2XlStX+rM5G9i6dSsnnHBCb34FgUAg6HcYWayhQqya6tXrICUJyZbR20Mb0PR6FuuDDz7IO++8wyWXXMKGDRu44447AFi6dCnPP/88ADfeeCNut5v58+fz2GOP8fTTTwNw8803k5eXx6WXXsp1113Hj3/8Y8aMGdPbX0EgEAj6F4oFJAV8HjSfJ+gtzdEEgGTPDNtMWRCaXq+DLC4uZsmSJV1ev+666wL/tlqtPPXUU12OMZvN3HfffT06PoFAIBhoSJKkt7xytaK5nUh2c+A9zdGoH5OS1TeDG8CI5YRAIBAMBsKIBagBAylKu2JFGEiBQCAYBIQTC9DaGgGQU4SWc6wIAykQCASDgEAt5HFiAZrwIONGGEiBQCAYBITrCRkwkKnCg4wV0c2jn/HMM0+xbdsWvF4PR4+WMXr0WAC+9a1rmT//sqRdZ+PGDfzxj3/gd7/7Q9hjPv/8U44ePcK11347adcVCAQ9g5SWA4BafxTGnhJ4XfVnscoiSSdmhIHsZ9x1188BvXHy7bf/IEhDtbfZs2dXn11bIBDEhmnENDzbV+Mt3Yx19pWB1zVHAyCyWONBGMgBQLh+jo899hA2m52tWzfT2trCf/3XXXz44Qr279/L2Wd/g9tvv5MVKz7gX//6mObmZhoa6jnzzLP5yU/uDDr/pk1f84c/vIjL1U5LSws/+tF/MWbMWJYv11tiFRUN49xzz+fZZ5/i4MEDqKrKDTfcxAUXXNwX0yEQCEKgDJ8MJgtqXSlqWwOyP6RqJOlIqVl9N7gBijCQx+FY+Sy+sq09cm5l5HRS5v2/mD/XXT/H2toaXnttKStX/o0nnniYpUvfw2q1csUVl/Dd7/4nALt37+RPf3qT9PQMbr/9B3z66Sekp3coavzf/73NokW/pKRkNF9//RXPP7+YP//5bS6/fCEA8+dfxksv/ZZJk07gF794mLa2Vn74w1s48cSplJSMSsLMCASCRJFMFkzFU/CWbsJbuhnLiecKFZ0EEQayn9O5n6NB536Op512BgCFhUWMGTOO7Gx9HyIjI4OWFr1X5plnnkNOTi4A3/zmhXz99Vd84xvfDJzvl798hC+//IxPPvmIHTu24XR2lavasGE9Llc7f//7+wC0t7dz6NBBYSAFgn6EUjJTN5BH/Aay1R9etWUIFZ04EAbyOLrz8HqzH6RBpH6OZnOHYobRA/J4FKXj16xpatDPALfd9p/MmnUyJ510MieffAoPP/yLEOPw8ctfPsKkSZMBqK+vEz0lBYJ+hmnUDFyAr3wnmteFt3wHAHK+kOSMB7Gk6Ocko5/junVf0traisvlYvXqDwNeJ0BzcxNlZaXceusPOf30s1i/fh2qqi8CFEXxN2OGWbNOYdmyvwBQW1vLd75zHVVVlcn4igKBIEnIKVm6MfR58JXvxFu6GQBTycw+HddARXiQA4AHH3yUX/9a7+doMpn51a8ej7qPJkB2djZ33/1fNDY2ctFFlzBnzuls3LgBgIyMTC699ApuvPFqUlNTmTJlOu3t7TidTmbOnMVjjz1ETk4Ot9zynzzzzFPceOPVqKrKj3/8XxQXj+iprywQCOLEVDITd80hPAe/wle+U39t1Iw+HtXARNI0TYt82ODh+H6QlZWlFBWVRPXZvgixJsqKFR+wadPX3H//Qz1y/lBzEsucDkZE78OuiDkJpifnw1dbiuO9jt6xct5oUhc+1CPXSiZDvh+kQCAQCHoWOXcUUnp+4GfTmNl9OJqBjQixDnIuuWQBl1yyoK+HIRAIeglJkkiZfze+yr1gsor9xwQQBlIgEAgGGXJGAXJGQV8PY8AjQqzAENuG7VE0TQWiTyASCASC/sqQN5Amk4W2tmZhJBNE0zS8Xg+NjbVYLLa+Ho5AIBAkzJAPsWZn59PQUENra2PEY2VZDtQICnQ6z4ksK9jtaaSlCQEBgUAw8BnyBlJRTOTlDYvqWJGq3hUxJwKBYLAy5EOsAoFAIBCEQhhIgUAgEAhCMORCrLKcWIZlop8fjIg56YqYk66IOQlGzEdXentOIl1vyEnNCQQCgUAQDSLEKhAIBAJBCISBFAgEAoEgBMJACgQCgUAQAmEgBQKBQCAIgTCQAoFAIBCEQBhIgUAgEAhCIAykQCAQCAQhEAZSIBAIBIIQCAMpEAgEAkEIhIGMkg8++IBLLrmECy64gDfeeKOvh9Mn3HTTTcyfP5/LL7+cyy+/nC1btgzZeWltbeXSSy/l6NGjAHz55ZcsWLCACy+8kOeeey5w3K5du/iP//gPLrroIu6//368Xm9fDbnHOX5O7r33Xi688MLA/bJ69Wog/FwNJn73u98xf/585s+fz9NPPw2IeyTUnPT7e0QTRKSyslI799xztYaGBq2trU1bsGCBtm/fvr4eVq+iqqp25plnah6PJ/DaUJ2XzZs3a5deeqk2ZcoUraysTHM6ndrcuXO1I0eOaB6PR7vlllu0NWvWaJqmafPnz9c2bdqkaZqm3Xvvvdobb7zRhyPvOY6fE03TtEsvvVSrqqoKOq67uRosfPHFF9o111yjuVwuze12azfddJP2wQcfDOl7JNSc/OMf/+j394jwIKPgyy+/5LTTTiMrK4uUlBQuuugiVq1a1dfD6lUOHjyIJEn853/+J5dddhmvv/76kJ2Xd955hwcffJCCggIAtm7dSklJCSNHjsRkMrFgwQJWrVpFeXk57e3tzJw5E4CFCxcO2vk5fk4cDgcVFRX88pe/ZMGCBbzwwguoqhp2rgYT+fn5LFq0CIvFgtlsZty4cRw+fHhI3yOh5qSioqLf3yNDrptHPFRXV5Ofnx/4uaCggK1bt/bhiHqf5uZmTj/9dB566CHa29u56aabmDdv3pCcl8ceeyzo51D3R1VVVZfX8/Pzqaqq6rVx9ibHz0ldXR2nnXYav/rVr0hJSeEHP/gBf/nLX0hJSQk5V4OJCRMmBP59+PBhVqxYwY033jik75FQc/Lmm2+yfv36fn2PCA8yCrQQDU8kaWi1qjnppJN4+umnSUlJIScnh6uuuooXXnihy3FDbV4g/P0xlO+bkSNH8vvf/57c3Fzsdjs33ngj//rXv4bUnOzbt49bbrmFn//854waNarL+0PxHuk8J2PHju3394gwkFFQWFhIbW1t4Ofq6upAKGmosGHDBtauXRv4WdM0iouLh/y8QPj74/jXa2pqhsz87Nmzhw8//DDws6ZpmEymIfO39PXXX3PzzTdz1113ceWVV4p7hK5zMhDuEWEgo+CMM85g7dq11NfX43Q6+cc//sE555zT18PqVVpaWnj66adxuVy0trby17/+lV//+tdDfl4AZsyYwaFDhygtLcXn8/G3v/2Nc845h+LiYqxWK19//TUAy5YtGzLzo2kajz/+OE1NTXg8Ht5++20uuOCCsHM1mDh27Bi33XYbixcvZv78+YC4R0LNyUC4R8QeZBQUFhZy5513ctNNN+HxeLjqqquYPn16Xw+rVzn33HPZsmULV1xxBaqqcv3113PyyScP+XkBsFqtPPnkk9x+++24XC7mzp3LxRdfDMDixYv5xS9+QVtbGyeeeCI33XRTH4+2d5g8eTLf//73ue666/B6vVx44YVceumlAGHnarDwyiuv4HK5ePLJJwOvXXvttUP6Hgk3J/39HpG0UAFfgUAgEAiGOCLEKhAIBAJBCISBFAgEAoEgBMJACgQCgUAQAmEgBQKBQCAIgTCQAoFAIBCEQBhIgSBOPB4PZ511FrfeemvgtX//+9+BVPXnn3+eZcuW9dHouqelpSVsOcFvf/tbTjvtNC6//HKuuOIKFixYwM0338yhQ4fiulZVVRXXXnstAGVlZdx+++1dXhcI+iOiDlIgiJPVq1czadIkduzYwYEDBxg3blzQ+z/96U/7aGSRaWpqYtu2bWHfv+SSS3jggQcCPy9ZsoS77rqL9957L+ZrFRYW8tZbbwFQUVERMLSdXxcI+iPCQAoEcbJ06VIuueQSSkpKeO211/jVr34V9P6iRYuYMGECt956K//6179YvHgxsixzwgkn8OWXXwbEmlevXo0sy5SWlmI2m3nqqaeYOHEiN954I1OmTGHdunXU1dVx0003UVdXx/r163E6nfzmN79h0qRJtLS08Nhjj7F37148Hg+nn34699xzDyaTiWnTpvH973+fL774gurqam666SZuvvlm7r33Xtrb27n88st57733UBSl2+96+umn8+yzzwJQWVnJQw89RHl5OZqmccUVV/C9730Pr9fLI488wsaNGzGbzYwYMYInnniChoYGFixYwIYNG/jFL35BVVUVt956Kw8//DALFixg06ZNeDwennzySdauXYuiKEyfPp17772XtLQ0zjvvPK688krWrl3LsWPHmDdvHvfcc0+P/V4FAgMRYhUI4mD//v1s3ryZefPmccUVV7B8+XIaGhpCHtvQ0MA999zDr3/9a5YvX86cOXOCuhN89dVX/PKXv+Rvf/sbs2bN4pVXXgm8V15ezrJly/jd737H4sWLOfXUU3nvvfc4++yzef311wF4/PHHmTJlCu+99x7Lli2joaGBP/3pTwC43W6ys7N56623eOGFF3jmmWdwuVw88cQT2Gw2li9fHtE4er1e/vKXvzBnzhwAfvaznzFnzhw++OADli5dyvvvv8/f//53Nm/ezPr163n//fd57733GDlyJHv27AmcR1EUHn30UUaNGhX0HQFeeuklqqurWb58OcuXL0dV1UBTXdDbZ7355pu89dZbvP7665SVlUXzaxIIEkIYSIEgDpYuXco3vvENsrKymD59OiNGjODtt98OeeyGDRsYN24ckydPBuDKK68kLS0t8P6UKVMoKioC4MQTT6SpqSnw3gUXXADo3TEAzj77bABGjRoVOG7NmjW8/fbbXH755SxcuJCtW7eyd+/ewDm++c1vBq7jdrtxOBwRv9+KFSsCXd4vu+wy6urqeOSRR3A4HGzcuJEbbrgBgPT0dBYuXMinn37KxIkTURSFb33rW/zmN7/hoosuYtasWVHMJnz66adce+21mM1mZFnmxhtv5LPPPuvyHQoLC8nNzQ2aI4GgpxAhVoEgRhwOB8uWLcNqtXLeeecB0NrayhtvvMG0adO6HK8oSpcWPrLcsTa12WyBfx/fAslisQR9zmw2dzm/qqo8//zzgT3Q5ubmoPZAVqs1cG4I3Z7reI7fgzRobW3t8nlVVfF6vWRkZLB8+XI2btzIunXruOOOO7jppps4//zzI15PVdUuP3s8ni7fwfgeQiFT0BsID1IgiJEPPviA7OxsPvvsMz7++GM+/vhjPvroIxwOB3V1dV2OnzVrFocPH2b37t0AfPjhh12MWCKcddZZvPrqq2iahtvt5kc/+lEg/BoOk8mEz+eL2dCkpaUxY8YM3njjDUDPhl22bBlnnHEGn3zyCTfffDMnnXQSt99+O1dccUXgOxsoihJk+AzOPvts3nrrLTweD6qq8sYbb3DmmWfGNDaBINkIAykQxMjSpUv57ne/G7R3l5GRwY033shrr73W5fisrCyeffZZfv7zn3PllVfy+eefYzKZsNvtSRnP/fffj8PhYMGCBSxYsICJEyfyve99r9vP5Ofnc+KJJzJv3rywe6fhWLx4MWvXrmXBggVcddVVXHjhhSxcuJBzzjmH8ePHc+mll7Jw4UI2bdrET37yk6DPTpgwAUVRuOqqq4KM849+9CPy8vK44oormDdvHl6vl/vvvz+mcQkEyUZ08xAIepjW1lZefPFFbr/9dux2Ozt27OAHP/gBn3322aDuHi8QDHTEHqRA0MOkpaVhNpu56qqrMJlMmEwmfvOb3wjjKBD0c4QHKRAIBAJBCMQepEAgEAgEIRAGUiAQCASCEAgDKRAIBAJBCISBFAgEAoEgBMJACgQCgUAQAmEgBQKBQCAIwf8HWyhtMWTZd1IAAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 504x288 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "hg.dope_plot(env,\"1t70A\",\"NSP4\",save=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Comments\n",
    "The DOPE profile of the model follows that of the template quite well. However, the protruding structure (reddish-orange) and the C-terminus (red) have high energy relative to the template indicating that these regions have been modelled poorly."
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.1"
  },
  "widgets": {
   "application/vnd.jupyter.widget-state+json": {
    "state": {
     "08fae6ec25304c8297384d41dcdd18f0": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "SliderStyleModel",
      "state": {
       "description_width": ""
      }
     },
     "09ebdc541c664802aa15120b6b3b0e0f": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "PlayModel",
      "state": {
       "layout": "IPY_MODEL_bbcb56f26d404a01b3a18ce02cf23ebc",
       "max": 0,
       "style": "IPY_MODEL_417a29b7da864854acfc8576bf4cb919"
      }
     },
     "0db08b1071ad4d4f90e698719511fd94": {
      "model_module": "@jupyter-widgets/base",
      "model_module_version": "1.2.0",
      "model_name": "LayoutModel",
      "state": {}
     },
     "16d8a7347c49453491cd94bb48d380f4": {
      "model_module": "@jupyter-widgets/base",
      "model_module_version": "1.2.0",
      "model_name": "LayoutModel",
      "state": {}
     },
     "1906c7f1dd7b4e89be2881c6a52b2e01": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "LinkModel",
      "state": {
       "source": [
        "IPY_MODEL_dc4c2913fc224986b8eafdcab1278219",
        "value"
       ],
       "target": [
        "IPY_MODEL_68adf8c3ebd649cca0c0b81626f28a36",
        "frame"
       ]
      }
     },
     "19e9593679d2499089ef47047fc01e42": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "LinkModel",
      "state": {
       "source": [
        "IPY_MODEL_dc4c2913fc224986b8eafdcab1278219",
        "max"
       ],
       "target": [
        "IPY_MODEL_68adf8c3ebd649cca0c0b81626f28a36",
        "max_frame"
       ]
      }
     },
     "1c6e494d218b4e14b65e11b65d1e79d7": {
      "model_module": "@jupyter-widgets/base",
      "model_module_version": "1.2.0",
      "model_name": "LayoutModel",
      "state": {}
     },
     "1f3beddbb917459aaf9affec0339d5ce": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "DescriptionStyleModel",
      "state": {
       "description_width": ""
      }
     },
     "2151f8609be64b4b8062e3fd14334761": {
      "model_module": "@jupyter-widgets/base",
      "model_module_version": "1.2.0",
      "model_name": "LayoutModel",
      "state": {
       "width": "34px"
      }
     },
     "257e2bf2ff8540308ef4cf62cfe844b6": {
      "model_module": "@jupyter-widgets/base",
      "model_module_version": "1.2.0",
      "model_name": "LayoutModel",
      "state": {}
     },
     "29c8848788294b8f9c8c3b8edeeab294": {
      "model_module": "nglview-js-widgets",
      "model_module_version": "2.7.7",
      "model_name": "NGLModel",
      "state": {
       "_camera_orientation": [
        90.28334037768687,
        0,
        0,
        0,
        0,
        90.28334037768687,
        0,
        0,
        0,
        0,
        90.28334037768687,
        0,
        -14.973885893821716,
        -32.70515298843384,
        -14.773627161979675,
        1
       ],
       "_camera_str": "orthographic",
       "_gui_theme": null,
       "_ibtn_fullscreen": "IPY_MODEL_4d8a710940d946d3896b43a978075941",
       "_igui": null,
       "_iplayer": "IPY_MODEL_ce533c82575f4df092c78f3ae3ece00d",
       "_ngl_color_dict": {},
       "_ngl_coordinate_resource": {},
       "_ngl_full_stage_parameters": {
        "ambientColor": 14540253,
        "ambientIntensity": 0.2,
        "backgroundColor": "white",
        "cameraEyeSep": 0.3,
        "cameraFov": 40,
        "cameraType": "perspective",
        "clipDist": 10,
        "clipFar": 100,
        "clipNear": 0,
        "fogFar": 100,
        "fogNear": 50,
        "hoverTimeout": 0,
        "impostor": true,
        "lightColor": 14540253,
        "lightIntensity": 1,
        "mousePreset": "default",
        "panSpeed": 1,
        "quality": "medium",
        "rotateSpeed": 2,
        "sampleLevel": 0,
        "tooltip": true,
        "workerDefault": true,
        "zoomSpeed": 1.2
       },
       "_ngl_msg_archive": [
        {
         "args": [
          {
           "binary": false,
           "data": "EXPDTA    THEORETICAL MODEL, MODELLER 9.25 2020/12/21 14:58:58\nREMARK   6 MODELLER OBJECTIVE FUNCTION:      1194.4651\nREMARK   6 MODELLER BEST TEMPLATE % SEQ ID:  30.323\nREMARK   6 SEQUENCE: NSP4\nREMARK   6 ALIGNMENT: NSP4-1t70A.ali\nREMARK   6 SCRIPT: /usr/local/lib/python3.9/site-packages/ipykernel_launcher.py\nREMARK   6 DOPE score: -12980.01562\nREMARK   6 GA341 score: 0.15301\nREMARK   6 TEMPLATE: 1t70A 84:A - 251:A MODELS 1: - 155: AT 30.3%\nATOM      1  N   ILE     1      22.189  21.165  11.666  1.00 40.43           N\nATOM      2  CA  ILE     1      22.332  19.781  11.171  1.00 40.43           C\nATOM      3  CB  ILE     1      22.850  18.900  12.268  1.00 40.43           C\nATOM      4  CG1 ILE     1      24.221  19.407  12.752  1.00 40.43           C\nATOM      5  CG2 ILE     1      22.892  17.460  11.732  1.00 40.43           C\nATOM      6  CD1 ILE     1      24.674  18.779  14.069  1.00 40.43           C\nATOM      7  C   ILE     1      20.993  19.299  10.726  1.00 40.43           C\nATOM      8  O   ILE     1      20.882  18.283  10.041  1.00 40.43           O\nATOM      9  N   THR     2      19.931  20.037  11.108  1.00111.45           N\nATOM     10  CA  THR     2      18.612  19.701  10.659  1.00111.45           C\nATOM     11  CB  THR     2      17.567  19.648  11.734  1.00111.45           C\nATOM     12  OG1 THR     2      16.340  19.195  11.179  1.00111.45           O\nATOM     13  CG2 THR     2      17.388  21.046  12.349  1.00111.45           C\nATOM     14  C   THR     2      18.205  20.761   9.687  1.00111.45           C\nATOM     15  O   THR     2      18.354  21.955   9.947  1.00111.45           O\nATOM     16  N   ARG     3      17.687  20.328   8.521  1.00125.13           N\nATOM     17  CA  ARG     3      17.270  21.210   7.471  1.00125.13           C\nATOM     18  CB  ARG     3      17.570  20.630   6.075  1.00125.13           C\nATOM     19  CG  ARG     3      17.274  21.570   4.907  1.00125.13           C\nATOM     20  CD  ARG     3      17.130  20.835   3.575  1.00125.13           C\nATOM     21  NE  ARG     3      15.868  20.052   3.667  1.00125.13           N\nATOM     22  CZ  ARG     3      15.902  18.694   3.779  1.00125.13           C\nATOM     23  NH1 ARG     3      17.086  18.028   3.649  1.00125.13           N\nATOM     24  NH2 ARG     3      14.745  18.007   4.012  1.00125.13           N\nATOM     25  C   ARG     3      15.785  21.369   7.578  1.00125.13           C\nATOM     26  O   ARG     3      15.121  20.633   8.306  1.00125.13           O\nATOM     27  N   GLU     4      15.225  22.362   6.860  1.00142.41           N\nATOM     28  CA  GLU     4      13.813  22.604   6.927  1.00142.41           C\nATOM     29  CB  GLU     4      13.377  23.843   6.130  1.00142.41           C\nATOM     30  CG  GLU     4      13.968  23.867   4.724  1.00142.41           C\nATOM     31  CD  GLU     4      15.364  24.465   4.844  1.00142.41           C\nATOM     32  OE1 GLU     4      15.572  25.297   5.768  1.00142.41           O\nATOM     33  OE2 GLU     4      16.241  24.101   4.017  1.00142.41           O\nATOM     34  C   GLU     4      13.073  21.414   6.398  1.00142.41           C\nATOM     35  O   GLU     4      13.452  20.808   5.399  1.00142.41           O\nATOM     36  N   ILE     5      11.989  21.056   7.108  1.00168.52           N\nATOM     37  CA  ILE     5      11.097  19.965   6.814  1.00168.52           C\nATOM     38  CB  ILE     5       9.994  19.858   7.821  1.00168.52           C\nATOM     39  CG1 ILE     5       8.914  18.871   7.334  1.00168.52           C\nATOM     40  CG2 ILE     5       9.470  21.277   8.112  1.00168.52           C\nATOM     41  CD1 ILE     5       9.232  17.381   7.408  1.00168.52           C\nATOM     42  C   ILE     5      10.372  20.191   5.534  1.00168.52           C\nATOM     43  O   ILE     5      10.208  19.280   4.726  1.00168.52           O\nATOM     44  N   GLY     6       9.960  21.447   5.323  1.00 87.96           N\nATOM     45  CA  GLY     6       9.045  21.887   4.312  1.00 87.96           C\nATOM     46  C   GLY     6       9.469  21.517   2.931  1.00 87.96           C\nATOM     47  O   GLY     6       8.622  21.537   2.043  1.00 87.96           O\nATOM     48  N   PHE     7      10.769  21.284   2.663  1.00 93.92           N\nATOM     49  CA  PHE     7      11.113  20.923   1.315  1.00 93.92           C\nATOM     50  CB  PHE     7      12.596  20.597   1.093  1.00 93.92           C\nATOM     51  CG  PHE     7      13.312  21.885   0.943  1.00 93.92           C\nATOM     52  CD1 PHE     7      13.173  22.598  -0.226  1.00 93.92           C\nATOM     53  CD2 PHE     7      14.128  22.372   1.932  1.00 93.92           C\nATOM     54  CE1 PHE     7      13.828  23.791  -0.401  1.00 93.92           C\nATOM     55  CE2 PHE     7      14.786  23.567   1.759  1.00 93.92           C\nATOM     56  CZ  PHE     7      14.638  24.279   0.595  1.00 93.92           C\nATOM     57  C   PHE     7      10.366  19.698   0.901  1.00 93.92           C\nATOM     58  O   PHE     7      10.539  18.617   1.462  1.00 93.92           O\nATOM     59  N   ILE     8       9.543  19.856  -0.150  1.00141.05           N\nATOM     60  CA  ILE     8       8.707  18.829  -0.697  1.00141.05           C\nATOM     61  CB  ILE     8       7.817  19.350  -1.783  1.00141.05           C\nATOM     62  CG1 ILE     8       7.074  18.184  -2.452  1.00141.05           C\nATOM     63  CG2 ILE     8       8.678  20.194  -2.738  1.00141.05           C\nATOM     64  CD1 ILE     8       6.119  17.450  -1.516  1.00141.05           C\nATOM     65  C   ILE     8       9.496  17.714  -1.304  1.00141.05           C\nATOM     66  O   ILE     8       9.177  16.543  -1.103  1.00141.05           O\nATOM     67  N   VAL     9      10.551  18.040  -2.069  1.00108.01           N\nATOM     68  CA  VAL     9      11.201  16.991  -2.795  1.00108.01           C\nATOM     69  CB  VAL     9      12.163  17.464  -3.844  1.00108.01           C\nATOM     70  CG1 VAL     9      11.379  18.281  -4.883  1.00108.01           C\nATOM     71  CG2 VAL     9      13.309  18.231  -3.169  1.00108.01           C\nATOM     72  C   VAL     9      11.937  16.090  -1.867  1.00108.01           C\nATOM     73  O   VAL     9      12.594  16.489  -0.908  1.00108.01           O\nATOM     74  N   PRO    10      11.753  14.846  -2.189  1.00140.72           N\nATOM     75  CA  PRO    10      12.389  13.745  -1.524  1.00140.72           C\nATOM     76  CD  PRO    10      10.577  14.436  -2.933  1.00140.72           C\nATOM     77  CB  PRO    10      11.654  12.488  -1.992  1.00140.72           C\nATOM     78  CG  PRO    10      10.818  12.947  -3.203  1.00140.72           C\nATOM     79  C   PRO    10      13.839  13.757  -1.876  1.00140.72           C\nATOM     80  O   PRO    10      14.614  13.030  -1.257  1.00140.72           O\nATOM     81  N   GLY    11      14.206  14.538  -2.907  1.00 28.38           N\nATOM     82  CA  GLY    11      15.560  14.631  -3.364  1.00 28.38           C\nATOM     83  C   GLY    11      16.427  15.270  -2.323  1.00 28.38           C\nATOM     84  O   GLY    11      17.582  14.881  -2.163  1.00 28.38           O\nATOM     85  N   LEU    12      15.913  16.281  -1.590  1.00 85.41           N\nATOM     86  CA  LEU    12      16.791  16.981  -0.696  1.00 85.41           C\nATOM     87  CB  LEU    12      16.196  18.269  -0.099  1.00 85.41           C\nATOM     88  CG  LEU    12      15.986  19.347  -1.183  1.00 85.41           C\nATOM     89  CD1 LEU    12      15.580  20.702  -0.581  1.00 85.41           C\nATOM     90  CD2 LEU    12      17.215  19.448  -2.101  1.00 85.41           C\nATOM     91  C   LEU    12      17.296  16.079   0.398  1.00 85.41           C\nATOM     92  O   LEU    12      16.568  15.412   1.130  1.00 85.41           O\nATOM     93  N   PRO    13      18.602  16.078   0.445  1.00254.10           N\nATOM     94  CA  PRO    13      19.421  15.259   1.302  1.00254.10           C\nATOM     95  CD  PRO    13      19.382  16.932  -0.434  1.00254.10           C\nATOM     96  CB  PRO    13      20.867  15.543   0.878  1.00254.10           C\nATOM     97  CG  PRO    13      20.795  16.906   0.162  1.00254.10           C\nATOM     98  C   PRO    13      19.255  15.297   2.792  1.00254.10           C\nATOM     99  O   PRO    13      18.905  14.274   3.372  1.00254.10           O\nATOM    100  N   GLY    14      19.403  16.459   3.445  1.00106.34           N\nATOM    101  CA  GLY    14      19.447  16.439   4.882  1.00106.34           C\nATOM    102  C   GLY    14      18.117  16.015   5.427  1.00106.34           C\nATOM    103  O   GLY    14      17.228  15.588   4.694  1.00106.34           O\nATOM    104  N   THR    15      17.961  16.092   6.765  1.00 46.84           N\nATOM    105  CA  THR    15      16.737  15.659   7.373  1.00 46.84           C\nATOM    106  CB  THR    15      16.916  14.599   8.421  1.00 46.84           C\nATOM    107  OG1 THR    15      17.716  15.089   9.487  1.00 46.84           O\nATOM    108  CG2 THR    15      17.578  13.368   7.777  1.00 46.84           C\nATOM    109  C   THR    15      16.055  16.817   8.035  1.00 46.84           C\nATOM    110  O   THR    15      16.677  17.795   8.447  1.00 46.84           O\nATOM    111  N   VAL    16      14.718  16.697   8.101  1.00 77.15           N\nATOM    112  CA  VAL    16      13.742  17.591   8.651  1.00 77.15           C\nATOM    113  CB  VAL    16      12.383  17.059   8.393  1.00 77.15           C\nATOM    114  CG1 VAL    16      11.415  17.791   9.330  1.00 77.15           C\nATOM    115  CG2 VAL    16      12.115  17.196   6.883  1.00 77.15           C\nATOM    116  C   VAL    16      13.851  17.715  10.135  1.00 77.15           C\nATOM    117  O   VAL    16      13.591  18.780  10.698  1.00 77.15           O\nATOM    118  N   LEU    17      14.140  16.607  10.826  1.00132.52           N\nATOM    119  CA  LEU    17      14.218  16.737  12.245  1.00132.52           C\nATOM    120  CB  LEU    17      12.839  16.783  12.950  1.00132.52           C\nATOM    121  CG  LEU    17      11.924  15.555  12.772  1.00132.52           C\nATOM    122  CD1 LEU    17      12.490  14.322  13.491  1.00132.52           C\nATOM    123  CD2 LEU    17      10.482  15.871  13.206  1.00132.52           C\nATOM    124  C   LEU    17      15.036  15.611  12.769  1.00132.52           C\nATOM    125  O   LEU    17      15.345  14.662  12.050  1.00132.52           O\nATOM    126  N   ARG    18      15.456  15.735  14.037  1.00 67.83           N\nATOM    127  CA  ARG    18      16.204  14.696  14.672  1.00 67.83           C\nATOM    128  CB  ARG    18      17.698  14.994  14.807  1.00 67.83           C\nATOM    129  CG  ARG    18      18.546  14.609  13.604  1.00 67.83           C\nATOM    130  CD  ARG    18      20.018  14.890  13.884  1.00 67.83           C\nATOM    131  NE  ARG    18      20.816  13.920  13.095  1.00 67.83           N\nATOM    132  CZ  ARG    18      20.928  12.651  13.583  1.00 67.83           C\nATOM    133  NH1 ARG    18      20.196  12.290  14.676  1.00 67.83           N\nATOM    134  NH2 ARG    18      21.757  11.746  12.985  1.00 67.83           N\nATOM    135  C   ARG    18      15.704  14.589  16.065  1.00 67.83           C\nATOM    136  O   ARG    18      15.265  15.576  16.654  1.00 67.83           O\nATOM    137  N   ALA    19      15.743  13.367  16.623  1.00 42.77           N\nATOM    138  CA  ALA    19      15.387  13.215  17.997  1.00 42.77           C\nATOM    139  CB  ALA    19      14.517  11.979  18.295  1.00 42.77           C\nATOM    140  C   ALA    19      16.691  13.028  18.699  1.00 42.77           C\nATOM    141  O   ALA    19      17.540  12.248  18.272  1.00 42.77           O\nATOM    142  N   ILE    20      16.885  13.775  19.788  1.00 79.64           N\nATOM    143  CA  ILE    20      18.075  13.738  20.580  1.00 79.64           C\nATOM    144  CB  ILE    20      18.987  14.884  20.255  1.00 79.64           C\nATOM    145  CG1 ILE    20      19.262  14.866  18.743  1.00 79.64           C\nATOM    146  CG2 ILE    20      20.283  14.759  21.075  1.00 79.64           C\nATOM    147  CD1 ILE    20      19.910  16.150  18.238  1.00 79.64           C\nATOM    148  C   ILE    20      17.485  13.894  21.931  1.00 79.64           C\nATOM    149  O   ILE    20      16.495  13.227  22.225  1.00 79.64           O\nATOM    150  N   ASN    21      18.071  14.703  22.827  1.00119.10           N\nATOM    151  CA  ASN    21      17.277  14.901  24.000  1.00119.10           C\nATOM    152  CB  ASN    21      17.984  15.707  25.103  1.00119.10           C\nATOM    153  CG  ASN    21      17.067  15.757  26.319  1.00119.10           C\nATOM    154  OD1 ASN    21      15.897  16.127  26.221  1.00119.10           O\nATOM    155  ND2 ASN    21      17.613  15.367  27.501  1.00119.10           N\nATOM    156  C   ASN    21      16.121  15.747  23.512  1.00119.10           C\nATOM    157  O   ASN    21      16.240  16.970  23.440  1.00119.10           O\nATOM    158  N   GLY    22      14.965  15.127  23.155  1.00 72.52           N\nATOM    159  CA  GLY    22      13.866  15.883  22.615  1.00 72.52           C\nATOM    160  C   GLY    22      14.031  15.892  21.124  1.00 72.52           C\nATOM    161  O   GLY    22      15.103  15.568  20.616  1.00 72.52           O\nATOM    162  N   ASP    23      12.968  16.261  20.375  1.00110.93           N\nATOM    163  CA  ASP    23      13.104  16.287  18.944  1.00110.93           C\nATOM    164  CB  ASP    23      12.045  15.488  18.151  1.00110.93           C\nATOM    165  CG  ASP    23      10.661  16.080  18.367  1.00110.93           C\nATOM    166  OD1 ASP    23      10.270  16.247  19.551  1.00110.93           O\nATOM    167  OD2 ASP    23       9.972  16.375  17.353  1.00110.93           O\nATOM    168  C   ASP    23      13.079  17.702  18.471  1.00110.93           C\nATOM    169  O   ASP    23      12.246  18.509  18.888  1.00110.93           O\nATOM    170  N   PHE    24      14.027  18.049  17.576  1.00101.52           N\nATOM    171  CA  PHE    24      14.042  19.398  17.101  1.00101.52           C\nATOM    172  CB  PHE    24      15.404  20.094  17.248  1.00101.52           C\nATOM    173  CG  PHE    24      15.760  19.962  18.687  1.00101.52           C\nATOM    174  CD1 PHE    24      15.088  20.675  19.656  1.00101.52           C\nATOM    175  CD2 PHE    24      16.739  19.078  19.075  1.00101.52           C\nATOM    176  CE1 PHE    24      15.416  20.538  20.985  1.00101.52           C\nATOM    177  CE2 PHE    24      17.068  18.935  20.400  1.00101.52           C\nATOM    178  CZ  PHE    24      16.412  19.667  21.359  1.00101.52           C\nATOM    179  C   PHE    24      13.670  19.380  15.651  1.00101.52           C\nATOM    180  O   PHE    24      14.171  18.561  14.880  1.00101.52           O\nATOM    181  N   LEU    25      12.756  20.289  15.251  1.00108.03           N\nATOM    182  CA  LEU    25      12.301  20.353  13.891  1.00108.03           C\nATOM    183  CB  LEU    25      10.778  20.104  13.741  1.00108.03           C\nATOM    184  CG  LEU    25      10.301  18.720  14.240  1.00108.03           C\nATOM    185  CD1 LEU    25      10.521  18.578  15.755  1.00108.03           C\nATOM    186  CD2 LEU    25       8.847  18.418  13.830  1.00108.03           C\nATOM    187  C   LEU    25      12.573  21.735  13.370  1.00108.03           C\nATOM    188  O   LEU    25      12.496  22.718  14.108  1.00108.03           O\nATOM    189  N   HIS    26      12.924  21.836  12.071  1.00 77.25           N\nATOM    190  CA  HIS    26      13.158  23.122  11.478  1.00 77.25           C\nATOM    191  ND1 HIS    26      14.421  25.347   9.305  1.00 77.25           N\nATOM    192  CG  HIS    26      14.866  24.688  10.430  1.00 77.25           C\nATOM    193  CB  HIS    26      14.526  23.267  10.785  1.00 77.25           C\nATOM    194  NE2 HIS    26      15.694  26.791  10.420  1.00 77.25           N\nATOM    195  CD2 HIS    26      15.644  25.587  11.100  1.00 77.25           C\nATOM    196  CE1 HIS    26      14.947  26.598   9.347  1.00 77.25           C\nATOM    197  C   HIS    26      12.105  23.321  10.430  1.00 77.25           C\nATOM    198  O   HIS    26      11.793  22.413   9.661  1.00 77.25           O\nATOM    199  N   PHE    27      11.520  24.535  10.390  1.00 54.21           N\nATOM    200  CA  PHE    27      10.456  24.830   9.470  1.00 54.21           C\nATOM    201  CB  PHE    27       9.099  25.001  10.171  1.00 54.21           C\nATOM    202  CG  PHE    27       8.698  23.663  10.688  1.00 54.21           C\nATOM    203  CD1 PHE    27       9.221  23.173  11.863  1.00 54.21           C\nATOM    204  CD2 PHE    27       7.808  22.885   9.983  1.00 54.21           C\nATOM    205  CE1 PHE    27       8.851  21.933  12.332  1.00 54.21           C\nATOM    206  CE2 PHE    27       7.435  21.648  10.448  1.00 54.21           C\nATOM    207  CZ  PHE    27       7.954  21.168  11.626  1.00 54.21           C\nATOM    208  C   PHE    27      10.749  26.123   8.765  1.00 54.21           C\nATOM    209  O   PHE    27      11.412  27.005   9.311  1.00 54.21           O\nATOM    210  N   LEU    28      10.239  26.253   7.516  1.00 78.71           N\nATOM    211  CA  LEU    28      10.449  27.427   6.705  1.00 78.71           C\nATOM    212  CB  LEU    28      11.080  27.124   5.328  1.00 78.71           C\nATOM    213  CG  LEU    28      12.581  26.787   5.373  1.00 78.71           C\nATOM    214  CD1 LEU    28      13.145  26.588   3.956  1.00 78.71           C\nATOM    215  CD2 LEU    28      13.358  27.853   6.163  1.00 78.71           C\nATOM    216  C   LEU    28       9.143  28.132   6.453  1.00 78.71           C\nATOM    217  O   LEU    28       8.080  27.520   6.328  1.00 78.71           O\nATOM    218  N   PRO    29       9.264  29.442   6.420  1.00138.15           N\nATOM    219  CA  PRO    29       8.158  30.360   6.260  1.00138.15           C\nATOM    220  CD  PRO    29      10.478  30.039   5.869  1.00138.15           C\nATOM    221  CB  PRO    29       8.778  31.721   6.448  1.00138.15           C\nATOM    222  CG  PRO    29      10.161  31.550   5.769  1.00138.15           C\nATOM    223  C   PRO    29       7.655  30.376   4.855  1.00138.15           C\nATOM    224  O   PRO    29       8.458  30.247   3.935  1.00138.15           O\nATOM    225  N   ARG    30       6.334  30.501   4.662  1.00179.97           N\nATOM    226  CA  ARG    30       5.772  30.708   3.362  1.00179.97           C\nATOM    227  CB  ARG    30       4.374  30.079   3.242  1.00179.97           C\nATOM    228  CG  ARG    30       4.466  28.546   3.226  1.00179.97           C\nATOM    229  CD  ARG    30       3.131  27.801   3.347  1.00179.97           C\nATOM    230  NE  ARG    30       2.126  28.505   2.499  1.00179.97           N\nATOM    231  CZ  ARG    30       1.200  27.800   1.780  1.00179.97           C\nATOM    232  NH1 ARG    30       1.229  26.436   1.761  1.00179.97           N\nATOM    233  NH2 ARG    30       0.240  28.467   1.076  1.00179.97           N\nATOM    234  C   ARG    30       5.774  32.160   2.947  1.00179.97           C\nATOM    235  O   ARG    30       5.904  32.475   1.767  1.00179.97           O\nATOM    236  N   VAL    31       5.615  33.104   3.895  1.00 87.82           N\nATOM    237  CA  VAL    31       5.500  34.487   3.507  1.00 87.82           C\nATOM    238  CB  VAL    31       5.007  35.360   4.625  1.00 87.82           C\nATOM    239  CG1 VAL    31       4.920  36.812   4.127  1.00 87.82           C\nATOM    240  CG2 VAL    31       3.674  34.794   5.134  1.00 87.82           C\nATOM    241  C   VAL    31       6.840  34.993   3.096  1.00 87.82           C\nATOM    242  O   VAL    31       7.838  34.784   3.780  1.00 87.82           O\nATOM    243  N   PHE    32       6.870  35.702   1.951  1.00225.68           N\nATOM    244  CA  PHE    32       8.063  36.278   1.407  1.00225.68           C\nATOM    245  CB  PHE    32       8.711  37.313   2.343  1.00225.68           C\nATOM    246  CG  PHE    32       7.665  38.327   2.662  1.00225.68           C\nATOM    247  CD1 PHE    32       7.087  39.082   1.666  1.00225.68           C\nATOM    248  CD2 PHE    32       7.284  38.552   3.965  1.00225.68           C\nATOM    249  CE1 PHE    32       6.126  40.019   1.964  1.00225.68           C\nATOM    250  CE2 PHE    32       6.321  39.486   4.267  1.00225.68           C\nATOM    251  CZ  PHE    32       5.736  40.223   3.266  1.00225.68           C\nATOM    252  C   PHE    32       9.055  35.185   1.130  1.00225.68           C\nATOM    253  O   PHE    32      10.262  35.420   1.158  1.00225.68           O\nATOM    254  N   SER    33       8.557  33.959   0.854  1.00 93.43           N\nATOM    255  CA  SER    33       9.387  32.831   0.501  1.00 93.43           C\nATOM    256  CB  SER    33      10.091  32.172   1.699  1.00 93.43           C\nATOM    257  OG  SER    33      11.052  33.069   2.243  1.00 93.43           O\nATOM    258  C   SER    33       8.482  31.798  -0.141  1.00 93.43           C\nATOM    259  O   SER    33       7.292  32.054  -0.301  1.00 93.43           O\nATOM    260  N   ALA    34       9.007  30.629  -0.598  1.00104.05           N\nATOM    261  CA  ALA    34       8.090  29.641  -1.131  1.00104.05           C\nATOM    262  CB  ALA    34       7.799  29.813  -2.631  1.00104.05           C\nATOM    263  C   ALA    34       8.627  28.244  -0.935  1.00104.05           C\nATOM    264  O   ALA    34       9.796  27.979  -1.213  1.00104.05           O\nATOM    265  N   VAL    35       7.745  27.320  -0.465  1.00123.52           N\nATOM    266  CA  VAL    35       7.998  25.915  -0.232  1.00123.52           C\nATOM    267  CB  VAL    35       8.897  25.585   0.929  1.00123.52           C\nATOM    268  CG1 VAL    35      10.339  26.020   0.617  1.00123.52           C\nATOM    269  CG2 VAL    35       8.292  26.204   2.201  1.00123.52           C\nATOM    270  C   VAL    35       6.684  25.267   0.093  1.00123.52           C\nATOM    271  O   VAL    35       5.617  25.831  -0.144  1.00123.52           O\nATOM    272  N   GLY    36       6.733  24.036   0.650  1.00 25.59           N\nATOM    273  CA  GLY    36       5.530  23.321   0.981  1.00 25.59           C\nATOM    274  C   GLY    36       4.889  23.985   2.161  1.00 25.59           C\nATOM    275  O   GLY    36       5.527  24.749   2.883  1.00 25.59           O\nATOM    276  N   ASN    37       3.593  23.685   2.397  1.00 66.08           N\nATOM    277  CA  ASN    37       2.882  24.325   3.467  1.00 66.08           C\nATOM    278  CB  ASN    37       1.382  23.986   3.517  1.00 66.08           C\nATOM    279  CG  ASN    37       0.751  24.799   4.640  1.00 66.08           C\nATOM    280  OD1 ASN    37       1.365  25.712   5.192  1.00 66.08           O\nATOM    281  ND2 ASN    37      -0.512  24.451   5.006  1.00 66.08           N\nATOM    282  C   ASN    37       3.468  23.908   4.771  1.00 66.08           C\nATOM    283  O   ASN    37       3.501  22.730   5.124  1.00 66.08           O\nATOM    284  N   ILE    38       3.925  24.915   5.534  1.00129.48           N\nATOM    285  CA  ILE    38       4.545  24.735   6.804  1.00129.48           C\nATOM    286  CB  ILE    38       5.084  26.018   7.349  1.00129.48           C\nATOM    287  CG1 ILE    38       5.768  25.824   8.710  1.00129.48           C\nATOM    288  CG2 ILE    38       3.915  26.998   7.381  1.00129.48           C\nATOM    289  CD1 ILE    38       6.437  27.101   9.219  1.00129.48           C\nATOM    290  C   ILE    38       3.577  24.203   7.791  1.00129.48           C\nATOM    291  O   ILE    38       3.991  23.344   8.545  1.00129.48           O\nATOM    292  N   CYS    39       2.323  24.721   7.847  1.00111.63           N\nATOM    293  CA  CYS    39       1.282  24.358   8.784  1.00111.63           C\nATOM    294  CB  CYS    39       0.116  25.354   8.799  1.00111.63           C\nATOM    295  SG  CYS    39      -1.015  25.029  10.180  1.00111.63           S\nATOM    296  C   CYS    39       0.710  22.994   8.508  1.00111.63           C\nATOM    297  O   CYS    39       0.221  22.320   9.417  1.00111.63           O\nATOM    298  N   TYR    40       0.674  22.571   7.236  1.00111.81           N\nATOM    299  CA  TYR    40       0.217  21.247   6.911  1.00111.81           C\nATOM    300  CB  TYR    40       0.060  21.076   5.385  1.00111.81           C\nATOM    301  CG  TYR    40      -0.595  19.782   5.026  1.00111.81           C\nATOM    302  CD1 TYR    40      -1.967  19.669   5.029  1.00111.81           C\nATOM    303  CD2 TYR    40       0.163  18.693   4.657  1.00111.81           C\nATOM    304  CE1 TYR    40      -2.577  18.484   4.688  1.00111.81           C\nATOM    305  CE2 TYR    40      -0.441  17.505   4.314  1.00111.81           C\nATOM    306  CZ  TYR    40      -1.810  17.400   4.331  1.00111.81           C\nATOM    307  OH  TYR    40      -2.432  16.184   3.980  1.00111.81           O\nATOM    308  C   TYR    40       1.273  20.305   7.381  1.00111.81           C\nATOM    309  O   TYR    40       0.998  19.252   7.955  1.00111.81           O\nATOM    310  N   THR    41       2.537  20.709   7.159  1.00 74.64           N\nATOM    311  CA  THR    41       3.675  19.895   7.436  1.00 74.64           C\nATOM    312  CB  THR    41       4.968  20.556   7.077  1.00 74.64           C\nATOM    313  OG1 THR    41       5.052  20.752   5.674  1.00 74.64           O\nATOM    314  CG2 THR    41       6.105  19.656   7.549  1.00 74.64           C\nATOM    315  C   THR    41       3.771  19.468   8.879  1.00 74.64           C\nATOM    316  O   THR    41       3.962  18.272   9.088  1.00 74.64           O\nATOM    317  N   PRO    42       3.698  20.301   9.897  1.00149.26           N\nATOM    318  CA  PRO    42       3.871  19.819  11.221  1.00149.26           C\nATOM    319  CD  PRO    42       2.777  21.393   9.962  1.00149.26           C\nATOM    320  CB  PRO    42       3.755  21.030  12.130  1.00149.26           C\nATOM    321  CG  PRO    42       2.642  21.814  11.434  1.00149.26           C\nATOM    322  C   PRO    42       2.776  18.861  11.561  1.00149.26           C\nATOM    323  O   PRO    42       2.999  17.991  12.395  1.00149.26           O\nATOM    324  N   SER    43       1.575  19.009  10.965  1.00 39.55           N\nATOM    325  CA  SER    43       0.496  18.140  11.285  1.00 39.55           C\nATOM    326  CB  SER    43      -0.775  18.462  10.479  1.00 39.55           C\nATOM    327  OG  SER    43      -1.239  19.766  10.806  1.00 39.55           O\nATOM    328  C   SER    43       0.970  16.773  10.925  1.00 39.55           C\nATOM    329  O   SER    43       0.688  15.800  11.622  1.00 39.55           O\nATOM    330  N   LYS    44       1.730  16.679   9.817  1.00 98.68           N\nATOM    331  CA  LYS    44       2.252  15.427   9.350  1.00 98.68           C\nATOM    332  CB  LYS    44       3.082  15.584   8.065  1.00 98.68           C\nATOM    333  CG  LYS    44       2.321  16.165   6.875  1.00 98.68           C\nATOM    334  CD  LYS    44       3.252  16.616   5.747  1.00 98.68           C\nATOM    335  CE  LYS    44       3.789  15.460   4.898  1.00 98.68           C\nATOM    336  NZ  LYS    44       4.544  14.507   5.742  1.00 98.68           N\nATOM    337  C   LYS    44       3.212  14.873  10.357  1.00 98.68           C\nATOM    338  O   LYS    44       3.081  13.726  10.784  1.00 98.68           O\nATOM    339  N   LEU    45       4.195  15.696  10.793  1.00167.18           N\nATOM    340  CA  LEU    45       5.211  15.186  11.675  1.00167.18           C\nATOM    341  CB  LEU    45       6.309  16.183  12.099  1.00167.18           C\nATOM    342  CG  LEU    45       7.138  16.791  10.955  1.00167.18           C\nATOM    343  CD1 LEU    45       7.650  15.706   9.994  1.00167.18           C\nATOM    344  CD2 LEU    45       6.417  17.967  10.282  1.00167.18           C\nATOM    345  C   LEU    45       4.574  14.779  12.955  1.00167.18           C\nATOM    346  O   LEU    45       4.887  13.737  13.525  1.00167.18           O\nATOM    347  N   ILE    46       3.600  15.576  13.400  1.00141.89           N\nATOM    348  CA  ILE    46       2.956  15.393  14.664  1.00141.89           C\nATOM    349  CB  ILE    46       1.860  16.393  14.883  1.00141.89           C\nATOM    350  CG1 ILE    46       2.479  17.788  15.078  1.00141.89           C\nATOM    351  CG2 ILE    46       0.975  15.931  16.051  1.00141.89           C\nATOM    352  CD1 ILE    46       3.411  17.869  16.286  1.00141.89           C\nATOM    353  C   ILE    46       2.418  13.999  14.756  1.00141.89           C\nATOM    354  O   ILE    46       2.209  13.488  15.853  1.00141.89           O\nATOM    355  N   GLU    47       2.131  13.347  13.620  1.00 98.07           N\nATOM    356  CA  GLU    47       1.552  12.030  13.632  1.00 98.07           C\nATOM    357  CB  GLU    47       1.295  11.522  12.199  1.00 98.07           C\nATOM    358  CG  GLU    47       0.428  10.265  12.127  1.00 98.07           C\nATOM    359  CD  GLU    47      -0.090  10.103  10.700  1.00 98.07           C\nATOM    360  OE1 GLU    47       0.747  10.012   9.763  1.00 98.07           O\nATOM    361  OE2 GLU    47      -1.338  10.074  10.532  1.00 98.07           O\nATOM    362  C   GLU    47       2.419  11.027  14.375  1.00 98.07           C\nATOM    363  O   GLU    47       1.880  10.171  15.078  1.00 98.07           O\nATOM    364  N   TYR    48       3.768  11.089  14.266  1.00119.09           N\nATOM    365  CA  TYR    48       4.632  10.100  14.882  1.00119.09           C\nATOM    366  CB  TYR    48       6.114  10.209  14.474  1.00119.09           C\nATOM    367  CG  TYR    48       6.848   9.034  15.032  1.00119.09           C\nATOM    368  CD1 TYR    48       6.684   7.799  14.449  1.00119.09           C\nATOM    369  CD2 TYR    48       7.702   9.156  16.104  1.00119.09           C\nATOM    370  CE1 TYR    48       7.348   6.699  14.936  1.00119.09           C\nATOM    371  CE2 TYR    48       8.373   8.056  16.595  1.00119.09           C\nATOM    372  CZ  TYR    48       8.195   6.825  16.012  1.00119.09           C\nATOM    373  OH  TYR    48       8.875   5.694  16.510  1.00119.09           O\nATOM    374  C   TYR    48       4.553  10.182  16.381  1.00119.09           C\nATOM    375  O   TYR    48       4.463  11.265  16.954  1.00119.09           O\nATOM    376  N   SER    49       4.529   9.001  17.040  1.00 78.58           N\nATOM    377  CA  SER    49       4.364   8.857  18.460  1.00 78.58           C\nATOM    378  CB  SER    49       3.898   7.447  18.860  1.00 78.58           C\nATOM    379  OG  SER    49       4.926   6.502  18.607  1.00 78.58           O\nATOM    380  C   SER    49       5.584   9.144  19.286  1.00 78.58           C\nATOM    381  O   SER    49       5.444   9.597  20.421  1.00 78.58           O\nATOM    382  N   ASP    50       6.817   8.894  18.795  1.00219.35           N\nATOM    383  CA  ASP    50       7.850   9.021  19.784  1.00219.35           C\nATOM    384  CB  ASP    50       8.733   7.774  19.912  1.00219.35           C\nATOM    385  CG  ASP    50       9.370   7.920  21.276  1.00219.35           C\nATOM    386  OD1 ASP    50       8.865   8.782  22.045  1.00219.35           O\nATOM    387  OD2 ASP    50      10.351   7.188  21.576  1.00219.35           O\nATOM    388  C   ASP    50       8.756  10.203  19.573  1.00219.35           C\nATOM    389  O   ASP    50       9.981  10.076  19.604  1.00219.35           O\nATOM    390  N   PHE    51       8.173  11.395  19.371  1.00171.59           N\nATOM    391  CA  PHE    51       8.932  12.610  19.266  1.00171.59           C\nATOM    392  CB  PHE    51       8.149  13.839  18.781  1.00171.59           C\nATOM    393  CG  PHE    51       7.878  13.653  17.341  1.00171.59           C\nATOM    394  CD1 PHE    51       8.807  14.038  16.400  1.00171.59           C\nATOM    395  CD2 PHE    51       6.702  13.079  16.936  1.00171.59           C\nATOM    396  CE1 PHE    51       8.562  13.863  15.062  1.00171.59           C\nATOM    397  CE2 PHE    51       6.456  12.905  15.601  1.00171.59           C\nATOM    398  CZ  PHE    51       7.379  13.294  14.660  1.00171.59           C\nATOM    399  C   PHE    51       9.448  13.013  20.608  1.00171.59           C\nATOM    400  O   PHE    51      10.530  13.592  20.710  1.00171.59           O\nATOM    401  N   ALA    52       8.640  12.797  21.669  1.00224.82           N\nATOM    402  CA  ALA    52       9.053  13.228  22.973  1.00224.82           C\nATOM    403  CB  ALA    52      10.518  12.922  23.352  1.00224.82           C\nATOM    404  C   ALA    52       8.807  14.700  22.997  1.00224.82           C\nATOM    405  O   ALA    52       7.836  15.175  22.415  1.00224.82           O\nATOM    406  N   THR    53       9.643  15.472  23.719  1.00100.37           N\nATOM    407  CA  THR    53       9.428  16.893  23.723  1.00100.37           C\nATOM    408  CB  THR    53      10.297  17.634  24.698  1.00100.37           C\nATOM    409  OG1 THR    53      10.050  17.181  26.020  1.00100.37           O\nATOM    410  CG2 THR    53      10.010  19.142  24.580  1.00100.37           C\nATOM    411  C   THR    53       9.791  17.403  22.366  1.00100.37           C\nATOM    412  O   THR    53      10.869  17.108  21.856  1.00100.37           O\nATOM    413  N   SER    54       8.900  18.214  21.767  1.00 71.00           N\nATOM    414  CA  SER    54       9.093  18.708  20.436  1.00 71.00           C\nATOM    415  CB  SER    54       7.846  18.547  19.551  1.00 71.00           C\nATOM    416  OG  SER    54       6.815  19.398  20.024  1.00 71.00           O\nATOM    417  C   SER    54       9.372  20.173  20.491  1.00 71.00           C\nATOM    418  O   SER    54       8.701  20.933  21.191  1.00 71.00           O\nATOM    419  N   ALA    55      10.403  20.591  19.734  1.00 57.18           N\nATOM    420  CA  ALA    55      10.732  21.980  19.651  1.00 57.18           C\nATOM    421  CB  ALA    55      12.123  22.313  20.214  1.00 57.18           C\nATOM    422  C   ALA    55      10.742  22.334  18.202  1.00 57.18           C\nATOM    423  O   ALA    55      11.322  21.623  17.379  1.00 57.18           O\nATOM    424  N   CYS    56      10.087  23.455  17.844  1.00106.65           N\nATOM    425  CA  CYS    56      10.088  23.794  16.454  1.00106.65           C\nATOM    426  CB  CYS    56       8.707  23.760  15.784  1.00106.65           C\nATOM    427  SG  CYS    56       7.531  24.903  16.540  1.00106.65           S\nATOM    428  C   CYS    56      10.727  25.121  16.239  1.00106.65           C\nATOM    429  O   CYS    56      10.436  26.107  16.917  1.00106.65           O\nATOM    430  N   VAL    57      11.659  25.141  15.270  1.00 60.43           N\nATOM    431  CA  VAL    57      12.335  26.351  14.920  1.00 60.43           C\nATOM    432  CB  VAL    57      13.794  26.148  14.645  1.00 60.43           C\nATOM    433  CG1 VAL    57      14.416  27.494  14.237  1.00 60.43           C\nATOM    434  CG2 VAL    57      14.440  25.503  15.882  1.00 60.43           C\nATOM    435  C   VAL    57      11.715  26.824  13.646  1.00 60.43           C\nATOM    436  O   VAL    57      12.017  26.320  12.565  1.00 60.43           O\nATOM    437  N   LEU    58      10.836  27.835  13.738  1.00130.61           N\nATOM    438  CA  LEU    58      10.220  28.326  12.547  1.00130.61           C\nATOM    439  CB  LEU    58       8.893  29.053  12.744  1.00130.61           C\nATOM    440  CG  LEU    58       7.739  28.174  13.204  1.00130.61           C\nATOM    441  CD1 LEU    58       7.940  27.655  14.637  1.00130.61           C\nATOM    442  CD2 LEU    58       6.437  28.937  12.980  1.00130.61           C\nATOM    443  C   LEU    58      11.077  29.402  11.994  1.00130.61           C\nATOM    444  O   LEU    58      11.223  30.464  12.598  1.00130.61           O\nATOM    445  N   ALA    59      11.648  29.167  10.805  1.00 78.94           N\nATOM    446  CA  ALA    59      12.391  30.224  10.208  1.00 78.94           C\nATOM    447  CB  ALA    59      13.509  29.744   9.270  1.00 78.94           C\nATOM    448  C   ALA    59      11.375  30.935   9.391  1.00 78.94           C\nATOM    449  O   ALA    59      10.682  30.291   8.617  1.00 78.94           O\nATOM    450  N   ALA    60      11.203  32.259   9.572  1.00 69.79           N\nATOM    451  CA  ALA    60      10.220  32.954   8.790  1.00 69.79           C\nATOM    452  CB  ALA    60       8.782  32.746   9.292  1.00 69.79           C\nATOM    453  C   ALA    60      10.479  34.423   8.858  1.00 69.79           C\nATOM    454  O   ALA    60      11.040  34.930   9.829  1.00 69.79           O\nATOM    455  N   GLU    61      10.171  35.126   7.751  1.00 88.98           N\nATOM    456  CA  GLU    61      10.303  36.553   7.700  1.00 88.98           C\nATOM    457  CB  GLU    61      10.527  37.078   6.276  1.00 88.98           C\nATOM    458  CG  GLU    61      10.875  38.566   6.251  1.00 88.98           C\nATOM    459  CD  GLU    61      11.515  38.889   4.911  1.00 88.98           C\nATOM    460  OE1 GLU    61      12.759  38.722   4.811  1.00 88.98           O\nATOM    461  OE2 GLU    61      10.780  39.306   3.976  1.00 88.98           O\nATOM    462  C   GLU    61       9.132  37.291   8.295  1.00 88.98           C\nATOM    463  O   GLU    61       9.320  38.245   9.048  1.00 88.98           O\nATOM    464  N   CYS    62       7.882  36.871   7.980  1.00 97.99           N\nATOM    465  CA  CYS    62       6.730  37.653   8.371  1.00 97.99           C\nATOM    466  CB  CYS    62       5.527  37.520   7.423  1.00 97.99           C\nATOM    467  SG  CYS    62       4.150  38.591   7.933  1.00 97.99           S\nATOM    468  C   CYS    62       6.259  37.327   9.756  1.00 97.99           C\nATOM    469  O   CYS    62       6.162  36.168  10.157  1.00 97.99           O\nATOM    470  N   THR    63       5.929  38.388  10.519  1.00 46.15           N\nATOM    471  CA  THR    63       5.467  38.269  11.874  1.00 46.15           C\nATOM    472  CB  THR    63       5.395  39.568  12.613  1.00 46.15           C\nATOM    473  OG1 THR    63       4.465  40.437  11.986  1.00 46.15           O\nATOM    474  CG2 THR    63       6.795  40.203  12.645  1.00 46.15           C\nATOM    475  C   THR    63       4.109  37.654  11.918  1.00 46.15           C\nATOM    476  O   THR    63       3.785  36.907  12.841  1.00 46.15           O\nATOM    477  N   ILE    64       3.258  37.961  10.930  1.00135.01           N\nATOM    478  CA  ILE    64       1.917  37.465  11.007  1.00135.01           C\nATOM    479  CB  ILE    64       1.029  37.997   9.905  1.00135.01           C\nATOM    480  CG1 ILE    64       1.376  37.425   8.520  1.00135.01           C\nATOM    481  CG2 ILE    64       1.160  39.532   9.938  1.00135.01           C\nATOM    482  CD1 ILE    64       0.817  36.028   8.240  1.00135.01           C\nATOM    483  C   ILE    64       1.973  35.968  10.949  1.00135.01           C\nATOM    484  O   ILE    64       1.240  35.274  11.654  1.00135.01           O\nATOM    485  N   PHE    65       2.864  35.444  10.087  1.00 78.37           N\nATOM    486  CA  PHE    65       3.011  34.040   9.831  1.00 78.37           C\nATOM    487  CB  PHE    65       3.974  33.778   8.667  1.00 78.37           C\nATOM    488  CG  PHE    65       3.709  32.373   8.302  1.00 78.37           C\nATOM    489  CD1 PHE    65       2.544  32.088   7.635  1.00 78.37           C\nATOM    490  CD2 PHE    65       4.593  31.369   8.613  1.00 78.37           C\nATOM    491  CE1 PHE    65       2.238  30.803   7.277  1.00 78.37           C\nATOM    492  CE2 PHE    65       4.293  30.078   8.258  1.00 78.37           C\nATOM    493  CZ  PHE    65       3.121  29.806   7.594  1.00 78.37           C\nATOM    494  C   PHE    65       3.523  33.329  11.046  1.00 78.37           C\nATOM    495  O   PHE    65       3.117  32.203  11.338  1.00 78.37           O\nATOM    496  N   LYS    66       4.486  33.941  11.760  1.00 65.80           N\nATOM    497  CA  LYS    66       5.010  33.294  12.927  1.00 65.80           C\nATOM    498  CB  LYS    66       6.306  33.910  13.479  1.00 65.80           C\nATOM    499  CG  LYS    66       7.549  33.459  12.705  1.00 65.80           C\nATOM    500  CD  LYS    66       8.836  34.196  13.086  1.00 65.80           C\nATOM    501  CE  LYS    66      10.059  33.734  12.293  1.00 65.80           C\nATOM    502  NZ  LYS    66      11.212  34.618  12.574  1.00 65.80           N\nATOM    503  C   LYS    66       3.991  33.262  14.014  1.00 65.80           C\nATOM    504  O   LYS    66       3.876  32.268  14.729  1.00 65.80           O\nATOM    505  N   ASP    67       3.215  34.349  14.167  1.00 46.49           N\nATOM    506  CA  ASP    67       2.238  34.413  15.210  1.00 46.49           C\nATOM    507  CB  ASP    67       1.434  35.726  15.154  1.00 46.49           C\nATOM    508  CG  ASP    67       2.385  36.881  15.436  1.00 46.49           C\nATOM    509  OD1 ASP    67       3.218  36.754  16.374  1.00 46.49           O\nATOM    510  OD2 ASP    67       2.304  37.903  14.702  1.00 46.49           O\nATOM    511  C   ASP    67       1.282  33.301  14.948  1.00 46.49           C\nATOM    512  O   ASP    67       0.814  32.621  15.863  1.00 46.49           O\nATOM    513  N   ALA    68       0.982  33.084  13.658  1.00 57.51           N\nATOM    514  CA  ALA    68       0.016  32.097  13.303  1.00 57.51           C\nATOM    515  CB  ALA    68      -0.210  32.025  11.783  1.00 57.51           C\nATOM    516  C   ALA    68       0.470  30.749  13.746  1.00 57.51           C\nATOM    517  O   ALA    68      -0.235  30.105  14.501  1.00 57.51           O\nATOM    518  N   MET    69       1.704  30.326  13.422  1.00168.34           N\nATOM    519  CA  MET    69       2.112  28.972  13.704  1.00168.34           C\nATOM    520  CB  MET    69       3.546  28.680  13.260  1.00168.34           C\nATOM    521  CG  MET    69       3.769  28.531  11.750  1.00168.34           C\nATOM    522  SD  MET    69       3.543  26.849  11.084  1.00168.34           S\nATOM    523  CE  MET    69       1.780  26.691  11.480  1.00168.34           C\nATOM    524  C   MET    69       2.049  28.742  15.174  1.00168.34           C\nATOM    525  O   MET    69       1.810  27.625  15.635  1.00168.34           O\nATOM    526  N   GLY    70       2.301  29.796  15.958  1.00 46.46           N\nATOM    527  CA  GLY    70       2.226  29.636  17.372  1.00 46.46           C\nATOM    528  C   GLY    70       0.832  29.218  17.732  1.00 46.46           C\nATOM    529  O   GLY    70       0.673  28.378  18.612  1.00 46.46           O\nATOM    530  N   LYS    71      -0.210  29.815  17.102  1.00111.98           N\nATOM    531  CA  LYS    71      -1.589  29.502  17.405  1.00111.98           C\nATOM    532  CB  LYS    71      -2.614  30.540  16.889  1.00111.98           C\nATOM    533  CG  LYS    71      -2.734  31.754  17.823  1.00111.98           C\nATOM    534  CD  LYS    71      -3.556  32.926  17.277  1.00111.98           C\nATOM    535  CE  LYS    71      -5.073  32.723  17.363  1.00111.98           C\nATOM    536  NZ  LYS    71      -5.777  33.971  16.988  1.00111.98           N\nATOM    537  C   LYS    71      -2.026  28.116  16.986  1.00111.98           C\nATOM    538  O   LYS    71      -2.797  27.519  17.733  1.00111.98           O\nATOM    539  N   PRO    72      -1.723  27.584  15.829  1.00180.65           N\nATOM    540  CA  PRO    72      -2.015  26.182  15.673  1.00180.65           C\nATOM    541  CD  PRO    72      -2.282  28.270  14.668  1.00180.65           C\nATOM    542  CB  PRO    72      -2.066  25.910  14.174  1.00180.65           C\nATOM    543  CG  PRO    72      -2.697  27.201  13.646  1.00180.65           C\nATOM    544  C   PRO    72      -1.234  25.169  16.455  1.00180.65           C\nATOM    545  O   PRO    72      -1.774  24.099  16.723  1.00180.65           O\nATOM    546  N   VAL    73       0.043  25.428  16.769  1.00 97.35           N\nATOM    547  CA  VAL    73       0.818  24.448  17.480  1.00 97.35           C\nATOM    548  CB  VAL    73       2.284  24.577  17.200  1.00 97.35           C\nATOM    549  CG1 VAL    73       2.997  23.377  17.829  1.00 97.35           C\nATOM    550  CG2 VAL    73       2.490  24.657  15.679  1.00 97.35           C\nATOM    551  C   VAL    73       0.587  24.381  18.989  1.00 97.35           C\nATOM    552  O   VAL    73       0.836  23.312  19.548  1.00 97.35           O\nATOM    553  N   PRO    74       0.188  25.414  19.724  1.00172.43           N\nATOM    554  CA  PRO    74       0.194  25.318  21.160  1.00172.43           C\nATOM    555  CD  PRO    74      -0.952  26.213  19.343  1.00172.43           C\nATOM    556  CB  PRO    74      -0.338  26.651  21.651  1.00172.43           C\nATOM    557  CG  PRO    74      -1.413  26.974  20.602  1.00172.43           C\nATOM    558  C   PRO    74      -0.663  24.200  21.659  1.00172.43           C\nATOM    559  O   PRO    74      -1.827  24.131  21.274  1.00172.43           O\nATOM    560  N   TYR    75      -0.088  23.357  22.544  1.00284.25           N\nATOM    561  CA  TYR    75      -0.650  22.203  23.194  1.00284.25           C\nATOM    562  CB  TYR    75      -1.244  21.103  22.282  1.00284.25           C\nATOM    563  CG  TYR    75      -2.584  21.487  21.764  1.00284.25           C\nATOM    564  CD1 TYR    75      -3.657  21.556  22.622  1.00284.25           C\nATOM    565  CD2 TYR    75      -2.793  21.725  20.428  1.00284.25           C\nATOM    566  CE1 TYR    75      -4.902  21.901  22.161  1.00284.25           C\nATOM    567  CE2 TYR    75      -4.037  22.068  19.955  1.00284.25           C\nATOM    568  CZ  TYR    75      -5.092  22.161  20.825  1.00284.25           C\nATOM    569  OH  TYR    75      -6.371  22.515  20.348  1.00284.25           O\nATOM    570  C   TYR    75       0.530  21.539  23.807  1.00284.25           C\nATOM    571  O   TYR    75       1.566  21.465  23.160  1.00284.25           O\nATOM    572  N   CYS    76       0.450  21.063  25.064  1.00137.39           N\nATOM    573  CA  CYS    76       1.600  20.409  25.616  1.00137.39           C\nATOM    574  CB  CYS    76       1.365  19.876  27.040  1.00137.39           C\nATOM    575  SG  CYS    76       2.728  18.844  27.669  1.00137.39           S\nATOM    576  C   CYS    76       1.822  19.242  24.734  1.00137.39           C\nATOM    577  O   CYS    76       2.938  18.937  24.311  1.00137.39           O\nATOM    578  N   TYR    77       0.704  18.587  24.402  1.00 93.24           N\nATOM    579  CA  TYR    77       0.751  17.464  23.544  1.00 93.24           C\nATOM    580  CB  TYR    77      -0.650  16.835  23.406  1.00 93.24           C\nATOM    581  CG  TYR    77      -0.612  15.646  22.509  1.00 93.24           C\nATOM    582  CD1 TYR    77      -0.201  14.414  22.974  1.00 93.24           C\nATOM    583  CD2 TYR    77      -1.013  15.769  21.204  1.00 93.24           C\nATOM    584  CE1 TYR    77      -0.183  13.324  22.135  1.00 93.24           C\nATOM    585  CE2 TYR    77      -0.997  14.686  20.360  1.00 93.24           C\nATOM    586  CZ  TYR    77      -0.581  13.463  20.824  1.00 93.24           C\nATOM    587  OH  TYR    77      -0.567  12.356  19.950  1.00 93.24           O\nATOM    588  C   TYR    77       1.245  17.966  22.224  1.00 93.24           C\nATOM    589  O   TYR    77       2.127  17.351  21.626  1.00 93.24           O\nATOM    590  N   ASP    78       0.724  19.114  21.731  1.00102.50           N\nATOM    591  CA  ASP    78       1.177  19.486  20.423  1.00102.50           C\nATOM    592  CB  ASP    78       0.366  20.599  19.740  1.00102.50           C\nATOM    593  CG  ASP    78      -0.802  19.910  19.045  1.00102.50           C\nATOM    594  OD1 ASP    78      -0.951  18.675  19.248  1.00102.50           O\nATOM    595  OD2 ASP    78      -1.539  20.590  18.282  1.00102.50           O\nATOM    596  C   ASP    78       2.630  19.839  20.361  1.00102.50           C\nATOM    597  O   ASP    78       3.377  19.205  19.619  1.00102.50           O\nATOM    598  N   THR    79       3.087  20.830  21.149  1.00138.83           N\nATOM    599  CA  THR    79       4.459  21.223  21.034  1.00138.83           C\nATOM    600  CB  THR    79       4.659  22.266  19.992  1.00138.83           C\nATOM    601  OG1 THR    79       6.037  22.439  19.703  1.00138.83           O\nATOM    602  CG2 THR    79       4.055  23.564  20.553  1.00138.83           C\nATOM    603  C   THR    79       4.907  21.854  22.316  1.00138.83           C\nATOM    604  O   THR    79       4.133  22.471  23.048  1.00138.83           O\nATOM    605  N   ASN    80       6.199  21.662  22.618  1.00110.63           N\nATOM    606  CA  ASN    80       6.854  22.245  23.746  1.00110.63           C\nATOM    607  CB  ASN    80       8.217  21.569  23.955  1.00110.63           C\nATOM    608  CG  ASN    80       8.863  22.124  25.204  1.00110.63           C\nATOM    609  OD1 ASN    80       9.966  22.664  25.157  1.00110.63           O\nATOM    610  ND2 ASN    80       8.166  21.976  26.358  1.00110.63           N\nATOM    611  C   ASN    80       7.097  23.703  23.502  1.00110.63           C\nATOM    612  O   ASN    80       6.864  24.533  24.379  1.00110.63           O\nATOM    613  N   LEU    81       7.572  24.060  22.283  1.00172.32           N\nATOM    614  CA  LEU    81       7.956  25.428  22.068  1.00172.32           C\nATOM    615  CB  LEU    81       9.372  25.667  22.647  1.00172.32           C\nATOM    616  CG  LEU    81       9.992  27.078  22.542  1.00172.32           C\nATOM    617  CD1 LEU    81      10.286  27.485  21.090  1.00172.32           C\nATOM    618  CD2 LEU    81       9.187  28.119  23.333  1.00172.32           C\nATOM    619  C   LEU    81       7.977  25.728  20.593  1.00172.32           C\nATOM    620  O   LEU    81       8.254  24.854  19.772  1.00172.32           O\nATOM    621  N   LEU    82       7.643  26.988  20.221  1.00127.66           N\nATOM    622  CA  LEU    82       7.736  27.461  18.861  1.00127.66           C\nATOM    623  CB  LEU    82       6.385  27.855  18.218  1.00127.66           C\nATOM    624  CG  LEU    82       5.445  26.681  17.886  1.00127.66           C\nATOM    625  CD1 LEU    82       5.006  25.959  19.158  1.00127.66           C\nATOM    626  CD2 LEU    82       4.273  27.110  16.987  1.00127.66           C\nATOM    627  C   LEU    82       8.544  28.724  18.894  1.00127.66           C\nATOM    628  O   LEU    82       8.046  29.775  19.294  1.00127.66           O\nATOM    629  N   GLU    83       9.817  28.665  18.456  1.00105.98           N\nATOM    630  CA  GLU    83      10.624  29.856  18.474  1.00105.98           C\nATOM    631  CB  GLU    83      11.924  29.705  19.274  1.00105.98           C\nATOM    632  CG  GLU    83      12.879  28.681  18.666  1.00105.98           C\nATOM    633  CD  GLU    83      14.255  28.953  19.246  1.00105.98           C\nATOM    634  OE1 GLU    83      14.327  29.444  20.405  1.00105.98           O\nATOM    635  OE2 GLU    83      15.258  28.680  18.533  1.00105.98           O\nATOM    636  C   GLU    83      11.021  30.187  17.061  1.00105.98           C\nATOM    637  O   GLU    83      11.518  29.332  16.332  1.00105.98           O\nATOM    638  N   GLY    84      10.812  31.453  16.640  1.00 49.75           N\nATOM    639  CA  GLY    84      11.101  31.869  15.287  1.00 49.75           C\nATOM    640  C   GLY    84      12.512  32.367  15.148  1.00 49.75           C\nATOM    641  O   GLY    84      13.157  32.726  16.133  1.00 49.75           O\nATOM    642  N   SER    85      13.008  32.424  13.885  1.00 87.87           N\nATOM    643  CA  SER    85      14.343  32.881  13.593  1.00 87.87           C\nATOM    644  CB  SER    85      15.397  31.769  13.743  1.00 87.87           C\nATOM    645  OG  SER    85      15.119  30.705  12.847  1.00 87.87           O\nATOM    646  C   SER    85      14.417  33.353  12.167  1.00 87.87           C\nATOM    647  O   SER    85      13.461  33.194  11.416  1.00 87.87           O\nATOM    648  N   ILE    86      15.521  34.054  11.816  1.00228.75           N\nATOM    649  CA  ILE    86      15.955  34.479  10.504  1.00228.75           C\nATOM    650  CB  ILE    86      15.171  34.028   9.298  1.00228.75           C\nATOM    651  CG1 ILE    86      13.802  34.700   9.171  1.00228.75           C\nATOM    652  CG2 ILE    86      15.105  32.492   9.333  1.00228.75           C\nATOM    653  CD1 ILE    86      13.214  34.495   7.779  1.00228.75           C\nATOM    654  C   ILE    86      16.003  35.973  10.510  1.00228.75           C\nATOM    655  O   ILE    86      16.721  36.596   9.727  1.00228.75           O\nATOM    656  N   SER    87      15.217  36.568  11.430  1.00136.16           N\nATOM    657  CA  SER    87      15.050  37.983  11.612  1.00136.16           C\nATOM    658  CB  SER    87      14.042  38.306  12.728  1.00136.16           C\nATOM    659  OG  SER    87      12.784  37.699  12.465  1.00136.16           O\nATOM    660  C   SER    87      16.344  38.592  12.045  1.00136.16           C\nATOM    661  O   SER    87      16.625  39.735  11.690  1.00136.16           O\nATOM    662  N   TYR    88      17.157  37.851  12.831  1.00298.94           N\nATOM    663  CA  TYR    88      18.420  38.371  13.285  1.00298.94           C\nATOM    664  CB  TYR    88      19.199  38.928  12.066  1.00298.94           C\nATOM    665  CG  TYR    88      20.414  39.757  12.321  1.00298.94           C\nATOM    666  CD1 TYR    88      21.654  39.190  12.502  1.00298.94           C\nATOM    667  CD2 TYR    88      20.307  41.129  12.319  1.00298.94           C\nATOM    668  CE1 TYR    88      22.761  39.983  12.701  1.00298.94           C\nATOM    669  CE2 TYR    88      21.408  41.927  12.517  1.00298.94           C\nATOM    670  CZ  TYR    88      22.638  41.353  12.707  1.00298.94           C\nATOM    671  OH  TYR    88      23.763  42.178  12.904  1.00298.94           O\nATOM    672  C   TYR    88      18.172  39.425  14.324  1.00298.94           C\nATOM    673  O   TYR    88      19.100  40.086  14.785  1.00298.94           O\nATOM    674  N   SER    89      16.911  39.566  14.776  1.00 67.81           N\nATOM    675  CA  SER    89      16.615  40.505  15.822  1.00 67.81           C\nATOM    676  CB  SER    89      15.797  41.717  15.348  1.00 67.81           C\nATOM    677  OG  SER    89      14.480  41.312  15.010  1.00 67.81           O\nATOM    678  C   SER    89      15.754  39.765  16.796  1.00 67.81           C\nATOM    679  O   SER    89      15.000  38.879  16.407  1.00 67.81           O\nATOM    680  N   GLU    90      15.838  40.095  18.099  1.00101.84           N\nATOM    681  CA  GLU    90      15.020  39.387  19.043  1.00101.84           C\nATOM    682  CB  GLU    90      15.708  39.069  20.376  1.00101.84           C\nATOM    683  CG  GLU    90      16.251  40.314  21.081  1.00101.84           C\nATOM    684  CD  GLU    90      16.604  39.952  22.517  1.00101.84           C\nATOM    685  OE1 GLU    90      15.685  39.512  23.259  1.00101.84           O\nATOM    686  OE2 GLU    90      17.795  40.116  22.897  1.00101.84           O\nATOM    687  C   GLU    90      13.858  40.240  19.407  1.00101.84           C\nATOM    688  O   GLU    90      13.969  41.460  19.516  1.00101.84           O\nATOM    689  N   LEU    91      12.689  39.606  19.598  1.00151.08           N\nATOM    690  CA  LEU    91      11.567  40.363  20.051  1.00151.08           C\nATOM    691  CB  LEU    91      10.254  39.905  19.389  1.00151.08           C\nATOM    692  CG  LEU    91       9.019  40.754  19.733  1.00151.08           C\nATOM    693  CD1 LEU    91       9.224  42.220  19.323  1.00151.08           C\nATOM    694  CD2 LEU    91       7.754  40.157  19.093  1.00151.08           C\nATOM    695  C   LEU    91      11.513  40.098  21.527  1.00151.08           C\nATOM    696  O   LEU    91      11.011  39.064  21.961  1.00151.08           O\nATOM    697  N   ARG    92      12.046  41.034  22.344  1.00124.44           N\nATOM    698  CA  ARG    92      12.130  40.805  23.763  1.00124.44           C\nATOM    699  CB  ARG    92      12.824  41.922  24.574  1.00124.44           C\nATOM    700  CG  ARG    92      14.344  41.835  24.686  1.00124.44           C\nATOM    701  CD  ARG    92      14.930  42.953  25.559  1.00124.44           C\nATOM    702  NE  ARG    92      14.381  42.812  26.942  1.00124.44           N\nATOM    703  CZ  ARG    92      15.116  42.243  27.945  1.00124.44           C\nATOM    704  NH1 ARG    92      16.378  41.783  27.708  1.00124.44           N\nATOM    705  NH2 ARG    92      14.585  42.149  29.200  1.00124.44           N\nATOM    706  C   ARG    92      10.773  40.662  24.366  1.00124.44           C\nATOM    707  O   ARG    92      10.557  39.759  25.172  1.00124.44           O\nATOM    708  N   PRO    93       9.841  41.501  24.024  1.00159.89           N\nATOM    709  CA  PRO    93       8.585  41.422  24.708  1.00159.89           C\nATOM    710  CD  PRO    93      10.136  42.862  23.587  1.00159.89           C\nATOM    711  CB  PRO    93       7.869  42.744  24.425  1.00159.89           C\nATOM    712  CG  PRO    93       9.010  43.738  24.158  1.00159.89           C\nATOM    713  C   PRO    93       7.753  40.232  24.385  1.00159.89           C\nATOM    714  O   PRO    93       6.877  39.882  25.173  1.00159.89           O\nATOM    715  N   ASP    94       8.003  39.601  23.231  1.00129.24           N\nATOM    716  CA  ASP    94       7.149  38.567  22.748  1.00129.24           C\nATOM    717  CB  ASP    94       7.557  38.128  21.332  1.00129.24           C\nATOM    718  CG  ASP    94       6.445  37.317  20.686  1.00129.24           C\nATOM    719  OD1 ASP    94       5.606  36.733  21.421  1.00129.24           O\nATOM    720  OD2 ASP    94       6.425  37.273  19.427  1.00129.24           O\nATOM    721  C   ASP    94       7.155  37.362  23.641  1.00129.24           C\nATOM    722  O   ASP    94       6.082  36.821  23.905  1.00129.24           O\nATOM    723  N   THR    95       8.347  36.963  24.152  1.00143.87           N\nATOM    724  CA  THR    95       8.551  35.730  24.881  1.00143.87           C\nATOM    725  CB  THR    95       9.844  35.670  25.648  1.00143.87           C\nATOM    726  OG1 THR    95       9.884  36.719  26.602  1.00143.87           O\nATOM    727  CG2 THR    95      11.040  35.798  24.697  1.00143.87           C\nATOM    728  C   THR    95       7.447  35.537  25.858  1.00143.87           C\nATOM    729  O   THR    95       7.334  36.254  26.850  1.00143.87           O\nATOM    730  N   ARG    96       6.577  34.550  25.557  1.00164.99           N\nATOM    731  CA  ARG    96       5.450  34.327  26.401  1.00164.99           C\nATOM    732  CB  ARG    96       4.192  35.103  25.961  1.00164.99           C\nATOM    733  CG  ARG    96       3.190  35.363  27.094  1.00164.99           C\nATOM    734  CD  ARG    96       1.849  34.636  26.972  1.00164.99           C\nATOM    735  NE  ARG    96       1.042  35.008  28.173  1.00164.99           N\nATOM    736  CZ  ARG    96       0.323  36.168  28.190  1.00164.99           C\nATOM    737  NH1 ARG    96       0.335  36.996  27.104  1.00164.99           N\nATOM    738  NH2 ARG    96      -0.401  36.515  29.293  1.00164.99           N\nATOM    739  C   ARG    96       5.144  32.868  26.386  1.00164.99           C\nATOM    740  O   ARG    96       5.681  32.093  25.594  1.00164.99           O\nATOM    741  N   TYR    97       4.245  32.484  27.299  1.00194.19           N\nATOM    742  CA  TYR    97       3.810  31.146  27.541  1.00194.19           C\nATOM    743  CB  TYR    97       3.639  31.006  29.063  1.00194.19           C\nATOM    744  CG  TYR    97       2.977  29.741  29.450  1.00194.19           C\nATOM    745  CD1 TYR    97       3.722  28.607  29.656  1.00194.19           C\nATOM    746  CD2 TYR    97       1.615  29.702  29.627  1.00194.19           C\nATOM    747  CE1 TYR    97       3.109  27.437  30.026  1.00194.19           C\nATOM    748  CE2 TYR    97       0.996  28.532  29.993  1.00194.19           C\nATOM    749  CZ  TYR    97       1.744  27.400  30.191  1.00194.19           C\nATOM    750  OH  TYR    97       1.109  26.201  30.568  1.00194.19           O\nATOM    751  C   TYR    97       2.460  31.064  26.914  1.00194.19           C\nATOM    752  O   TYR    97       1.510  31.676  27.394  1.00194.19           O\nATOM    753  N   VAL    98       2.331  30.300  25.816  1.00 73.57           N\nATOM    754  CA  VAL    98       1.041  30.252  25.209  1.00 73.57           C\nATOM    755  CB  VAL    98       1.005  29.464  23.940  1.00 73.57           C\nATOM    756  CG1 VAL    98      -0.439  29.429  23.409  1.00 73.57           C\nATOM    757  CG2 VAL    98       2.016  30.106  22.983  1.00 73.57           C\nATOM    758  C   VAL    98       0.156  29.596  26.195  1.00 73.57           C\nATOM    759  O   VAL    98       0.531  28.612  26.828  1.00 73.57           O\nATOM    760  N   LEU    99      -1.052  30.155  26.345  1.00244.47           N\nATOM    761  CA  LEU    99      -1.980  29.630  27.292  1.00244.47           C\nATOM    762  CB  LEU    99      -3.311  30.413  27.290  1.00244.47           C\nATOM    763  CG  LEU    99      -4.420  29.891  28.236  1.00244.47           C\nATOM    764  CD1 LEU    99      -5.047  28.582  27.726  1.00244.47           C\nATOM    765  CD2 LEU    99      -3.933  29.785  29.689  1.00244.47           C\nATOM    766  C   LEU    99      -2.251  28.237  26.876  1.00244.47           C\nATOM    767  O   LEU    99      -2.291  27.328  27.703  1.00244.47           O\nATOM    768  N   MET   100      -2.403  28.005  25.565  1.00265.73           N\nATOM    769  CA  MET   100      -2.762  26.668  25.229  1.00265.73           C\nATOM    770  CB  MET   100      -3.232  26.470  23.779  1.00265.73           C\nATOM    771  CG  MET   100      -4.699  26.860  23.634  1.00265.73           C\nATOM    772  SD  MET   100      -5.763  25.894  24.749  1.00265.73           S\nATOM    773  CE  MET   100      -5.413  24.300  23.952  1.00265.73           C\nATOM    774  C   MET   100      -1.668  25.707  25.549  1.00265.73           C\nATOM    775  O   MET   100      -0.629  25.635  24.896  1.00265.73           O\nATOM    776  N   ASP   101      -1.918  24.962  26.636  1.00108.33           N\nATOM    777  CA  ASP   101      -1.179  23.831  27.098  1.00108.33           C\nATOM    778  CB  ASP   101      -1.566  22.544  26.355  1.00108.33           C\nATOM    779  CG  ASP   101      -2.967  22.154  26.808  1.00108.33           C\nATOM    780  OD1 ASP   101      -3.095  21.646  27.952  1.00108.33           O\nATOM    781  OD2 ASP   101      -3.921  22.358  26.013  1.00108.33           O\nATOM    782  C   ASP   101       0.316  23.978  27.094  1.00108.33           C\nATOM    783  O   ASP   101       1.013  23.123  26.560  1.00108.33           O\nATOM    784  N   GLY   102       0.872  25.053  27.679  1.00 50.17           N\nATOM    785  CA  GLY   102       2.294  25.060  27.912  1.00 50.17           C\nATOM    786  C   GLY   102       3.130  25.082  26.670  1.00 50.17           C\nATOM    787  O   GLY   102       4.139  24.377  26.594  1.00 50.17           O\nATOM    788  N   SER   103       2.756  25.872  25.648  1.00 62.67           N\nATOM    789  CA  SER   103       3.647  25.965  24.529  1.00 62.67           C\nATOM    790  CB  SER   103       2.947  25.860  23.162  1.00 62.67           C\nATOM    791  OG  SER   103       2.304  24.601  23.032  1.00 62.67           O\nATOM    792  C   SER   103       4.249  27.332  24.604  1.00 62.67           C\nATOM    793  O   SER   103       3.537  28.334  24.562  1.00 62.67           O\nATOM    794  N   ILE   104       5.585  27.430  24.716  1.00101.22           N\nATOM    795  CA  ILE   104       6.121  28.754  24.815  1.00101.22           C\nATOM    796  CB  ILE   104       7.239  28.927  25.804  1.00101.22           C\nATOM    797  CG1 ILE   104       6.626  28.750  27.210  1.00101.22           C\nATOM    798  CG2 ILE   104       7.889  30.308  25.605  1.00101.22           C\nATOM    799  CD1 ILE   104       7.608  28.874  28.367  1.00101.22           C\nATOM    800  C   ILE   104       6.412  29.282  23.446  1.00101.22           C\nATOM    801  O   ILE   104       6.620  28.521  22.503  1.00101.22           O\nATOM    802  N   ILE   105       6.294  30.619  23.286  1.00 82.74           N\nATOM    803  CA  ILE   105       6.456  31.234  21.999  1.00 82.74           C\nATOM    804  CB  ILE   105       5.147  31.683  21.390  1.00 82.74           C\nATOM    805  CG1 ILE   105       4.318  30.477  20.934  1.00 82.74           C\nATOM    806  CG2 ILE   105       5.402  32.711  20.278  1.00 82.74           C\nATOM    807  CD1 ILE   105       2.982  30.895  20.321  1.00 82.74           C\nATOM    808  C   ILE   105       7.321  32.460  22.073  1.00 82.74           C\nATOM    809  O   ILE   105       7.308  33.191  23.061  1.00 82.74           O\nATOM    810  N   GLN   106       8.121  32.682  21.006  1.00145.78           N\nATOM    811  CA  GLN   106       8.949  33.850  20.830  1.00145.78           C\nATOM    812  CB  GLN   106      10.377  33.651  21.374  1.00145.78           C\nATOM    813  CG  GLN   106      11.398  34.691  20.896  1.00145.78           C\nATOM    814  CD  GLN   106      11.322  35.966  21.722  1.00145.78           C\nATOM    815  OE1 GLN   106      10.249  36.390  22.144  1.00145.78           O\nATOM    816  NE2 GLN   106      12.504  36.602  21.944  1.00145.78           N\nATOM    817  C   GLN   106       9.052  34.083  19.351  1.00145.78           C\nATOM    818  O   GLN   106       9.563  33.245  18.612  1.00145.78           O\nATOM    819  N   PHE   107       8.598  35.256  18.873  1.00200.38           N\nATOM    820  CA  PHE   107       8.590  35.513  17.466  1.00200.38           C\nATOM    821  CB  PHE   107       7.962  36.882  17.153  1.00200.38           C\nATOM    822  CG  PHE   107       8.072  37.098  15.690  1.00200.38           C\nATOM    823  CD1 PHE   107       7.204  36.471  14.837  1.00200.38           C\nATOM    824  CD2 PHE   107       9.037  37.932  15.177  1.00200.38           C\nATOM    825  CE1 PHE   107       7.314  36.680  13.487  1.00200.38           C\nATOM    826  CE2 PHE   107       9.151  38.143  13.826  1.00200.38           C\nATOM    827  CZ  PHE   107       8.283  37.508  12.975  1.00200.38           C\nATOM    828  C   PHE   107       9.991  35.420  16.907  1.00200.38           C\nATOM    829  O   PHE   107      10.143  34.730  15.901  1.00200.38           O\nATOM    830  N   PRO   108      11.016  36.107  17.400  1.00155.42           N\nATOM    831  CA  PRO   108      12.336  35.796  16.888  1.00155.42           C\nATOM    832  CD  PRO   108      10.875  37.551  17.539  1.00155.42           C\nATOM    833  CB  PRO   108      12.828  37.054  16.191  1.00155.42           C\nATOM    834  CG  PRO   108      12.202  38.161  17.052  1.00155.42           C\nATOM    835  C   PRO   108      13.266  35.452  18.023  1.00155.42           C\nATOM    836  O   PRO   108      13.299  36.218  18.987  1.00155.42           O\nATOM    837  N   ASN   109      14.114  34.413  17.927  1.00118.63           N\nATOM    838  CA  ASN   109      14.969  34.197  19.058  1.00118.63           C\nATOM    839  CB  ASN   109      15.663  32.827  19.059  1.00118.63           C\nATOM    840  CG  ASN   109      16.055  32.549  20.501  1.00118.63           C\nATOM    841  OD1 ASN   109      16.030  33.444  21.345  1.00118.63           O\nATOM    842  ND2 ASN   109      16.420  31.273  20.802  1.00118.63           N\nATOM    843  C   ASN   109      16.034  35.246  19.112  1.00118.63           C\nATOM    844  O   ASN   109      16.228  35.876  20.147  1.00118.63           O\nATOM    845  N   THR   110      16.756  35.430  17.982  1.00171.31           N\nATOM    846  CA  THR   110      17.849  36.355  17.780  1.00171.31           C\nATOM    847  CB  THR   110      18.787  36.641  18.931  1.00171.31           C\nATOM    848  OG1 THR   110      19.197  35.431  19.547  1.00171.31           O\nATOM    849  CG2 THR   110      18.249  37.673  19.925  1.00171.31           C\nATOM    850  C   THR   110      18.786  35.692  16.831  1.00171.31           C\nATOM    851  O   THR   110      18.545  34.577  16.375  1.00171.31           O\nATOM    852  N   TYR   111      19.923  36.363  16.567  1.00147.77           N\nATOM    853  CA  TYR   111      20.942  35.809  15.722  1.00147.77           C\nATOM    854  CB  TYR   111      21.377  36.745  14.576  1.00147.77           C\nATOM    855  CG  TYR   111      22.036  37.922  15.204  1.00147.77           C\nATOM    856  CD1 TYR   111      21.295  38.811  15.945  1.00147.77           C\nATOM    857  CD2 TYR   111      23.383  38.146  15.043  1.00147.77           C\nATOM    858  CE1 TYR   111      21.898  39.899  16.533  1.00147.77           C\nATOM    859  CE2 TYR   111      23.991  39.234  15.627  1.00147.77           C\nATOM    860  CZ  TYR   111      23.247  40.112  16.376  1.00147.77           C\nATOM    861  OH  TYR   111      23.861  41.228  16.980  1.00147.77           O\nATOM    862  C   TYR   111      22.136  35.580  16.598  1.00147.77           C\nATOM    863  O   TYR   111      22.465  36.406  17.449  1.00147.77           O\nATOM    864  N   LEU   112      22.792  34.409  16.445  1.00176.93           N\nATOM    865  CA  LEU   112      23.910  34.088  17.291  1.00176.93           C\nATOM    866  CB  LEU   112      24.489  32.683  17.036  1.00176.93           C\nATOM    867  CG  LEU   112      23.569  31.516  17.431  1.00176.93           C\nATOM    868  CD1 LEU   112      22.269  31.523  16.613  1.00176.93           C\nATOM    869  CD2 LEU   112      24.321  30.175  17.357  1.00176.93           C\nATOM    870  C   LEU   112      25.046  35.025  17.059  1.00176.93           C\nATOM    871  O   LEU   112      25.527  35.667  17.990  1.00176.93           O\nATOM    872  N   GLU   113      25.484  35.175  15.796  1.00161.35           N\nATOM    873  CA  GLU   113      26.668  35.953  15.574  1.00161.35           C\nATOM    874  CB  GLU   113      27.900  35.107  15.206  1.00161.35           C\nATOM    875  CG  GLU   113      27.785  34.436  13.835  1.00161.35           C\nATOM    876  CD  GLU   113      26.868  33.223  13.937  1.00161.35           C\nATOM    877  OE1 GLU   113      25.619  33.390  13.895  1.00161.35           O\nATOM    878  OE2 GLU   113      27.418  32.096  14.052  1.00161.35           O\nATOM    879  C   GLU   113      26.435  36.876  14.427  1.00161.35           C\nATOM    880  O   GLU   113      25.588  36.628  13.572  1.00161.35           O\nATOM    881  N   GLY   114      27.203  37.984  14.404  1.00120.20           N\nATOM    882  CA  GLY   114      27.112  38.968  13.370  1.00120.20           C\nATOM    883  C   GLY   114      27.534  40.270  13.973  1.00120.20           C\nATOM    884  O   GLY   114      27.913  40.340  15.142  1.00120.20           O\nATOM    885  N   SER   115      27.490  41.342  13.166  1.00159.79           N\nATOM    886  CA  SER   115      27.804  42.649  13.652  1.00159.79           C\nATOM    887  CB  SER   115      28.294  43.623  12.565  1.00159.79           C\nATOM    888  OG  SER   115      29.567  43.218  12.090  1.00159.79           O\nATOM    889  C   SER   115      26.523  43.176  14.198  1.00159.79           C\nATOM    890  O   SER   115      25.664  42.408  14.623  1.00159.79           O\nATOM    891  N   VAL   116      26.398  44.513  14.257  1.00 92.51           N\nATOM    892  CA  VAL   116      25.189  45.153  14.687  1.00 92.51           C\nATOM    893  CB  VAL   116      25.417  46.577  15.115  1.00 92.51           C\nATOM    894  CG1 VAL   116      24.085  47.200  15.566  1.00 92.51           C\nATOM    895  CG2 VAL   116      26.494  46.580  16.215  1.00 92.51           C\nATOM    896  C   VAL   116      24.264  45.108  13.500  1.00 92.51           C\nATOM    897  O   VAL   116      24.690  44.731  12.410  1.00 92.51           O\nATOM    898  N   ARG   117      22.965  45.446  13.662  1.00336.37           N\nATOM    899  CA  ARG   117      22.094  45.301  12.525  1.00336.37           C\nATOM    900  CB  ARG   117      20.581  45.431  12.790  1.00336.37           C\nATOM    901  CG  ARG   117      19.932  44.298  13.586  1.00336.37           C\nATOM    902  CD  ARG   117      19.895  44.519  15.100  1.00336.37           C\nATOM    903  NE  ARG   117      18.647  45.291  15.377  1.00336.37           N\nATOM    904  CZ  ARG   117      17.808  44.926  16.391  1.00336.37           C\nATOM    905  NH1 ARG   117      18.152  43.894  17.216  1.00336.37           N\nATOM    906  NH2 ARG   117      16.622  45.583  16.561  1.00336.37           N\nATOM    907  C   ARG   117      22.402  46.337  11.494  1.00336.37           C\nATOM    908  O   ARG   117      22.201  47.534  11.701  1.00336.37           O\nATOM    909  N   VAL   118      22.914  45.863  10.345  1.00 64.31           N\nATOM    910  CA  VAL   118      23.241  46.650   9.192  1.00 64.31           C\nATOM    911  CB  VAL   118      23.979  45.836   8.174  1.00 64.31           C\nATOM    912  CG1 VAL   118      24.271  46.698   6.933  1.00 64.31           C\nATOM    913  CG2 VAL   118      25.232  45.265   8.857  1.00 64.31           C\nATOM    914  C   VAL   118      21.981  47.150   8.554  1.00 64.31           C\nATOM    915  O   VAL   118      21.904  48.294   8.110  1.00 64.31           O\nATOM    916  N   VAL   119      20.942  46.300   8.504  1.00 96.31           N\nATOM    917  CA  VAL   119      19.739  46.655   7.806  1.00 96.31           C\nATOM    918  CB  VAL   119      18.707  45.559   7.804  1.00 96.31           C\nATOM    919  CG1 VAL   119      19.319  44.301   7.160  1.00 96.31           C\nATOM    920  CG2 VAL   119      18.179  45.355   9.232  1.00 96.31           C\nATOM    921  C   VAL   119      19.126  47.860   8.447  1.00 96.31           C\nATOM    922  O   VAL   119      18.620  48.742   7.756  1.00 96.31           O\nATOM    923  N   THR   120      19.183  47.945   9.787  1.00 96.26           N\nATOM    924  CA  THR   120      18.523  49.007  10.487  1.00 96.26           C\nATOM    925  CB  THR   120      18.653  48.899  11.981  1.00 96.26           C\nATOM    926  OG1 THR   120      17.720  49.760  12.614  1.00 96.26           O\nATOM    927  CG2 THR   120      20.085  49.278  12.390  1.00 96.26           C\nATOM    928  C   THR   120      19.089  50.320  10.045  1.00 96.26           C\nATOM    929  O   THR   120      18.359  51.303   9.922  1.00 96.26           O\nATOM    930  N   THR   121      20.409  50.375   9.784  1.00102.65           N\nATOM    931  CA  THR   121      21.004  51.614   9.373  1.00102.65           C\nATOM    932  CB  THR   121      22.500  51.566   9.211  1.00102.65           C\nATOM    933  OG1 THR   121      23.026  52.884   9.185  1.00102.65           O\nATOM    934  CG2 THR   121      22.842  50.856   7.892  1.00102.65           C\nATOM    935  C   THR   121      20.398  51.995   8.063  1.00102.65           C\nATOM    936  O   THR   121      20.247  53.177   7.761  1.00102.65           O\nATOM    937  N   PHE   122      20.042  50.996   7.234  1.00104.50           N\nATOM    938  CA  PHE   122      19.435  51.317   5.977  1.00104.50           C\nATOM    939  CB  PHE   122      18.951  50.062   5.216  1.00104.50           C\nATOM    940  CG  PHE   122      20.075  49.420   4.473  1.00104.50           C\nATOM    941  CD1 PHE   122      21.098  48.780   5.135  1.00104.50           C\nATOM    942  CD2 PHE   122      20.110  49.485   3.098  1.00104.50           C\nATOM    943  CE1 PHE   122      22.129  48.198   4.433  1.00104.50           C\nATOM    944  CE2 PHE   122      21.136  48.906   2.391  1.00104.50           C\nATOM    945  CZ  PHE   122      22.147  48.259   3.060  1.00104.50           C\nATOM    946  C   PHE   122      18.215  52.145   6.238  1.00104.50           C\nATOM    947  O   PHE   122      18.113  53.268   5.746  1.00104.50           O\nATOM    948  N   ASP   123      17.264  51.627   7.043  1.00 54.91           N\nATOM    949  CA  ASP   123      16.044  52.357   7.241  1.00 54.91           C\nATOM    950  CB  ASP   123      14.960  51.526   7.959  1.00 54.91           C\nATOM    951  CG  ASP   123      14.486  50.449   6.986  1.00 54.91           C\nATOM    952  OD1 ASP   123      15.352  49.847   6.295  1.00 54.91           O\nATOM    953  OD2 ASP   123      13.247  50.233   6.905  1.00 54.91           O\nATOM    954  C   ASP   123      16.259  53.625   8.016  1.00 54.91           C\nATOM    955  O   ASP   123      15.931  54.710   7.536  1.00 54.91           O\nATOM    956  N   ALA   124      16.849  53.527   9.227  1.00 65.92           N\nATOM    957  CA  ALA   124      17.003  54.663  10.101  1.00 65.92           C\nATOM    958  CB  ALA   124      17.580  54.287  11.478  1.00 65.92           C\nATOM    959  C   ALA   124      17.942  55.665   9.517  1.00 65.92           C\nATOM    960  O   ALA   124      17.675  56.866   9.509  1.00 65.92           O\nATOM    961  N   GLU   125      19.064  55.169   8.975  1.00228.14           N\nATOM    962  CA  GLU   125      20.124  55.991   8.473  1.00228.14           C\nATOM    963  CB  GLU   125      19.646  57.176   7.618  1.00228.14           C\nATOM    964  CG  GLU   125      19.156  56.824   6.214  1.00228.14           C\nATOM    965  CD  GLU   125      20.329  56.927   5.248  1.00228.14           C\nATOM    966  OE1 GLU   125      21.459  57.257   5.699  1.00228.14           O\nATOM    967  OE2 GLU   125      20.104  56.675   4.034  1.00228.14           O\nATOM    968  C   GLU   125      20.865  56.603   9.624  1.00228.14           C\nATOM    969  O   GLU   125      21.746  57.435   9.411  1.00228.14           O\nATOM    970  N   TYR   126      20.542  56.229  10.876  1.00 98.22           N\nATOM    971  CA  TYR   126      21.367  56.702  11.950  1.00 98.22           C\nATOM    972  CB  TYR   126      20.719  57.800  12.813  1.00 98.22           C\nATOM    973  CG  TYR   126      20.655  59.066  12.033  1.00 98.22           C\nATOM    974  CD1 TYR   126      19.716  59.243  11.042  1.00 98.22           C\nATOM    975  CD2 TYR   126      21.532  60.086  12.322  1.00 98.22           C\nATOM    976  CE1 TYR   126      19.671  60.423  10.339  1.00 98.22           C\nATOM    977  CE2 TYR   126      21.493  61.268  11.623  1.00 98.22           C\nATOM    978  CZ  TYR   126      20.561  61.433  10.630  1.00 98.22           C\nATOM    979  OH  TYR   126      20.521  62.647   9.914  1.00 98.22           O\nATOM    980  C   TYR   126      21.585  55.544  12.854  1.00 98.22           C\nATOM    981  O   TYR   126      20.713  55.230  13.660  1.00 98.22           O\nATOM    982  N   CYS   127      22.765  54.899  12.771  1.00 58.95           N\nATOM    983  CA  CYS   127      23.026  53.775  13.623  1.00 58.95           C\nATOM    984  CB  CYS   127      22.337  52.466  13.193  1.00 58.95           C\nATOM    985  SG  CYS   127      20.571  52.384  13.618  1.00 58.95           S\nATOM    986  C   CYS   127      24.483  53.505  13.575  1.00 58.95           C\nATOM    987  O   CYS   127      25.164  53.828  12.603  1.00 58.95           O\nATOM    988  N   ARG   128      25.006  52.898  14.653  1.00132.52           N\nATOM    989  CA  ARG   128      26.393  52.575  14.655  1.00132.52           C\nATOM    990  CB  ARG   128      27.147  53.193  15.840  1.00132.52           C\nATOM    991  CG  ARG   128      28.658  53.012  15.737  1.00132.52           C\nATOM    992  CD  ARG   128      29.456  53.697  16.848  1.00132.52           C\nATOM    993  NE  ARG   128      29.586  52.727  17.969  1.00132.52           N\nATOM    994  CZ  ARG   128      30.517  51.732  17.894  1.00132.52           C\nATOM    995  NH1 ARG   128      31.273  51.586  16.767  1.00132.52           N\nATOM    996  NH2 ARG   128      30.673  50.870  18.941  1.00132.52           N\nATOM    997  C   ARG   128      26.482  51.094  14.789  1.00132.52           C\nATOM    998  O   ARG   128      26.070  50.531  15.802  1.00132.52           O\nATOM    999  N   HIS   129      27.019  50.411  13.762  1.00 79.00           N\nATOM   1000  CA  HIS   129      27.159  48.995  13.887  1.00 79.00           C\nATOM   1001  ND1 HIS   129      27.557  47.712  10.310  1.00 79.00           N\nATOM   1002  CG  HIS   129      27.298  48.590  11.340  1.00 79.00           C\nATOM   1003  CB  HIS   129      26.693  48.180  12.656  1.00 79.00           C\nATOM   1004  NE2 HIS   129      28.152  49.731   9.592  1.00 79.00           N\nATOM   1005  CD2 HIS   129      27.667  49.816  10.886  1.00 79.00           C\nATOM   1006  CE1 HIS   129      28.067  48.450   9.288  1.00 79.00           C\nATOM   1007  C   HIS   129      28.606  48.749  14.136  1.00 79.00           C\nATOM   1008  O   HIS   129      29.461  49.495  13.663  1.00 79.00           O\nATOM   1009  N   GLY   130      28.917  47.718  14.939  1.00 50.47           N\nATOM   1010  CA  GLY   130      30.287  47.445  15.228  1.00 50.47           C\nATOM   1011  C   GLY   130      30.367  46.002  15.563  1.00 50.47           C\nATOM   1012  O   GLY   130      29.371  45.377  15.925  1.00 50.47           O\nATOM   1013  N   THR   131      31.578  45.436  15.469  1.00123.02           N\nATOM   1014  CA  THR   131      31.691  44.042  15.734  1.00123.02           C\nATOM   1015  CB  THR   131      33.049  43.483  15.419  1.00123.02           C\nATOM   1016  OG1 THR   131      34.050  44.171  16.156  1.00123.02           O\nATOM   1017  CG2 THR   131      33.302  43.611  13.907  1.00123.02           C\nATOM   1018  C   THR   131      31.409  43.811  17.179  1.00123.02           C\nATOM   1019  O   THR   131      31.700  44.645  18.037  1.00123.02           O\nATOM   1020  N   CYS   132      30.773  42.659  17.456  1.00 91.61           N\nATOM   1021  CA  CYS   132      30.490  42.198  18.780  1.00 91.61           C\nATOM   1022  CB  CYS   132      29.036  41.711  18.950  1.00 91.61           C\nATOM   1023  SG  CYS   132      28.661  41.088  20.618  1.00 91.61           S\nATOM   1024  C   CYS   132      31.371  41.004  18.923  1.00 91.61           C\nATOM   1025  O   CYS   132      31.718  40.384  17.920  1.00 91.61           O\nATOM   1026  N   GLU   133      31.796  40.663  20.155  1.00 65.30           N\nATOM   1027  CA  GLU   133      32.629  39.500  20.272  1.00 65.30           C\nATOM   1028  CB  GLU   133      33.992  39.721  20.954  1.00 65.30           C\nATOM   1029  CG  GLU   133      33.951  39.947  22.466  1.00 65.30           C\nATOM   1030  CD  GLU   133      35.375  39.728  22.960  1.00 65.30           C\nATOM   1031  OE1 GLU   133      36.162  39.083  22.215  1.00 65.30           O\nATOM   1032  OE2 GLU   133      35.699  40.197  24.083  1.00 65.30           O\nATOM   1033  C   GLU   133      31.877  38.519  21.102  1.00 65.30           C\nATOM   1034  O   GLU   133      31.006  38.919  21.872  1.00 65.30           O\nATOM   1035  N   ARG   134      32.192  37.213  20.937  1.00158.64           N\nATOM   1036  CA  ARG   134      31.515  36.145  21.620  1.00158.64           C\nATOM   1037  CB  ARG   134      31.079  36.473  23.065  1.00158.64           C\nATOM   1038  CG  ARG   134      32.211  36.795  24.049  1.00158.64           C\nATOM   1039  CD  ARG   134      31.776  37.700  25.213  1.00158.64           C\nATOM   1040  NE  ARG   134      30.323  37.472  25.469  1.00158.64           N\nATOM   1041  CZ  ARG   134      29.559  38.416  26.097  1.00158.64           C\nATOM   1042  NH1 ARG   134      30.126  39.569  26.566  1.00158.64           N\nATOM   1043  NH2 ARG   134      28.219  38.208  26.251  1.00158.64           N\nATOM   1044  C   ARG   134      30.273  35.925  20.819  1.00158.64           C\nATOM   1045  O   ARG   134      29.762  36.864  20.214  1.00158.64           O\nATOM   1046  N   SER   135      29.758  34.683  20.773  1.00 56.82           N\nATOM   1047  CA  SER   135      28.567  34.457  20.001  1.00 56.82           C\nATOM   1048  CB  SER   135      28.408  33.017  19.484  1.00 56.82           C\nATOM   1049  OG  SER   135      28.282  32.114  20.576  1.00 56.82           O\nATOM   1050  C   SER   135      27.388  34.778  20.864  1.00 56.82           C\nATOM   1051  O   SER   135      27.543  35.182  22.015  1.00 56.82           O\nATOM   1052  N   GLU   136      26.164  34.637  20.323  1.00127.47           N\nATOM   1053  CA  GLU   136      25.022  34.929  21.141  1.00127.47           C\nATOM   1054  CB  GLU   136      24.007  35.921  20.549  1.00127.47           C\nATOM   1055  CG  GLU   136      24.490  37.368  20.512  1.00127.47           C\nATOM   1056  CD  GLU   136      23.242  38.233  20.452  1.00127.47           C\nATOM   1057  OE1 GLU   136      22.149  37.701  20.784  1.00127.47           O\nATOM   1058  OE2 GLU   136      23.360  39.433  20.089  1.00127.47           O\nATOM   1059  C   GLU   136      24.271  33.666  21.380  1.00127.47           C\nATOM   1060  O   GLU   136      24.570  32.624  20.798  1.00127.47           O\nATOM   1061  N   VAL   137      23.274  33.744  22.285  1.00 68.47           N\nATOM   1062  CA  VAL   137      22.471  32.614  22.648  1.00 68.47           C\nATOM   1063  CB  VAL   137      22.704  32.207  24.075  1.00 68.47           C\nATOM   1064  CG1 VAL   137      21.833  30.993  24.456  1.00 68.47           C\nATOM   1065  CG2 VAL   137      24.216  31.986  24.237  1.00 68.47           C\nATOM   1066  C   VAL   137      21.042  33.055  22.458  1.00 68.47           C\nATOM   1067  O   VAL   137      20.793  34.231  22.193  1.00 68.47           O\nATOM   1068  N   GLY   138      20.077  32.112  22.566  1.00 73.30           N\nATOM   1069  CA  GLY   138      18.683  32.341  22.298  1.00 73.30           C\nATOM   1070  C   GLY   138      17.853  32.462  23.544  1.00 73.30           C\nATOM   1071  O   GLY   138      18.073  33.338  24.375  1.00 73.30           O\nATOM   1072  N   ILE   139      16.829  31.582  23.665  1.00107.12           N\nATOM   1073  CA  ILE   139      15.856  31.636  24.721  1.00107.12           C\nATOM   1074  CB  ILE   139      14.440  31.558  24.208  1.00107.12           C\nATOM   1075  CG1 ILE   139      14.132  32.724  23.247  1.00107.12           C\nATOM   1076  CG2 ILE   139      13.489  31.478  25.412  1.00107.12           C\nATOM   1077  CD1 ILE   139      14.227  34.111  23.882  1.00107.12           C\nATOM   1078  C   ILE   139      16.058  30.464  25.632  1.00107.12           C\nATOM   1079  O   ILE   139      16.384  29.363  25.190  1.00107.12           O\nATOM   1080  N   CYS   140      15.892  30.696  26.949  1.00 42.56           N\nATOM   1081  CA  CYS   140      16.045  29.657  27.922  1.00 42.56           C\nATOM   1082  CB  CYS   140      16.541  30.153  29.293  1.00 42.56           C\nATOM   1083  SG  CYS   140      18.226  30.832  29.248  1.00 42.56           S\nATOM   1084  C   CYS   140      14.692  29.085  28.171  1.00 42.56           C\nATOM   1085  O   CYS   140      13.717  29.815  28.348  1.00 42.56           O\nATOM   1086  N   LEU   141      14.588  27.746  28.165  1.00106.69           N\nATOM   1087  CA  LEU   141      13.303  27.183  28.438  1.00106.69           C\nATOM   1088  CB  LEU   141      12.579  26.573  27.229  1.00106.69           C\nATOM   1089  CG  LEU   141      12.122  27.555  26.141  1.00106.69           C\nATOM   1090  CD1 LEU   141      13.285  28.074  25.285  1.00106.69           C\nATOM   1091  CD2 LEU   141      10.998  26.922  25.317  1.00106.69           C\nATOM   1092  C   LEU   141      13.472  26.035  29.364  1.00106.69           C\nATOM   1093  O   LEU   141      14.550  25.453  29.476  1.00106.69           O\nATOM   1094  N   SER   142      12.384  25.703  30.083  1.00 42.98           N\nATOM   1095  CA  SER   142      12.405  24.510  30.867  1.00 42.98           C\nATOM   1096  CB  SER   142      12.255  24.723  32.384  1.00 42.98           C\nATOM   1097  OG  SER   142      13.379  25.430  32.888  1.00 42.98           O\nATOM   1098  C   SER   142      11.236  23.710  30.405  1.00 42.98           C\nATOM   1099  O   SER   142      10.120  24.217  30.293  1.00 42.98           O\nATOM   1100  N   THR   143      11.473  22.421  30.104  1.00 47.78           N\nATOM   1101  CA  THR   143      10.394  21.592  29.656  1.00 47.78           C\nATOM   1102  CB  THR   143      10.679  20.829  28.392  1.00 47.78           C\nATOM   1103  OG1 THR   143      10.985  21.718  27.330  1.00 47.78           O\nATOM   1104  CG2 THR   143       9.439  19.981  28.048  1.00 47.78           C\nATOM   1105  C   THR   143      10.188  20.537  30.690  1.00 47.78           C\nATOM   1106  O   THR   143      11.139  20.028  31.277  1.00 47.78           O\nATOM   1107  N   SER   144       8.914  20.210  30.968  1.00 59.04           N\nATOM   1108  CA  SER   144       8.626  19.112  31.840  1.00 59.04           C\nATOM   1109  CB  SER   144       8.035  19.509  33.202  1.00 59.04           C\nATOM   1110  OG  SER   144       6.729  20.029  33.028  1.00 59.04           O\nATOM   1111  C   SER   144       7.558  18.384  31.114  1.00 59.04           C\nATOM   1112  O   SER   144       6.682  19.037  30.564  1.00 59.04           O\nATOM   1113  N   GLY   145       7.584  17.040  31.084  1.00116.09           N\nATOM   1114  CA  GLY   145       6.588  16.397  30.278  1.00116.09           C\nATOM   1115  C   GLY   145       6.898  16.894  28.903  1.00116.09           C\nATOM   1116  O   GLY   145       8.066  17.050  28.567  1.00116.09           O\nATOM   1117  N   ARG   146       5.884  17.196  28.077  1.00366.35           N\nATOM   1118  CA  ARG   146       6.221  17.752  26.797  1.00366.35           C\nATOM   1119  CB  ARG   146       5.502  17.054  25.632  1.00366.35           C\nATOM   1120  CG  ARG   146       5.827  15.565  25.500  1.00366.35           C\nATOM   1121  CD  ARG   146       5.393  14.739  26.713  1.00366.35           C\nATOM   1122  NE  ARG   146       3.985  15.110  27.037  1.00366.35           N\nATOM   1123  CZ  ARG   146       2.945  14.500  26.396  1.00366.35           C\nATOM   1124  NH1 ARG   146       3.191  13.563  25.434  1.00366.35           N\nATOM   1125  NH2 ARG   146       1.661  14.824  26.731  1.00366.35           N\nATOM   1126  C   ARG   146       5.735  19.172  26.763  1.00366.35           C\nATOM   1127  O   ARG   146       5.364  19.655  25.695  1.00366.35           O\nATOM   1128  N   TRP   147       5.750  19.902  27.903  1.00178.23           N\nATOM   1129  CA  TRP   147       5.287  21.268  27.842  1.00178.23           C\nATOM   1130  CB  TRP   147       3.907  21.553  28.479  1.00178.23           C\nATOM   1131  CG  TRP   147       3.735  21.236  29.949  1.00178.23           C\nATOM   1132  CD2 TRP   147       2.964  22.044  30.845  1.00178.23           C\nATOM   1133  CD1 TRP   147       4.239  20.217  30.693  1.00178.23           C\nATOM   1134  NE1 TRP   147       3.848  20.344  32.003  1.00178.23           N\nATOM   1135  CE2 TRP   147       3.056  21.466  32.114  1.00178.23           C\nATOM   1136  CE3 TRP   147       2.241  23.179  30.635  1.00178.23           C\nATOM   1137  CZ2 TRP   147       2.426  22.014  33.193  1.00178.23           C\nATOM   1138  CZ3 TRP   147       1.606  23.725  31.725  1.00178.23           C\nATOM   1139  CH2 TRP   147       1.693  23.159  32.979  1.00178.23           C\nATOM   1140  C   TRP   147       6.293  22.163  28.496  1.00178.23           C\nATOM   1141  O   TRP   147       7.143  21.699  29.254  1.00178.23           O\nATOM   1142  N   VAL   148       6.259  23.481  28.181  1.00 72.55           N\nATOM   1143  CA  VAL   148       7.249  24.325  28.783  1.00 72.55           C\nATOM   1144  CB  VAL   148       7.811  25.427  27.934  1.00 72.55           C\nATOM   1145  CG1 VAL   148       8.783  26.220  28.824  1.00 72.55           C\nATOM   1146  CG2 VAL   148       8.507  24.816  26.705  1.00 72.55           C\nATOM   1147  C   VAL   148       6.680  24.961  30.008  1.00 72.55           C\nATOM   1148  O   VAL   148       5.583  25.521  29.983  1.00 72.55           O\nATOM   1149  N   LEU   149       7.408  24.791  31.136  1.00114.96           N\nATOM   1150  CA  LEU   149       7.070  25.370  32.404  1.00114.96           C\nATOM   1151  CB  LEU   149       7.866  24.818  33.606  1.00114.96           C\nATOM   1152  CG  LEU   149       7.836  23.290  33.794  1.00114.96           C\nATOM   1153  CD1 LEU   149       6.400  22.743  33.809  1.00114.96           C\nATOM   1154  CD2 LEU   149       8.786  22.580  32.817  1.00114.96           C\nATOM   1155  C   LEU   149       7.370  26.834  32.393  1.00114.96           C\nATOM   1156  O   LEU   149       6.597  27.633  32.921  1.00114.96           O\nATOM   1157  N   ASN   150       8.530  27.231  31.828  1.00 84.27           N\nATOM   1158  CA  ASN   150       8.832  28.633  31.878  1.00 84.27           C\nATOM   1159  CB  ASN   150       9.551  29.065  33.169  1.00 84.27           C\nATOM   1160  CG  ASN   150       9.304  30.555  33.387  1.00 84.27           C\nATOM   1161  OD1 ASN   150       8.599  31.210  32.619  1.00 84.27           O\nATOM   1162  ND2 ASN   150       9.894  31.109  34.481  1.00 84.27           N\nATOM   1163  C   ASN   150       9.707  28.974  30.719  1.00 84.27           C\nATOM   1164  O   ASN   150      10.384  28.114  30.153  1.00 84.27           O\nATOM   1165  N   ASN   151       9.689  30.267  30.334  1.00 99.45           N\nATOM   1166  CA  ASN   151      10.449  30.736  29.214  1.00 99.45           C\nATOM   1167  CB  ASN   151       9.571  31.030  27.992  1.00 99.45           C\nATOM   1168  CG  ASN   151      10.486  31.403  26.856  1.00 99.45           C\nATOM   1169  OD1 ASN   151      11.028  30.535  26.174  1.00 99.45           O\nATOM   1170  ND2 ASN   151      10.675  32.735  26.641  1.00 99.45           N\nATOM   1171  C   ASN   151      11.088  32.038  29.590  1.00 99.45           C\nATOM   1172  O   ASN   151      10.469  32.867  30.257  1.00 99.45           O\nATOM   1173  N   GLU   152      12.354  32.243  29.161  1.00123.71           N\nATOM   1174  CA  GLU   152      13.072  33.460  29.432  1.00123.71           C\nATOM   1175  CB  GLU   152      14.023  33.372  30.637  1.00123.71           C\nATOM   1176  CG  GLU   152      13.349  33.553  31.992  1.00123.71           C\nATOM   1177  CD  GLU   152      13.475  35.021  32.370  1.00123.71           C\nATOM   1178  OE1 GLU   152      14.370  35.707  31.805  1.00123.71           O\nATOM   1179  OE2 GLU   152      12.682  35.476  33.237  1.00123.71           O\nATOM   1180  C   GLU   152      13.967  33.751  28.275  1.00123.71           C\nATOM   1181  O   GLU   152      14.433  32.838  27.597  1.00123.71           O\nATOM   1182  N   HIS   153      14.221  35.052  28.024  1.00 91.31           N\nATOM   1183  CA  HIS   153      15.130  35.464  26.992  1.00 91.31           C\nATOM   1184  ND1 HIS   153      14.621  38.258  28.751  1.00 91.31           N\nATOM   1185  CG  HIS   153      15.465  37.832  27.750  1.00 91.31           C\nATOM   1186  CB  HIS   153      15.031  36.952  26.615  1.00 91.31           C\nATOM   1187  NE2 HIS   153      16.627  39.091  29.224  1.00 91.31           N\nATOM   1188  CD2 HIS   153      16.690  38.351  28.057  1.00 91.31           C\nATOM   1189  CE1 HIS   153      15.362  39.006  29.603  1.00 91.31           C\nATOM   1190  C   HIS   153      16.501  35.305  27.557  1.00 91.31           C\nATOM   1191  O   HIS   153      16.695  35.421  28.765  1.00 91.31           O\nATOM   1192  N   TYR   154      17.489  35.013  26.693  1.00146.69           N\nATOM   1193  CA  TYR   154      18.838  34.871  27.155  1.00146.69           C\nATOM   1194  CB  TYR   154      19.318  33.411  27.087  1.00146.69           C\nATOM   1195  CG  TYR   154      20.672  33.249  27.697  1.00146.69           C\nATOM   1196  CD1 TYR   154      20.813  33.069  29.054  1.00146.69           C\nATOM   1197  CD2 TYR   154      21.796  33.254  26.905  1.00146.69           C\nATOM   1198  CE1 TYR   154      22.061  32.907  29.614  1.00146.69           C\nATOM   1199  CE2 TYR   154      23.046  33.094  27.457  1.00146.69           C\nATOM   1200  CZ  TYR   154      23.179  32.920  28.815  1.00146.69           C\nATOM   1201  OH  TYR   154      24.459  32.754  29.384  1.00146.69           O\nATOM   1202  C   TYR   154      19.663  35.685  26.214  1.00146.69           C\nATOM   1203  O   TYR   154      19.397  35.717  25.013  1.00146.69           O\nATOM   1204  N   ARG   155      20.676  36.389  26.738  1.00142.61           N\nATOM   1205  CA  ARG   155      21.459  37.192  25.852  1.00142.61           C\nATOM   1206  CB  ARG   155      21.663  38.628  26.358  1.00142.61           C\nATOM   1207  CG  ARG   155      20.381  39.461  26.379  1.00142.61           C\nATOM   1208  CD  ARG   155      20.644  40.933  26.696  1.00142.61           C\nATOM   1209  NE  ARG   155      21.245  41.565  25.487  1.00142.61           N\nATOM   1210  CZ  ARG   155      20.441  42.174  24.567  1.00142.61           C\nATOM   1211  NH1 ARG   155      19.085  42.146  24.718  1.00142.61           N\nATOM   1212  NH2 ARG   155      20.995  42.822  23.501  1.00142.61           N\nATOM   1213  C   ARG   155      22.826  36.545  25.737  1.00142.61           C\nATOM   1214  O   ARG   155      23.824  37.222  26.099  1.00142.61           O\nATOM   1215  OXT ARG   155      22.896  35.370  25.283  1.00142.61           O\nTER    1216      ARG   155\nEND\n",
           "type": "blob"
          }
         ],
         "kwargs": {
          "defaultRepresentation": true,
          "ext": "pdb"
         },
         "methodName": "loadFile",
         "reconstruc_color_scheme": false,
         "target": "Stage",
         "type": "call_method"
        }
       ],
       "_ngl_original_stage_parameters": {
        "ambientColor": 14540253,
        "ambientIntensity": 0.2,
        "backgroundColor": "white",
        "cameraEyeSep": 0.3,
        "cameraFov": 40,
        "cameraType": "perspective",
        "clipDist": 10,
        "clipFar": 100,
        "clipNear": 0,
        "fogFar": 100,
        "fogNear": 50,
        "hoverTimeout": 0,
        "impostor": true,
        "lightColor": 14540253,
        "lightIntensity": 1,
        "mousePreset": "default",
        "panSpeed": 1,
        "quality": "medium",
        "rotateSpeed": 2,
        "sampleLevel": 0,
        "tooltip": true,
        "workerDefault": true,
        "zoomSpeed": 1.2
       },
       "_ngl_repr_dict": {
        "0": {
         "0": {
          "params": {
           "aspectRatio": 5,
           "assembly": "default",
           "capped": true,
           "clipCenter": {
            "x": 0,
            "y": 0,
            "z": 0
           },
           "clipNear": 0,
           "clipRadius": 0,
           "colorMode": "hcl",
           "colorReverse": true,
           "colorScale": "spectral",
           "colorScheme": "residueindex",
           "colorValue": 9474192,
           "defaultAssembly": "",
           "depthWrite": true,
           "diffuse": 16777215,
           "diffuseInterior": false,
           "disablePicking": false,
           "flatShaded": false,
           "interiorColor": 2236962,
           "interiorDarkening": 0,
           "lazy": false,
           "matrix": {
            "elements": [
             1,
             0,
             0,
             0,
             0,
             1,
             0,
             0,
             0,
             0,
             1,
             0,
             0,
             0,
             0,
             1
            ]
           },
           "metalness": 0,
           "opacity": 1,
           "quality": "high",
           "radialSegments": 20,
           "radiusData": {},
           "radiusScale": 0.7,
           "radiusSize": 1,
           "radiusType": "sstruc",
           "roughness": 0.4,
           "sele": "",
           "side": "double",
           "smoothSheet": false,
           "subdiv": 6,
           "tension": null,
           "useInteriorColor": true,
           "visible": true,
           "wireframe": false
          },
          "type": "cartoon"
         },
         "1": {
          "params": {
           "aspectRatio": 1,
           "assembly": "default",
           "bondScale": 0.4,
           "clipCenter": {
            "x": 0,
            "y": 0,
            "z": 0
           },
           "clipNear": 0,
           "clipRadius": 0,
           "colorMode": "hcl",
           "colorReverse": true,
           "colorScale": "spectral",
           "colorScheme": "residueindex",
           "colorValue": 9474192,
           "cylinderOnly": false,
           "defaultAssembly": "",
           "depthWrite": true,
           "diffuse": 16777215,
           "diffuseInterior": false,
           "disableImpostor": false,
           "disablePicking": false,
           "flatShaded": false,
           "interiorColor": 2236962,
           "interiorDarkening": 0,
           "lazy": false,
           "lineOnly": false,
           "linewidth": 2,
           "matrix": {
            "elements": [
             1,
             0,
             0,
             0,
             0,
             1,
             0,
             0,
             0,
             0,
             1,
             0,
             0,
             0,
             0,
             1
            ]
           },
           "metalness": 0,
           "opacity": 1,
           "openEnded": true,
           "quality": "high",
           "radialSegments": 20,
           "radiusData": {},
           "radiusScale": 1,
           "radiusSize": 0.3,
           "radiusType": "size",
           "roughness": 0.4,
           "sele": "",
           "side": "double",
           "sphereDetail": 2,
           "useInteriorColor": true,
           "visible": true,
           "wireframe": false
          },
          "type": "base"
         },
         "2": {
          "params": {
           "aspectRatio": 1.5,
           "assembly": "default",
           "bondScale": 0.3,
           "bondSpacing": 0.75,
           "clipCenter": {
            "x": 0,
            "y": 0,
            "z": 0
           },
           "clipNear": 0,
           "clipRadius": 0,
           "colorMode": "hcl",
           "colorReverse": false,
           "colorScale": "",
           "colorScheme": "element",
           "colorValue": 9474192,
           "cylinderOnly": false,
           "defaultAssembly": "",
           "depthWrite": true,
           "diffuse": 16777215,
           "diffuseInterior": false,
           "disableImpostor": false,
           "disablePicking": false,
           "flatShaded": false,
           "interiorColor": 2236962,
           "interiorDarkening": 0,
           "lazy": false,
           "lineOnly": false,
           "linewidth": 2,
           "matrix": {
            "elements": [
             1,
             0,
             0,
             0,
             0,
             1,
             0,
             0,
             0,
             0,
             1,
             0,
             0,
             0,
             0,
             1
            ]
           },
           "metalness": 0,
           "multipleBond": "off",
           "opacity": 1,
           "openEnded": true,
           "quality": "high",
           "radialSegments": 20,
           "radiusData": {},
           "radiusScale": 2,
           "radiusSize": 0.15,
           "radiusType": "size",
           "roughness": 0.4,
           "sele": "ligand",
           "side": "double",
           "sphereDetail": 2,
           "useInteriorColor": true,
           "visible": true,
           "wireframe": false
          },
          "type": "ball+stick"
         }
        }
       },
       "_ngl_serialize": false,
       "_ngl_version": "2.0.0-dev.36",
       "_ngl_view_id": [
        "96F251F7-9FA1-4A9E-85C4-771B718AD2A8"
       ],
       "_player_dict": {},
       "_scene_position": {},
       "_scene_rotation": {},
       "_synced_model_ids": [],
       "_synced_repr_model_ids": [],
       "_view_height": "",
       "_view_width": "",
       "background": "white",
       "frame": 0,
       "gui_style": null,
       "layout": "IPY_MODEL_5dfaf163cbba4f9a9a32c830a0743b64",
       "max_frame": 0,
       "n_components": 1,
       "picked": {}
      }
     },
     "2e56223b06e6444f9add47a331531f15": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "IntSliderModel",
      "state": {
       "layout": "IPY_MODEL_30f8d55e5c7c4ab1b5891e31fe75e714",
       "max": 0,
       "style": "IPY_MODEL_08fae6ec25304c8297384d41dcdd18f0"
      }
     },
     "30f8d55e5c7c4ab1b5891e31fe75e714": {
      "model_module": "@jupyter-widgets/base",
      "model_module_version": "1.2.0",
      "model_name": "LayoutModel",
      "state": {}
     },
     "36aea798f34049fe94b689d34dc09683": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "LinkModel",
      "state": {
       "source": [
        "IPY_MODEL_09ebdc541c664802aa15120b6b3b0e0f",
        "value"
       ],
       "target": [
        "IPY_MODEL_af804eb2a94e4c6e96628a3b2407ccd4",
        "value"
       ]
      }
     },
     "3d3d42ec8c104cdcb7bcb02ecc9c4823": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "ButtonStyleModel",
      "state": {}
     },
     "417a29b7da864854acfc8576bf4cb919": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "DescriptionStyleModel",
      "state": {
       "description_width": ""
      }
     },
     "4d8a710940d946d3896b43a978075941": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "ButtonModel",
      "state": {
       "icon": "compress",
       "layout": "IPY_MODEL_bf658c21ff794afa9013ffaca07caac9",
       "style": "IPY_MODEL_7be2599b2b664f6da60d19e172d2e2e2"
      }
     },
     "51f826ab880644b3bf2b2a928b120e84": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "LinkModel",
      "state": {
       "source": [
        "IPY_MODEL_af804eb2a94e4c6e96628a3b2407ccd4",
        "max"
       ],
       "target": [
        "IPY_MODEL_29c8848788294b8f9c8c3b8edeeab294",
        "max_frame"
       ]
      }
     },
     "5dfaf163cbba4f9a9a32c830a0743b64": {
      "model_module": "@jupyter-widgets/base",
      "model_module_version": "1.2.0",
      "model_name": "LayoutModel",
      "state": {}
     },
     "5f5726af8e7e48878aff4f76c57d3be9": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "HBoxModel",
      "state": {
       "children": [
        "IPY_MODEL_dc4c2913fc224986b8eafdcab1278219",
        "IPY_MODEL_2e56223b06e6444f9add47a331531f15"
       ],
       "layout": "IPY_MODEL_16d8a7347c49453491cd94bb48d380f4"
      }
     },
     "63310cba8e144b0db4559434ab1c2fca": {
      "model_module": "@jupyter-widgets/base",
      "model_module_version": "1.2.0",
      "model_name": "LayoutModel",
      "state": {}
     },
     "655d671cfd5a488091623461aa729741": {
      "model_module": "nglview-js-widgets",
      "model_module_version": "2.7.7",
      "model_name": "ColormakerRegistryModel",
      "state": {
       "_msg_ar": [],
       "_msg_q": [],
       "_ready": true,
       "layout": "IPY_MODEL_9ca7560536f249c5b530a48981858aeb"
      }
     },
     "68adf8c3ebd649cca0c0b81626f28a36": {
      "model_module": "nglview-js-widgets",
      "model_module_version": "2.7.7",
      "model_name": "NGLModel",
      "state": {
       "_camera_orientation": [
        95.56174366094167,
        0,
        0,
        0,
        0,
        95.56174366094167,
        0,
        0,
        0,
        0,
        95.56174366094167,
        0,
        -13.290690660476685,
        -35.494890213012695,
        -14.702915906906128,
        1
       ],
       "_camera_str": "orthographic",
       "_gui_theme": null,
       "_ibtn_fullscreen": "IPY_MODEL_eb93fc0f2f99418bba22a238f335ab4c",
       "_igui": null,
       "_iplayer": "IPY_MODEL_5f5726af8e7e48878aff4f76c57d3be9",
       "_ngl_color_dict": {},
       "_ngl_coordinate_resource": {},
       "_ngl_full_stage_parameters": {
        "ambientColor": 14540253,
        "ambientIntensity": 0.2,
        "backgroundColor": "white",
        "cameraEyeSep": 0.3,
        "cameraFov": 40,
        "cameraType": "perspective",
        "clipDist": 10,
        "clipFar": 100,
        "clipNear": 0,
        "fogFar": 100,
        "fogNear": 50,
        "hoverTimeout": 0,
        "impostor": true,
        "lightColor": 14540253,
        "lightIntensity": 1,
        "mousePreset": "default",
        "panSpeed": 1,
        "quality": "medium",
        "rotateSpeed": 2,
        "sampleLevel": 0,
        "tooltip": true,
        "workerDefault": true,
        "zoomSpeed": 1.2
       },
       "_ngl_msg_archive": [
        {
         "args": [
          {
           "binary": false,
           "data": "ATOM      1  N   MET A   1      15.188  20.000  29.458  1.00 41.84           N  \nATOM      2  CA  MET A   1      14.144  20.675  30.276  1.00 42.28           C  \nATOM      3  C   MET A   1      14.352  22.193  30.295  1.00 41.10           C  \nATOM      4  O   MET A   1      13.390  22.953  30.323  1.00 41.61           O  \nATOM      5  CB  MET A   1      14.141  20.112  31.700  1.00 51.56           C  \nATOM      6  CG  MET A   1      13.936  18.600  31.770  1.00 54.52           C  \nATOM      7  SD  MET A   1      13.094  18.041  33.287  1.00 61.21           S  \nATOM      8  CE  MET A   1      14.055  18.913  34.546  1.00 54.82           C  \nATOM      9  N   ARG A   2      15.608  22.631  30.277  1.00 40.74           N  \nATOM     10  CA  ARG A   2      15.941  24.058  30.255  1.00 37.10           C  \nATOM     11  C   ARG A   2      16.450  24.393  28.852  1.00 35.35           C  \nATOM     12  O   ARG A   2      17.458  23.837  28.408  1.00 36.47           O  \nATOM     13  CB  ARG A   2      17.063  24.381  31.258  1.00 33.13           C  \nATOM     14  CG  ARG A   2      16.745  24.147  32.732  1.00 35.07           C  \nATOM     15  CD  ARG A   2      17.998  24.333  33.588  1.00 33.25           C  \nATOM     16  NE  ARG A   2      19.116  23.592  33.010  1.00 36.01           N  \nATOM     17  CZ  ARG A   2      20.346  23.536  33.513  1.00 35.77           C  \nATOM     18  NH1 ARG A   2      20.650  24.177  34.634  1.00 38.10           N  \nATOM     19  NH2 ARG A   2      21.281  22.845  32.879  1.00 32.68           N  \nATOM     20  N   VAL A   3      15.762  25.271  28.131  1.00 31.66           N  \nATOM     21  CA  VAL A   3      16.243  25.642  26.802  1.00 28.60           C  \nATOM     22  C   VAL A   3      16.063  27.131  26.564  1.00 28.06           C  \nATOM     23  O   VAL A   3      15.174  27.773  27.132  1.00 27.85           O  \nATOM     24  CB  VAL A   3      15.559  24.827  25.654  1.00 23.82           C  \nATOM     25  CG1 VAL A   3      14.631  23.777  26.227  1.00 20.78           C  \nATOM     26  CG2 VAL A   3      14.830  25.758  24.692  1.00 21.64           C  \nATOM     27  N   LEU A   4      16.938  27.672  25.729  1.00 25.07           N  \nATOM     28  CA  LEU A   4      16.922  29.085  25.401  1.00 23.19           C  \nATOM     29  C   LEU A   4      16.727  29.263  23.916  1.00 23.61           C  \nATOM     30  O   LEU A   4      17.332  28.539  23.121  1.00 23.34           O  \nATOM     31  CB  LEU A   4      18.254  29.730  25.792  1.00 22.80           C  \nATOM     32  CG  LEU A   4      18.439  31.208  25.418  1.00 24.05           C  \nATOM     33  CD1 LEU A   4      17.565  32.090  26.332  1.00 21.39           C  \nATOM     34  CD2 LEU A   4      19.911  31.590  25.543  1.00 22.16           C  \nATOM     35  N   PHE A   5      15.869  30.210  23.540  1.00 23.66           N  \nATOM     36  CA  PHE A   5      15.658  30.521  22.136  1.00 22.40           C  \nATOM     37  C   PHE A   5      16.113  31.971  21.949  1.00 24.10           C  \nATOM     38  O   PHE A   5      15.683  32.870  22.683  1.00 22.91           O  \nATOM     39  CB  PHE A   5      14.188  30.369  21.731  1.00 21.92           C  \nATOM     40  CG  PHE A   5      13.906  30.780  20.300  1.00 22.58           C  \nATOM     41  CD1 PHE A   5      13.390  32.044  20.009  1.00 23.87           C  \nATOM     42  CD2 PHE A   5      14.189  29.916  19.246  1.00 21.77           C  \nATOM     43  CE1 PHE A   5      13.155  32.442  18.681  1.00 20.75           C  \nATOM     44  CE2 PHE A   5      13.965  30.295  17.927  1.00 20.60           C  \nATOM     45  CZ  PHE A   5      13.447  31.563  17.642  1.00 23.84           C  \nATOM     46  N   ILE A   6      16.991  32.190  20.972  1.00 25.94           N  \nATOM     47  CA  ILE A   6      17.521  33.517  20.695  1.00 25.61           C  \nATOM     48  C   ILE A   6      17.002  34.014  19.347  1.00 27.55           C  \nATOM     49  O   ILE A   6      17.100  33.314  18.337  1.00 29.08           O  \nATOM     50  CB  ILE A   6      19.060  33.488  20.694  1.00 21.06           C  \nATOM     51  CG1 ILE A   6      19.563  33.088  22.077  1.00 20.37           C  \nATOM     52  CG2 ILE A   6      19.610  34.845  20.310  1.00 24.23           C  \nATOM     53  CD1 ILE A   6      21.072  33.040  22.197  1.00 18.13           C  \nATOM     54  N   GLY A   7      16.456  35.226  19.344  1.00 27.22           N  \nATOM     55  CA  GLY A   7      15.896  35.798  18.133  1.00 28.27           C  \nATOM     56  C   GLY A   7      16.858  36.103  16.999  1.00 29.47           C  \nATOM     57  O   GLY A   7      18.072  36.133  17.198  1.00 29.81           O  \nATOM     58  N   ASP A   8      16.287  36.336  15.814  1.00 23.71           N  \nATOM     59  CA  ASP A   8      17.006  36.651  14.577  1.00 25.24           C  \nATOM     60  C   ASP A   8      18.369  37.343  14.728  1.00 23.46           C  \nATOM     61  O   ASP A   8      18.430  38.536  15.025  1.00 23.27           O  \nATOM     62  CB  ASP A   8      16.131  37.544  13.684  1.00 39.09           C  \nATOM     63  CG  ASP A   8      14.773  36.936  13.380  1.00 42.32           C  \nATOM     64  OD1 ASP A   8      14.113  36.453  14.322  1.00 44.69           O  \nATOM     65  OD2 ASP A   8      14.354  36.956  12.200  1.00 40.75           O  \nATOM     66  N   VAL A   9      19.454  36.605  14.510  1.00 23.15           N  \nATOM     67  CA  VAL A   9      20.791  37.197  14.578  1.00 22.04           C  \nATOM     68  C   VAL A   9      20.939  38.054  13.328  1.00 22.16           C  \nATOM     69  O   VAL A   9      20.771  37.565  12.206  1.00 20.99           O  \nATOM     70  CB  VAL A   9      21.876  36.124  14.602  1.00 25.18           C  \nATOM     71  CG1 VAL A   9      23.245  36.770  14.443  1.00 20.89           C  \nATOM     72  CG2 VAL A   9      21.793  35.357  15.915  1.00 23.53           C  \nATOM     73  N   PHE A  10      21.261  39.330  13.522  1.00 28.20           N  \nATOM     74  CA  PHE A  10      21.354  40.263  12.410  1.00 28.12           C  \nATOM     75  C   PHE A  10      22.720  40.874  12.138  1.00 29.60           C  \nATOM     76  O   PHE A  10      23.135  41.802  12.829  1.00 31.86           O  \nATOM     77  CB  PHE A  10      20.346  41.392  12.632  1.00 19.23           C  \nATOM     78  CG  PHE A  10      19.969  42.133  11.384  1.00 20.75           C  \nATOM     79  CD1 PHE A  10      18.714  41.934  10.798  1.00 19.21           C  \nATOM     80  CD2 PHE A  10      20.830  43.075  10.823  1.00 20.42           C  \nATOM     81  CE1 PHE A  10      18.315  42.661   9.672  1.00 17.89           C  \nATOM     82  CE2 PHE A  10      20.440  43.809   9.690  1.00 19.55           C  \nATOM     83  CZ  PHE A  10      19.178  43.604   9.120  1.00 18.04           C  \nATOM     84  N   GLY A  11      23.396  40.371  11.108  1.00 29.37           N  \nATOM     85  CA  GLY A  11      24.696  40.891  10.722  1.00 30.98           C  \nATOM     86  C   GLY A  11      25.809  40.773  11.743  1.00 31.62           C  \nATOM     87  O   GLY A  11      25.700  40.032  12.720  1.00 31.55           O  \nATOM     88  N   GLN A  12      26.894  41.507  11.506  1.00 32.75           N  \nATOM     89  CA  GLN A  12      28.039  41.491  12.408  1.00 33.90           C  \nATOM     90  C   GLN A  12      27.681  41.929  13.825  1.00 31.31           C  \nATOM     91  O   GLN A  12      28.133  41.324  14.793  1.00 32.19           O  \nATOM     92  CB  GLN A  12      29.168  42.362  11.855  1.00 39.94           C  \nATOM     93  CG  GLN A  12      29.890  41.719  10.682  1.00 47.13           C  \nATOM     94  CD  GLN A  12      31.005  42.590  10.132  1.00 51.50           C  \nATOM     95  OE1 GLN A  12      31.816  43.132  10.888  1.00 54.09           O  \nATOM     96  NE2 GLN A  12      31.058  42.720   8.808  1.00 51.61           N  \nATOM     97  N   PRO A  13      26.880  42.998  13.974  1.00 28.99           N  \nATOM     98  CA  PRO A  13      26.549  43.384  15.348  1.00 27.07           C  \nATOM     99  C   PRO A  13      25.887  42.194  16.058  1.00 27.16           C  \nATOM    100  O   PRO A  13      26.213  41.874  17.206  1.00 26.62           O  \nATOM    101  CB  PRO A  13      25.601  44.565  15.146  1.00 22.14           C  \nATOM    102  CG  PRO A  13      26.143  45.203  13.895  1.00 19.47           C  \nATOM    103  CD  PRO A  13      26.396  44.005  13.013  1.00 20.16           C  \nATOM    104  N   GLY A  14      24.978  41.529  15.349  1.00 24.22           N  \nATOM    105  CA  GLY A  14      24.295  40.378  15.909  1.00 23.99           C  \nATOM    106  C   GLY A  14      25.215  39.221  16.256  1.00 22.47           C  \nATOM    107  O   GLY A  14      25.082  38.630  17.324  1.00 22.42           O  \nATOM    108  N   ARG A  15      26.137  38.888  15.355  1.00 24.02           N  \nATOM    109  CA  ARG A  15      27.086  37.797  15.582  1.00 26.82           C  \nATOM    110  C   ARG A  15      28.052  38.139  16.718  1.00 26.63           C  \nATOM    111  O   ARG A  15      28.488  37.259  17.464  1.00 24.60           O  \nATOM    112  CB  ARG A  15      27.889  37.514  14.317  1.00 38.40           C  \nATOM    113  CG  ARG A  15      27.154  36.745  13.237  1.00 43.84           C  \nATOM    114  CD  ARG A  15      28.056  36.604  12.020  1.00 48.22           C  \nATOM    115  NE  ARG A  15      27.768  37.626  11.020  1.00 56.69           N  \nATOM    116  CZ  ARG A  15      28.665  38.144  10.185  1.00 59.43           C  \nATOM    117  NH1 ARG A  15      29.932  37.746  10.234  1.00 55.91           N  \nATOM    118  NH2 ARG A  15      28.285  39.042   9.278  1.00 60.98           N  \nATOM    119  N   ARG A  16      28.389  39.421  16.838  1.00 27.52           N  \nATOM    120  CA  ARG A  16      29.289  39.853  17.887  1.00 29.08           C  \nATOM    121  C   ARG A  16      28.672  39.574  19.251  1.00 28.80           C  \nATOM    122  O   ARG A  16      29.271  38.879  20.050  1.00 32.23           O  \nATOM    123  CB  ARG A  16      29.603  41.341  17.758  1.00 27.02           C  \nATOM    124  CG  ARG A  16      30.564  41.848  18.826  1.00 30.49           C  \nATOM    125  CD  ARG A  16      30.248  43.293  19.211  1.00 29.98           C  \nATOM    126  NE  ARG A  16      30.345  44.160  18.049  1.00 32.66           N  \nATOM    127  CZ  ARG A  16      29.436  45.063  17.709  1.00 30.10           C  \nATOM    128  NH1 ARG A  16      28.347  45.232  18.444  1.00 30.54           N  \nATOM    129  NH2 ARG A  16      29.613  45.791  16.620  1.00 32.16           N  \nATOM    130  N   VAL A  17      27.480  40.103  19.525  1.00 23.76           N  \nATOM    131  CA  VAL A  17      26.854  39.858  20.824  1.00 23.85           C  \nATOM    132  C   VAL A  17      26.549  38.388  21.052  1.00 22.40           C  \nATOM    133  O   VAL A  17      26.636  37.899  22.170  1.00 22.37           O  \nATOM    134  CB  VAL A  17      25.526  40.611  21.009  1.00 40.35           C  \nATOM    135  CG1 VAL A  17      25.767  41.912  21.730  1.00 41.96           C  \nATOM    136  CG2 VAL A  17      24.862  40.835  19.672  1.00 41.51           C  \nATOM    137  N   LEU A  18      26.173  37.688  19.993  1.00 24.86           N  \nATOM    138  CA  LEU A  18      25.844  36.275  20.110  1.00 25.70           C  \nATOM    139  C   LEU A  18      27.038  35.500  20.645  1.00 26.12           C  \nATOM    140  O   LEU A  18      26.953  34.842  21.676  1.00 26.66           O  \nATOM    141  CB  LEU A  18      25.452  35.712  18.738  1.00 30.34           C  \nATOM    142  CG  LEU A  18      24.675  34.390  18.629  1.00 32.11           C  \nATOM    143  CD1 LEU A  18      25.381  33.480  17.658  1.00 33.45           C  \nATOM    144  CD2 LEU A  18      24.530  33.721  19.991  1.00 30.05           C  \nATOM    145  N   GLN A  19      28.155  35.619  19.937  1.00 26.48           N  \nATOM    146  CA  GLN A  19      29.377  34.905  20.249  1.00 27.78           C  \nATOM    147  C   GLN A  19      30.064  35.302  21.545  1.00 28.57           C  \nATOM    148  O   GLN A  19      30.618  34.440  22.226  1.00 27.44           O  \nATOM    149  CB  GLN A  19      30.322  35.023  19.049  1.00 29.08           C  \nATOM    150  CG  GLN A  19      29.664  34.470  17.791  1.00 28.59           C  \nATOM    151  CD  GLN A  19      30.488  34.609  16.531  1.00 29.77           C  \nATOM    152  OE1 GLN A  19      30.024  34.263  15.442  1.00 30.02           O  \nATOM    153  NE2 GLN A  19      31.710  35.116  16.662  1.00 30.25           N  \nATOM    154  N   ASN A  20      30.022  36.588  21.893  1.00 26.65           N  \nATOM    155  CA  ASN A  20      30.635  37.052  23.124  1.00 27.53           C  \nATOM    156  C   ASN A  20      29.773  36.709  24.318  1.00 28.19           C  \nATOM    157  O   ASN A  20      30.267  36.230  25.344  1.00 26.39           O  \nATOM    158  CB  ASN A  20      30.815  38.574  23.128  1.00 45.66           C  \nATOM    159  CG  ASN A  20      32.007  39.032  22.325  1.00 49.15           C  \nATOM    160  OD1 ASN A  20      32.866  38.232  21.937  1.00 54.58           O  \nATOM    161  ND2 ASN A  20      32.078  40.336  22.081  1.00 50.61           N  \nATOM    162  N   HIS A  21      28.476  36.951  24.184  1.00 27.47           N  \nATOM    163  CA  HIS A  21      27.580  36.739  25.303  1.00 29.27           C  \nATOM    164  C   HIS A  21      27.013  35.344  25.576  1.00 29.53           C  \nATOM    165  O   HIS A  21      26.929  34.930  26.736  1.00 28.18           O  \nATOM    166  CB  HIS A  21      26.450  37.758  25.205  1.00 36.53           C  \nATOM    167  CG  HIS A  21      25.693  37.944  26.479  1.00 36.98           C  \nATOM    168  ND1 HIS A  21      24.683  37.094  26.875  1.00 38.39           N  \nATOM    169  CD2 HIS A  21      25.788  38.892  27.441  1.00 38.24           C  \nATOM    170  CE1 HIS A  21      24.185  37.513  28.024  1.00 39.37           C  \nATOM    171  NE2 HIS A  21      24.836  38.602  28.389  1.00 38.52           N  \nATOM    172  N   LEU A  22      26.644  34.608  24.533  1.00 31.90           N  \nATOM    173  CA  LEU A  22      26.058  33.282  24.732  1.00 32.38           C  \nATOM    174  C   LEU A  22      26.854  32.366  25.669  1.00 31.95           C  \nATOM    175  O   LEU A  22      26.294  31.787  26.602  1.00 30.38           O  \nATOM    176  CB  LEU A  22      25.856  32.566  23.383  1.00 29.75           C  \nATOM    177  CG  LEU A  22      24.664  31.605  23.240  1.00 30.08           C  \nATOM    178  CD1 LEU A  22      24.873  30.705  22.019  1.00 29.15           C  \nATOM    179  CD2 LEU A  22      24.497  30.759  24.482  1.00 26.85           C  \nATOM    180  N   PRO A  23      28.168  32.214  25.429  1.00 40.08           N  \nATOM    181  CA  PRO A  23      28.995  31.345  26.277  1.00 39.55           C  \nATOM    182  C   PRO A  23      28.954  31.581  27.790  1.00 39.47           C  \nATOM    183  O   PRO A  23      29.273  30.683  28.569  1.00 40.20           O  \nATOM    184  CB  PRO A  23      30.399  31.504  25.681  1.00 32.65           C  \nATOM    185  CG  PRO A  23      30.334  32.801  24.943  1.00 34.15           C  \nATOM    186  CD  PRO A  23      28.971  32.809  24.348  1.00 31.08           C  \nATOM    187  N   THR A  24      28.556  32.769  28.219  1.00 33.14           N  \nATOM    188  CA  THR A  24      28.487  33.024  29.646  1.00 33.75           C  \nATOM    189  C   THR A  24      27.144  32.570  30.240  1.00 33.74           C  \nATOM    190  O   THR A  24      27.051  32.331  31.448  1.00 33.56           O  \nATOM    191  CB  THR A  24      28.716  34.520  29.972  1.00 35.52           C  \nATOM    192  OG1 THR A  24      27.746  35.315  29.282  1.00 38.73           O  \nATOM    193  CG2 THR A  24      30.106  34.956  29.551  1.00 32.89           C  \nATOM    194  N   ILE A  25      26.109  32.448  29.407  1.00 30.44           N  \nATOM    195  CA  ILE A  25      24.795  32.002  29.891  1.00 30.91           C  \nATOM    196  C   ILE A  25      24.526  30.555  29.525  1.00 31.36           C  \nATOM    197  O   ILE A  25      23.715  29.873  30.154  1.00 32.41           O  \nATOM    198  CB  ILE A  25      23.599  32.857  29.337  1.00 32.10           C  \nATOM    199  CG1 ILE A  25      23.895  33.410  27.951  1.00 28.60           C  \nATOM    200  CG2 ILE A  25      23.269  33.979  30.279  1.00 31.92           C  \nATOM    201  CD1 ILE A  25      22.807  34.365  27.472  1.00 30.37           C  \nATOM    202  N   ARG A  26      25.220  30.093  28.497  1.00 30.68           N  \nATOM    203  CA  ARG A  26      25.074  28.733  28.014  1.00 31.36           C  \nATOM    204  C   ARG A  26      25.003  27.681  29.148  1.00 31.44           C  \nATOM    205  O   ARG A  26      24.227  26.723  29.072  1.00 31.06           O  \nATOM    206  CB  ARG A  26      26.238  28.425  27.063  1.00 37.53           C  \nATOM    207  CG  ARG A  26      25.890  27.479  25.928  1.00 38.53           C  \nATOM    208  CD  ARG A  26      25.150  26.293  26.469  1.00 40.05           C  \nATOM    209  NE  ARG A  26      24.900  25.262  25.485  1.00 37.15           N  \nATOM    210  CZ  ARG A  26      24.519  24.039  25.818  1.00 40.72           C  \nATOM    211  NH1 ARG A  26      24.356  23.731  27.099  1.00 40.84           N  \nATOM    212  NH2 ARG A  26      24.318  23.124  24.882  1.00 42.64           N  \nATOM    213  N   PRO A  27      25.808  27.846  30.214  1.00 33.93           N  \nATOM    214  CA  PRO A  27      25.784  26.872  31.316  1.00 34.58           C  \nATOM    215  C   PRO A  27      24.429  26.667  31.987  1.00 34.21           C  \nATOM    216  O   PRO A  27      24.184  25.615  32.581  1.00 35.31           O  \nATOM    217  CB  PRO A  27      26.822  27.428  32.294  1.00 38.04           C  \nATOM    218  CG  PRO A  27      27.800  28.105  31.392  1.00 38.28           C  \nATOM    219  CD  PRO A  27      26.902  28.816  30.402  1.00 36.08           C  \nATOM    220  N   GLN A  28      23.558  27.669  31.905  1.00 31.46           N  \nATOM    221  CA  GLN A  28      22.235  27.584  32.524  1.00 31.59           C  \nATOM    222  C   GLN A  28      21.258  26.754  31.695  1.00 30.71           C  \nATOM    223  O   GLN A  28      20.161  26.437  32.151  1.00 29.99           O  \nATOM    224  CB  GLN A  28      21.626  28.980  32.709  1.00 38.79           C  \nATOM    225  CG  GLN A  28      22.536  30.019  33.353  1.00 42.42           C  \nATOM    226  CD  GLN A  28      21.835  31.358  33.555  1.00 44.66           C  \nATOM    227  OE1 GLN A  28      22.481  32.397  33.730  1.00 44.06           O  \nATOM    228  NE2 GLN A  28      20.504  31.335  33.538  1.00 45.09           N  \nATOM    229  N   PHE A  29      21.646  26.398  30.478  1.00 32.64           N  \nATOM    230  CA  PHE A  29      20.740  25.645  29.633  1.00 32.69           C  \nATOM    231  C   PHE A  29      21.268  24.306  29.152  1.00 32.33           C  \nATOM    232  O   PHE A  29      22.474  24.070  29.103  1.00 33.36           O  \nATOM    233  CB  PHE A  29      20.340  26.505  28.431  1.00 25.30           C  \nATOM    234  CG  PHE A  29      19.780  27.841  28.820  1.00 24.78           C  \nATOM    235  CD1 PHE A  29      18.443  27.971  29.168  1.00 21.87           C  \nATOM    236  CD2 PHE A  29      20.612  28.958  28.914  1.00 21.96           C  \nATOM    237  CE1 PHE A  29      17.941  29.195  29.607  1.00 21.59           C  \nATOM    238  CE2 PHE A  29      20.115  30.176  29.350  1.00 21.23           C  \nATOM    239  CZ  PHE A  29      18.778  30.293  29.698  1.00 19.54           C  \nATOM    240  N   ASP A  30      20.324  23.444  28.791  1.00 29.56           N  \nATOM    241  CA  ASP A  30      20.598  22.115  28.279  1.00 28.95           C  \nATOM    242  C   ASP A  30      20.697  22.182  26.764  1.00 29.73           C  \nATOM    243  O   ASP A  30      21.418  21.408  26.147  1.00 29.38           O  \nATOM    244  CB  ASP A  30      19.466  21.180  28.674  1.00 30.42           C  \nATOM    245  CG  ASP A  30      19.353  21.018  30.171  1.00 33.95           C  \nATOM    246  OD1 ASP A  30      20.354  20.584  30.781  1.00 33.88           O  \nATOM    247  OD2 ASP A  30      18.276  21.324  30.735  1.00 33.12           O  \nATOM    248  N   PHE A  31      19.978  23.130  26.173  1.00 33.40           N  \nATOM    249  CA  PHE A  31      19.975  23.295  24.731  1.00 32.98           C  \nATOM    250  C   PHE A  31      19.607  24.732  24.352  1.00 32.73           C  \nATOM    251  O   PHE A  31      18.761  25.364  24.986  1.00 31.97           O  \nATOM    252  CB  PHE A  31      18.969  22.314  24.116  1.00 28.39           C  \nATOM    253  CG  PHE A  31      19.172  22.073  22.659  1.00 25.66           C  \nATOM    254  CD1 PHE A  31      18.293  22.602  21.723  1.00 27.73           C  \nATOM    255  CD2 PHE A  31      20.230  21.296  22.222  1.00 25.34           C  \nATOM    256  CE1 PHE A  31      18.468  22.359  20.371  1.00 25.21           C  \nATOM    257  CE2 PHE A  31      20.418  21.044  20.876  1.00 25.91           C  \nATOM    258  CZ  PHE A  31      19.532  21.577  19.945  1.00 28.44           C  \nATOM    259  N   VAL A  32      20.237  25.235  23.301  1.00 27.18           N  \nATOM    260  CA  VAL A  32      19.986  26.590  22.848  1.00 25.82           C  \nATOM    261  C   VAL A  32      19.666  26.668  21.354  1.00 24.92           C  \nATOM    262  O   VAL A  32      20.495  26.303  20.508  1.00 25.41           O  \nATOM    263  CB  VAL A  32      21.203  27.476  23.166  1.00 24.31           C  \nATOM    264  CG1 VAL A  32      21.007  28.875  22.617  1.00 20.18           C  \nATOM    265  CG2 VAL A  32      21.407  27.522  24.675  1.00 24.52           C  \nATOM    266  N   ILE A  33      18.459  27.151  21.052  1.00 27.25           N  \nATOM    267  CA  ILE A  33      17.971  27.317  19.682  1.00 28.09           C  \nATOM    268  C   ILE A  33      18.227  28.760  19.230  1.00 27.82           C  \nATOM    269  O   ILE A  33      17.884  29.706  19.933  1.00 28.38           O  \nATOM    270  CB  ILE A  33      16.454  27.040  19.608  1.00 23.71           C  \nATOM    271  CG1 ILE A  33      16.148  25.688  20.269  1.00 24.93           C  \nATOM    272  CG2 ILE A  33      15.983  27.082  18.152  1.00 23.05           C  \nATOM    273  CD1 ILE A  33      14.662  25.312  20.313  1.00 24.17           C  \nATOM    274  N   VAL A  34      18.825  28.934  18.060  1.00 31.91           N  \nATOM    275  CA  VAL A  34      19.123  30.275  17.579  1.00 32.54           C  \nATOM    276  C   VAL A  34      18.643  30.470  16.146  1.00 33.24           C  \nATOM    277  O   VAL A  34      18.870  29.616  15.281  1.00 33.77           O  \nATOM    278  CB  VAL A  34      20.657  30.561  17.665  1.00 23.49           C  \nATOM    279  CG1 VAL A  34      20.966  32.001  17.242  1.00 21.52           C  \nATOM    280  CG2 VAL A  34      21.145  30.333  19.095  1.00 25.97           C  \nATOM    281  N   ASN A  35      17.956  31.582  15.906  1.00 24.10           N  \nATOM    282  CA  ASN A  35      17.470  31.898  14.565  1.00 24.99           C  \nATOM    283  C   ASN A  35      18.553  32.748  13.876  1.00 25.10           C  \nATOM    284  O   ASN A  35      18.872  33.857  14.319  1.00 25.49           O  \nATOM    285  CB  ASN A  35      16.138  32.652  14.660  1.00 23.88           C  \nATOM    286  CG  ASN A  35      15.577  33.033  13.299  1.00 25.70           C  \nATOM    287  OD1 ASN A  35      16.150  33.863  12.592  1.00 25.19           O  \nATOM    288  ND2 ASN A  35      14.449  32.428  12.927  1.00 24.99           N  \nATOM    289  N   MET A  36      19.106  32.222  12.789  1.00 25.47           N  \nATOM    290  CA  MET A  36      20.185  32.882  12.055  1.00 27.10           C  \nATOM    291  C   MET A  36      19.792  33.441  10.674  1.00 30.63           C  \nATOM    292  O   MET A  36      20.665  33.680   9.826  1.00 30.65           O  \nATOM    293  CB  MET A  36      21.331  31.877  11.858  1.00 22.58           C  \nATOM    294  CG  MET A  36      21.916  31.270  13.127  1.00 22.80           C  \nATOM    295  SD  MET A  36      22.905  32.430  14.096  1.00 17.52           S  \nATOM    296  CE  MET A  36      24.329  32.542  13.060  1.00 23.23           C  \nATOM    297  N   GLU A  37      18.498  33.662  10.447  1.00 24.57           N  \nATOM    298  CA  GLU A  37      18.031  34.140   9.144  1.00 25.93           C  \nATOM    299  C   GLU A  37      18.642  35.455   8.631  1.00 24.40           C  \nATOM    300  O   GLU A  37      18.607  35.729   7.427  1.00 21.63           O  \nATOM    301  CB  GLU A  37      16.496  34.272   9.145  1.00 39.44           C  \nATOM    302  CG  GLU A  37      15.992  35.696   9.361  1.00 43.38           C  \nATOM    303  CD  GLU A  37      14.780  36.032   8.510  1.00 50.07           C  \nATOM    304  OE1 GLU A  37      14.676  35.493   7.382  1.00 51.97           O  \nATOM    305  OE2 GLU A  37      13.940  36.850   8.957  1.00 53.68           O  \nATOM    306  N   ASN A  38      19.202  36.265   9.527  1.00 31.86           N  \nATOM    307  CA  ASN A  38      19.771  37.555   9.126  1.00 32.15           C  \nATOM    308  C   ASN A  38      21.288  37.670   9.368  1.00 33.48           C  \nATOM    309  O   ASN A  38      21.843  38.764   9.278  1.00 32.84           O  \nATOM    310  CB  ASN A  38      19.050  38.686   9.878  1.00 32.03           C  \nATOM    311  CG  ASN A  38      17.530  38.674   9.667  1.00 32.88           C  \nATOM    312  OD1 ASN A  38      17.038  38.866   8.551  1.00 34.03           O  \nATOM    313  ND2 ASN A  38      16.790  38.456  10.742  1.00 30.70           N  \nATOM    314  N   SER A  39      21.941  36.543   9.659  1.00 37.59           N  \nATOM    315  CA  SER A  39      23.385  36.475   9.936  1.00 38.57           C  \nATOM    316  C   SER A  39      24.305  37.134   8.910  1.00 38.32           C  \nATOM    317  O   SER A  39      25.198  37.904   9.264  1.00 39.47           O  \nATOM    318  CB  SER A  39      23.826  35.016  10.052  1.00 29.68           C  \nATOM    319  OG  SER A  39      22.916  34.271  10.815  1.00 36.35           O  \nATOM    320  N   ALA A  40      24.095  36.785   7.645  1.00 26.81           N  \nATOM    321  CA  ALA A  40      24.903  37.274   6.534  1.00 26.47           C  \nATOM    322  C   ALA A  40      24.595  38.702   6.091  1.00 25.67           C  \nATOM    323  O   ALA A  40      23.960  38.912   5.063  1.00 26.35           O  \nATOM    324  CB  ALA A  40      24.744  36.316   5.346  1.00 16.70           C  \nATOM    325  N   GLY A  41      25.050  39.685   6.854  1.00 27.93           N  \nATOM    326  CA  GLY A  41      24.795  41.063   6.479  1.00 27.55           C  \nATOM    327  C   GLY A  41      23.313  41.392   6.410  1.00 29.79           C  \nATOM    328  O   GLY A  41      22.880  42.178   5.570  1.00 29.63           O  \nATOM    329  N   GLY A  42      22.532  40.773   7.291  1.00 33.37           N  \nATOM    330  CA  GLY A  42      21.108  41.027   7.330  1.00 34.24           C  \nATOM    331  C   GLY A  42      20.264  40.114   6.464  1.00 34.74           C  \nATOM    332  O   GLY A  42      19.048  40.039   6.655  1.00 34.43           O  \nATOM    333  N   PHE A  43      20.899  39.400   5.540  1.00 31.26           N  \nATOM    334  CA  PHE A  43      20.169  38.523   4.625  1.00 32.88           C  \nATOM    335  C   PHE A  43      20.766  37.117   4.471  1.00 31.75           C  \nATOM    336  O   PHE A  43      21.742  36.931   3.744  1.00 31.43           O  \nATOM    337  CB  PHE A  43      20.113  39.179   3.247  1.00 44.20           C  \nATOM    338  CG  PHE A  43      19.532  40.564   3.252  1.00 46.42           C  \nATOM    339  CD1 PHE A  43      18.153  40.758   3.260  1.00 48.48           C  \nATOM    340  CD2 PHE A  43      20.365  41.682   3.235  1.00 48.64           C  \nATOM    341  CE1 PHE A  43      17.611  42.048   3.250  1.00 49.18           C  \nATOM    342  CE2 PHE A  43      19.830  42.976   3.227  1.00 48.39           C  \nATOM    343  CZ  PHE A  43      18.452  43.157   3.232  1.00 47.41           C  \nATOM    344  N   GLY A  44      20.176  36.131   5.140  1.00 29.50           N  \nATOM    345  CA  GLY A  44      20.677  34.767   5.032  1.00 28.85           C  \nATOM    346  C   GLY A  44      21.758  34.372   6.030  1.00 29.37           C  \nATOM    347  O   GLY A  44      22.061  35.131   6.960  1.00 25.95           O  \nATOM    348  N   MET A  45      22.348  33.188   5.824  1.00 33.74           N  \nATOM    349  CA  MET A  45      23.399  32.657   6.705  1.00 33.06           C  \nATOM    350  C   MET A  45      24.593  32.082   5.940  1.00 33.83           C  \nATOM    351  O   MET A  45      24.428  31.200   5.096  1.00 33.50           O  \nATOM    352  CB  MET A  45      22.811  31.564   7.622  1.00 27.51           C  \nATOM    353  CG  MET A  45      23.837  30.825   8.509  1.00 28.47           C  \nATOM    354  SD  MET A  45      23.118  29.741   9.803  1.00 22.19           S  \nATOM    355  CE  MET A  45      22.696  28.329   8.858  1.00 25.96           C  \nATOM    356  N   HIS A  46      25.793  32.580   6.225  1.00 28.88           N  \nATOM    357  CA  HIS A  46      26.996  32.060   5.560  1.00 30.35           C  \nATOM    358  C   HIS A  46      27.739  31.105   6.497  1.00 30.02           C  \nATOM    359  O   HIS A  46      27.480  31.091   7.700  1.00 27.91           O  \nATOM    360  CB  HIS A  46      27.908  33.200   5.091  1.00 35.41           C  \nATOM    361  CG  HIS A  46      28.309  34.148   6.172  1.00 36.12           C  \nATOM    362  ND1 HIS A  46      28.349  35.512   5.982  1.00 38.10           N  \nATOM    363  CD2 HIS A  46      28.712  33.932   7.445  1.00 37.70           C  \nATOM    364  CE1 HIS A  46      28.758  36.098   7.093  1.00 36.74           C  \nATOM    365  NE2 HIS A  46      28.984  35.161   7.996  1.00 39.86           N  \nATOM    366  N   ARG A  47      28.647  30.302   5.946  1.00 31.38           N  \nATOM    367  CA  ARG A  47      29.368  29.302   6.739  1.00 33.03           C  \nATOM    368  C   ARG A  47      30.048  29.786   8.019  1.00 33.73           C  \nATOM    369  O   ARG A  47      30.022  29.077   9.037  1.00 33.06           O  \nATOM    370  CB  ARG A  47      30.387  28.560   5.871  1.00 37.99           C  \nATOM    371  CG  ARG A  47      30.988  27.336   6.565  1.00 41.11           C  \nATOM    372  CD  ARG A  47      31.945  26.584   5.651  1.00 42.06           C  \nATOM    373  NE  ARG A  47      31.262  26.013   4.492  1.00 42.74           N  \nATOM    374  CZ  ARG A  47      30.539  24.895   4.518  1.00 43.53           C  \nATOM    375  NH1 ARG A  47      30.402  24.207   5.645  1.00 41.52           N  \nATOM    376  NH2 ARG A  47      29.941  24.469   3.413  1.00 43.28           N  \nATOM    377  N   ASP A  48      30.660  30.970   7.982  1.00 35.85           N  \nATOM    378  CA  ASP A  48      31.318  31.493   9.178  1.00 38.59           C  \nATOM    379  C   ASP A  48      30.280  31.764  10.263  1.00 37.75           C  \nATOM    380  O   ASP A  48      30.459  31.385  11.427  1.00 38.30           O  \nATOM    381  CB  ASP A  48      32.085  32.789   8.869  1.00 49.82           C  \nATOM    382  CG  ASP A  48      33.413  32.541   8.165  1.00 55.60           C  \nATOM    383  OD1 ASP A  48      34.039  33.532   7.724  1.00 57.65           O  \nATOM    384  OD2 ASP A  48      33.837  31.366   8.056  1.00 56.79           O  \nATOM    385  N   ALA A  49      29.197  32.424   9.867  1.00 32.85           N  \nATOM    386  CA  ALA A  49      28.122  32.753  10.784  1.00 30.68           C  \nATOM    387  C   ALA A  49      27.667  31.500  11.510  1.00 29.79           C  \nATOM    388  O   ALA A  49      27.542  31.487  12.732  1.00 31.53           O  \nATOM    389  CB  ALA A  49      26.951  33.369  10.015  1.00 19.88           C  \nATOM    390  N   ALA A  50      27.423  30.435  10.755  1.00 23.49           N  \nATOM    391  CA  ALA A  50      26.971  29.192  11.359  1.00 24.19           C  \nATOM    392  C   ALA A  50      28.040  28.591  12.269  1.00 25.36           C  \nATOM    393  O   ALA A  50      27.725  28.091  13.358  1.00 23.00           O  \nATOM    394  CB  ALA A  50      26.574  28.197  10.274  1.00 27.63           C  \nATOM    395  N   ARG A  51      29.301  28.642  11.832  1.00 30.70           N  \nATOM    396  CA  ARG A  51      30.387  28.088  12.634  1.00 31.44           C  \nATOM    397  C   ARG A  51      30.519  28.855  13.941  1.00 30.92           C  \nATOM    398  O   ARG A  51      30.663  28.255  15.002  1.00 29.80           O  \nATOM    399  CB  ARG A  51      31.716  28.116  11.872  1.00 43.13           C  \nATOM    400  CG  ARG A  51      32.329  26.728  11.736  1.00 49.31           C  \nATOM    401  CD  ARG A  51      33.777  26.714  11.247  1.00 52.96           C  \nATOM    402  NE  ARG A  51      33.910  27.193   9.878  1.00 57.08           N  \nATOM    403  CZ  ARG A  51      34.213  28.447   9.553  1.00 61.85           C  \nATOM    404  NH1 ARG A  51      34.421  29.352  10.508  1.00 62.40           N  \nATOM    405  NH2 ARG A  51      34.298  28.800   8.273  1.00 61.11           N  \nATOM    406  N   GLY A  52      30.450  30.182  13.862  1.00 36.15           N  \nATOM    407  CA  GLY A  52      30.563  31.000  15.057  1.00 33.26           C  \nATOM    408  C   GLY A  52      29.468  30.727  16.073  1.00 33.45           C  \nATOM    409  O   GLY A  52      29.730  30.631  17.274  1.00 32.68           O  \nATOM    410  N   ALA A  53      28.237  30.591  15.584  1.00 28.31           N  \nATOM    411  CA  ALA A  53      27.080  30.340  16.432  1.00 27.78           C  \nATOM    412  C   ALA A  53      27.190  29.016  17.160  1.00 28.05           C  \nATOM    413  O   ALA A  53      26.912  28.949  18.354  1.00 29.27           O  \nATOM    414  CB  ALA A  53      25.811  30.366  15.602  1.00 27.85           C  \nATOM    415  N   LEU A  54      27.582  27.962  16.448  1.00 30.70           N  \nATOM    416  CA  LEU A  54      27.721  26.642  17.064  1.00 33.55           C  \nATOM    417  C   LEU A  54      28.843  26.664  18.108  1.00 35.03           C  \nATOM    418  O   LEU A  54      28.671  26.194  19.234  1.00 33.78           O  \nATOM    419  CB  LEU A  54      28.007  25.588  15.992  1.00 31.94           C  \nATOM    420  CG  LEU A  54      26.854  25.340  15.012  1.00 31.93           C  \nATOM    421  CD1 LEU A  54      27.357  24.650  13.762  1.00 32.30           C  \nATOM    422  CD2 LEU A  54      25.783  24.506  15.694  1.00 33.34           C  \nATOM    423  N   GLU A  55      29.991  27.217  17.731  1.00 38.44           N  \nATOM    424  CA  GLU A  55      31.114  27.330  18.649  1.00 39.86           C  \nATOM    425  C   GLU A  55      30.614  28.045  19.901  1.00 39.81           C  \nATOM    426  O   GLU A  55      30.979  27.683  21.019  1.00 41.60           O  \nATOM    427  CB  GLU A  55      32.239  28.146  18.012  1.00 75.27           C  \nATOM    428  CG  GLU A  55      33.416  28.416  18.933  1.00 84.28           C  \nATOM    429  CD  GLU A  55      34.543  27.416  18.763  1.00 91.39           C  \nATOM    430  OE1 GLU A  55      35.471  27.421  19.605  1.00 93.60           O  \nATOM    431  OE2 GLU A  55      34.508  26.632  17.785  1.00 94.40           O  \nATOM    432  N   ALA A  56      29.771  29.058  19.709  1.00 27.59           N  \nATOM    433  CA  ALA A  56      29.220  29.827  20.825  1.00 25.33           C  \nATOM    434  C   ALA A  56      28.355  29.012  21.790  1.00 25.27           C  \nATOM    435  O   ALA A  56      28.185  29.398  22.945  1.00 26.63           O  \nATOM    436  CB  ALA A  56      28.417  31.003  20.296  1.00 39.70           C  \nATOM    437  N   GLY A  57      27.790  27.900  21.328  1.00 27.17           N  \nATOM    438  CA  GLY A  57      26.960  27.094  22.207  1.00 25.21           C  \nATOM    439  C   GLY A  57      25.583  26.728  21.673  1.00 26.67           C  \nATOM    440  O   GLY A  57      24.827  26.015  22.343  1.00 26.59           O  \nATOM    441  N   ALA A  58      25.241  27.202  20.479  1.00 32.99           N  \nATOM    442  CA  ALA A  58      23.940  26.894  19.892  1.00 34.37           C  \nATOM    443  C   ALA A  58      23.888  25.443  19.415  1.00 34.98           C  \nATOM    444  O   ALA A  58      24.829  24.959  18.780  1.00 34.44           O  \nATOM    445  CB  ALA A  58      23.651  27.829  18.723  1.00 22.97           C  \nATOM    446  N   GLY A  59      22.778  24.767  19.715  1.00 34.39           N  \nATOM    447  CA  GLY A  59      22.603  23.378  19.310  1.00 33.66           C  \nATOM    448  C   GLY A  59      21.742  23.231  18.061  1.00 33.63           C  \nATOM    449  O   GLY A  59      21.795  22.212  17.376  1.00 32.71           O  \nATOM    450  N   CYS A  60      20.936  24.248  17.775  1.00 28.67           N  \nATOM    451  CA  CYS A  60      20.078  24.245  16.587  1.00 30.88           C  \nATOM    452  C   CYS A  60      20.019  25.636  15.967  1.00 29.38           C  \nATOM    453  O   CYS A  60      19.914  26.646  16.674  1.00 29.97           O  \nATOM    454  CB  CYS A  60      18.654  23.785  16.920  1.00 34.69           C  \nATOM    455  SG  CYS A  60      17.573  23.647  15.461  1.00 39.46           S  \nATOM    456  N   LEU A  61      20.100  25.671  14.643  1.00 25.41           N  \nATOM    457  CA  LEU A  61      20.058  26.909  13.879  1.00 25.00           C  \nATOM    458  C   LEU A  61      18.794  26.935  13.031  1.00 25.87           C  \nATOM    459  O   LEU A  61      18.669  26.160  12.073  1.00 24.78           O  \nATOM    460  CB  LEU A  61      21.279  27.010  12.953  1.00 27.69           C  \nATOM    461  CG  LEU A  61      22.679  27.261  13.534  1.00 28.58           C  \nATOM    462  CD1 LEU A  61      22.809  26.631  14.888  1.00 27.38           C  \nATOM    463  CD2 LEU A  61      23.736  26.689  12.579  1.00 27.44           C  \nATOM    464  N   THR A  62      17.854  27.813  13.380  1.00 28.50           N  \nATOM    465  CA  THR A  62      16.626  27.924  12.598  1.00 27.87           C  \nATOM    466  C   THR A  62      16.760  29.123  11.678  1.00 27.80           C  \nATOM    467  O   THR A  62      17.511  30.056  11.967  1.00 26.05           O  \nATOM    468  CB  THR A  62      15.375  28.109  13.486  1.00 23.75           C  \nATOM    469  OG1 THR A  62      15.506  29.304  14.257  1.00 22.39           O  \nATOM    470  CG2 THR A  62      15.211  26.921  14.411  1.00 22.39           C  \nATOM    471  N   LEU A  63      16.042  29.089  10.562  1.00 28.55           N  \nATOM    472  CA  LEU A  63      16.096  30.178   9.604  1.00 30.75           C  \nATOM    473  C   LEU A  63      14.717  30.799   9.413  1.00 32.12           C  \nATOM    474  O   LEU A  63      13.951  30.999  10.371  1.00 32.45           O  \nATOM    475  CB  LEU A  63      16.618  29.663   8.264  1.00 32.08           C  \nATOM    476  CG  LEU A  63      17.997  28.987   8.256  1.00 33.66           C  \nATOM    477  CD1 LEU A  63      18.257  28.349   6.892  1.00 31.38           C  \nATOM    478  CD2 LEU A  63      19.056  30.016   8.593  1.00 32.72           C  \nATOM    479  N   GLY A  64      14.409  31.115   8.166  1.00 27.78           N  \nATOM    480  CA  GLY A  64      13.121  31.688   7.889  1.00 29.38           C  \nATOM    481  C   GLY A  64      13.022  32.240   6.494  1.00 30.77           C  \nATOM    482  O   GLY A  64      13.451  31.631   5.517  1.00 29.50           O  \nATOM    483  N   ASN A  65      12.427  33.421   6.449  1.00 41.48           N  \nATOM    484  CA  ASN A  65      12.189  34.204   5.258  1.00 42.61           C  \nATOM    485  C   ASN A  65      13.364  34.212   4.271  1.00 42.38           C  \nATOM    486  O   ASN A  65      13.234  33.757   3.130  1.00 40.41           O  \nATOM    487  CB  ASN A  65      11.886  35.630   5.709  1.00 46.56           C  \nATOM    488  CG  ASN A  65      11.190  36.425   4.663  1.00 49.30           C  \nATOM    489  OD1 ASN A  65      11.559  36.383   3.493  1.00 54.19           O  \nATOM    490  ND2 ASN A  65      10.174  37.173   5.070  1.00 52.59           N  \nATOM    491  N   HIS A  66      14.504  34.737   4.725  1.00 43.23           N  \nATOM    492  CA  HIS A  66      15.713  34.853   3.902  1.00 41.86           C  \nATOM    493  C   HIS A  66      16.637  33.652   3.993  1.00 40.05           C  \nATOM    494  O   HIS A  66      17.834  33.772   3.756  1.00 41.48           O  \nATOM    495  CB  HIS A  66      16.523  36.083   4.308  1.00 33.24           C  \nATOM    496  CG  HIS A  66      15.708  37.325   4.467  1.00 35.25           C  \nATOM    497  ND1 HIS A  66      14.923  37.562   5.574  1.00 36.79           N  \nATOM    498  CD2 HIS A  66      15.554  38.399   3.657  1.00 36.18           C  \nATOM    499  CE1 HIS A  66      14.320  38.730   5.440  1.00 37.62           C  \nATOM    500  NE2 HIS A  66      14.686  39.259   4.286  1.00 36.53           N  \nATOM    501  N   ALA A  67      16.090  32.497   4.331  1.00 23.98           N  \nATOM    502  CA  ALA A  67      16.904  31.301   4.458  1.00 22.60           C  \nATOM    503  C   ALA A  67      17.780  31.003   3.248  1.00 22.72           C  \nATOM    504  O   ALA A  67      18.778  30.303   3.377  1.00 20.78           O  \nATOM    505  CB  ALA A  67      16.015  30.111   4.735  1.00 21.21           C  \nATOM    506  N   TRP A  68      17.418  31.514   2.074  1.00 29.26           N  \nATOM    507  CA  TRP A  68      18.192  31.216   0.869  1.00 29.14           C  \nATOM    508  C   TRP A  68      18.979  32.364   0.274  1.00 29.39           C  \nATOM    509  O   TRP A  68      19.536  32.213  -0.806  1.00 31.93           O  \nATOM    510  CB  TRP A  68      17.274  30.674  -0.234  1.00 28.48           C  \nATOM    511  CG  TRP A  68      16.206  29.786   0.266  1.00 27.94           C  \nATOM    512  CD1 TRP A  68      14.939  30.153   0.632  1.00 28.40           C  \nATOM    513  CD2 TRP A  68      16.336  28.402   0.595  1.00 26.95           C  \nATOM    514  NE1 TRP A  68      14.276  29.083   1.179  1.00 28.28           N  \nATOM    515  CE2 TRP A  68      15.110  27.994   1.172  1.00 28.81           C  \nATOM    516  CE3 TRP A  68      17.372  27.466   0.469  1.00 25.80           C  \nATOM    517  CZ2 TRP A  68      14.894  26.685   1.625  1.00 26.39           C  \nATOM    518  CZ3 TRP A  68      17.157  26.168   0.919  1.00 26.37           C  \nATOM    519  CH2 TRP A  68      15.926  25.790   1.491  1.00 25.90           C  \nATOM    520  N   HIS A  69      19.049  33.503   0.947  1.00 27.86           N  \nATOM    521  CA  HIS A  69      19.754  34.626   0.338  1.00 29.65           C  \nATOM    522  C   HIS A  69      21.264  34.530   0.192  1.00 29.36           C  \nATOM    523  O   HIS A  69      21.855  35.287  -0.578  1.00 30.47           O  \nATOM    524  CB  HIS A  69      19.352  35.939   1.011  1.00 37.41           C  \nATOM    525  CG  HIS A  69      17.949  36.357   0.685  1.00 44.07           C  \nATOM    526  ND1 HIS A  69      17.615  37.646   0.330  1.00 45.03           N  \nATOM    527  CD2 HIS A  69      16.799  35.640   0.623  1.00 44.01           C  \nATOM    528  CE1 HIS A  69      16.322  37.706   0.060  1.00 45.72           C  \nATOM    529  NE2 HIS A  69      15.804  36.502   0.229  1.00 45.77           N  \nATOM    530  N   HIS A  70      21.897  33.605   0.906  1.00 31.03           N  \nATOM    531  CA  HIS A  70      23.344  33.449   0.771  1.00 31.50           C  \nATOM    532  C   HIS A  70      23.650  32.072   0.195  1.00 31.38           C  \nATOM    533  O   HIS A  70      23.200  31.055   0.715  1.00 31.14           O  \nATOM    534  CB  HIS A  70      24.042  33.644   2.114  1.00 27.10           C  \nATOM    535  CG  HIS A  70      25.485  34.005   1.982  1.00 28.34           C  \nATOM    536  ND1 HIS A  70      26.467  33.073   1.719  1.00 28.17           N  \nATOM    537  CD2 HIS A  70      26.107  35.208   2.018  1.00 27.57           C  \nATOM    538  CE1 HIS A  70      27.630  33.689   1.596  1.00 27.19           C  \nATOM    539  NE2 HIS A  70      27.438  34.985   1.772  1.00 26.99           N  \nATOM    540  N   LYS A  71      24.420  32.046  -0.887  1.00 30.53           N  \nATOM    541  CA  LYS A  71      24.752  30.787  -1.554  1.00 33.13           C  \nATOM    542  C   LYS A  71      25.293  29.684  -0.632  1.00 31.54           C  \nATOM    543  O   LYS A  71      25.101  28.493  -0.903  1.00 31.49           O  \nATOM    544  CB  LYS A  71      25.739  31.043  -2.706  1.00 63.96           C  \nATOM    545  CG  LYS A  71      27.209  31.148  -2.303  1.00 71.43           C  \nATOM    546  CD  LYS A  71      27.534  32.425  -1.542  1.00 76.02           C  \nATOM    547  CE  LYS A  71      27.627  33.638  -2.463  1.00 79.12           C  \nATOM    548  NZ  LYS A  71      28.102  34.857  -1.734  1.00 78.87           N  \nATOM    549  N   ASP A  72      25.945  30.073   0.461  1.00 32.89           N  \nATOM    550  CA  ASP A  72      26.521  29.107   1.394  1.00 33.05           C  \nATOM    551  C   ASP A  72      25.529  28.129   1.994  1.00 32.92           C  \nATOM    552  O   ASP A  72      25.911  27.039   2.420  1.00 32.98           O  \nATOM    553  CB  ASP A  72      27.255  29.839   2.517  1.00 33.67           C  \nATOM    554  CG  ASP A  72      28.644  30.293   2.105  1.00 36.04           C  \nATOM    555  OD1 ASP A  72      28.917  30.360   0.890  1.00 37.09           O  \nATOM    556  OD2 ASP A  72      29.464  30.589   2.997  1.00 39.06           O  \nATOM    557  N   ILE A  73      24.253  28.504   1.997  1.00 28.80           N  \nATOM    558  CA  ILE A  73      23.210  27.669   2.573  1.00 27.70           C  \nATOM    559  C   ILE A  73      22.942  26.380   1.794  1.00 26.31           C  \nATOM    560  O   ILE A  73      22.433  25.401   2.352  1.00 22.95           O  \nATOM    561  CB  ILE A  73      21.898  28.460   2.687  1.00 47.79           C  \nATOM    562  CG1 ILE A  73      22.215  29.902   3.112  1.00 54.48           C  \nATOM    563  CG2 ILE A  73      21.002  27.814   3.739  1.00 45.47           C  \nATOM    564  CD1 ILE A  73      21.188  30.976   2.690  1.00 51.94           C  \nATOM    565  N   TYR A  74      23.269  26.367   0.506  1.00 28.59           N  \nATOM    566  CA  TYR A  74      23.021  25.170  -0.290  1.00 30.23           C  \nATOM    567  C   TYR A  74      24.072  24.106   0.021  1.00 29.26           C  \nATOM    568  O   TYR A  74      23.728  22.955   0.302  1.00 29.11           O  \nATOM    569  CB  TYR A  74      22.958  25.528  -1.786  1.00 28.33           C  \nATOM    570  CG  TYR A  74      21.660  26.219  -2.166  1.00 30.36           C  \nATOM    571  CD1 TYR A  74      20.462  25.509  -2.212  1.00 30.65           C  \nATOM    572  CD2 TYR A  74      21.624  27.590  -2.437  1.00 32.11           C  \nATOM    573  CE1 TYR A  74      19.265  26.136  -2.514  1.00 30.41           C  \nATOM    574  CE2 TYR A  74      20.422  28.234  -2.739  1.00 33.43           C  \nATOM    575  CZ  TYR A  74      19.246  27.502  -2.776  1.00 32.97           C  \nATOM    576  OH  TYR A  74      18.044  28.128  -3.072  1.00 34.01           O  \nATOM    577  N   PRO A  75      25.365  24.459  -0.045  1.00 31.63           N  \nATOM    578  CA  PRO A  75      26.315  23.397   0.290  1.00 32.47           C  \nATOM    579  C   PRO A  75      26.157  22.998   1.766  1.00 33.08           C  \nATOM    580  O   PRO A  75      26.226  21.817   2.095  1.00 33.05           O  \nATOM    581  CB  PRO A  75      27.679  24.019  -0.022  1.00 30.60           C  \nATOM    582  CG  PRO A  75      27.416  25.501  -0.052  1.00 32.06           C  \nATOM    583  CD  PRO A  75      26.049  25.607  -0.661  1.00 30.13           C  \nATOM    584  N   MET A  76      25.915  23.964   2.653  1.00 33.61           N  \nATOM    585  CA  MET A  76      25.742  23.624   4.069  1.00 35.30           C  \nATOM    586  C   MET A  76      24.594  22.638   4.285  1.00 35.48           C  \nATOM    587  O   MET A  76      24.696  21.728   5.114  1.00 35.46           O  \nATOM    588  CB  MET A  76      25.497  24.876   4.931  1.00 30.44           C  \nATOM    589  CG  MET A  76      26.726  25.724   5.189  1.00 29.24           C  \nATOM    590  SD  MET A  76      26.509  26.948   6.511  1.00 24.50           S  \nATOM    591  CE  MET A  76      26.292  25.900   7.810  1.00 33.61           C  \nATOM    592  N   LEU A  77      23.495  22.815   3.557  1.00 29.46           N  \nATOM    593  CA  LEU A  77      22.364  21.903   3.712  1.00 32.21           C  \nATOM    594  C   LEU A  77      22.648  20.565   3.029  1.00 32.60           C  \nATOM    595  O   LEU A  77      22.224  19.513   3.504  1.00 32.60           O  \nATOM    596  CB  LEU A  77      21.085  22.512   3.130  1.00 33.89           C  \nATOM    597  CG  LEU A  77      20.460  23.703   3.862  1.00 33.00           C  \nATOM    598  CD1 LEU A  77      19.273  24.202   3.050  1.00 30.58           C  \nATOM    599  CD2 LEU A  77      20.031  23.303   5.259  1.00 28.84           C  \nATOM    600  N   SER A  78      23.372  20.613   1.917  1.00 32.75           N  \nATOM    601  CA  SER A  78      23.714  19.408   1.181  1.00 34.93           C  \nATOM    602  C   SER A  78      24.579  18.436   1.998  1.00 37.39           C  \nATOM    603  O   SER A  78      24.333  17.225   1.982  1.00 37.56           O  \nATOM    604  CB  SER A  78      24.446  19.774  -0.099  1.00 37.40           C  \nATOM    605  OG  SER A  78      24.729  18.606  -0.841  1.00 42.64           O  \nATOM    606  N   GLU A  79      25.584  18.956   2.707  1.00 48.83           N  \nATOM    607  CA  GLU A  79      26.461  18.103   3.513  1.00 50.45           C  \nATOM    608  C   GLU A  79      25.843  17.671   4.839  1.00 50.60           C  \nATOM    609  O   GLU A  79      26.451  16.932   5.608  1.00 52.60           O  \nATOM    610  CB  GLU A  79      27.828  18.766   3.736  1.00 49.29           C  \nATOM    611  CG  GLU A  79      27.807  20.268   3.825  1.00 52.19           C  \nATOM    612  CD  GLU A  79      29.026  20.896   3.178  1.00 52.68           C  \nATOM    613  OE1 GLU A  79      29.336  20.539   2.024  1.00 52.55           O  \nATOM    614  OE2 GLU A  79      29.668  21.752   3.818  1.00 56.97           O  \nATOM    615  N   ASP A  80      24.626  18.137   5.091  1.00 39.55           N  \nATOM    616  CA  ASP A  80      23.855  17.768   6.276  1.00 39.95           C  \nATOM    617  C   ASP A  80      24.649  17.498   7.562  1.00 39.28           C  \nATOM    618  O   ASP A  80      24.426  16.490   8.234  1.00 40.38           O  \nATOM    619  CB  ASP A  80      22.991  16.544   5.912  1.00 51.25           C  \nATOM    620  CG  ASP A  80      22.093  16.082   7.046  1.00 54.07           C  \nATOM    621  OD1 ASP A  80      21.399  16.935   7.642  1.00 56.90           O  \nATOM    622  OD2 ASP A  80      22.072  14.861   7.333  1.00 53.16           O  \nATOM    623  N   THR A  81      25.567  18.389   7.920  1.00 34.09           N  \nATOM    624  CA  THR A  81      26.323  18.180   9.151  1.00 32.68           C  \nATOM    625  C   THR A  81      25.929  19.198  10.227  1.00 30.81           C  \nATOM    626  O   THR A  81      25.685  18.815  11.374  1.00 28.93           O  \nATOM    627  CB  THR A  81      27.853  18.208   8.894  1.00 46.99           C  \nATOM    628  OG1 THR A  81      28.285  19.538   8.593  1.00 49.88           O  \nATOM    629  CG2 THR A  81      28.201  17.305   7.718  1.00 45.93           C  \nATOM    630  N   TYR A  82      25.849  20.481   9.862  1.00 27.69           N  \nATOM    631  CA  TYR A  82      25.443  21.526  10.811  1.00 28.34           C  \nATOM    632  C   TYR A  82      23.965  21.317  11.156  1.00 26.46           C  \nATOM    633  O   TYR A  82      23.149  21.046  10.272  1.00 24.58           O  \nATOM    634  CB  TYR A  82      25.585  22.921  10.199  1.00 49.01           C  \nATOM    635  CG  TYR A  82      26.990  23.337   9.850  1.00 55.32           C  \nATOM    636  CD1 TYR A  82      27.757  22.596   8.959  1.00 58.98           C  \nATOM    637  CD2 TYR A  82      27.540  24.497  10.379  1.00 57.64           C  \nATOM    638  CE1 TYR A  82      29.037  23.002   8.598  1.00 60.48           C  \nATOM    639  CE2 TYR A  82      28.818  24.913  10.024  1.00 60.01           C  \nATOM    640  CZ  TYR A  82      29.559  24.160   9.132  1.00 59.47           C  \nATOM    641  OH  TYR A  82      30.818  24.571   8.762  1.00 59.57           O  \nATOM    642  N   PRO A  83      23.600  21.443  12.439  1.00 32.28           N  \nATOM    643  CA  PRO A  83      22.201  21.257  12.830  1.00 32.50           C  \nATOM    644  C   PRO A  83      21.346  22.433  12.373  1.00 33.42           C  \nATOM    645  O   PRO A  83      21.097  23.364  13.132  1.00 34.87           O  \nATOM    646  CB  PRO A  83      22.282  21.126  14.351  1.00 28.21           C  \nATOM    647  CG  PRO A  83      23.449  21.973  14.696  1.00 29.42           C  \nATOM    648  CD  PRO A  83      24.458  21.644  13.619  1.00 29.01           C  \nATOM    649  N   ILE A  84      20.900  22.378  11.123  1.00 32.59           N  \nATOM    650  CA  ILE A  84      20.093  23.443  10.546  1.00 31.57           C  \nATOM    651  C   ILE A  84      18.655  23.011  10.292  1.00 31.23           C  \nATOM    652  O   ILE A  84      18.391  21.869   9.899  1.00 31.49           O  \nATOM    653  CB  ILE A  84      20.687  23.917   9.207  1.00 32.29           C  \nATOM    654  CG1 ILE A  84      22.149  24.317   9.390  1.00 31.90           C  \nATOM    655  CG2 ILE A  84      19.891  25.095   8.679  1.00 33.38           C  \nATOM    656  CD1 ILE A  84      22.890  24.514   8.080  1.00 32.04           C  \nATOM    657  N   VAL A  85      17.725  23.934  10.518  1.00 32.20           N  \nATOM    658  CA  VAL A  85      16.309  23.668  10.291  1.00 30.39           C  \nATOM    659  C   VAL A  85      15.702  24.836   9.522  1.00 30.11           C  \nATOM    660  O   VAL A  85      15.659  25.954  10.029  1.00 34.13           O  \nATOM    661  CB  VAL A  85      15.542  23.507  11.621  1.00 20.76           C  \nATOM    662  CG1 VAL A  85      14.062  23.284  11.342  1.00 21.87           C  \nATOM    663  CG2 VAL A  85      16.106  22.340  12.413  1.00 21.45           C  \nATOM    664  N   ARG A  86      15.257  24.591   8.294  1.00 26.96           N  \nATOM    665  CA  ARG A  86      14.641  25.650   7.506  1.00 25.63           C  \nATOM    666  C   ARG A  86      13.110  25.520   7.555  1.00 26.09           C  \nATOM    667  O   ARG A  86      12.587  24.499   8.024  1.00 27.14           O  \nATOM    668  CB  ARG A  86      15.176  25.637   6.061  1.00 21.24           C  \nATOM    669  CG  ARG A  86      15.146  24.315   5.341  1.00 22.74           C  \nATOM    670  CD  ARG A  86      13.740  23.840   5.027  1.00 21.69           C  \nATOM    671  NE  ARG A  86      13.806  22.614   4.249  1.00 21.52           N  \nATOM    672  CZ  ARG A  86      12.763  21.859   3.926  1.00 22.84           C  \nATOM    673  NH1 ARG A  86      11.534  22.204   4.320  1.00 19.54           N  \nATOM    674  NH2 ARG A  86      12.957  20.756   3.209  1.00 18.48           N  \nATOM    675  N   PRO A  87      12.373  26.553   7.093  1.00 26.27           N  \nATOM    676  CA  PRO A  87      10.905  26.507   7.119  1.00 26.24           C  \nATOM    677  C   PRO A  87      10.339  25.190   6.585  1.00 27.04           C  \nATOM    678  O   PRO A  87      10.738  24.708   5.522  1.00 24.76           O  \nATOM    679  CB  PRO A  87      10.506  27.715   6.269  1.00 20.84           C  \nATOM    680  CG  PRO A  87      11.637  28.677   6.509  1.00 20.08           C  \nATOM    681  CD  PRO A  87      12.839  27.764   6.390  1.00 19.14           C  \nATOM    682  N   LEU A  88       9.409  24.613   7.340  1.00 27.92           N  \nATOM    683  CA  LEU A  88       8.803  23.345   6.969  1.00 29.30           C  \nATOM    684  C   LEU A  88       7.775  23.446   5.853  1.00 31.63           C  \nATOM    685  O   LEU A  88       7.697  22.567   4.994  1.00 31.95           O  \nATOM    686  CB  LEU A  88       8.138  22.706   8.187  1.00 25.47           C  \nATOM    687  CG  LEU A  88       7.396  21.401   7.877  1.00 25.11           C  \nATOM    688  CD1 LEU A  88       8.366  20.384   7.293  1.00 24.81           C  \nATOM    689  CD2 LEU A  88       6.764  20.870   9.141  1.00 23.01           C  \nATOM    690  N   ASN A  89       6.995  24.522   5.866  1.00 39.30           N  \nATOM    691  CA  ASN A  89       5.943  24.704   4.877  1.00 40.35           C  \nATOM    692  C   ASN A  89       6.325  25.169   3.471  1.00 42.27           C  \nATOM    693  O   ASN A  89       5.603  25.938   2.846  1.00 41.78           O  \nATOM    694  CB  ASN A  89       4.825  25.584   5.458  1.00 30.91           C  \nATOM    695  CG  ASN A  89       5.335  26.881   6.051  1.00 29.80           C  \nATOM    696  OD1 ASN A  89       6.406  26.930   6.659  1.00 30.31           O  \nATOM    697  ND2 ASN A  89       4.550  27.937   5.903  1.00 28.68           N  \nATOM    698  N   TYR A  90       7.472  24.708   2.987  1.00 43.28           N  \nATOM    699  CA  TYR A  90       7.897  24.987   1.618  1.00 45.89           C  \nATOM    700  C   TYR A  90       7.432  23.691   0.945  1.00 48.65           C  \nATOM    701  O   TYR A  90       8.069  22.648   1.081  1.00 51.43           O  \nATOM    702  CB  TYR A  90       9.421  25.124   1.511  1.00 32.83           C  \nATOM    703  CG  TYR A  90       9.943  26.533   1.703  1.00 33.10           C  \nATOM    704  CD1 TYR A  90       9.512  27.575   0.881  1.00 34.49           C  \nATOM    705  CD2 TYR A  90      10.884  26.826   2.692  1.00 34.07           C  \nATOM    706  CE1 TYR A  90      10.005  28.873   1.038  1.00 33.76           C  \nATOM    707  CE2 TYR A  90      11.388  28.121   2.861  1.00 32.19           C  \nATOM    708  CZ  TYR A  90      10.944  29.139   2.033  1.00 36.13           C  \nATOM    709  OH  TYR A  90      11.426  30.426   2.197  1.00 37.57           O  \nATOM    710  N   ALA A  91       6.310  23.761   0.239  1.00 58.78           N  \nATOM    711  CA  ALA A  91       5.720  22.590  -0.404  1.00 60.51           C  \nATOM    712  C   ALA A  91       6.541  21.790  -1.420  1.00 60.52           C  \nATOM    713  O   ALA A  91       6.216  20.632  -1.691  1.00 60.44           O  \nATOM    714  CB  ALA A  91       4.392  22.981  -1.024  1.00 52.37           C  \nATOM    715  N   ASP A  92       7.579  22.384  -1.998  1.00 37.12           N  \nATOM    716  CA  ASP A  92       8.376  21.655  -2.977  1.00 37.08           C  \nATOM    717  C   ASP A  92       9.113  20.519  -2.276  1.00 36.73           C  \nATOM    718  O   ASP A  92       9.886  20.745  -1.335  1.00 34.66           O  \nATOM    719  CB  ASP A  92       9.374  22.573  -3.675  1.00 62.09           C  \nATOM    720  CG  ASP A  92       9.849  22.005  -4.998  1.00 65.99           C  \nATOM    721  OD1 ASP A  92       9.015  21.877  -5.923  1.00 69.78           O  \nATOM    722  OD2 ASP A  92      11.050  21.686  -5.118  1.00 67.15           O  \nATOM    723  N   PRO A  93       8.875  19.274  -2.726  1.00 41.84           N  \nATOM    724  CA  PRO A  93       9.480  18.054  -2.181  1.00 41.41           C  \nATOM    725  C   PRO A  93      10.983  18.001  -2.389  1.00 39.91           C  \nATOM    726  O   PRO A  93      11.673  17.221  -1.755  1.00 40.06           O  \nATOM    727  CB  PRO A  93       8.777  16.936  -2.951  1.00 57.99           C  \nATOM    728  CG  PRO A  93       7.494  17.572  -3.422  1.00 57.47           C  \nATOM    729  CD  PRO A  93       7.939  18.943  -3.814  1.00 57.14           C  \nATOM    730  N   GLY A  94      11.491  18.826  -3.289  1.00 50.40           N  \nATOM    731  CA  GLY A  94      12.915  18.811  -3.533  1.00 48.69           C  \nATOM    732  C   GLY A  94      13.672  19.900  -2.811  1.00 49.02           C  \nATOM    733  O   GLY A  94      14.895  19.976  -2.931  1.00 51.25           O  \nATOM    734  N   THR A  95      12.972  20.743  -2.054  1.00 34.29           N  \nATOM    735  CA  THR A  95      13.660  21.823  -1.356  1.00 30.33           C  \nATOM    736  C   THR A  95      14.683  21.276  -0.373  1.00 28.39           C  \nATOM    737  O   THR A  95      14.376  20.422   0.455  1.00 26.10           O  \nATOM    738  CB  THR A  95      12.672  22.775  -0.633  1.00 36.96           C  \nATOM    739  OG1 THR A  95      11.689  22.007   0.054  1.00 40.85           O  \nATOM    740  CG2 THR A  95      11.980  23.702  -1.641  1.00 37.96           C  \nATOM    741  N   PRO A  96      15.926  21.760  -0.474  1.00 32.70           N  \nATOM    742  CA  PRO A  96      17.058  21.365   0.373  1.00 31.14           C  \nATOM    743  C   PRO A  96      16.777  21.416   1.870  1.00 29.67           C  \nATOM    744  O   PRO A  96      15.845  22.077   2.324  1.00 28.49           O  \nATOM    745  CB  PRO A  96      18.144  22.352  -0.029  1.00 28.82           C  \nATOM    746  CG  PRO A  96      17.862  22.557  -1.498  1.00 30.74           C  \nATOM    747  CD  PRO A  96      16.355  22.714  -1.515  1.00 30.81           C  \nATOM    748  N   GLY A  97      17.591  20.686   2.624  1.00 24.46           N  \nATOM    749  CA  GLY A  97      17.472  20.664   4.068  1.00 23.40           C  \nATOM    750  C   GLY A  97      16.224  20.007   4.595  1.00 22.47           C  \nATOM    751  O   GLY A  97      15.563  19.257   3.874  1.00 24.16           O  \nATOM    752  N   VAL A  98      15.889  20.306   5.848  1.00 20.42           N  \nATOM    753  CA  VAL A  98      14.716  19.712   6.489  1.00 20.52           C  \nATOM    754  C   VAL A  98      13.897  20.728   7.293  1.00 22.10           C  \nATOM    755  O   VAL A  98      14.415  21.760   7.710  1.00 21.72           O  \nATOM    756  CB  VAL A  98      15.142  18.545   7.428  1.00 23.31           C  \nATOM    757  CG1 VAL A  98      15.730  17.403   6.610  1.00 20.89           C  \nATOM    758  CG2 VAL A  98      16.179  19.028   8.424  1.00 19.32           C  \nATOM    759  N   GLY A  99      12.622  20.418   7.519  1.00 27.39           N  \nATOM    760  CA  GLY A  99      11.754  21.326   8.248  1.00 28.66           C  \nATOM    761  C   GLY A  99      11.766  21.137   9.748  1.00 30.71           C  \nATOM    762  O   GLY A  99      11.113  21.878  10.486  1.00 31.66           O  \nATOM    763  N   TRP A 100      12.504  20.135  10.203  1.00 32.01           N  \nATOM    764  CA  TRP A 100      12.608  19.846  11.623  1.00 33.50           C  \nATOM    765  C   TRP A 100      13.758  18.896  11.871  1.00 32.56           C  \nATOM    766  O   TRP A 100      14.236  18.236  10.954  1.00 33.01           O  \nATOM    767  CB  TRP A 100      11.325  19.199  12.144  1.00 41.75           C  \nATOM    768  CG  TRP A 100      10.926  17.986  11.372  1.00 45.69           C  \nATOM    769  CD1 TRP A 100      10.208  17.955  10.213  1.00 47.55           C  \nATOM    770  CD2 TRP A 100      11.251  16.627  11.678  1.00 48.00           C  \nATOM    771  NE1 TRP A 100      10.063  16.662   9.778  1.00 49.43           N  \nATOM    772  CE2 TRP A 100      10.693  15.825  10.660  1.00 51.36           C  \nATOM    773  CE3 TRP A 100      11.960  16.008  12.715  1.00 51.84           C  \nATOM    774  CZ2 TRP A 100      10.819  14.431  10.647  1.00 53.28           C  \nATOM    775  CZ3 TRP A 100      12.088  14.618  12.704  1.00 53.49           C  \nATOM    776  CH2 TRP A 100      11.518  13.847  11.676  1.00 54.57           C  \nATOM    777  N   ARG A 101      14.188  18.854  13.125  1.00 31.52           N  \nATOM    778  CA  ARG A 101      15.258  17.981  13.597  1.00 31.56           C  \nATOM    779  C   ARG A 101      14.934  17.708  15.050  1.00 31.68           C  \nATOM    780  O   ARG A 101      14.522  18.615  15.769  1.00 31.13           O  \nATOM    781  CB  ARG A 101      16.623  18.668  13.554  1.00 29.76           C  \nATOM    782  CG  ARG A 101      17.217  18.860  12.184  1.00 28.69           C  \nATOM    783  CD  ARG A 101      18.574  19.542  12.321  1.00 28.74           C  \nATOM    784  NE  ARG A 101      19.269  19.639  11.045  1.00 30.85           N  \nATOM    785  CZ  ARG A 101      19.729  18.600  10.361  1.00 32.43           C  \nATOM    786  NH1 ARG A 101      19.575  17.371  10.834  1.00 33.37           N  \nATOM    787  NH2 ARG A 101      20.327  18.789   9.191  1.00 33.54           N  \nATOM    788  N   THR A 102      15.085  16.461  15.477  1.00 31.09           N  \nATOM    789  CA  THR A 102      14.845  16.140  16.867  1.00 32.33           C  \nATOM    790  C   THR A 102      16.237  15.864  17.424  1.00 31.93           C  \nATOM    791  O   THR A 102      17.071  15.226  16.770  1.00 31.65           O  \nATOM    792  CB  THR A 102      13.943  14.912  17.033  1.00 40.42           C  \nATOM    793  OG1 THR A 102      14.744  13.733  17.080  1.00 46.02           O  \nATOM    794  CG2 THR A 102      12.992  14.806  15.879  1.00 38.98           C  \nATOM    795  N   PHE A 103      16.496  16.380  18.616  1.00 30.60           N  \nATOM    796  CA  PHE A 103      17.793  16.226  19.258  1.00 30.66           C  \nATOM    797  C   PHE A 103      17.639  15.492  20.575  1.00 32.53           C  \nATOM    798  O   PHE A 103      16.564  15.511  21.180  1.00 30.32           O  \nATOM    799  CB  PHE A 103      18.384  17.598  19.558  1.00 28.93           C  \nATOM    800  CG  PHE A 103      18.523  18.483  18.361  1.00 30.76           C  \nATOM    801  CD1 PHE A 103      19.684  18.469  17.596  1.00 30.41           C  \nATOM    802  CD2 PHE A 103      17.508  19.369  18.017  1.00 29.98           C  \nATOM    803  CE1 PHE A 103      19.830  19.335  16.509  1.00 30.32           C  \nATOM    804  CE2 PHE A 103      17.652  20.234  16.933  1.00 30.08           C  \nATOM    805  CZ  PHE A 103      18.816  20.216  16.180  1.00 26.63           C  \nATOM    806  N   ASP A 104      18.722  14.852  21.010  1.00 46.31           N  \nATOM    807  CA  ASP A 104      18.756  14.156  22.293  1.00 50.21           C  \nATOM    808  C   ASP A 104      19.239  15.195  23.287  1.00 52.25           C  \nATOM    809  O   ASP A 104      20.369  15.670  23.179  1.00 52.90           O  \nATOM    810  CB  ASP A 104      19.764  13.012  22.274  1.00 55.91           C  \nATOM    811  CG  ASP A 104      19.254  11.802  21.546  1.00 59.32           C  \nATOM    812  OD1 ASP A 104      19.938  11.341  20.605  1.00 61.06           O  \nATOM    813  OD2 ASP A 104      18.168  11.307  21.923  1.00 61.54           O  \nATOM    814  N   VAL A 105      18.395  15.562  24.242  1.00 53.66           N  \nATOM    815  CA  VAL A 105      18.795  16.554  25.224  1.00 56.73           C  \nATOM    816  C   VAL A 105      19.303  15.870  26.493  1.00 58.96           C  \nATOM    817  O   VAL A 105      20.510  15.664  26.649  1.00 61.18           O  \nATOM    818  CB  VAL A 105      17.635  17.503  25.536  1.00 42.84           C  \nATOM    819  CG1 VAL A 105      18.113  18.636  26.419  1.00 43.33           C  \nATOM    820  CG2 VAL A 105      17.077  18.062  24.237  1.00 41.72           C  \nATOM    821  N   ASN A 106      18.411  15.510  27.405  1.00 56.56           N  \nATOM    822  CA  ASN A 106      18.873  14.828  28.608  1.00 57.87           C  \nATOM    823  C   ASN A 106      18.084  13.557  28.828  1.00 57.07           C  \nATOM    824  O   ASN A 106      17.343  13.435  29.807  1.00 57.07           O  \nATOM    825  CB  ASN A 106      18.764  15.736  29.831  1.00 78.74           C  \nATOM    826  CG  ASN A 106      20.054  16.486  30.109  1.00 81.86           C  \nATOM    827  OD1 ASN A 106      21.126  15.885  30.205  1.00 82.37           O  \nATOM    828  ND2 ASN A 106      19.957  17.802  30.245  1.00 82.37           N  \nATOM    829  N   GLY A 107      18.262  12.608  27.912  1.00 50.81           N  \nATOM    830  CA  GLY A 107      17.541  11.351  27.993  1.00 49.74           C  \nATOM    831  C   GLY A 107      16.184  11.485  27.325  1.00 47.79           C  \nATOM    832  O   GLY A 107      15.440  10.509  27.190  1.00 48.84           O  \nATOM    833  N   GLU A 108      15.864  12.711  26.913  1.00 39.69           N  \nATOM    834  CA  GLU A 108      14.599  13.018  26.245  1.00 37.19           C  \nATOM    835  C   GLU A 108      14.841  13.616  24.864  1.00 33.22           C  \nATOM    836  O   GLU A 108      15.935  14.082  24.558  1.00 30.53           O  \nATOM    837  CB  GLU A 108      13.775  13.994  27.089  1.00 82.68           C  \nATOM    838  CG  GLU A 108      13.042  13.344  28.250  1.00 89.82           C  \nATOM    839  CD  GLU A 108      12.224  14.341  29.054  1.00 95.81           C  \nATOM    840  OE1 GLU A 108      12.825  15.149  29.795  1.00 98.99           O  \nATOM    841  OE2 GLU A 108      10.978  14.324  28.939  1.00 96.01           O  \nATOM    842  N   LYS A 109      13.815  13.593  24.028  1.00 48.10           N  \nATOM    843  CA  LYS A 109      13.927  14.141  22.686  1.00 47.51           C  \nATOM    844  C   LYS A 109      13.288  15.526  22.595  1.00 45.48           C  \nATOM    845  O   LYS A 109      12.207  15.768  23.139  1.00 44.57           O  \nATOM    846  CB  LYS A 109      13.253  13.204  21.685  1.00 51.95           C  \nATOM    847  CG  LYS A 109      13.818  11.802  21.667  1.00 55.68           C  \nATOM    848  CD  LYS A 109      12.762  10.817  21.198  1.00 59.46           C  \nATOM    849  CE  LYS A 109      11.579  10.782  22.174  1.00 63.32           C  \nATOM    850  NZ  LYS A 109      10.459   9.896  21.724  1.00 64.79           N  \nATOM    851  N   LEU A 110      13.982  16.435  21.922  1.00 36.88           N  \nATOM    852  CA  LEU A 110      13.489  17.792  21.700  1.00 34.82           C  \nATOM    853  C   LEU A 110      13.442  17.996  20.186  1.00 33.74           C  \nATOM    854  O   LEU A 110      14.456  17.882  19.496  1.00 32.03           O  \nATOM    855  CB  LEU A 110      14.422  18.834  22.330  1.00 28.53           C  \nATOM    856  CG  LEU A 110      14.214  20.293  21.895  1.00 28.56           C  \nATOM    857  CD1 LEU A 110      12.839  20.771  22.330  1.00 27.59           C  \nATOM    858  CD2 LEU A 110      15.285  21.177  22.512  1.00 27.80           C  \nATOM    859  N   THR A 111      12.259  18.282  19.665  1.00 36.37           N  \nATOM    860  CA  THR A 111      12.114  18.498  18.237  1.00 35.40           C  \nATOM    861  C   THR A 111      11.917  19.971  17.937  1.00 34.70           C  \nATOM    862  O   THR A 111      11.008  20.616  18.466  1.00 34.13           O  \nATOM    863  CB  THR A 111      10.934  17.674  17.663  1.00 34.97           C  \nATOM    864  OG1 THR A 111      11.325  16.295  17.570  1.00 37.89           O  \nATOM    865  CG2 THR A 111      10.537  18.176  16.279  1.00 35.35           C  \nATOM    866  N   VAL A 112      12.792  20.503  17.093  1.00 31.14           N  \nATOM    867  CA  VAL A 112      12.704  21.898  16.711  1.00 29.67           C  \nATOM    868  C   VAL A 112      12.124  22.019  15.305  1.00 29.84           C  \nATOM    869  O   VAL A 112      12.594  21.377  14.360  1.00 27.66           O  \nATOM    870  CB  VAL A 112      14.087  22.572  16.773  1.00 22.28           C  \nATOM    871  CG1 VAL A 112      13.990  24.006  16.321  1.00 18.35           C  \nATOM    872  CG2 VAL A 112      14.617  22.526  18.207  1.00 23.37           C  \nATOM    873  N   VAL A 113      11.075  22.822  15.176  1.00 33.08           N  \nATOM    874  CA  VAL A 113      10.451  23.024  13.880  1.00 34.35           C  \nATOM    875  C   VAL A 113      10.415  24.523  13.553  1.00 35.21           C  \nATOM    876  O   VAL A 113      10.214  25.364  14.433  1.00 36.19           O  \nATOM    877  CB  VAL A 113       9.032  22.408  13.854  1.00 41.12           C  \nATOM    878  CG1 VAL A 113       8.188  23.012  14.940  1.00 44.33           C  \nATOM    879  CG2 VAL A 113       8.392  22.614  12.493  1.00 41.41           C  \nATOM    880  N   ASN A 114      10.651  24.842  12.285  1.00 31.77           N  \nATOM    881  CA  ASN A 114      10.668  26.218  11.785  1.00 30.63           C  \nATOM    882  C   ASN A 114       9.457  26.356  10.844  1.00 29.75           C  \nATOM    883  O   ASN A 114       9.239  25.508   9.983  1.00 30.48           O  \nATOM    884  CB  ASN A 114      11.997  26.442  11.038  1.00 28.96           C  \nATOM    885  CG  ASN A 114      12.312  27.909  10.788  1.00 30.33           C  \nATOM    886  OD1 ASN A 114      11.935  28.796  11.562  1.00 32.25           O  \nATOM    887  ND2 ASN A 114      13.040  28.167   9.711  1.00 29.07           N  \nATOM    888  N   LEU A 115       8.671  27.411  11.021  1.00 27.27           N  \nATOM    889  CA  LEU A 115       7.467  27.643  10.218  1.00 27.57           C  \nATOM    890  C   LEU A 115       7.399  29.062   9.658  1.00 27.97           C  \nATOM    891  O   LEU A 115       7.851  30.010  10.306  1.00 29.09           O  \nATOM    892  CB  LEU A 115       6.208  27.446  11.074  1.00 26.13           C  \nATOM    893  CG  LEU A 115       5.291  26.231  11.008  1.00 24.71           C  \nATOM    894  CD1 LEU A 115       4.010  26.596  11.756  1.00 25.04           C  \nATOM    895  CD2 LEU A 115       4.970  25.843   9.579  1.00 20.13           C  \nATOM    896  N   LEU A 116       6.804  29.207   8.474  1.00 24.99           N  \nATOM    897  CA  LEU A 116       6.650  30.523   7.846  1.00 24.05           C  \nATOM    898  C   LEU A 116       5.188  30.974   7.813  1.00 25.00           C  \nATOM    899  O   LEU A 116       4.284  30.184   7.498  1.00 23.45           O  \nATOM    900  CB  LEU A 116       7.204  30.514   6.416  1.00 24.17           C  \nATOM    901  CG  LEU A 116       8.675  30.916   6.258  1.00 24.95           C  \nATOM    902  CD1 LEU A 116       9.105  30.804   4.806  1.00 24.83           C  \nATOM    903  CD2 LEU A 116       8.855  32.336   6.751  1.00 23.24           C  \nATOM    904  N   GLY A 117       4.969  32.245   8.153  1.00 31.18           N  \nATOM    905  CA  GLY A 117       3.634  32.820   8.137  1.00 30.21           C  \nATOM    906  C   GLY A 117       3.362  33.249   6.707  1.00 32.39           C  \nATOM    907  O   GLY A 117       4.273  33.205   5.877  1.00 33.58           O  \nATOM    908  N   ARG A 118       2.136  33.665   6.403  1.00 29.45           N  \nATOM    909  CA  ARG A 118       1.808  34.064   5.039  1.00 31.50           C  \nATOM    910  C   ARG A 118       1.196  35.454   4.918  1.00 31.80           C  \nATOM    911  O   ARG A 118       1.212  36.043   3.832  1.00 31.16           O  \nATOM    912  CB  ARG A 118       0.846  33.063   4.413  1.00 30.56           C  \nATOM    913  CG  ARG A 118       1.285  31.625   4.510  1.00 30.89           C  \nATOM    914  CD  ARG A 118       0.061  30.739   4.528  1.00 31.19           C  \nATOM    915  NE  ARG A 118      -0.233  30.178   3.227  1.00 28.95           N  \nATOM    916  CZ  ARG A 118      -1.312  29.451   2.965  1.00 31.98           C  \nATOM    917  NH1 ARG A 118      -2.200  29.214   3.919  1.00 29.65           N  \nATOM    918  NH2 ARG A 118      -1.482  28.923   1.761  1.00 31.82           N  \nATOM    919  N   VAL A 119       0.646  35.985   6.005  1.00 25.99           N  \nATOM    920  CA  VAL A 119       0.057  37.312   5.915  1.00 24.84           C  \nATOM    921  C   VAL A 119       1.152  38.361   5.718  1.00 24.55           C  \nATOM    922  O   VAL A 119       2.167  38.371   6.434  1.00 23.73           O  \nATOM    923  CB  VAL A 119      -0.761  37.658   7.171  1.00 24.34           C  \nATOM    924  CG1 VAL A 119      -1.594  38.893   6.908  1.00 22.15           C  \nATOM    925  CG2 VAL A 119      -1.653  36.495   7.548  1.00 26.68           C  \nATOM    926  N   PHE A 120       0.949  39.242   4.743  1.00 23.79           N  \nATOM    927  CA  PHE A 120       1.924  40.288   4.446  1.00 26.88           C  \nATOM    928  C   PHE A 120       3.267  39.660   4.057  1.00 27.62           C  \nATOM    929  O   PHE A 120       4.329  40.268   4.244  1.00 27.25           O  \nATOM    930  CB  PHE A 120       2.131  41.202   5.668  1.00 28.20           C  \nATOM    931  CG  PHE A 120       0.858  41.774   6.226  1.00 28.79           C  \nATOM    932  CD1 PHE A 120      -0.149  42.234   5.379  1.00 29.83           C  \nATOM    933  CD2 PHE A 120       0.668  41.867   7.599  1.00 30.06           C  \nATOM    934  CE1 PHE A 120      -1.324  42.770   5.898  1.00 30.68           C  \nATOM    935  CE2 PHE A 120      -0.509  42.406   8.128  1.00 30.86           C  \nATOM    936  CZ  PHE A 120      -1.504  42.858   7.278  1.00 30.70           C  \nATOM    937  N   MET A 121       3.217  38.439   3.533  1.00 27.09           N  \nATOM    938  CA  MET A 121       4.433  37.739   3.131  1.00 28.01           C  \nATOM    939  C   MET A 121       4.193  36.918   1.872  1.00 28.17           C  \nATOM    940  O   MET A 121       3.102  36.910   1.330  1.00 27.83           O  \nATOM    941  CB  MET A 121       4.931  36.824   4.266  1.00 25.18           C  \nATOM    942  CG  MET A 121       5.572  37.565   5.440  1.00 25.93           C  \nATOM    943  SD  MET A 121       6.082  36.471   6.780  1.00 26.65           S  \nATOM    944  CE  MET A 121       7.550  35.769   6.080  1.00 26.96           C  \nATOM    945  N   GLU A 122       5.226  36.231   1.408  1.00 28.12           N  \nATOM    946  CA  GLU A 122       5.110  35.413   0.218  1.00 29.89           C  \nATOM    947  C   GLU A 122       4.295  34.154   0.497  1.00 27.93           C  \nATOM    948  O   GLU A 122       4.366  33.574   1.578  1.00 27.33           O  \nATOM    949  CB  GLU A 122       6.501  35.042  -0.284  1.00 62.64           C  \nATOM    950  CG  GLU A 122       6.498  34.220  -1.545  1.00 72.71           C  \nATOM    951  CD  GLU A 122       7.888  34.043  -2.102  1.00 80.59           C  \nATOM    952  OE1 GLU A 122       8.515  35.067  -2.452  1.00 83.80           O  \nATOM    953  OE2 GLU A 122       8.356  32.885  -2.186  1.00 84.75           O  \nATOM    954  N   ALA A 123       3.519  33.727  -0.488  1.00 28.10           N  \nATOM    955  CA  ALA A 123       2.695  32.544  -0.316  1.00 27.32           C  \nATOM    956  C   ALA A 123       3.508  31.265  -0.399  1.00 26.71           C  \nATOM    957  O   ALA A 123       4.282  31.066  -1.336  1.00 24.81           O  \nATOM    958  CB  ALA A 123       1.586  32.516  -1.357  1.00 20.63           C  \nATOM    959  N   VAL A 124       3.324  30.418   0.609  1.00 27.71           N  \nATOM    960  CA  VAL A 124       3.966  29.114   0.703  1.00 27.14           C  \nATOM    961  C   VAL A 124       2.838  28.213   1.205  1.00 27.85           C  \nATOM    962  O   VAL A 124       1.681  28.633   1.214  1.00 25.06           O  \nATOM    963  CB  VAL A 124       5.122  29.131   1.722  1.00 31.05           C  \nATOM    964  CG1 VAL A 124       6.245  30.017   1.215  1.00 27.98           C  \nATOM    965  CG2 VAL A 124       4.628  29.649   3.058  1.00 29.49           C  \nATOM    966  N   ASP A 125       3.146  26.991   1.625  1.00 30.47           N  \nATOM    967  CA  ASP A 125       2.093  26.109   2.119  1.00 31.61           C  \nATOM    968  C   ASP A 125       1.437  26.639   3.389  1.00 30.05           C  \nATOM    969  O   ASP A 125       1.977  27.510   4.060  1.00 29.94           O  \nATOM    970  CB  ASP A 125       2.632  24.700   2.365  1.00 39.80           C  \nATOM    971  CG  ASP A 125       2.455  23.792   1.162  1.00 44.46           C  \nATOM    972  OD1 ASP A 125       2.752  22.580   1.273  1.00 46.62           O  \nATOM    973  OD2 ASP A 125       2.019  24.290   0.099  1.00 49.56           O  \nATOM    974  N   ASN A 126       0.259  26.107   3.701  1.00 26.76           N  \nATOM    975  CA  ASN A 126      -0.505  26.509   4.877  1.00 24.74           C  \nATOM    976  C   ASN A 126       0.209  26.028   6.136  1.00 26.00           C  \nATOM    977  O   ASN A 126       0.283  24.836   6.396  1.00 25.09           O  \nATOM    978  CB  ASN A 126      -1.905  25.902   4.799  1.00 25.70           C  \nATOM    979  CG  ASN A 126      -2.794  26.354   5.924  1.00 28.25           C  \nATOM    980  OD1 ASN A 126      -2.419  26.262   7.094  1.00 28.70           O  \nATOM    981  ND2 ASN A 126      -3.986  26.849   5.583  1.00 25.01           N  \nATOM    982  N   PRO A 127       0.743  26.956   6.941  1.00 30.56           N  \nATOM    983  CA  PRO A 127       1.447  26.570   8.165  1.00 30.78           C  \nATOM    984  C   PRO A 127       0.618  25.799   9.187  1.00 31.25           C  \nATOM    985  O   PRO A 127       1.117  24.866   9.807  1.00 32.73           O  \nATOM    986  CB  PRO A 127       1.934  27.907   8.708  1.00 27.66           C  \nATOM    987  CG  PRO A 127       0.855  28.840   8.286  1.00 26.92           C  \nATOM    988  CD  PRO A 127       0.653  28.421   6.844  1.00 28.05           C  \nATOM    989  N   PHE A 128      -0.641  26.184   9.356  1.00 31.12           N  \nATOM    990  CA  PHE A 128      -1.521  25.528  10.324  1.00 32.76           C  \nATOM    991  C   PHE A 128      -1.718  24.043  10.041  1.00 33.66           C  \nATOM    992  O   PHE A 128      -1.629  23.212  10.945  1.00 34.49           O  \nATOM    993  CB  PHE A 128      -2.877  26.238  10.356  1.00 25.87           C  \nATOM    994  CG  PHE A 128      -2.788  27.685  10.752  1.00 22.89           C  \nATOM    995  CD1 PHE A 128      -2.883  28.061  12.085  1.00 21.88           C  \nATOM    996  CD2 PHE A 128      -2.539  28.665   9.797  1.00 22.18           C  \nATOM    997  CE1 PHE A 128      -2.721  29.396  12.462  1.00 21.98           C  \nATOM    998  CE2 PHE A 128      -2.377  29.996  10.162  1.00 21.45           C  \nATOM    999  CZ  PHE A 128      -2.464  30.363  11.497  1.00 20.24           C  \nATOM   1000  N   ARG A 129      -1.981  23.714   8.783  1.00 32.31           N  \nATOM   1001  CA  ARG A 129      -2.189  22.332   8.406  1.00 34.08           C  \nATOM   1002  C   ARG A 129      -0.870  21.600   8.232  1.00 33.10           C  \nATOM   1003  O   ARG A 129      -0.806  20.386   8.399  1.00 34.03           O  \nATOM   1004  CB  ARG A 129      -3.050  22.254   7.144  1.00 34.61           C  \nATOM   1005  CG  ARG A 129      -4.504  22.549   7.443  1.00 39.52           C  \nATOM   1006  CD  ARG A 129      -5.385  22.278   6.264  1.00 41.60           C  \nATOM   1007  NE  ARG A 129      -5.343  23.384   5.324  1.00 48.28           N  \nATOM   1008  CZ  ARG A 129      -5.176  23.235   4.016  1.00 51.80           C  \nATOM   1009  NH1 ARG A 129      -5.030  22.016   3.511  1.00 54.84           N  \nATOM   1010  NH2 ARG A 129      -5.165  24.297   3.219  1.00 50.72           N  \nATOM   1011  N   THR A 130       0.182  22.334   7.894  1.00 26.82           N  \nATOM   1012  CA  THR A 130       1.494  21.717   7.774  1.00 26.67           C  \nATOM   1013  C   THR A 130       1.872  21.185   9.170  1.00 27.44           C  \nATOM   1014  O   THR A 130       2.457  20.117   9.285  1.00 27.04           O  \nATOM   1015  CB  THR A 130       2.560  22.732   7.288  1.00 26.07           C  \nATOM   1016  OG1 THR A 130       2.324  23.051   5.908  1.00 24.00           O  \nATOM   1017  CG2 THR A 130       3.960  22.155   7.434  1.00 23.80           C  \nATOM   1018  N   MET A 131       1.516  21.921  10.224  1.00 36.93           N  \nATOM   1019  CA  MET A 131       1.803  21.489  11.598  1.00 38.60           C  \nATOM   1020  C   MET A 131       0.914  20.315  12.003  1.00 39.36           C  \nATOM   1021  O   MET A 131       1.359  19.402  12.694  1.00 41.08           O  \nATOM   1022  CB  MET A 131       1.595  22.635  12.611  1.00 28.60           C  \nATOM   1023  CG  MET A 131       2.765  23.594  12.758  1.00 29.97           C  \nATOM   1024  SD  MET A 131       4.314  22.798  13.268  1.00 23.16           S  \nATOM   1025  CE  MET A 131       4.974  22.492  11.755  1.00 28.98           C  \nATOM   1026  N   ASP A 132      -0.348  20.345  11.593  1.00 34.26           N  \nATOM   1027  CA  ASP A 132      -1.254  19.253  11.928  1.00 36.11           C  \nATOM   1028  C   ASP A 132      -0.655  17.954  11.410  1.00 36.88           C  \nATOM   1029  O   ASP A 132      -0.706  16.920  12.078  1.00 38.84           O  \nATOM   1030  CB  ASP A 132      -2.642  19.490  11.322  1.00 30.89           C  \nATOM   1031  CG  ASP A 132      -3.481  20.454  12.151  1.00 30.24           C  \nATOM   1032  OD1 ASP A 132      -4.448  21.033  11.621  1.00 34.39           O  \nATOM   1033  OD2 ASP A 132      -3.181  20.628  13.346  1.00 28.77           O  \nATOM   1034  N   ALA A 133      -0.060  18.028  10.226  1.00 37.14           N  \nATOM   1035  CA  ALA A 133       0.570  16.873   9.616  1.00 37.21           C  \nATOM   1036  C   ALA A 133       1.833  16.440  10.376  1.00 36.95           C  \nATOM   1037  O   ALA A 133       2.002  15.251  10.658  1.00 37.48           O  \nATOM   1038  CB  ALA A 133       0.906  17.180   8.166  1.00 37.59           C  \nATOM   1039  N   LEU A 134       2.718  17.381  10.716  1.00 31.34           N  \nATOM   1040  CA  LEU A 134       3.935  17.000  11.433  1.00 31.89           C  \nATOM   1041  C   LEU A 134       3.522  16.312  12.717  1.00 33.83           C  \nATOM   1042  O   LEU A 134       3.975  15.205  13.019  1.00 33.60           O  \nATOM   1043  CB  LEU A 134       4.812  18.216  11.775  1.00 25.14           C  \nATOM   1044  CG  LEU A 134       6.175  17.954  12.472  1.00 21.15           C  \nATOM   1045  CD1 LEU A 134       7.340  17.844  11.494  1.00 11.30           C  \nATOM   1046  CD2 LEU A 134       6.443  19.089  13.380  1.00 21.62           C  \nATOM   1047  N   LEU A 135       2.629  16.974  13.447  1.00 45.51           N  \nATOM   1048  CA  LEU A 135       2.132  16.487  14.723  1.00 47.30           C  \nATOM   1049  C   LEU A 135       1.450  15.119  14.725  1.00 50.47           C  \nATOM   1050  O   LEU A 135       0.941  14.692  15.755  1.00 51.99           O  \nATOM   1051  CB  LEU A 135       1.205  17.536  15.341  1.00 43.38           C  \nATOM   1052  CG  LEU A 135       1.877  18.873  15.685  1.00 43.30           C  \nATOM   1053  CD1 LEU A 135       0.855  19.839  16.266  1.00 41.00           C  \nATOM   1054  CD2 LEU A 135       3.008  18.643  16.675  1.00 42.03           C  \nATOM   1055  N   GLU A 136       1.426  14.429  13.589  1.00 46.61           N  \nATOM   1056  CA  GLU A 136       0.823  13.096  13.549  1.00 49.07           C  \nATOM   1057  C   GLU A 136       1.895  12.046  13.850  1.00 50.08           C  \nATOM   1058  O   GLU A 136       1.583  10.895  14.159  1.00 49.45           O  \nATOM   1059  CB  GLU A 136       0.155  12.825  12.191  1.00 55.81           C  \nATOM   1060  CG  GLU A 136      -1.277  13.373  12.115  1.00 58.73           C  \nATOM   1061  CD  GLU A 136      -1.975  13.105  10.790  1.00 59.92           C  \nATOM   1062  OE1 GLU A 136      -2.015  11.937  10.342  1.00 60.19           O  \nATOM   1063  OE2 GLU A 136      -2.500  14.071  10.199  1.00 61.51           O  \nATOM   1064  N   ARG A 137       3.159  12.461  13.772  1.00 47.97           N  \nATOM   1065  CA  ARG A 137       4.286  11.581  14.064  1.00 49.55           C  \nATOM   1066  C   ARG A 137       4.249  11.241  15.554  1.00 50.69           C  \nATOM   1067  O   ARG A 137       3.573  11.916  16.332  1.00 50.52           O  \nATOM   1068  CB  ARG A 137       5.603  12.277  13.733  1.00 54.59           C  \nATOM   1069  CG  ARG A 137       5.722  12.754  12.304  1.00 56.27           C  \nATOM   1070  CD  ARG A 137       6.972  13.615  12.129  1.00 60.34           C  \nATOM   1071  NE  ARG A 137       7.083  14.150  10.775  1.00 63.10           N  \nATOM   1072  CZ  ARG A 137       7.355  13.413   9.701  1.00 64.71           C  \nATOM   1073  NH1 ARG A 137       7.552  12.105   9.822  1.00 64.25           N  \nATOM   1074  NH2 ARG A 137       7.418  13.981   8.505  1.00 65.76           N  \nATOM   1075  N   ASP A 138       4.984  10.208  15.955  1.00 64.98           N  \nATOM   1076  CA  ASP A 138       4.997   9.787  17.354  1.00 66.00           C  \nATOM   1077  C   ASP A 138       6.379   9.772  17.998  1.00 64.86           C  \nATOM   1078  O   ASP A 138       6.532   9.374  19.153  1.00 65.36           O  \nATOM   1079  CB  ASP A 138       4.353   8.402  17.477  1.00 80.99           C  \nATOM   1080  CG  ASP A 138       4.657   7.508  16.287  1.00 83.92           C  \nATOM   1081  OD1 ASP A 138       4.248   7.858  15.158  1.00 85.36           O  \nATOM   1082  OD2 ASP A 138       5.300   6.453  16.473  1.00 85.70           O  \nATOM   1083  N   ASP A 139       7.380  10.225  17.255  1.00 48.84           N  \nATOM   1084  CA  ASP A 139       8.745  10.243  17.754  1.00 48.26           C  \nATOM   1085  C   ASP A 139       9.288  11.659  17.877  1.00 45.70           C  \nATOM   1086  O   ASP A 139      10.486  11.880  17.700  1.00 45.70           O  \nATOM   1087  CB  ASP A 139       9.639   9.441  16.805  1.00 80.91           C  \nATOM   1088  CG  ASP A 139       9.636  10.001  15.391  1.00 83.57           C  \nATOM   1089  OD1 ASP A 139      10.323   9.429  14.517  1.00 85.24           O  \nATOM   1090  OD2 ASP A 139       8.946  11.015  15.151  1.00 84.13           O  \nATOM   1091  N   LEU A 140       8.418  12.616  18.184  1.00 53.08           N  \nATOM   1092  CA  LEU A 140       8.845  14.012  18.278  1.00 49.07           C  \nATOM   1093  C   LEU A 140       9.258  14.452  19.677  1.00 46.69           C  \nATOM   1094  O   LEU A 140      10.087  15.349  19.827  1.00 44.77           O  \nATOM   1095  CB  LEU A 140       7.736  14.934  17.757  1.00 31.54           C  \nATOM   1096  CG  LEU A 140       7.186  14.635  16.351  1.00 31.01           C  \nATOM   1097  CD1 LEU A 140       5.906  15.424  16.153  1.00 31.24           C  \nATOM   1098  CD2 LEU A 140       8.200  14.986  15.269  1.00 30.55           C  \nATOM   1099  N   GLY A 141       8.690  13.817  20.697  1.00 40.23           N  \nATOM   1100  CA  GLY A 141       9.020  14.192  22.057  1.00 38.54           C  \nATOM   1101  C   GLY A 141       8.537  15.602  22.343  1.00 37.22           C  \nATOM   1102  O   GLY A 141       7.398  15.954  22.030  1.00 39.02           O  \nATOM   1103  N   THR A 142       9.395  16.416  22.943  1.00 31.42           N  \nATOM   1104  CA  THR A 142       9.038  17.792  23.252  1.00 29.33           C  \nATOM   1105  C   THR A 142       9.213  18.632  21.990  1.00 29.10           C  \nATOM   1106  O   THR A 142      10.317  18.732  21.450  1.00 27.63           O  \nATOM   1107  CB  THR A 142       9.930  18.322  24.379  1.00 31.79           C  \nATOM   1108  OG1 THR A 142       9.675  17.555  25.555  1.00 32.88           O  \nATOM   1109  CG2 THR A 142       9.649  19.788  24.670  1.00 31.83           C  \nATOM   1110  N   VAL A 143       8.120  19.222  21.515  1.00 26.18           N  \nATOM   1111  CA  VAL A 143       8.172  20.019  20.298  1.00 24.99           C  \nATOM   1112  C   VAL A 143       8.239  21.532  20.512  1.00 23.66           C  \nATOM   1113  O   VAL A 143       7.493  22.101  21.318  1.00 20.95           O  \nATOM   1114  CB  VAL A 143       6.962  19.715  19.389  1.00 31.23           C  \nATOM   1115  CG1 VAL A 143       7.069  20.516  18.098  1.00 31.58           C  \nATOM   1116  CG2 VAL A 143       6.899  18.234  19.082  1.00 31.47           C  \nATOM   1117  N   PHE A 144       9.150  22.170  19.781  1.00 30.38           N  \nATOM   1118  CA  PHE A 144       9.325  23.618  19.823  1.00 30.87           C  \nATOM   1119  C   PHE A 144       9.070  24.165  18.416  1.00 30.79           C  \nATOM   1120  O   PHE A 144       9.600  23.645  17.433  1.00 32.80           O  \nATOM   1121  CB  PHE A 144      10.746  23.976  20.270  1.00 27.59           C  \nATOM   1122  CG  PHE A 144      11.043  25.449  20.228  1.00 26.82           C  \nATOM   1123  CD1 PHE A 144      11.354  26.079  19.026  1.00 27.54           C  \nATOM   1124  CD2 PHE A 144      10.991  26.213  21.389  1.00 28.38           C  \nATOM   1125  CE1 PHE A 144      11.604  27.448  18.985  1.00 27.10           C  \nATOM   1126  CE2 PHE A 144      11.240  27.582  21.356  1.00 26.84           C  \nATOM   1127  CZ  PHE A 144      11.545  28.197  20.157  1.00 25.60           C  \nATOM   1128  N   VAL A 145       8.258  25.207  18.311  1.00 23.73           N  \nATOM   1129  CA  VAL A 145       7.967  25.784  17.009  1.00 22.51           C  \nATOM   1130  C   VAL A 145       8.444  27.224  16.923  1.00 23.25           C  \nATOM   1131  O   VAL A 145       8.113  28.038  17.788  1.00 22.18           O  \nATOM   1132  CB  VAL A 145       6.457  25.785  16.719  1.00 25.19           C  \nATOM   1133  CG1 VAL A 145       6.211  26.235  15.289  1.00 25.03           C  \nATOM   1134  CG2 VAL A 145       5.871  24.406  16.972  1.00 24.76           C  \nATOM   1135  N   ASP A 146       9.248  27.517  15.901  1.00 31.13           N  \nATOM   1136  CA  ASP A 146       9.742  28.871  15.646  1.00 31.93           C  \nATOM   1137  C   ASP A 146       8.900  29.390  14.466  1.00 31.46           C  \nATOM   1138  O   ASP A 146       9.216  29.122  13.307  1.00 32.56           O  \nATOM   1139  CB  ASP A 146      11.235  28.840  15.278  1.00 37.16           C  \nATOM   1140  CG  ASP A 146      11.750  30.187  14.763  1.00 41.60           C  \nATOM   1141  OD1 ASP A 146      12.984  30.341  14.643  1.00 43.25           O  \nATOM   1142  OD2 ASP A 146      10.933  31.089  14.468  1.00 41.27           O  \nATOM   1143  N   PHE A 147       7.826  30.117  14.770  1.00 27.24           N  \nATOM   1144  CA  PHE A 147       6.922  30.633  13.735  1.00 27.49           C  \nATOM   1145  C   PHE A 147       7.407  31.991  13.253  1.00 27.12           C  \nATOM   1146  O   PHE A 147       7.337  32.992  13.969  1.00 28.64           O  \nATOM   1147  CB  PHE A 147       5.490  30.702  14.293  1.00 27.72           C  \nATOM   1148  CG  PHE A 147       4.423  30.946  13.249  1.00 28.69           C  \nATOM   1149  CD1 PHE A 147       4.572  30.475  11.949  1.00 28.47           C  \nATOM   1150  CD2 PHE A 147       3.251  31.626  13.585  1.00 27.63           C  \nATOM   1151  CE1 PHE A 147       3.582  30.682  10.990  1.00 25.94           C  \nATOM   1152  CE2 PHE A 147       2.248  31.840  12.636  1.00 28.49           C  \nATOM   1153  CZ  PHE A 147       2.413  31.362  11.335  1.00 28.93           C  \nATOM   1154  N   HIS A 148       7.924  31.993  12.031  1.00 24.56           N  \nATOM   1155  CA  HIS A 148       8.477  33.173  11.391  1.00 24.95           C  \nATOM   1156  C   HIS A 148       7.367  33.857  10.585  1.00 26.08           C  \nATOM   1157  O   HIS A 148       7.049  33.452   9.452  1.00 25.17           O  \nATOM   1158  CB  HIS A 148       9.635  32.729  10.487  1.00 26.77           C  \nATOM   1159  CG  HIS A 148      10.639  33.804  10.202  1.00 28.03           C  \nATOM   1160  ND1 HIS A 148      10.649  34.523   9.027  1.00 27.34           N  \nATOM   1161  CD2 HIS A 148      11.669  34.281  10.939  1.00 27.14           C  \nATOM   1162  CE1 HIS A 148      11.640  35.393   9.052  1.00 24.24           C  \nATOM   1163  NE2 HIS A 148      12.275  35.267  10.201  1.00 25.99           N  \nATOM   1164  N   ALA A 149       6.783  34.896  11.181  1.00 27.50           N  \nATOM   1165  CA  ALA A 149       5.682  35.622  10.571  1.00 25.87           C  \nATOM   1166  C   ALA A 149       5.640  37.120  10.899  1.00 27.47           C  \nATOM   1167  O   ALA A 149       6.301  37.597  11.823  1.00 26.62           O  \nATOM   1168  CB  ALA A 149       4.390  34.973  10.981  1.00 15.41           C  \nATOM   1169  N   GLU A 150       4.823  37.844  10.139  1.00 26.84           N  \nATOM   1170  CA  GLU A 150       4.672  39.291  10.274  1.00 27.85           C  \nATOM   1171  C   GLU A 150       3.429  39.732  11.062  1.00 27.49           C  \nATOM   1172  O   GLU A 150       3.508  40.583  11.958  1.00 25.45           O  \nATOM   1173  CB  GLU A 150       4.623  39.905   8.874  1.00 34.41           C  \nATOM   1174  CG  GLU A 150       4.449  41.406   8.842  1.00 40.02           C  \nATOM   1175  CD  GLU A 150       5.728  42.145   9.167  1.00 41.81           C  \nATOM   1176  OE1 GLU A 150       6.706  42.006   8.398  1.00 42.05           O  \nATOM   1177  OE2 GLU A 150       5.750  42.862  10.189  1.00 45.39           O  \nATOM   1178  N   ALA A 151       2.291  39.139  10.718  1.00 27.39           N  \nATOM   1179  CA  ALA A 151       1.004  39.465  11.322  1.00 27.46           C  \nATOM   1180  C   ALA A 151       0.767  38.938  12.732  1.00 27.72           C  \nATOM   1181  O   ALA A 151       0.902  37.740  12.976  1.00 29.25           O  \nATOM   1182  CB  ALA A 151      -0.109  38.977  10.403  1.00 22.77           C  \nATOM   1183  N   THR A 152       0.380  39.819  13.655  1.00 30.33           N  \nATOM   1184  CA  THR A 152       0.115  39.372  15.021  1.00 32.70           C  \nATOM   1185  C   THR A 152      -1.105  38.448  15.049  1.00 33.33           C  \nATOM   1186  O   THR A 152      -1.219  37.592  15.924  1.00 33.04           O  \nATOM   1187  CB  THR A 152      -0.127  40.552  16.009  1.00 28.80           C  \nATOM   1188  OG1 THR A 152      -1.304  41.271  15.639  1.00 23.57           O  \nATOM   1189  CG2 THR A 152       1.064  41.504  16.015  1.00 30.63           C  \nATOM   1190  N   SER A 153      -2.010  38.612  14.086  1.00 31.54           N  \nATOM   1191  CA  SER A 153      -3.207  37.767  14.018  1.00 32.24           C  \nATOM   1192  C   SER A 153      -2.850  36.315  13.689  1.00 30.76           C  \nATOM   1193  O   SER A 153      -3.407  35.384  14.262  1.00 28.97           O  \nATOM   1194  CB  SER A 153      -4.176  38.302  12.964  1.00 44.76           C  \nATOM   1195  OG  SER A 153      -3.525  38.466  11.721  1.00 49.19           O  \nATOM   1196  N   GLU A 154      -1.915  36.131  12.765  1.00 28.36           N  \nATOM   1197  CA  GLU A 154      -1.481  34.797  12.358  1.00 28.57           C  \nATOM   1198  C   GLU A 154      -0.646  34.133  13.464  1.00 28.71           C  \nATOM   1199  O   GLU A 154      -0.652  32.907  13.621  1.00 26.84           O  \nATOM   1200  CB  GLU A 154      -0.682  34.912  11.048  1.00 31.98           C  \nATOM   1201  CG  GLU A 154      -0.112  33.610  10.498  1.00 35.47           C  \nATOM   1202  CD  GLU A 154       0.190  33.680   9.000  1.00 35.33           C  \nATOM   1203  OE1 GLU A 154       0.656  34.733   8.513  1.00 35.03           O  \nATOM   1204  OE2 GLU A 154      -0.031  32.668   8.306  1.00 34.44           O  \nATOM   1205  N   LYS A 155       0.066  34.945  14.238  1.00 31.32           N  \nATOM   1206  CA  LYS A 155       0.878  34.405  15.310  1.00 30.68           C  \nATOM   1207  C   LYS A 155       0.016  33.981  16.492  1.00 30.57           C  \nATOM   1208  O   LYS A 155       0.208  32.895  17.037  1.00 30.56           O  \nATOM   1209  CB  LYS A 155       1.929  35.424  15.752  1.00 30.40           C  \nATOM   1210  CG  LYS A 155       3.013  35.667  14.722  1.00 31.65           C  \nATOM   1211  CD  LYS A 155       3.994  36.729  15.184  1.00 32.44           C  \nATOM   1212  CE  LYS A 155       5.296  36.635  14.398  1.00 34.23           C  \nATOM   1213  NZ  LYS A 155       6.315  37.622  14.826  1.00 32.81           N  \nATOM   1214  N   GLU A 156      -0.939  34.818  16.884  1.00 24.89           N  \nATOM   1215  CA  GLU A 156      -1.800  34.468  18.012  1.00 25.19           C  \nATOM   1216  C   GLU A 156      -2.619  33.237  17.682  1.00 23.07           C  \nATOM   1217  O   GLU A 156      -2.867  32.386  18.542  1.00 19.48           O  \nATOM   1218  CB  GLU A 156      -2.726  35.631  18.383  1.00 29.54           C  \nATOM   1219  CG  GLU A 156      -1.973  36.838  18.931  1.00 34.22           C  \nATOM   1220  CD  GLU A 156      -2.639  37.448  20.154  1.00 38.39           C  \nATOM   1221  OE1 GLU A 156      -3.802  37.887  20.044  1.00 41.31           O  \nATOM   1222  OE2 GLU A 156      -1.996  37.488  21.229  1.00 42.86           O  \nATOM   1223  N   ALA A 157      -3.019  33.130  16.419  1.00 27.32           N  \nATOM   1224  CA  ALA A 157      -3.802  31.992  15.986  1.00 26.08           C  \nATOM   1225  C   ALA A 157      -3.000  30.690  16.057  1.00 27.67           C  \nATOM   1226  O   ALA A 157      -3.522  29.672  16.510  1.00 27.83           O  \nATOM   1227  CB  ALA A 157      -4.306  32.218  14.578  1.00 30.16           C  \nATOM   1228  N   MET A 158      -1.738  30.714  15.624  1.00 26.75           N  \nATOM   1229  CA  MET A 158      -0.924  29.499  15.644  1.00 26.35           C  \nATOM   1230  C   MET A 158      -0.710  29.065  17.088  1.00 26.13           C  \nATOM   1231  O   MET A 158      -0.557  27.878  17.379  1.00 28.17           O  \nATOM   1232  CB  MET A 158       0.424  29.729  14.933  1.00 25.26           C  \nATOM   1233  CG  MET A 158       1.327  28.478  14.832  1.00 21.00           C  \nATOM   1234  SD  MET A 158       0.615  27.073  13.904  1.00 13.81           S  \nATOM   1235  CE  MET A 158       0.559  27.715  12.325  1.00 11.61           C  \nATOM   1236  N   GLY A 159      -0.719  30.034  17.994  1.00 26.83           N  \nATOM   1237  CA  GLY A 159      -0.554  29.734  19.401  1.00 27.05           C  \nATOM   1238  C   GLY A 159      -1.799  29.042  19.932  1.00 29.29           C  \nATOM   1239  O   GLY A 159      -1.718  28.103  20.724  1.00 28.67           O  \nATOM   1240  N   TRP A 160      -2.969  29.508  19.510  1.00 36.10           N  \nATOM   1241  CA  TRP A 160      -4.199  28.876  19.962  1.00 36.73           C  \nATOM   1242  C   TRP A 160      -4.358  27.526  19.263  1.00 35.72           C  \nATOM   1243  O   TRP A 160      -4.869  26.582  19.854  1.00 35.25           O  \nATOM   1244  CB  TRP A 160      -5.405  29.790  19.692  1.00 26.35           C  \nATOM   1245  CG  TRP A 160      -5.407  31.025  20.571  1.00 27.15           C  \nATOM   1246  CD1 TRP A 160      -5.443  32.335  20.163  1.00 27.50           C  \nATOM   1247  CD2 TRP A 160      -5.334  31.058  22.007  1.00 26.77           C  \nATOM   1248  NE1 TRP A 160      -5.392  33.176  21.253  1.00 27.11           N  \nATOM   1249  CE2 TRP A 160      -5.328  32.421  22.396  1.00 25.72           C  \nATOM   1250  CE3 TRP A 160      -5.273  30.070  23.001  1.00 26.59           C  \nATOM   1251  CZ2 TRP A 160      -5.263  32.820  23.732  1.00 26.31           C  \nATOM   1252  CZ3 TRP A 160      -5.205  30.471  24.347  1.00 27.81           C  \nATOM   1253  CH2 TRP A 160      -5.202  31.835  24.694  1.00 27.18           C  \nATOM   1254  N   HIS A 161      -3.880  27.431  18.021  1.00 30.81           N  \nATOM   1255  CA  HIS A 161      -3.970  26.199  17.236  1.00 30.05           C  \nATOM   1256  C   HIS A 161      -3.117  25.039  17.767  1.00 31.58           C  \nATOM   1257  O   HIS A 161      -3.478  23.868  17.590  1.00 31.73           O  \nATOM   1258  CB  HIS A 161      -3.578  26.478  15.785  1.00 25.00           C  \nATOM   1259  CG  HIS A 161      -3.635  25.273  14.897  1.00 25.02           C  \nATOM   1260  ND1 HIS A 161      -4.787  24.539  14.706  1.00 26.60           N  \nATOM   1261  CD2 HIS A 161      -2.687  24.684  14.130  1.00 24.34           C  \nATOM   1262  CE1 HIS A 161      -4.546  23.552  13.860  1.00 22.37           C  \nATOM   1263  NE2 HIS A 161      -3.279  23.617  13.495  1.00 21.43           N  \nATOM   1264  N   LEU A 162      -1.997  25.370  18.413  1.00 28.36           N  \nATOM   1265  CA  LEU A 162      -1.073  24.374  18.955  1.00 28.87           C  \nATOM   1266  C   LEU A 162      -1.098  24.255  20.476  1.00 27.41           C  \nATOM   1267  O   LEU A 162      -0.379  23.440  21.043  1.00 29.81           O  \nATOM   1268  CB  LEU A 162       0.368  24.695  18.511  1.00 23.52           C  \nATOM   1269  CG  LEU A 162       0.902  24.229  17.149  1.00 25.12           C  \nATOM   1270  CD1 LEU A 162      -0.142  24.358  16.086  1.00 25.18           C  \nATOM   1271  CD2 LEU A 162       2.140  25.029  16.783  1.00 25.92           C  \nATOM   1272  N   ALA A 163      -1.905  25.059  21.150  1.00 20.93           N  \nATOM   1273  CA  ALA A 163      -1.931  24.980  22.601  1.00 23.86           C  \nATOM   1274  C   ALA A 163      -2.306  23.580  23.060  1.00 25.96           C  \nATOM   1275  O   ALA A 163      -3.325  23.029  22.642  1.00 26.96           O  \nATOM   1276  CB  ALA A 163      -2.898  25.991  23.171  1.00 23.18           C  \nATOM   1277  N   GLY A 164      -1.474  22.997  23.916  1.00 33.89           N  \nATOM   1278  CA  GLY A 164      -1.764  21.666  24.412  1.00 32.75           C  \nATOM   1279  C   GLY A 164      -1.123  20.539  23.624  1.00 34.83           C  \nATOM   1280  O   GLY A 164      -1.243  19.379  24.003  1.00 36.79           O  \nATOM   1281  N   ARG A 165      -0.457  20.845  22.519  1.00 28.77           N  \nATOM   1282  CA  ARG A 165       0.176  19.781  21.761  1.00 29.09           C  \nATOM   1283  C   ARG A 165       1.554  20.129  21.231  1.00 28.06           C  \nATOM   1284  O   ARG A 165       2.082  19.481  20.335  1.00 27.46           O  \nATOM   1285  CB  ARG A 165      -0.736  19.303  20.630  1.00 34.32           C  \nATOM   1286  CG  ARG A 165      -1.423  20.359  19.809  1.00 34.40           C  \nATOM   1287  CD  ARG A 165      -2.310  19.649  18.784  1.00 34.48           C  \nATOM   1288  NE  ARG A 165      -2.949  20.577  17.866  1.00 34.97           N  \nATOM   1289  CZ  ARG A 165      -3.146  20.334  16.573  1.00 36.86           C  \nATOM   1290  NH1 ARG A 165      -2.750  19.183  16.039  1.00 33.97           N  \nATOM   1291  NH2 ARG A 165      -3.726  21.256  15.811  1.00 34.59           N  \nATOM   1292  N   VAL A 166       2.136  21.150  21.829  1.00 36.51           N  \nATOM   1293  CA  VAL A 166       3.459  21.617  21.473  1.00 35.50           C  \nATOM   1294  C   VAL A 166       3.919  22.283  22.760  1.00 35.77           C  \nATOM   1295  O   VAL A 166       3.110  22.859  23.491  1.00 35.26           O  \nATOM   1296  CB  VAL A 166       3.369  22.607  20.281  1.00 28.74           C  \nATOM   1297  CG1 VAL A 166       4.179  23.845  20.539  1.00 29.04           C  \nATOM   1298  CG2 VAL A 166       3.831  21.915  19.003  1.00 28.82           C  \nATOM   1299  N   ALA A 167       5.204  22.180  23.065  1.00 32.54           N  \nATOM   1300  CA  ALA A 167       5.703  22.770  24.288  1.00 32.40           C  \nATOM   1301  C   ALA A 167       5.860  24.285  24.187  1.00 31.45           C  \nATOM   1302  O   ALA A 167       5.745  24.993  25.185  1.00 30.14           O  \nATOM   1303  CB  ALA A 167       7.017  22.130  24.662  1.00 27.24           C  \nATOM   1304  N   ALA A 168       6.108  24.787  22.985  1.00 32.62           N  \nATOM   1305  CA  ALA A 168       6.280  26.220  22.813  1.00 32.35           C  \nATOM   1306  C   ALA A 168       6.081  26.723  21.376  1.00 30.66           C  \nATOM   1307  O   ALA A 168       6.339  26.005  20.405  1.00 31.41           O  \nATOM   1308  CB  ALA A 168       7.668  26.625  23.325  1.00 25.56           C  \nATOM   1309  N   VAL A 169       5.612  27.963  21.261  1.00 23.58           N  \nATOM   1310  CA  VAL A 169       5.385  28.624  19.971  1.00 22.29           C  \nATOM   1311  C   VAL A 169       5.920  30.052  20.120  1.00 23.43           C  \nATOM   1312  O   VAL A 169       5.241  30.928  20.657  1.00 21.10           O  \nATOM   1313  CB  VAL A 169       3.878  28.675  19.606  1.00 20.35           C  \nATOM   1314  CG1 VAL A 169       3.655  29.517  18.347  1.00 17.85           C  \nATOM   1315  CG2 VAL A 169       3.363  27.274  19.379  1.00 19.74           C  \nATOM   1316  N   ILE A 170       7.154  30.261  19.666  1.00 28.77           N  \nATOM   1317  CA  ILE A 170       7.818  31.559  19.746  1.00 27.09           C  \nATOM   1318  C   ILE A 170       7.981  32.151  18.344  1.00 27.46           C  \nATOM   1319  O   ILE A 170       8.616  31.547  17.481  1.00 28.26           O  \nATOM   1320  CB  ILE A 170       9.211  31.415  20.410  1.00 21.97           C  \nATOM   1321  CG1 ILE A 170       9.075  31.271  21.928  1.00 23.47           C  \nATOM   1322  CG2 ILE A 170      10.051  32.656  20.136  1.00 23.06           C  \nATOM   1323  CD1 ILE A 170       8.310  30.076  22.386  1.00 30.94           C  \nATOM   1324  N   GLY A 171       7.421  33.339  18.125  1.00 25.95           N  \nATOM   1325  CA  GLY A 171       7.507  33.959  16.814  1.00 25.13           C  \nATOM   1326  C   GLY A 171       8.749  34.798  16.604  1.00 25.68           C  \nATOM   1327  O   GLY A 171       9.391  35.218  17.565  1.00 25.48           O  \nATOM   1328  N   THR A 172       9.079  35.039  15.340  1.00 26.84           N  \nATOM   1329  CA  THR A 172      10.246  35.833  14.972  1.00 28.32           C  \nATOM   1330  C   THR A 172       9.975  36.611  13.691  1.00 29.74           C  \nATOM   1331  O   THR A 172       8.878  36.538  13.129  1.00 28.54           O  \nATOM   1332  CB  THR A 172      11.486  34.937  14.711  1.00 28.10           C  \nATOM   1333  OG1 THR A 172      11.127  33.857  13.837  1.00 28.51           O  \nATOM   1334  CG2 THR A 172      12.047  34.399  16.009  1.00 24.32           C  \nATOM   1335  N   HIS A 173      10.989  37.349  13.245  1.00 37.69           N  \nATOM   1336  CA  HIS A 173      10.947  38.135  12.012  1.00 38.47           C  \nATOM   1337  C   HIS A 173      10.628  39.621  12.141  1.00 38.78           C  \nATOM   1338  O   HIS A 173      11.002  40.402  11.264  1.00 39.87           O  \nATOM   1339  CB  HIS A 173       9.978  37.498  11.007  1.00 30.61           C  \nATOM   1340  CG  HIS A 173       9.913  38.209   9.689  1.00 35.05           C  \nATOM   1341  ND1 HIS A 173       8.758  38.803   9.218  1.00 34.86           N  \nATOM   1342  CD2 HIS A 173      10.856  38.414   8.736  1.00 36.23           C  \nATOM   1343  CE1 HIS A 173       8.993  39.341   8.034  1.00 34.01           C  \nATOM   1344  NE2 HIS A 173      10.257  39.119   7.718  1.00 34.53           N  \nATOM   1345  N   THR A 174       9.952  40.031  13.212  1.00 26.29           N  \nATOM   1346  CA  THR A 174       9.608  41.448  13.336  1.00 25.51           C  \nATOM   1347  C   THR A 174      10.629  42.291  14.082  1.00 25.02           C  \nATOM   1348  O   THR A 174      10.522  43.513  14.110  1.00 25.95           O  \nATOM   1349  CB  THR A 174       8.215  41.640  13.956  1.00 31.57           C  \nATOM   1350  OG1 THR A 174       8.204  41.151  15.296  1.00 30.96           O  \nATOM   1351  CG2 THR A 174       7.184  40.886  13.144  1.00 30.78           C  \nATOM   1352  N   HIS A 175      11.619  41.626  14.668  1.00 29.03           N  \nATOM   1353  CA  HIS A 175      12.720  42.276  15.373  1.00 29.57           C  \nATOM   1354  C   HIS A 175      12.438  43.023  16.674  1.00 30.48           C  \nATOM   1355  O   HIS A 175      13.367  43.417  17.372  1.00 32.99           O  \nATOM   1356  CB  HIS A 175      13.478  43.188  14.398  1.00 29.19           C  \nATOM   1357  CG  HIS A 175      14.017  42.464  13.200  1.00 30.47           C  \nATOM   1358  ND1 HIS A 175      14.059  41.087  13.118  1.00 31.31           N  \nATOM   1359  CD2 HIS A 175      14.572  42.923  12.053  1.00 31.62           C  \nATOM   1360  CE1 HIS A 175      14.618  40.730  11.974  1.00 31.05           C  \nATOM   1361  NE2 HIS A 175      14.938  41.825  11.310  1.00 31.29           N  \nATOM   1362  N   VAL A 176      11.179  43.249  17.011  1.00 31.54           N  \nATOM   1363  CA  VAL A 176      10.913  43.912  18.275  1.00 29.74           C  \nATOM   1364  C   VAL A 176      10.275  42.889  19.183  1.00 28.00           C  \nATOM   1365  O   VAL A 176       9.250  42.322  18.849  1.00 30.58           O  \nATOM   1366  CB  VAL A 176       9.968  45.116  18.123  1.00 26.11           C  \nATOM   1367  CG1 VAL A 176       9.671  45.704  19.493  1.00 26.45           C  \nATOM   1368  CG2 VAL A 176      10.609  46.178  17.258  1.00 25.07           C  \nATOM   1369  N   PRO A 177      10.882  42.624  20.343  1.00 26.10           N  \nATOM   1370  CA  PRO A 177      10.280  41.633  21.236  1.00 25.47           C  \nATOM   1371  C   PRO A 177       8.924  42.086  21.759  1.00 27.36           C  \nATOM   1372  O   PRO A 177       8.657  43.281  21.897  1.00 27.76           O  \nATOM   1373  CB  PRO A 177      11.316  41.498  22.353  1.00 19.38           C  \nATOM   1374  CG  PRO A 177      11.894  42.892  22.437  1.00 20.32           C  \nATOM   1375  CD  PRO A 177      12.055  43.266  20.970  1.00 20.39           C  \nATOM   1376  N   THR A 178       8.062  41.124  22.044  1.00 25.86           N  \nATOM   1377  CA  THR A 178       6.752  41.442  22.568  1.00 27.20           C  \nATOM   1378  C   THR A 178       6.733  41.099  24.050  1.00 28.31           C  \nATOM   1379  O   THR A 178       7.503  40.251  24.510  1.00 27.91           O  \nATOM   1380  CB  THR A 178       5.673  40.668  21.812  1.00 36.04           C  \nATOM   1381  OG1 THR A 178       5.925  39.262  21.914  1.00 36.18           O  \nATOM   1382  CG2 THR A 178       5.697  41.076  20.344  1.00 35.99           C  \nATOM   1383  N   ALA A 179       5.858  41.770  24.787  1.00 29.83           N  \nATOM   1384  CA  ALA A 179       5.749  41.580  26.223  1.00 31.32           C  \nATOM   1385  C   ALA A 179       4.625  40.651  26.648  1.00 31.98           C  \nATOM   1386  O   ALA A 179       4.274  40.598  27.828  1.00 35.25           O  \nATOM   1387  CB  ALA A 179       5.576  42.941  26.899  1.00 18.22           C  \nATOM   1388  N   ASP A 180       4.066  39.910  25.699  1.00 32.99           N  \nATOM   1389  CA  ASP A 180       2.970  39.009  26.014  1.00 31.35           C  \nATOM   1390  C   ASP A 180       3.391  37.594  26.416  1.00 30.22           C  \nATOM   1391  O   ASP A 180       2.560  36.698  26.462  1.00 31.20           O  \nATOM   1392  CB  ASP A 180       1.983  38.953  24.838  1.00 36.18           C  \nATOM   1393  CG  ASP A 180       2.627  38.478  23.549  1.00 38.58           C  \nATOM   1394  OD1 ASP A 180       1.930  38.457  22.514  1.00 40.33           O  \nATOM   1395  OD2 ASP A 180       3.823  38.122  23.563  1.00 39.61           O  \nATOM   1396  N   THR A 181       4.672  37.391  26.707  1.00 32.55           N  \nATOM   1397  CA  THR A 181       5.166  36.075  27.126  1.00 31.84           C  \nATOM   1398  C   THR A 181       4.142  35.426  28.063  1.00 32.73           C  \nATOM   1399  O   THR A 181       3.824  35.974  29.119  1.00 32.18           O  \nATOM   1400  CB  THR A 181       6.498  36.214  27.878  1.00 25.79           C  \nATOM   1401  OG1 THR A 181       7.356  37.091  27.142  1.00 24.70           O  \nATOM   1402  CG2 THR A 181       7.183  34.860  28.034  1.00 24.77           C  \nATOM   1403  N   ARG A 182       3.632  34.264  27.674  1.00 30.41           N  \nATOM   1404  CA  ARG A 182       2.629  33.563  28.473  1.00 31.81           C  \nATOM   1405  C   ARG A 182       2.688  32.063  28.220  1.00 30.10           C  \nATOM   1406  O   ARG A 182       3.314  31.602  27.265  1.00 30.00           O  \nATOM   1407  CB  ARG A 182       1.222  34.069  28.106  1.00 42.75           C  \nATOM   1408  CG  ARG A 182       0.105  33.654  29.082  1.00 50.72           C  \nATOM   1409  CD  ARG A 182      -0.942  32.702  28.458  1.00 53.05           C  \nATOM   1410  NE  ARG A 182      -2.206  33.346  28.081  1.00 52.23           N  \nATOM   1411  CZ  ARG A 182      -2.316  34.326  27.191  1.00 52.50           C  \nATOM   1412  NH1 ARG A 182      -1.232  34.791  26.582  1.00 53.06           N  \nATOM   1413  NH2 ARG A 182      -3.509  34.824  26.888  1.00 48.28           N  \nATOM   1414  N   ILE A 183       2.047  31.303  29.095  1.00 28.38           N  \nATOM   1415  CA  ILE A 183       1.964  29.863  28.923  1.00 28.28           C  \nATOM   1416  C   ILE A 183       0.483  29.597  28.732  1.00 28.04           C  \nATOM   1417  O   ILE A 183      -0.295  29.722  29.669  1.00 28.79           O  \nATOM   1418  CB  ILE A 183       2.467  29.090  30.151  1.00 26.29           C  \nATOM   1419  CG1 ILE A 183       3.979  29.286  30.292  1.00 22.97           C  \nATOM   1420  CG2 ILE A 183       2.126  27.593  30.005  1.00 21.69           C  \nATOM   1421  CD1 ILE A 183       4.589  28.464  31.378  1.00 23.59           C  \nATOM   1422  N   LEU A 184       0.103  29.264  27.503  1.00 28.04           N  \nATOM   1423  CA  LEU A 184      -1.286  28.988  27.158  1.00 29.33           C  \nATOM   1424  C   LEU A 184      -1.762  27.688  27.783  1.00 29.86           C  \nATOM   1425  O   LEU A 184      -0.990  26.734  27.910  1.00 27.37           O  \nATOM   1426  CB  LEU A 184      -1.440  28.899  25.638  1.00 24.80           C  \nATOM   1427  CG  LEU A 184      -0.827  30.062  24.841  1.00 27.15           C  \nATOM   1428  CD1 LEU A 184      -0.856  29.729  23.356  1.00 26.32           C  \nATOM   1429  CD2 LEU A 184      -1.581  31.363  25.138  1.00 22.29           C  \nATOM   1430  N   LYS A 185      -3.035  27.658  28.171  1.00 42.85           N  \nATOM   1431  CA  LYS A 185      -3.606  26.458  28.769  1.00 45.61           C  \nATOM   1432  C   LYS A 185      -3.153  25.258  27.939  1.00 44.55           C  \nATOM   1433  O   LYS A 185      -3.078  25.329  26.711  1.00 45.20           O  \nATOM   1434  CB  LYS A 185      -5.135  26.532  28.789  1.00 63.35           C  \nATOM   1435  CG  LYS A 185      -5.781  25.405  29.587  1.00 69.16           C  \nATOM   1436  CD  LYS A 185      -7.302  25.495  29.595  1.00 73.85           C  \nATOM   1437  CE  LYS A 185      -7.884  25.205  28.223  1.00 76.19           C  \nATOM   1438  NZ  LYS A 185      -9.368  25.285  28.227  1.00 79.22           N  \nATOM   1439  N   GLY A 186      -2.845  24.158  28.608  1.00 36.97           N  \nATOM   1440  CA  GLY A 186      -2.382  22.992  27.890  1.00 36.31           C  \nATOM   1441  C   GLY A 186      -0.869  22.909  27.953  1.00 35.57           C  \nATOM   1442  O   GLY A 186      -0.271  22.003  27.381  1.00 36.66           O  \nATOM   1443  N   GLY A 187      -0.250  23.869  28.635  1.00 30.98           N  \nATOM   1444  CA  GLY A 187       1.200  23.874  28.793  1.00 30.02           C  \nATOM   1445  C   GLY A 187       2.040  24.296  27.598  1.00 29.95           C  \nATOM   1446  O   GLY A 187       3.091  23.711  27.334  1.00 29.43           O  \nATOM   1447  N   THR A 188       1.591  25.320  26.882  1.00 33.49           N  \nATOM   1448  CA  THR A 188       2.320  25.802  25.720  1.00 31.79           C  \nATOM   1449  C   THR A 188       2.798  27.236  25.911  1.00 31.98           C  \nATOM   1450  O   THR A 188       1.992  28.162  25.994  1.00 32.09           O  \nATOM   1451  CB  THR A 188       1.439  25.724  24.463  1.00 24.88           C  \nATOM   1452  OG1 THR A 188       0.985  24.371  24.289  1.00 24.05           O  \nATOM   1453  CG2 THR A 188       2.228  26.150  23.228  1.00 21.98           C  \nATOM   1454  N   ALA A 189       4.113  27.417  25.994  1.00 27.09           N  \nATOM   1455  CA  ALA A 189       4.670  28.752  26.165  1.00 26.63           C  \nATOM   1456  C   ALA A 189       4.415  29.504  24.877  1.00 26.58           C  \nATOM   1457  O   ALA A 189       4.369  28.916  23.800  1.00 25.61           O  \nATOM   1458  CB  ALA A 189       6.145  28.683  26.434  1.00 25.70           C  \nATOM   1459  N   TYR A 190       4.244  30.811  24.980  1.00 26.35           N  \nATOM   1460  CA  TYR A 190       3.981  31.582  23.790  1.00 25.74           C  \nATOM   1461  C   TYR A 190       4.466  33.005  23.926  1.00 26.62           C  \nATOM   1462  O   TYR A 190       4.433  33.582  25.015  1.00 26.66           O  \nATOM   1463  CB  TYR A 190       2.481  31.570  23.490  1.00 24.85           C  \nATOM   1464  CG  TYR A 190       2.098  32.446  22.325  1.00 25.16           C  \nATOM   1465  CD1 TYR A 190       1.648  33.749  22.531  1.00 27.78           C  \nATOM   1466  CD2 TYR A 190       2.268  32.007  21.014  1.00 24.23           C  \nATOM   1467  CE1 TYR A 190       1.382  34.606  21.454  1.00 26.68           C  \nATOM   1468  CE2 TYR A 190       2.012  32.857  19.927  1.00 26.28           C  \nATOM   1469  CZ  TYR A 190       1.571  34.154  20.160  1.00 25.70           C  \nATOM   1470  OH  TYR A 190       1.346  35.009  19.111  1.00 24.38           O  \nATOM   1471  N   GLN A 191       4.936  33.549  22.808  1.00 27.77           N  \nATOM   1472  CA  GLN A 191       5.394  34.930  22.724  1.00 28.94           C  \nATOM   1473  C   GLN A 191       5.315  35.299  21.241  1.00 27.70           C  \nATOM   1474  O   GLN A 191       5.959  34.660  20.405  1.00 27.25           O  \nATOM   1475  CB  GLN A 191       6.824  35.081  23.268  1.00 26.56           C  \nATOM   1476  CG  GLN A 191       7.196  36.550  23.506  1.00 28.67           C  \nATOM   1477  CD  GLN A 191       8.509  36.756  24.232  1.00 27.86           C  \nATOM   1478  OE1 GLN A 191       8.712  36.240  25.333  1.00 28.74           O  \nATOM   1479  NE2 GLN A 191       9.408  37.524  23.625  1.00 27.25           N  \nATOM   1480  N   THR A 192       4.501  36.303  20.911  1.00 23.07           N  \nATOM   1481  CA  THR A 192       4.333  36.714  19.516  1.00 24.97           C  \nATOM   1482  C   THR A 192       5.649  36.998  18.785  1.00 25.45           C  \nATOM   1483  O   THR A 192       5.740  36.760  17.579  1.00 27.24           O  \nATOM   1484  CB  THR A 192       3.372  37.931  19.367  1.00 35.83           C  \nATOM   1485  OG1 THR A 192       3.877  39.044  20.101  1.00 44.38           O  \nATOM   1486  CG2 THR A 192       2.006  37.599  19.903  1.00 32.81           C  \nATOM   1487  N   ASP A 193       6.661  37.513  19.483  1.00 29.24           N  \nATOM   1488  CA  ASP A 193       7.969  37.722  18.846  1.00 30.11           C  \nATOM   1489  C   ASP A 193       9.116  37.784  19.834  1.00 29.57           C  \nATOM   1490  O   ASP A 193       9.141  38.638  20.713  1.00 29.30           O  \nATOM   1491  CB  ASP A 193       8.003  38.986  17.986  1.00 32.83           C  \nATOM   1492  CG  ASP A 193       9.064  38.903  16.891  1.00 34.92           C  \nATOM   1493  OD1 ASP A 193       8.679  38.695  15.720  1.00 39.37           O  \nATOM   1494  OD2 ASP A 193      10.277  39.020  17.189  1.00 31.58           O  \nATOM   1495  N   ALA A 194      10.075  36.879  19.669  1.00 30.96           N  \nATOM   1496  CA  ALA A 194      11.249  36.808  20.542  1.00 30.08           C  \nATOM   1497  C   ALA A 194      12.067  38.091  20.568  1.00 29.98           C  \nATOM   1498  O   ALA A 194      12.683  38.426  21.586  1.00 29.27           O  \nATOM   1499  CB  ALA A 194      12.152  35.652  20.103  1.00 20.46           C  \nATOM   1500  N   GLY A 195      12.057  38.811  19.449  1.00 28.71           N  \nATOM   1501  CA  GLY A 195      12.857  40.018  19.324  1.00 26.67           C  \nATOM   1502  C   GLY A 195      14.090  39.561  18.559  1.00 26.57           C  \nATOM   1503  O   GLY A 195      14.287  38.356  18.371  1.00 25.90           O  \nATOM   1504  N   PHE A 196      14.934  40.473  18.107  1.00 25.67           N  \nATOM   1505  CA  PHE A 196      16.093  40.007  17.374  1.00 24.98           C  \nATOM   1506  C   PHE A 196      17.403  40.233  18.116  1.00 25.33           C  \nATOM   1507  O   PHE A 196      17.427  40.794  19.219  1.00 22.69           O  \nATOM   1508  CB  PHE A 196      16.110  40.632  15.965  1.00 25.00           C  \nATOM   1509  CG  PHE A 196      16.803  41.976  15.859  1.00 23.28           C  \nATOM   1510  CD1 PHE A 196      16.870  42.856  16.937  1.00 22.85           C  \nATOM   1511  CD2 PHE A 196      17.356  42.374  14.639  1.00 22.20           C  \nATOM   1512  CE1 PHE A 196      17.465  44.118  16.796  1.00 24.03           C  \nATOM   1513  CE2 PHE A 196      17.948  43.622  14.487  1.00 20.53           C  \nATOM   1514  CZ  PHE A 196      18.006  44.496  15.571  1.00 22.74           C  \nATOM   1515  N   THR A 197      18.483  39.752  17.513  1.00 23.79           N  \nATOM   1516  CA  THR A 197      19.818  39.884  18.069  1.00 23.32           C  \nATOM   1517  C   THR A 197      20.631  40.621  17.027  1.00 24.55           C  \nATOM   1518  O   THR A 197      21.102  40.040  16.051  1.00 23.96           O  \nATOM   1519  CB  THR A 197      20.447  38.518  18.350  1.00 19.15           C  \nATOM   1520  OG1 THR A 197      19.686  37.855  19.362  1.00 20.37           O  \nATOM   1521  CG2 THR A 197      21.875  38.675  18.829  1.00 19.65           C  \nATOM   1522  N   GLY A 198      20.765  41.921  17.247  1.00 23.35           N  \nATOM   1523  CA  GLY A 198      21.482  42.766  16.330  1.00 23.32           C  \nATOM   1524  C   GLY A 198      21.546  44.165  16.893  1.00 23.53           C  \nATOM   1525  O   GLY A 198      21.212  44.386  18.052  1.00 24.62           O  \nATOM   1526  N   PRO A 199      21.961  45.136  16.075  1.00 24.30           N  \nATOM   1527  CA  PRO A 199      22.110  46.556  16.403  1.00 24.30           C  \nATOM   1528  C   PRO A 199      20.922  47.175  17.134  1.00 24.59           C  \nATOM   1529  O   PRO A 199      19.803  47.139  16.646  1.00 24.60           O  \nATOM   1530  CB  PRO A 199      22.358  47.194  15.034  1.00 19.46           C  \nATOM   1531  CG  PRO A 199      21.640  46.285  14.108  1.00 20.87           C  \nATOM   1532  CD  PRO A 199      22.026  44.935  14.617  1.00 18.40           C  \nATOM   1533  N   HIS A 200      21.207  47.770  18.287  1.00 32.13           N  \nATOM   1534  CA  HIS A 200      20.228  48.415  19.164  1.00 33.53           C  \nATOM   1535  C   HIS A 200      19.646  49.720  18.633  1.00 34.84           C  \nATOM   1536  O   HIS A 200      18.470  50.005  18.839  1.00 35.03           O  \nATOM   1537  CB  HIS A 200      20.869  48.675  20.530  1.00 29.39           C  \nATOM   1538  CG  HIS A 200      19.905  49.136  21.579  1.00 30.03           C  \nATOM   1539  ND1 HIS A 200      18.688  48.521  21.792  1.00 29.53           N  \nATOM   1540  CD2 HIS A 200      20.004  50.109  22.516  1.00 30.25           C  \nATOM   1541  CE1 HIS A 200      18.082  49.092  22.818  1.00 30.77           C  \nATOM   1542  NE2 HIS A 200      18.860  50.058  23.276  1.00 31.96           N  \nATOM   1543  N   ASP A 201      20.475  50.526  17.978  1.00 34.98           N  \nATOM   1544  CA  ASP A 201      20.005  51.787  17.417  1.00 36.66           C  \nATOM   1545  C   ASP A 201      19.950  51.676  15.897  1.00 36.90           C  \nATOM   1546  O   ASP A 201      20.966  51.807  15.212  1.00 36.26           O  \nATOM   1547  CB  ASP A 201      20.918  52.943  17.840  1.00 45.39           C  \nATOM   1548  CG  ASP A 201      20.894  53.178  19.338  1.00 47.73           C  \nATOM   1549  OD1 ASP A 201      19.836  52.940  19.949  1.00 51.28           O  \nATOM   1550  OD2 ASP A 201      21.920  53.611  19.907  1.00 52.02           O  \nATOM   1551  N   SER A 202      18.752  51.431  15.375  1.00 31.40           N  \nATOM   1552  CA  SER A 202      18.582  51.272  13.942  1.00 29.94           C  \nATOM   1553  C   SER A 202      17.116  51.229  13.527  1.00 29.29           C  \nATOM   1554  O   SER A 202      16.203  51.489  14.320  1.00 29.11           O  \nATOM   1555  CB  SER A 202      19.227  49.958  13.506  1.00 34.68           C  \nATOM   1556  OG  SER A 202      18.519  48.852  14.048  1.00 27.77           O  \nATOM   1557  N   ILE A 203      16.905  50.902  12.260  1.00 31.45           N  \nATOM   1558  CA  ILE A 203      15.557  50.742  11.747  1.00 33.00           C  \nATOM   1559  C   ILE A 203      15.462  49.301  11.277  1.00 33.38           C  \nATOM   1560  O   ILE A 203      15.899  48.972  10.180  1.00 32.75           O  \nATOM   1561  CB  ILE A 203      15.249  51.680  10.574  1.00 25.40           C  \nATOM   1562  CG1 ILE A 203      15.155  53.122  11.074  1.00 23.69           C  \nATOM   1563  CG2 ILE A 203      13.951  51.278   9.945  1.00 25.62           C  \nATOM   1564  CD1 ILE A 203      14.827  54.114   9.999  1.00 24.49           C  \nATOM   1565  N   ILE A 204      14.931  48.449  12.148  1.00 46.86           N  \nATOM   1566  CA  ILE A 204      14.752  47.031  11.866  1.00 49.01           C  \nATOM   1567  C   ILE A 204      16.078  46.281  11.673  1.00 48.80           C  \nATOM   1568  O   ILE A 204      16.093  45.112  11.292  1.00 50.25           O  \nATOM   1569  CB  ILE A 204      13.812  46.840  10.626  1.00 54.02           C  \nATOM   1570  CG1 ILE A 204      12.846  45.686  10.893  1.00 55.03           C  \nATOM   1571  CG2 ILE A 204      14.623  46.574   9.349  1.00 56.63           C  \nATOM   1572  CD1 ILE A 204      11.890  45.413   9.757  1.00 57.46           C  \nATOM   1573  N   GLY A 205      17.189  46.955  11.957  1.00 35.36           N  \nATOM   1574  CA  GLY A 205      18.493  46.329  11.816  1.00 31.76           C  \nATOM   1575  C   GLY A 205      19.373  47.075  10.833  1.00 31.46           C  \nATOM   1576  O   GLY A 205      20.580  46.865  10.778  1.00 29.67           O  \nATOM   1577  N   SER A 206      18.756  47.951  10.049  1.00 26.23           N  \nATOM   1578  CA  SER A 206      19.478  48.741   9.063  1.00 27.53           C  \nATOM   1579  C   SER A 206      19.919  50.068   9.666  1.00 27.32           C  \nATOM   1580  O   SER A 206      19.389  50.497  10.693  1.00 25.41           O  \nATOM   1581  CB  SER A 206      18.574  49.024   7.859  1.00 39.28           C  \nATOM   1582  OG  SER A 206      18.112  47.820   7.272  1.00 44.91           O  \nATOM   1583  N   ALA A 207      20.896  50.709   9.029  1.00 36.59           N  \nATOM   1584  CA  ALA A 207      21.359  52.009   9.487  1.00 39.20           C  \nATOM   1585  C   ALA A 207      20.161  52.926   9.296  1.00 40.61           C  \nATOM   1586  O   ALA A 207      19.400  52.774   8.335  1.00 40.68           O  \nATOM   1587  CB  ALA A 207      22.527  52.486   8.649  1.00 29.15           C  \nATOM   1588  N   ILE A 208      20.001  53.881  10.202  1.00 42.88           N  \nATOM   1589  CA  ILE A 208      18.871  54.795  10.170  1.00 43.69           C  \nATOM   1590  C   ILE A 208      18.691  55.680   8.938  1.00 44.41           C  \nATOM   1591  O   ILE A 208      17.576  55.814   8.447  1.00 45.24           O  \nATOM   1592  CB  ILE A 208      18.877  55.673  11.441  1.00 34.81           C  \nATOM   1593  CG1 ILE A 208      18.815  54.765  12.678  1.00 33.99           C  \nATOM   1594  CG2 ILE A 208      17.701  56.654  11.423  1.00 32.56           C  \nATOM   1595  CD1 ILE A 208      18.995  55.488  14.002  1.00 33.61           C  \nATOM   1596  N   GLU A 209      19.763  56.267   8.419  1.00 44.62           N  \nATOM   1597  CA  GLU A 209      19.625  57.162   7.267  1.00 45.78           C  \nATOM   1598  C   GLU A 209      18.958  56.589   6.025  1.00 43.61           C  \nATOM   1599  O   GLU A 209      18.017  57.179   5.506  1.00 43.73           O  \nATOM   1600  CB  GLU A 209      20.974  57.750   6.861  1.00 71.48           C  \nATOM   1601  CG  GLU A 209      20.893  58.548   5.562  1.00 77.58           C  \nATOM   1602  CD  GLU A 209      22.226  59.127   5.135  1.00 81.92           C  \nATOM   1603  OE1 GLU A 209      22.664  60.129   5.743  1.00 83.20           O  \nATOM   1604  OE2 GLU A 209      22.837  58.574   4.193  1.00 84.38           O  \nATOM   1605  N   GLY A 210      19.460  55.460   5.535  1.00 43.50           N  \nATOM   1606  CA  GLY A 210      18.891  54.843   4.346  1.00 40.50           C  \nATOM   1607  C   GLY A 210      17.374  54.736   4.369  1.00 39.54           C  \nATOM   1608  O   GLY A 210      16.699  55.401   3.585  1.00 38.96           O  \nATOM   1609  N   PRO A 211      16.805  53.901   5.254  1.00 37.92           N  \nATOM   1610  CA  PRO A 211      15.355  53.717   5.368  1.00 35.60           C  \nATOM   1611  C   PRO A 211      14.581  55.008   5.619  1.00 36.05           C  \nATOM   1612  O   PRO A 211      13.491  55.198   5.072  1.00 34.05           O  \nATOM   1613  CB  PRO A 211      15.229  52.740   6.525  1.00 33.57           C  \nATOM   1614  CG  PRO A 211      16.441  51.884   6.356  1.00 35.06           C  \nATOM   1615  CD  PRO A 211      17.515  52.925   6.099  1.00 34.40           C  \nATOM   1616  N   LEU A 212      15.138  55.897   6.436  1.00 31.69           N  \nATOM   1617  CA  LEU A 212      14.453  57.154   6.728  1.00 32.71           C  \nATOM   1618  C   LEU A 212      14.251  57.971   5.462  1.00 32.22           C  \nATOM   1619  O   LEU A 212      13.155  58.469   5.206  1.00 31.71           O  \nATOM   1620  CB  LEU A 212      15.231  57.986   7.751  1.00 43.73           C  \nATOM   1621  CG  LEU A 212      14.378  58.451   8.937  1.00 46.30           C  \nATOM   1622  CD1 LEU A 212      15.214  59.325   9.864  1.00 44.18           C  \nATOM   1623  CD2 LEU A 212      13.152  59.211   8.439  1.00 45.82           C  \nATOM   1624  N   GLN A 213      15.311  58.103   4.672  1.00 32.05           N  \nATOM   1625  CA  GLN A 213      15.243  58.846   3.433  1.00 32.13           C  \nATOM   1626  C   GLN A 213      14.274  58.179   2.453  1.00 32.49           C  \nATOM   1627  O   GLN A 213      13.630  58.861   1.648  1.00 30.70           O  \nATOM   1628  CB  GLN A 213      16.637  58.963   2.810  1.00 39.02           C  \nATOM   1629  CG  GLN A 213      17.567  59.893   3.567  1.00 39.22           C  \nATOM   1630  CD  GLN A 213      16.924  61.244   3.861  1.00 43.42           C  \nATOM   1631  OE1 GLN A 213      16.573  62.008   2.943  1.00 43.22           O  \nATOM   1632  NE2 GLN A 213      16.760  61.546   5.149  1.00 41.63           N  \nATOM   1633  N   ARG A 214      14.181  56.851   2.514  1.00 29.55           N  \nATOM   1634  CA  ARG A 214      13.264  56.121   1.647  1.00 29.10           C  \nATOM   1635  C   ARG A 214      11.859  56.520   2.046  1.00 28.00           C  \nATOM   1636  O   ARG A 214      11.032  56.853   1.202  1.00 28.03           O  \nATOM   1637  CB  ARG A 214      13.397  54.615   1.842  1.00 46.02           C  \nATOM   1638  CG  ARG A 214      14.141  53.882   0.747  1.00 53.34           C  \nATOM   1639  CD  ARG A 214      13.248  52.858   0.061  1.00 54.32           C  \nATOM   1640  NE  ARG A 214      12.284  53.482  -0.837  1.00 56.04           N  \nATOM   1641  CZ  ARG A 214      11.493  52.813  -1.669  1.00 56.07           C  \nATOM   1642  NH1 ARG A 214      11.539  51.490  -1.722  1.00 54.49           N  \nATOM   1643  NH2 ARG A 214      10.674  53.470  -2.471  1.00 56.30           N  \nATOM   1644  N   PHE A 215      11.594  56.478   3.345  1.00 27.95           N  \nATOM   1645  CA  PHE A 215      10.284  56.813   3.856  1.00 29.86           C  \nATOM   1646  C   PHE A 215       9.850  58.234   3.556  1.00 31.17           C  \nATOM   1647  O   PHE A 215       8.677  58.464   3.250  1.00 32.58           O  \nATOM   1648  CB  PHE A 215      10.221  56.548   5.359  1.00 29.66           C  \nATOM   1649  CG  PHE A 215       9.950  55.109   5.707  1.00 30.33           C  \nATOM   1650  CD1 PHE A 215       8.879  54.435   5.131  1.00 30.65           C  \nATOM   1651  CD2 PHE A 215      10.746  54.434   6.627  1.00 30.03           C  \nATOM   1652  CE1 PHE A 215       8.596  53.111   5.474  1.00 30.74           C  \nATOM   1653  CE2 PHE A 215      10.472  53.108   6.978  1.00 30.21           C  \nATOM   1654  CZ  PHE A 215       9.396  52.444   6.399  1.00 28.91           C  \nATOM   1655  N   LEU A 216      10.791  59.176   3.625  1.00 34.47           N  \nATOM   1656  CA  LEU A 216      10.511  60.591   3.368  1.00 34.33           C  \nATOM   1657  C   LEU A 216      10.407  60.983   1.886  1.00 34.60           C  \nATOM   1658  O   LEU A 216       9.482  61.691   1.493  1.00 34.31           O  \nATOM   1659  CB  LEU A 216      11.581  61.472   4.025  1.00 43.47           C  \nATOM   1660  CG  LEU A 216      11.504  61.805   5.519  1.00 46.49           C  \nATOM   1661  CD1 LEU A 216      12.844  62.345   5.982  1.00 45.53           C  \nATOM   1662  CD2 LEU A 216      10.398  62.820   5.770  1.00 45.80           C  \nATOM   1663  N   THR A 217      11.351  60.534   1.067  1.00 25.29           N  \nATOM   1664  CA  THR A 217      11.355  60.888  -0.355  1.00 27.21           C  \nATOM   1665  C   THR A 217      10.658  59.893  -1.280  1.00 27.04           C  \nATOM   1666  O   THR A 217      10.467  60.165  -2.469  1.00 25.90           O  \nATOM   1667  CB  THR A 217      12.797  61.065  -0.863  1.00 34.73           C  \nATOM   1668  OG1 THR A 217      13.442  59.786  -0.908  1.00 34.19           O  \nATOM   1669  CG2 THR A 217      13.581  61.987   0.069  1.00 34.39           C  \nATOM   1670  N   GLU A 218      10.301  58.731  -0.746  1.00 33.72           N  \nATOM   1671  CA  GLU A 218       9.637  57.707  -1.542  1.00 36.24           C  \nATOM   1672  C   GLU A 218      10.487  57.313  -2.766  1.00 36.95           C  \nATOM   1673  O   GLU A 218       9.976  56.836  -3.773  1.00 37.54           O  \nATOM   1674  CB  GLU A 218       8.248  58.219  -1.957  1.00 41.56           C  \nATOM   1675  CG  GLU A 218       7.517  58.892  -0.789  1.00 43.50           C  \nATOM   1676  CD  GLU A 218       6.208  59.568  -1.173  1.00 44.88           C  \nATOM   1677  OE1 GLU A 218       6.181  60.336  -2.163  1.00 44.80           O  \nATOM   1678  OE2 GLU A 218       5.208  59.342  -0.461  1.00 45.79           O  \nATOM   1679  N   ARG A 219      11.792  57.536  -2.666  1.00 43.70           N  \nATOM   1680  CA  ARG A 219      12.738  57.171  -3.720  1.00 44.68           C  \nATOM   1681  C   ARG A 219      13.754  56.217  -3.089  1.00 45.28           C  \nATOM   1682  O   ARG A 219      14.193  56.424  -1.955  1.00 45.56           O  \nATOM   1683  CB  ARG A 219      13.477  58.398  -4.267  1.00 42.01           C  \nATOM   1684  CG  ARG A 219      12.622  59.381  -5.039  1.00 44.83           C  \nATOM   1685  CD  ARG A 219      13.497  60.317  -5.865  1.00 45.97           C  \nATOM   1686  NE  ARG A 219      14.461  61.063  -5.055  1.00 47.94           N  \nATOM   1687  CZ  ARG A 219      14.126  61.997  -4.168  1.00 49.64           C  \nATOM   1688  NH1 ARG A 219      12.844  62.305  -3.973  1.00 49.03           N  \nATOM   1689  NH2 ARG A 219      15.074  62.624  -3.477  1.00 47.30           N  \nATOM   1690  N   PRO A 220      14.146  55.164  -3.815  1.00 41.31           N  \nATOM   1691  CA  PRO A 220      15.112  54.184  -3.306  1.00 42.99           C  \nATOM   1692  C   PRO A 220      16.465  54.722  -2.825  1.00 44.07           C  \nATOM   1693  O   PRO A 220      17.064  55.598  -3.448  1.00 44.70           O  \nATOM   1694  CB  PRO A 220      15.250  53.201  -4.471  1.00 51.62           C  \nATOM   1695  CG  PRO A 220      14.954  54.044  -5.671  1.00 51.58           C  \nATOM   1696  CD  PRO A 220      13.779  54.864  -5.209  1.00 50.68           C  \nATOM   1697  N   HIS A 221      16.928  54.193  -1.696  1.00 38.63           N  \nATOM   1698  CA  HIS A 221      18.210  54.575  -1.126  1.00 40.30           C  \nATOM   1699  C   HIS A 221      18.993  53.334  -0.702  1.00 41.79           C  \nATOM   1700  O   HIS A 221      18.416  52.344  -0.251  1.00 40.29           O  \nATOM   1701  CB  HIS A 221      18.023  55.504   0.076  1.00 47.08           C  \nATOM   1702  CG  HIS A 221      17.830  56.941  -0.297  1.00 47.69           C  \nATOM   1703  ND1 HIS A 221      16.701  57.401  -0.940  1.00 47.21           N  \nATOM   1704  CD2 HIS A 221      18.634  58.019  -0.133  1.00 47.65           C  \nATOM   1705  CE1 HIS A 221      16.817  58.700  -1.155  1.00 47.27           C  \nATOM   1706  NE2 HIS A 221      17.981  59.099  -0.674  1.00 47.09           N  \nATOM   1707  N   ARG A 222      20.311  53.395  -0.861  1.00 44.46           N  \nATOM   1708  CA  ARG A 222      21.178  52.286  -0.493  1.00 46.88           C  \nATOM   1709  C   ARG A 222      21.342  52.284   1.012  1.00 46.73           C  \nATOM   1710  O   ARG A 222      21.764  53.288   1.589  1.00 46.73           O  \nATOM   1711  CB  ARG A 222      22.551  52.453  -1.145  1.00 94.50           C  \nATOM   1712  CG  ARG A 222      23.553  51.356  -0.815  1.00 99.59           C  \nATOM   1713  CD  ARG A 222      24.907  51.649  -1.447  1.00103.54           C  \nATOM   1714  NE  ARG A 222      24.782  52.026  -2.855  1.00107.23           N  \nATOM   1715  CZ  ARG A 222      25.809  52.236  -3.675  1.00109.24           C  \nATOM   1716  NH1 ARG A 222      27.054  52.105  -3.237  1.00109.29           N  \nATOM   1717  NH2 ARG A 222      25.589  52.584  -4.936  1.00110.37           N  \nATOM   1718  N   TYR A 223      20.997  51.172   1.654  1.00 53.90           N  \nATOM   1719  CA  TYR A 223      21.149  51.073   3.099  1.00 54.26           C  \nATOM   1720  C   TYR A 223      21.786  49.760   3.530  1.00 51.41           C  \nATOM   1721  O   TYR A 223      21.592  48.718   2.899  1.00 52.33           O  \nATOM   1722  CB  TYR A 223      19.803  51.277   3.803  1.00102.65           C  \nATOM   1723  CG  TYR A 223      18.616  50.620   3.136  1.00108.18           C  \nATOM   1724  CD1 TYR A 223      17.408  51.308   3.003  1.00110.28           C  \nATOM   1725  CD2 TYR A 223      18.685  49.312   2.664  1.00110.44           C  \nATOM   1726  CE1 TYR A 223      16.297  50.711   2.417  1.00113.38           C  \nATOM   1727  CE2 TYR A 223      17.578  48.702   2.077  1.00114.29           C  \nATOM   1728  CZ  TYR A 223      16.387  49.409   1.957  1.00114.51           C  \nATOM   1729  OH  TYR A 223      15.285  48.814   1.387  1.00116.00           O  \nATOM   1730  N   GLY A 224      22.558  49.825   4.607  1.00 49.84           N  \nATOM   1731  CA  GLY A 224      23.222  48.643   5.115  1.00 45.79           C  \nATOM   1732  C   GLY A 224      22.962  48.410   6.589  1.00 42.70           C  \nATOM   1733  O   GLY A 224      22.155  49.102   7.216  1.00 42.55           O  \nATOM   1734  N   VAL A 225      23.651  47.416   7.137  1.00 34.69           N  \nATOM   1735  CA  VAL A 225      23.531  47.054   8.537  1.00 30.69           C  \nATOM   1736  C   VAL A 225      24.030  48.190   9.416  1.00 30.26           C  \nATOM   1737  O   VAL A 225      25.044  48.828   9.117  1.00 27.97           O  \nATOM   1738  CB  VAL A 225      24.351  45.776   8.840  1.00 24.10           C  \nATOM   1739  CG1 VAL A 225      24.263  45.420  10.318  1.00 20.98           C  \nATOM   1740  CG2 VAL A 225      23.833  44.630   7.986  1.00 22.06           C  \nATOM   1741  N   ALA A 226      23.303  48.434  10.500  1.00 29.91           N  \nATOM   1742  CA  ALA A 226      23.644  49.480  11.443  1.00 31.87           C  \nATOM   1743  C   ALA A 226      24.867  49.093  12.256  1.00 33.87           C  \nATOM   1744  O   ALA A 226      25.240  47.920  12.330  1.00 35.63           O  \nATOM   1745  CB  ALA A 226      22.474  49.735  12.368  1.00 21.56           C  \nATOM   1746  N   GLU A 227      25.499  50.090  12.859  1.00 42.50           N  \nATOM   1747  CA  GLU A 227      26.670  49.855  13.694  1.00 44.98           C  \nATOM   1748  C   GLU A 227      26.384  50.367  15.103  1.00 43.32           C  \nATOM   1749  O   GLU A 227      25.371  51.029  15.336  1.00 43.84           O  \nATOM   1750  CB  GLU A 227      27.909  50.542  13.098  1.00 76.77           C  \nATOM   1751  CG  GLU A 227      27.621  51.676  12.120  1.00 85.38           C  \nATOM   1752  CD  GLU A 227      26.952  52.871  12.773  1.00 90.38           C  \nATOM   1753  OE1 GLU A 227      27.550  53.450  13.707  1.00 93.54           O  \nATOM   1754  OE2 GLU A 227      25.830  53.234  12.349  1.00 92.47           O  \nATOM   1755  N   GLY A 228      27.262  50.045  16.045  1.00 31.50           N  \nATOM   1756  CA  GLY A 228      27.056  50.488  17.408  1.00 28.22           C  \nATOM   1757  C   GLY A 228      26.640  49.361  18.335  1.00 27.69           C  \nATOM   1758  O   GLY A 228      26.829  48.184  18.020  1.00 26.45           O  \nATOM   1759  N   ARG A 229      26.072  49.726  19.480  1.00 35.30           N  \nATOM   1760  CA  ARG A 229      25.624  48.754  20.477  1.00 36.73           C  \nATOM   1761  C   ARG A 229      24.703  47.704  19.857  1.00 36.29           C  \nATOM   1762  O   ARG A 229      23.890  48.013  18.982  1.00 35.18           O  \nATOM   1763  CB  ARG A 229      24.855  49.461  21.591  1.00 42.57           C  \nATOM   1764  CG  ARG A 229      25.573  50.602  22.222  1.00 43.97           C  \nATOM   1765  CD  ARG A 229      24.566  51.559  22.807  1.00 49.40           C  \nATOM   1766  NE  ARG A 229      23.822  50.997  23.932  1.00 51.00           N  \nATOM   1767  CZ  ARG A 229      22.742  51.566  24.461  1.00 51.94           C  \nATOM   1768  NH1 ARG A 229      22.275  52.705  23.962  1.00 50.39           N  \nATOM   1769  NH2 ARG A 229      22.141  51.006  25.499  1.00 53.73           N  \nATOM   1770  N   ALA A 230      24.830  46.468  20.325  1.00 30.24           N  \nATOM   1771  CA  ALA A 230      23.999  45.378  19.839  1.00 30.37           C  \nATOM   1772  C   ALA A 230      23.125  44.901  20.982  1.00 30.97           C  \nATOM   1773  O   ALA A 230      23.506  44.999  22.144  1.00 30.74           O  \nATOM   1774  CB  ALA A 230      24.866  44.226  19.336  1.00 18.19           C  \nATOM   1775  N   GLU A 231      21.944  44.399  20.652  1.00 34.85           N  \nATOM   1776  CA  GLU A 231      21.040  43.888  21.673  1.00 34.50           C  \nATOM   1777  C   GLU A 231      20.816  42.407  21.390  1.00 32.95           C  \nATOM   1778  O   GLU A 231      20.940  41.955  20.252  1.00 33.62           O  \nATOM   1779  CB  GLU A 231      19.703  44.653  21.655  1.00 30.59           C  \nATOM   1780  CG  GLU A 231      18.859  44.464  20.400  1.00 33.09           C  \nATOM   1781  CD  GLU A 231      17.527  45.198  20.468  1.00 35.67           C  \nATOM   1782  OE1 GLU A 231      17.523  46.444  20.396  1.00 36.38           O  \nATOM   1783  OE2 GLU A 231      16.478  44.534  20.596  1.00 36.98           O  \nATOM   1784  N   LEU A 232      20.511  41.649  22.431  1.00 29.36           N  \nATOM   1785  CA  LEU A 232      20.260  40.230  22.274  1.00 28.34           C  \nATOM   1786  C   LEU A 232      18.971  39.875  23.008  1.00 27.69           C  \nATOM   1787  O   LEU A 232      18.852  40.109  24.216  1.00 26.18           O  \nATOM   1788  CB  LEU A 232      21.435  39.421  22.828  1.00 21.63           C  \nATOM   1789  CG  LEU A 232      21.320  37.911  22.603  1.00 23.57           C  \nATOM   1790  CD1 LEU A 232      22.694  37.284  22.597  1.00 23.81           C  \nATOM   1791  CD2 LEU A 232      20.432  37.290  23.684  1.00 26.45           C  \nATOM   1792  N   ASN A 233      18.016  39.312  22.266  1.00 26.98           N  \nATOM   1793  CA  ASN A 233      16.712  38.920  22.808  1.00 27.06           C  \nATOM   1794  C   ASN A 233      16.422  37.430  22.670  1.00 27.05           C  \nATOM   1795  O   ASN A 233      16.749  36.795  21.666  1.00 29.07           O  \nATOM   1796  CB  ASN A 233      15.597  39.690  22.105  1.00 22.69           C  \nATOM   1797  CG  ASN A 233      15.618  41.167  22.419  1.00 22.31           C  \nATOM   1798  OD1 ASN A 233      15.175  41.592  23.484  1.00 22.02           O  \nATOM   1799  ND2 ASN A 233      16.142  41.960  21.496  1.00 24.32           N  \nATOM   1800  N   GLY A 234      15.786  36.873  23.685  1.00 23.52           N  \nATOM   1801  CA  GLY A 234      15.471  35.470  23.639  1.00 21.74           C  \nATOM   1802  C   GLY A 234      14.434  35.143  24.676  1.00 22.11           C  \nATOM   1803  O   GLY A 234      13.958  36.025  25.391  1.00 22.43           O  \nATOM   1804  N   VAL A 235      14.076  33.869  24.749  1.00 25.52           N  \nATOM   1805  CA  VAL A 235      13.099  33.409  25.709  1.00 25.54           C  \nATOM   1806  C   VAL A 235      13.703  32.203  26.417  1.00 27.45           C  \nATOM   1807  O   VAL A 235      14.134  31.247  25.763  1.00 28.02           O  \nATOM   1808  CB  VAL A 235      11.789  32.970  25.011  1.00 21.95           C  \nATOM   1809  CG1 VAL A 235      10.727  32.661  26.050  1.00 18.23           C  \nATOM   1810  CG2 VAL A 235      11.318  34.052  24.042  1.00 19.10           C  \nATOM   1811  N   ALA A 236      13.771  32.262  27.744  1.00 21.91           N  \nATOM   1812  CA  ALA A 236      14.289  31.152  28.521  1.00 22.92           C  \nATOM   1813  C   ALA A 236      13.039  30.340  28.842  1.00 25.99           C  \nATOM   1814  O   ALA A 236      12.008  30.909  29.204  1.00 25.48           O  \nATOM   1815  CB  ALA A 236      14.935  31.660  29.789  1.00 25.55           C  \nATOM   1816  N   LEU A 237      13.124  29.020  28.712  1.00 22.73           N  \nATOM   1817  CA  LEU A 237      11.966  28.174  28.949  1.00 23.46           C  \nATOM   1818  C   LEU A 237      12.301  26.954  29.769  1.00 25.25           C  \nATOM   1819  O   LEU A 237      13.452  26.518  29.817  1.00 25.59           O  \nATOM   1820  CB  LEU A 237      11.396  27.691  27.618  1.00 25.40           C  \nATOM   1821  CG  LEU A 237      11.083  28.707  26.523  1.00 25.05           C  \nATOM   1822  CD1 LEU A 237      11.157  28.016  25.183  1.00 24.02           C  \nATOM   1823  CD2 LEU A 237       9.711  29.320  26.750  1.00 24.27           C  \nATOM   1824  N   HIS A 238      11.275  26.391  30.400  1.00 27.55           N  \nATOM   1825  CA  HIS A 238      11.429  25.178  31.190  1.00 30.24           C  \nATOM   1826  C   HIS A 238      10.402  24.186  30.689  1.00 29.25           C  \nATOM   1827  O   HIS A 238       9.200  24.401  30.838  1.00 30.49           O  \nATOM   1828  CB  HIS A 238      11.211  25.453  32.685  1.00 53.25           C  \nATOM   1829  CG  HIS A 238      12.414  26.014  33.380  1.00 60.07           C  \nATOM   1830  ND1 HIS A 238      12.366  26.520  34.662  1.00 63.86           N  \nATOM   1831  CD2 HIS A 238      13.699  26.145  32.971  1.00 62.16           C  \nATOM   1832  CE1 HIS A 238      13.570  26.941  35.011  1.00 64.11           C  \nATOM   1833  NE2 HIS A 238      14.396  26.725  34.004  1.00 64.44           N  \nATOM   1834  N   PHE A 239      10.870  23.111  30.071  1.00 30.89           N  \nATOM   1835  CA  PHE A 239       9.964  22.098  29.561  1.00 34.19           C  \nATOM   1836  C   PHE A 239       9.950  20.897  30.483  1.00 35.70           C  \nATOM   1837  O   PHE A 239      10.971  20.536  31.063  1.00 38.50           O  \nATOM   1838  CB  PHE A 239      10.375  21.643  28.155  1.00 36.32           C  \nATOM   1839  CG  PHE A 239      10.289  22.720  27.117  1.00 36.24           C  \nATOM   1840  CD1 PHE A 239       9.274  23.667  27.158  1.00 35.97           C  \nATOM   1841  CD2 PHE A 239      11.222  22.786  26.091  1.00 36.79           C  \nATOM   1842  CE1 PHE A 239       9.195  24.669  26.188  1.00 35.79           C  \nATOM   1843  CE2 PHE A 239      11.147  23.782  25.120  1.00 35.60           C  \nATOM   1844  CZ  PHE A 239      10.132  24.723  25.171  1.00 34.60           C  \nATOM   1845  N   GLU A 240       8.787  20.273  30.609  1.00 38.83           N  \nATOM   1846  CA  GLU A 240       8.653  19.101  31.450  1.00 39.80           C  \nATOM   1847  C   GLU A 240       7.480  18.262  30.959  1.00 38.19           C  \nATOM   1848  O   GLU A 240       6.333  18.708  30.972  1.00 36.59           O  \nATOM   1849  CB  GLU A 240       8.446  19.533  32.901  1.00 58.50           C  \nATOM   1850  CG  GLU A 240       8.631  18.433  33.917  1.00 62.13           C  \nATOM   1851  CD  GLU A 240       8.599  18.962  35.335  1.00 66.25           C  \nATOM   1852  OE1 GLU A 240       8.791  18.152  36.271  1.00 67.75           O  \nATOM   1853  OE2 GLU A 240       8.383  20.185  35.510  1.00 65.30           O  \nATOM   1854  N   GLY A 241       7.789  17.048  30.517  1.00 37.11           N  \nATOM   1855  CA  GLY A 241       6.768  16.147  30.025  1.00 36.85           C  \nATOM   1856  C   GLY A 241       6.130  16.641  28.746  1.00 36.84           C  \nATOM   1857  O   GLY A 241       4.943  16.414  28.513  1.00 40.31           O  \nATOM   1858  N   GLY A 242       6.910  17.320  27.914  1.00 41.38           N  \nATOM   1859  CA  GLY A 242       6.389  17.830  26.655  1.00 40.38           C  \nATOM   1860  C   GLY A 242       5.613  19.124  26.813  1.00 41.28           C  \nATOM   1861  O   GLY A 242       5.003  19.626  25.861  1.00 41.87           O  \nATOM   1862  N   LYS A 243       5.639  19.679  28.019  1.00 42.19           N  \nATOM   1863  CA  LYS A 243       4.925  20.913  28.291  1.00 40.59           C  \nATOM   1864  C   LYS A 243       5.854  21.954  28.873  1.00 38.67           C  \nATOM   1865  O   LYS A 243       6.887  21.619  29.435  1.00 40.61           O  \nATOM   1866  CB  LYS A 243       3.782  20.653  29.272  1.00 32.73           C  \nATOM   1867  CG  LYS A 243       2.721  19.709  28.761  1.00 36.10           C  \nATOM   1868  CD  LYS A 243       1.529  19.679  29.708  1.00 39.45           C  \nATOM   1869  CE  LYS A 243       0.309  19.051  29.042  1.00 41.07           C  \nATOM   1870  NZ  LYS A 243      -0.954  19.352  29.792  1.00 40.76           N  \nATOM   1871  N   ALA A 244       5.476  23.220  28.735  1.00 32.51           N  \nATOM   1872  CA  ALA A 244       6.258  24.323  29.279  1.00 31.45           C  \nATOM   1873  C   ALA A 244       5.705  24.652  30.670  1.00 32.14           C  \nATOM   1874  O   ALA A 244       4.496  24.840  30.840  1.00 31.93           O  \nATOM   1875  CB  ALA A 244       6.144  25.529  28.374  1.00 33.46           C  \nATOM   1876  N   THR A 245       6.582  24.702  31.670  1.00 38.12           N  \nATOM   1877  CA  THR A 245       6.144  25.007  33.026  1.00 38.23           C  \nATOM   1878  C   THR A 245       6.636  26.380  33.441  1.00 38.17           C  \nATOM   1879  O   THR A 245       6.277  26.890  34.502  1.00 37.77           O  \nATOM   1880  CB  THR A 245       6.664  23.970  34.030  1.00 36.09           C  \nATOM   1881  OG1 THR A 245       8.098  23.943  34.003  1.00 36.94           O  \nATOM   1882  CG2 THR A 245       6.128  22.594  33.680  1.00 36.16           C  \nATOM   1883  N   ALA A 246       7.459  26.981  32.592  1.00 37.02           N  \nATOM   1884  CA  ALA A 246       7.988  28.302  32.880  1.00 38.48           C  \nATOM   1885  C   ALA A 246       8.548  28.947  31.629  1.00 37.96           C  \nATOM   1886  O   ALA A 246       9.233  28.302  30.841  1.00 38.69           O  \nATOM   1887  CB  ALA A 246       9.072  28.217  33.956  1.00 35.03           C  \nATOM   1888  N   ALA A 247       8.248  30.228  31.462  1.00 31.76           N  \nATOM   1889  CA  ALA A 247       8.718  30.994  30.326  1.00 31.28           C  \nATOM   1890  C   ALA A 247       8.922  32.454  30.727  1.00 31.93           C  \nATOM   1891  O   ALA A 247       8.037  33.068  31.322  1.00 31.91           O  \nATOM   1892  CB  ALA A 247       7.710  30.892  29.185  1.00 27.88           C  \nATOM   1893  N   GLU A 248      10.098  33.001  30.428  1.00 29.23           N  \nATOM   1894  CA  GLU A 248      10.385  34.406  30.731  1.00 31.17           C  \nATOM   1895  C   GLU A 248      11.341  34.946  29.676  1.00 29.51           C  \nATOM   1896  O   GLU A 248      12.187  34.216  29.158  1.00 28.13           O  \nATOM   1897  CB  GLU A 248      11.002  34.575  32.133  1.00 39.40           C  \nATOM   1898  CG  GLU A 248      12.439  34.103  32.276  1.00 43.65           C  \nATOM   1899  CD  GLU A 248      12.964  34.234  33.707  1.00 48.03           C  \nATOM   1900  OE1 GLU A 248      12.419  33.561  34.612  1.00 50.83           O  \nATOM   1901  OE2 GLU A 248      13.924  35.009  33.932  1.00 49.33           O  \nATOM   1902  N   ARG A 249      11.192  36.222  29.349  1.00 31.26           N  \nATOM   1903  CA  ARG A 249      12.042  36.844  28.345  1.00 32.33           C  \nATOM   1904  C   ARG A 249      13.487  37.038  28.781  1.00 31.32           C  \nATOM   1905  O   ARG A 249      13.775  37.226  29.962  1.00 32.02           O  \nATOM   1906  CB  ARG A 249      11.466  38.203  27.927  1.00 31.90           C  \nATOM   1907  CG  ARG A 249      10.211  38.119  27.086  1.00 29.89           C  \nATOM   1908  CD  ARG A 249       9.818  39.493  26.561  1.00 30.10           C  \nATOM   1909  NE  ARG A 249       9.712  40.465  27.641  1.00 30.01           N  \nATOM   1910  CZ  ARG A 249       8.708  40.526  28.509  1.00 29.91           C  \nATOM   1911  NH1 ARG A 249       7.697  39.673  28.432  1.00 28.19           N  \nATOM   1912  NH2 ARG A 249       8.729  41.440  29.470  1.00 30.69           N  \nATOM   1913  N   TYR A 250      14.397  36.969  27.821  1.00 25.17           N  \nATOM   1914  CA  TYR A 250      15.806  37.195  28.113  1.00 26.34           C  \nATOM   1915  C   TYR A 250      16.270  38.342  27.231  1.00 26.21           C  \nATOM   1916  O   TYR A 250      16.103  38.308  26.009  1.00 26.26           O  \nATOM   1917  CB  TYR A 250      16.664  35.957  27.838  1.00 40.28           C  \nATOM   1918  CG  TYR A 250      18.117  36.196  28.205  1.00 43.86           C  \nATOM   1919  CD1 TYR A 250      18.993  36.830  27.317  1.00 43.59           C  \nATOM   1920  CD2 TYR A 250      18.581  35.906  29.489  1.00 44.76           C  \nATOM   1921  CE1 TYR A 250      20.285  37.176  27.705  1.00 43.80           C  \nATOM   1922  CE2 TYR A 250      19.871  36.251  29.887  1.00 44.07           C  \nATOM   1923  CZ  TYR A 250      20.712  36.886  28.995  1.00 45.06           C  \nATOM   1924  OH  TYR A 250      21.967  37.252  29.412  1.00 48.93           O  \nATOM   1925  N   ARG A 251      16.843  39.370  27.839  1.00 32.24           N  \nATOM   1926  CA  ARG A 251      17.283  40.509  27.059  1.00 32.34           C  \nATOM   1927  C   ARG A 251      18.575  41.085  27.582  1.00 31.56           C  \nATOM   1928  O   ARG A 251      18.756  41.236  28.784  1.00 32.54           O  \nATOM   1929  CB  ARG A 251      16.210  41.604  27.055  1.00 34.01           C  \nATOM   1930  CG  ARG A 251      16.543  42.723  26.090  1.00 35.96           C  \nATOM   1931  CD  ARG A 251      15.579  43.900  26.119  1.00 36.80           C  \nATOM   1932  NE  ARG A 251      15.683  44.616  24.848  1.00 42.40           N  \nATOM   1933  CZ  ARG A 251      15.088  45.766  24.566  1.00 42.13           C  \nATOM   1934  NH1 ARG A 251      14.332  46.365  25.469  1.00 45.47           N  \nATOM   1935  NH2 ARG A 251      15.242  46.310  23.372  1.00 43.82           N  \nATOM   1936  N   PHE A 252      19.477  41.403  26.666  1.00 28.98           N  \nATOM   1937  CA  PHE A 252      20.754  41.986  27.030  1.00 27.51           C  \nATOM   1938  C   PHE A 252      21.123  43.038  25.994  1.00 27.08           C  \nATOM   1939  O   PHE A 252      20.963  42.812  24.806  1.00 27.73           O  \nATOM   1940  CB  PHE A 252      21.827  40.901  27.089  1.00 28.16           C  \nATOM   1941  CG  PHE A 252      23.229  41.439  27.192  1.00 29.16           C  \nATOM   1942  CD1 PHE A 252      24.029  41.559  26.058  1.00 28.28           C  \nATOM   1943  CD2 PHE A 252      23.731  41.869  28.418  1.00 28.96           C  \nATOM   1944  CE1 PHE A 252      25.309  42.100  26.143  1.00 29.74           C  \nATOM   1945  CE2 PHE A 252      25.014  42.414  28.517  1.00 31.06           C  \nATOM   1946  CZ  PHE A 252      25.805  42.533  27.374  1.00 31.26           C  \nATOM   1947  N   ILE A 253      21.592  44.193  26.454  1.00 29.39           N  \nATOM   1948  CA  ILE A 253      22.007  45.281  25.572  1.00 31.20           C  \nATOM   1949  C   ILE A 253      23.458  45.640  25.905  1.00 33.69           C  \nATOM   1950  O   ILE A 253      23.733  46.105  27.010  1.00 35.10           O  \nATOM   1951  CB  ILE A 253      21.185  46.552  25.815  1.00 27.90           C  \nATOM   1952  CG1 ILE A 253      19.691  46.228  25.856  1.00 25.42           C  \nATOM   1953  CG2 ILE A 253      21.523  47.581  24.744  1.00 25.26           C  \nATOM   1954  CD1 ILE A 253      19.112  45.876  24.535  1.00 28.00           C  \nATOM   1955  N   GLU A 254      24.390  45.452  24.978  1.00 38.13           N  \nATOM   1956  CA  GLU A 254      25.771  45.794  25.306  1.00 41.23           C  \nATOM   1957  C   GLU A 254      25.913  47.300  25.498  1.00 43.65           C  \nATOM   1958  O   GLU A 254      25.159  48.072  24.906  1.00 42.95           O  \nATOM   1959  CB  GLU A 254      26.743  45.267  24.235  1.00 40.07           C  \nATOM   1960  CG  GLU A 254      26.534  45.768  22.834  1.00 40.85           C  \nATOM   1961  CD  GLU A 254      27.346  44.978  21.812  1.00 39.62           C  \nATOM   1962  OE1 GLU A 254      27.303  45.322  20.618  1.00 40.66           O  \nATOM   1963  OE2 GLU A 254      28.025  44.008  22.195  1.00 40.86           O  \nATOM   1964  N   ASP A 255      26.861  47.708  26.347  1.00 54.69           N  \nATOM   1965  CA  ASP A 255      27.090  49.128  26.635  1.00 57.07           C  \nATOM   1966  C   ASP A 255      27.565  49.928  25.428  1.00 58.93           C  \nATOM   1967  O   ASP A 255      28.143  49.315  24.503  1.00 59.28           O  \nATOM   1968  CB  ASP A 255      28.119  49.294  27.757  1.00 57.55           C  \nATOM   1969  CG  ASP A 255      27.598  48.836  29.100  1.00 60.69           C  \nATOM   1970  OD1 ASP A 255      26.477  49.252  29.471  1.00 62.03           O  \nATOM   1971  OD2 ASP A 255      28.311  48.070  29.787  1.00 61.12           O  \nATOM   1972  OXT ASP A 255      27.373  51.166  25.437  1.00 59.59           O  \nHETATM 1973  O   HOH A 256      28.542  40.122   7.032  1.00 18.44           O  \nHETATM 1974  O   HOH A 257      -2.767  23.845  31.408  1.00 33.27           O  \nHETATM 1975  O   HOH A 258       6.972  42.898  17.181  1.00 28.95           O  \nHETATM 1976  O   HOH A 259       6.618  56.552   3.274  1.00 22.46           O  \nHETATM 1977  O   HOH A 260      12.332  38.821  15.424  1.00 25.45           O  \nHETATM 1978  O   HOH A 261      31.761  31.623  18.685  1.00 29.00           O  \nHETATM 1979  O   HOH A 262       5.976  33.044   3.652  1.00 31.01           O  \nHETATM 1980  O   HOH A 263      22.928  24.176  22.630  1.00 25.90           O  \nHETATM 1981  O   HOH A 264       9.135  37.542  30.915  1.00 32.08           O  \nHETATM 1982  O   HOH A 265      -2.033  32.342  21.300  1.00 29.24           O  \nHETATM 1983  O   HOH A 266      17.748  22.093   6.946  1.00 28.45           O  \nHETATM 1984  O   HOH A 267      23.291  20.026  18.055  1.00 25.05           O  \nHETATM 1985  O   HOH A 268      -4.733  40.911  10.190  1.00 33.62           O  \nHETATM 1986  O   HOH A 269      11.887  29.873  32.393  1.00 36.78           O  \nHETATM 1987  O   HOH A 270       5.748  18.980  22.876  1.00 34.91           O  \nHETATM 1988  O   HOH A 271       2.869  60.756  -0.284  1.00 30.72           O  \nHETATM 1989  O   HOH A 272      24.712  20.972  27.854  1.00 31.53           O  \nHETATM 1990  O   HOH A 273      13.233  40.483  25.325  1.00 31.67           O  \nHETATM 1991  O   HOH A 274      12.291  41.799  27.371  1.00 27.85           O  \nHETATM 1992  O   HOH A 275      11.273  18.547   6.238  1.00 24.96           O  \nHETATM 1993  O   HOH A 276      22.802  20.912   7.578  1.00 34.28           O  \nHETATM 1994  O   HOH A 277       2.434  36.986   8.678  1.00 46.55           O  \nHETATM 1995  O   HOH A 278      29.204  41.447  23.295  1.00 26.42           O  \nHETATM 1996  O   HOH A 279      20.202  20.072   6.558  1.00 34.58           O  \nHETATM 1997  O   HOH A 280       6.885  41.090   5.878  1.00 37.45           O  \nHETATM 1998  O   HOH A 281      32.197  43.739  15.641  1.00 32.01           O  \nHETATM 1999  O   HOH A 282      10.462  14.910  25.492  1.00 29.23           O  \nHETATM 2000  O   HOH A 283      12.425  38.041  24.116  1.00 30.90           O  \nHETATM 2001  O   HOH A 284      19.855  46.012   5.752  1.00 42.45           O  \nHETATM 2002  O   HOH A 285      -6.686  21.006  12.981  1.00 38.07           O  \nHETATM 2003  O   HOH A 286      28.832  47.783  15.030  1.00 33.95           O  \nHETATM 2004  O   HOH A 287      23.145  49.990  17.279  1.00 32.86           O  \nHETATM 2005  O   HOH A 288      21.790  17.954  22.162  1.00 35.92           O  \nHETATM 2006  O   HOH A 289      15.541  35.274  31.677  1.00 43.88           O  \nHETATM 2007  O   HOH A 290       1.392  39.051   1.066  1.00 35.40           O  \nHETATM 2008  O   HOH A 291      26.005  20.966   7.316  1.00 25.70           O  \nHETATM 2009  O   HOH A 292      29.756  28.104  27.700  1.00 32.51           O  \nHETATM 2010  O   HOH A 293      24.009  38.163   2.081  1.00 24.49           O  \nHETATM 2011  O   HOH A 294      27.295  33.761  14.435  1.00 29.85           O  \nHETATM 2012  O   HOH A 295      28.023  53.705  16.215  1.00 41.97           O  \nHETATM 2013  O   HOH A 296      22.570  54.113  11.925  1.00 40.79           O  \nHETATM 2014  O   HOH A 297      22.730  19.552  29.739  1.00 41.79           O  \nHETATM 2015  O   HOH A 298      30.647  31.882  -0.273  1.00 35.67           O  \nHETATM 2016  O   HOH A 299      30.566  35.050  12.977  1.00 32.73           O  \nHETATM 2017  O   HOH A 300      13.083  49.719  -0.438  1.00 40.14           O  \nHETATM 2018  O   HOH A 301      26.769  23.739  23.485  1.00 37.75           O  \nHETATM 2019  O   HOH A 302      17.825  27.757  33.245  1.00 39.96           O  \nHETATM 2020  O   HOH A 303       7.287  11.077  21.486  1.00 45.53           O  \nHETATM 2021  O   HOH A 304      32.207  38.239  19.214  1.00 38.32           O  \nHETATM 2022  O   HOH A 305      14.949  15.648  10.507  1.00 43.54           O  \nHETATM 2023  O   HOH A 306      28.627  45.516  26.773  1.00 35.36           O  \nHETATM 2024  O   HOH A 307      25.998  46.031  29.004  1.00 43.85           O  \nHETATM 2025  O   HOH A 308       1.410  21.436  25.387  1.00 33.31           O  \nHETATM 2026  O   HOH A 309      17.788  61.524   0.149  1.00 47.20           O  \nHETATM 2027  O   HOH A 310      15.213  33.189   1.153  1.00 25.44           O  \nHETATM 2028  O   HOH A 311      22.481  56.403   9.831  1.00 48.45           O  \nHETATM 2029  O   HOH A 312      22.540  35.287  34.557  1.00 51.50           O  \nHETATM 2030  O   HOH A 313      17.035  39.566  30.635  1.00 44.47           O  \nTER    2031      HOH A 313                                                       \nEND   \n",
           "type": "blob"
          }
         ],
         "kwargs": {
          "defaultRepresentation": true,
          "ext": "pdb"
         },
         "methodName": "loadFile",
         "reconstruc_color_scheme": false,
         "target": "Stage",
         "type": "call_method"
        }
       ],
       "_ngl_original_stage_parameters": {
        "ambientColor": 14540253,
        "ambientIntensity": 0.2,
        "backgroundColor": "white",
        "cameraEyeSep": 0.3,
        "cameraFov": 40,
        "cameraType": "perspective",
        "clipDist": 10,
        "clipFar": 100,
        "clipNear": 0,
        "fogFar": 100,
        "fogNear": 50,
        "hoverTimeout": 0,
        "impostor": true,
        "lightColor": 14540253,
        "lightIntensity": 1,
        "mousePreset": "default",
        "panSpeed": 1,
        "quality": "medium",
        "rotateSpeed": 2,
        "sampleLevel": 0,
        "tooltip": true,
        "workerDefault": true,
        "zoomSpeed": 1.2
       },
       "_ngl_repr_dict": {
        "0": {
         "0": {
          "params": {
           "aspectRatio": 5,
           "assembly": "default",
           "capped": true,
           "clipCenter": {
            "x": 0,
            "y": 0,
            "z": 0
           },
           "clipNear": 0,
           "clipRadius": 0,
           "colorMode": "hcl",
           "colorReverse": true,
           "colorScale": "spectral",
           "colorScheme": "residueindex",
           "colorValue": 9474192,
           "defaultAssembly": "",
           "depthWrite": true,
           "diffuse": 16777215,
           "diffuseInterior": false,
           "disablePicking": false,
           "flatShaded": false,
           "interiorColor": 2236962,
           "interiorDarkening": 0,
           "lazy": false,
           "matrix": {
            "elements": [
             1,
             0,
             0,
             0,
             0,
             1,
             0,
             0,
             0,
             0,
             1,
             0,
             0,
             0,
             0,
             1
            ]
           },
           "metalness": 0,
           "opacity": 1,
           "quality": "high",
           "radialSegments": 20,
           "radiusData": {},
           "radiusScale": 0.7,
           "radiusSize": 1,
           "radiusType": "sstruc",
           "roughness": 0.4,
           "sele": "",
           "side": "double",
           "smoothSheet": false,
           "subdiv": 6,
           "tension": null,
           "useInteriorColor": true,
           "visible": true,
           "wireframe": false
          },
          "type": "cartoon"
         },
         "1": {
          "params": {
           "aspectRatio": 1,
           "assembly": "default",
           "bondScale": 0.4,
           "clipCenter": {
            "x": 0,
            "y": 0,
            "z": 0
           },
           "clipNear": 0,
           "clipRadius": 0,
           "colorMode": "hcl",
           "colorReverse": true,
           "colorScale": "spectral",
           "colorScheme": "residueindex",
           "colorValue": 9474192,
           "cylinderOnly": false,
           "defaultAssembly": "",
           "depthWrite": true,
           "diffuse": 16777215,
           "diffuseInterior": false,
           "disableImpostor": false,
           "disablePicking": false,
           "flatShaded": false,
           "interiorColor": 2236962,
           "interiorDarkening": 0,
           "lazy": false,
           "lineOnly": false,
           "linewidth": 2,
           "matrix": {
            "elements": [
             1,
             0,
             0,
             0,
             0,
             1,
             0,
             0,
             0,
             0,
             1,
             0,
             0,
             0,
             0,
             1
            ]
           },
           "metalness": 0,
           "opacity": 1,
           "openEnded": true,
           "quality": "high",
           "radialSegments": 20,
           "radiusData": {},
           "radiusScale": 1,
           "radiusSize": 0.3,
           "radiusType": "size",
           "roughness": 0.4,
           "sele": "",
           "side": "double",
           "sphereDetail": 2,
           "useInteriorColor": true,
           "visible": true,
           "wireframe": false
          },
          "type": "base"
         },
         "2": {
          "params": {
           "aspectRatio": 1.5,
           "assembly": "default",
           "bondScale": 0.3,
           "bondSpacing": 0.75,
           "clipCenter": {
            "x": 0,
            "y": 0,
            "z": 0
           },
           "clipNear": 0,
           "clipRadius": 0,
           "colorMode": "hcl",
           "colorReverse": false,
           "colorScale": "",
           "colorScheme": "element",
           "colorValue": 9474192,
           "cylinderOnly": false,
           "defaultAssembly": "",
           "depthWrite": true,
           "diffuse": 16777215,
           "diffuseInterior": false,
           "disableImpostor": false,
           "disablePicking": false,
           "flatShaded": false,
           "interiorColor": 2236962,
           "interiorDarkening": 0,
           "lazy": false,
           "lineOnly": false,
           "linewidth": 2,
           "matrix": {
            "elements": [
             1,
             0,
             0,
             0,
             0,
             1,
             0,
             0,
             0,
             0,
             1,
             0,
             0,
             0,
             0,
             1
            ]
           },
           "metalness": 0,
           "multipleBond": "off",
           "opacity": 1,
           "openEnded": true,
           "quality": "high",
           "radialSegments": 20,
           "radiusData": {},
           "radiusScale": 2,
           "radiusSize": 0.15,
           "radiusType": "size",
           "roughness": 0.4,
           "sele": "ligand",
           "side": "double",
           "sphereDetail": 2,
           "useInteriorColor": true,
           "visible": true,
           "wireframe": false
          },
          "type": "ball+stick"
         }
        }
       },
       "_ngl_serialize": false,
       "_ngl_version": "2.0.0-dev.36",
       "_ngl_view_id": [
        "AB307DE2-1DEE-46EE-95B8-8DF37C3852EA"
       ],
       "_player_dict": {},
       "_scene_position": {},
       "_scene_rotation": {},
       "_synced_model_ids": [],
       "_synced_repr_model_ids": [],
       "_view_height": "",
       "_view_width": "",
       "background": "white",
       "frame": 0,
       "gui_style": null,
       "layout": "IPY_MODEL_1c6e494d218b4e14b65e11b65d1e79d7",
       "max_frame": 0,
       "n_components": 1,
       "picked": {}
      }
     },
     "7be2599b2b664f6da60d19e172d2e2e2": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "ButtonStyleModel",
      "state": {}
     },
     "8f810c0b5689480fac8c3acc26bb2029": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "ImageModel",
      "state": {
       "layout": "IPY_MODEL_c7cf354bcdd64d08adb47d47b0cf8c5f",
       "width": "900.0"
      }
     },
     "9885f63704de4e0797f0018f34b49cbd": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "LinkModel",
      "state": {
       "source": [
        "IPY_MODEL_09ebdc541c664802aa15120b6b3b0e0f",
        "max"
       ],
       "target": [
        "IPY_MODEL_29c8848788294b8f9c8c3b8edeeab294",
        "max_frame"
       ]
      }
     },
     "9ca7560536f249c5b530a48981858aeb": {
      "model_module": "@jupyter-widgets/base",
      "model_module_version": "1.2.0",
      "model_name": "LayoutModel",
      "state": {}
     },
     "9e424fd7259e46048fb3f55d2286f16d": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "LinkModel",
      "state": {
       "source": [
        "IPY_MODEL_2e56223b06e6444f9add47a331531f15",
        "max"
       ],
       "target": [
        "IPY_MODEL_68adf8c3ebd649cca0c0b81626f28a36",
        "max_frame"
       ]
      }
     },
     "a0b760e52e8a4356a18223f5e74ac156": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "LinkModel",
      "state": {
       "source": [
        "IPY_MODEL_09ebdc541c664802aa15120b6b3b0e0f",
        "value"
       ],
       "target": [
        "IPY_MODEL_29c8848788294b8f9c8c3b8edeeab294",
        "frame"
       ]
      }
     },
     "a22cb3edcf8c4dcaa928cccab78cdcb8": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "LinkModel",
      "state": {
       "source": [
        "IPY_MODEL_dc4c2913fc224986b8eafdcab1278219",
        "value"
       ],
       "target": [
        "IPY_MODEL_2e56223b06e6444f9add47a331531f15",
        "value"
       ]
      }
     },
     "af804eb2a94e4c6e96628a3b2407ccd4": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "IntSliderModel",
      "state": {
       "layout": "IPY_MODEL_0db08b1071ad4d4f90e698719511fd94",
       "max": 0,
       "style": "IPY_MODEL_bfd7f5b26cf84457ba031c59940c5b7d"
      }
     },
     "bbcb56f26d404a01b3a18ce02cf23ebc": {
      "model_module": "@jupyter-widgets/base",
      "model_module_version": "1.2.0",
      "model_name": "LayoutModel",
      "state": {}
     },
     "bf658c21ff794afa9013ffaca07caac9": {
      "model_module": "@jupyter-widgets/base",
      "model_module_version": "1.2.0",
      "model_name": "LayoutModel",
      "state": {
       "width": "34px"
      }
     },
     "bfd7f5b26cf84457ba031c59940c5b7d": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "SliderStyleModel",
      "state": {
       "description_width": ""
      }
     },
     "c7cf354bcdd64d08adb47d47b0cf8c5f": {
      "model_module": "@jupyter-widgets/base",
      "model_module_version": "1.2.0",
      "model_name": "LayoutModel",
      "state": {}
     },
     "c98e220293c341fbbe32b7057d68d4b7": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "ImageModel",
      "state": {
       "layout": "IPY_MODEL_63310cba8e144b0db4559434ab1c2fca",
       "width": "900.0"
      }
     },
     "ce533c82575f4df092c78f3ae3ece00d": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "HBoxModel",
      "state": {
       "children": [
        "IPY_MODEL_09ebdc541c664802aa15120b6b3b0e0f",
        "IPY_MODEL_af804eb2a94e4c6e96628a3b2407ccd4"
       ],
       "layout": "IPY_MODEL_e444878c93794c66aaeb2adcfb3bc68b"
      }
     },
     "dc4c2913fc224986b8eafdcab1278219": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "PlayModel",
      "state": {
       "layout": "IPY_MODEL_257e2bf2ff8540308ef4cf62cfe844b6",
       "max": 0,
       "style": "IPY_MODEL_1f3beddbb917459aaf9affec0339d5ce"
      }
     },
     "e444878c93794c66aaeb2adcfb3bc68b": {
      "model_module": "@jupyter-widgets/base",
      "model_module_version": "1.2.0",
      "model_name": "LayoutModel",
      "state": {}
     },
     "eb93fc0f2f99418bba22a238f335ab4c": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "ButtonModel",
      "state": {
       "icon": "compress",
       "layout": "IPY_MODEL_2151f8609be64b4b8062e3fd14334761",
       "style": "IPY_MODEL_3d3d42ec8c104cdcb7bcb02ecc9c4823"
      }
     }
    },
    "version_major": 2,
    "version_minor": 0
   }
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
