{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# SARSCov2 NSP4 Structure Prediction\n",
    "\n",
    "We will perform comparative homology modelling via the satisfaction of spatial restraints using the `modeller` package."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false,
    "jupyter": {
     "outputs_hidden": false
    }
   },
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "6988be637aa141faa48861d7b59d23af",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": []
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "                         MODELLER 9.25, 2020/09/03, r11894\n",
      "\n",
      "     PROTEIN STRUCTURE MODELLING BY SATISFACTION OF SPATIAL RESTRAINTS\n",
      "\n",
      "\n",
      "                     Copyright(c) 1989-2020 Andrej Sali\n",
      "                            All Rights Reserved\n",
      "\n",
      "                             Written by A. Sali\n",
      "                               with help from\n",
      "              B. Webb, M.S. Madhusudhan, M-Y. Shen, G.Q. Dong,\n",
      "          M.A. Marti-Renom, N. Eswar, F. Alber, M. Topf, B. Oliva,\n",
      "             A. Fiser, R. Sanchez, B. Yerkovich, A. Badretdinov,\n",
      "                     F. Melo, J.P. Overington, E. Feyfant\n",
      "                 University of California, San Francisco, USA\n",
      "                    Rockefeller University, New York, USA\n",
      "                      Harvard University, Cambridge, USA\n",
      "                   Imperial Cancer Research Fund, London, UK\n",
      "              Birkbeck College, University of London, London, UK\n",
      "\n",
      "\n",
      "Kind, OS, HostName, Kernel, Processor: 4, Darwin Skynet.local 19.6.0 x86_64\n",
      "Date and time of compilation         : 2020/09/03 08:25:47\n",
      "MODELLER executable type             : mac10v4-intel64\n",
      "Job starting time (YY/MM/DD HH:MM:SS): 2020/12/24 18:39:36\n",
      "\n"
     ]
    }
   ],
   "source": [
    "from modeller import *\n",
    "from Bio.PDB import *\n",
    "import numpy as np\n",
    "import homology_modelling as hg\n",
    "import seaborn as sns; sns.set()\n",
    "import nglview as ngv\n",
    "\n",
    "env = environ()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### NSP4 amino acid sequence\n",
    "Observe the amino acid sequence of the region of the NSP4 protein specified in the `.ali` file."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false,
    "jupyter": {
     "outputs_hidden": false
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">P1;NSP4\n",
      "sequence:NSP4:::::::0.00: 0.00\n",
      "ITREIGFIVPGLPGTVLRAINGDFLHFLPRVFSAVGNICYTPSKLIEYSDFATSACVLAAECTIFKDAMGKPVPY\n",
      "CYDTNLLEGSISYSELRPDTRYVLMDGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCERSEVGICLSTSGRWVLN\n",
      "NEHYR*\n"
     ]
    }
   ],
   "source": [
    "!cat NSP4.ali"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1. PSI-BLAST\n",
    "Read the `.ali` file and modify the string into the required NCBI format `.faa` for PSI-BLAST."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "hg.ali_to_faa(\"NSP4.ali\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Call `psiblast` from the command line. Run PSI-BLAST over the `pdbaa` database which contains non-redundant sequences of all proteins with structures in the Protein Data Bank.\n",
    "\n",
    "*Note*: In this case we will run PSI-BLAST to convergence. To speed up the process we can run it in the local directory."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PSIBLAST 2.11.0+\n",
      "\n",
      "\n",
      "Reference: Stephen F. Altschul, Thomas L. Madden, Alejandro A.\n",
      "Schaffer, Jinghui Zhang, Zheng Zhang, Webb Miller, and David J.\n",
      "Lipman (1997), \"Gapped BLAST and PSI-BLAST: a new generation of\n",
      "protein database search programs\", Nucleic Acids Res. 25:3389-3402.\n",
      "\n",
      "\n",
      "Reference for compositional score matrix adjustment: Stephen F.\n",
      "Altschul, John C. Wootton, E. Michael Gertz, Richa Agarwala,\n",
      "Aleksandr Morgulis, Alejandro A. Schaffer, and Yi-Kuo Yu (2005)\n",
      "\"Protein database searches using compositionally adjusted\n",
      "substitution matrices\", FEBS J. 272:5101-5109.\n",
      "\n",
      "\n",
      "Reference for composition-based statistics starting in round 2:\n",
      "Alejandro A. Schaffer, L. Aravind, Thomas L. Madden, Sergei\n",
      "Shavirin, John L. Spouge, Yuri I. Wolf, Eugene V. Koonin, and\n",
      "Stephen F. Altschul (2001), \"Improving the accuracy of PSI-BLAST\n",
      "protein database searches with composition-based statistics and\n",
      "other refinements\", Nucleic Acids Res. 29:2994-3005.\n",
      "\n",
      "\n",
      "\n",
      "Database: PDB protein database\n",
      "           120,434 sequences; 31,663,421 total letters\n",
      "\n",
      "Results from round 1\n",
      "\n",
      "\n",
      "Query= P1;NSP4\n",
      "\n",
      "Length=156\n",
      "                                                                      Score     E\n",
      "Sequences producing significant alignments:                          (Bits)  Value\n",
      "\n",
      "1T70_A Crystal structure of a novel phosphatase from Deinococcus ...  42.0    1e-04\n",
      "\n",
      "\n",
      ">1T70_A Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_B Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_C Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_D Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_E Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_F Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_G Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_H Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      "Length=255\n",
      "\n",
      " Score = 42.0 bits (97),  Expect = 1e-04, Method: Compositional matrix adjust.\n",
      " Identities = 54/177 (31%), Positives = 74/177 (42%), Gaps = 31/177 (18%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTVLRA--INGDFL---HFLPRVF-SAVGNICYTPSKLIEYSDFATS  54\n",
      "            I R + +  PG PG   R   +NG+ L   + L RVF  AV N   T   L+E  D  T \n",
      "Sbjct  84   IVRPLNYADPGTPGVGWRTFDVNGEKLTVVNLLGRVFMEAVDNPFRTMDALLERDDLGTV  143\n",
      "\n",
      "Query  55   ACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYVLM  100\n",
      "                 AE T  K+AMG                VP   DT +L+G  +Y   + D  +   \n",
      "Sbjct  144  FVDFHAEATSEKEAMGWHLAGRVAAVIGTHTHVPTA-DTRILKGGTAY---QTDAGFTGP  199\n",
      "\n",
      "Query  101  DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCE-RSEV-GICLSTSGRWVLNNEHYR  155\n",
      "              SII    + +EG ++   T      R+G  E R+E+ G+ L   G      E YR\n",
      "Sbjct  200  HDSII---GSAIEGPLQRFLT--ERPHRYGVAEGRAELNGVALHFEGGKATAAERYR  251\n",
      "\n",
      "\n",
      "\n",
      "Lambda      K        H        a         alpha\n",
      "   0.327    0.143    0.454    0.792     4.96 \n",
      "\n",
      "Gapped\n",
      "Lambda      K        H        a         alpha    sigma\n",
      "   0.267   0.0410    0.140     1.90     42.6     43.6 \n",
      "\n",
      "Effective search space used: 1206105420\n",
      "Warning: [psiblast] Query_1 P1;NSP4: Composition-based score adjustment conditioned on sequence properties and unconditional composition-based score adjustment is not supported with PSSMs, resetting to default value of standard composition-based statistics \n",
      "Results from round 2\n",
      "\n",
      "\n",
      "Query= P1;NSP4\n",
      "\n",
      "Length=156\n",
      "                                                                      Score     E\n",
      "Sequences producing significant alignments:                          (Bits)  Value\n",
      "Sequences used in model and found again:\n",
      "\n",
      "1T70_A Crystal structure of a novel phosphatase from Deinococcus ...  194     6e-63\n",
      "\n",
      "Sequences not found previously or not previously below threshold:\n",
      "\n",
      "2Z06_A Crystal structure of uncharacterized conserved protein fro...  74.7    2e-16\n",
      "2CV9_A Crystal structure of a hypothetical protein from Thermus t...  73.1    7e-16\n",
      "4B2O_A Crystal structure of Bacillus subtilis YmdB, a global regu...  59.3    1e-10\n",
      "\n",
      "\n",
      ">1T70_A Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_B Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_C Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_D Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_E Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_F Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_G Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_H Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      "Length=255\n",
      "\n",
      " Score = 194 bits (494),  Expect = 6e-63, Method: Composition-based stats.\n",
      " Identities = 54/177 (31%), Positives = 74/177 (42%), Gaps = 31/177 (18%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTVLRA--INGDFL---HFLPRVF-SAVGNICYTPSKLIEYSDFATS  54\n",
      "            I R + +  PG PG   R   +NG+ L   + L RVF  AV N   T   L+E  D  T \n",
      "Sbjct  84   IVRPLNYADPGTPGVGWRTFDVNGEKLTVVNLLGRVFMEAVDNPFRTMDALLERDDLGTV  143\n",
      "\n",
      "Query  55   ACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYVLM  100\n",
      "                 AE T  K+AMG                VP   DT +L+G  +Y   + D  +   \n",
      "Sbjct  144  FVDFHAEATSEKEAMGWHLAGRVAAVIGTHTHVPTA-DTRILKGGTAY---QTDAGFTGP  199\n",
      "\n",
      "Query  101  DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCE-RSEV-GICLSTSGRWVLNNEHYR  155\n",
      "              SII    + +EG ++   T      R+G  E R+E+ G+ L   G      E YR\n",
      "Sbjct  200  HDSII---GSAIEGPLQRFLT--ERPHRYGVAEGRAELNGVALHFEGGKATAAERYR  251\n",
      "\n",
      "\n",
      ">2Z06_A Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      " 2Z06_B Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      " 2Z06_C Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      " 2Z06_D Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      "Length=252\n",
      "\n",
      " Score = 74.7 bits (182),  Expect = 2e-16, Method: Composition-based stats.\n",
      " Identities = 32/174 (18%), Positives = 55/174 (32%), Gaps = 28/174 (16%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTV-LR-AINGD---FLHFLPRVF-SAVGNICYTPSKLIEYSDFATS  54\n",
      "            + R + +  PG PG    R  + G+   F+  + R+F   + +      +L+E       \n",
      "Sbjct  82   VVRPLNYP-PGTPGKGFWRLEVGGESLLFVQVMGRIFMDPLDDPFRALDRLLEEEKADYV  140\n",
      "\n",
      "Query  55   ACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYVLM  100\n",
      "               + AE T  K A+                 VP    T L +G++     + D      \n",
      "Sbjct  141  LVEVHAEATSEKMALAHYLDGRASAVLGTHTHVPTLDATRLPKGTL----YQTDVGMTGT  196\n",
      "\n",
      "Query  101  DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCERSEVGICLSTSGRWVLNNEHY  154\n",
      "              SII      +E  +    T   +  R    +       L   G   +    Y\n",
      "Sbjct  197  YHSII---GGEVETFLARFLTGRPQPFRAAQGKARFHATELVFEGGRPVAISPY  247\n",
      "\n",
      "\n",
      ">2CV9_A Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      " 2CV9_B Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      " 2CV9_C Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      " 2CV9_D Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      "Length=252\n",
      "\n",
      " Score = 73.1 bits (178),  Expect = 7e-16, Method: Composition-based stats.\n",
      " Identities = 33/174 (19%), Positives = 55/174 (32%), Gaps = 28/174 (16%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTV-LR-AINGDFLHFL---PRVF-SAVGNICYTPSKLIEYSDFATS  54\n",
      "            + R + +  PG PG    R  + G+ L F+    R+F   + +      +L+E       \n",
      "Sbjct  82   VVRPLNYP-PGTPGKGFWRLEVGGESLLFVQVXGRIFXDPLDDPFRALDRLLEEEKADYV  140\n",
      "\n",
      "Query  55   ACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYVLM  100\n",
      "               + AE T  K A+                 VP    T L +G++     + D      \n",
      "Sbjct  141  LVEVHAEATSEKXALAHYLDGRASAVLGTHTHVPTLDATRLPKGTL----YQTDVGXTGT  196\n",
      "\n",
      "Query  101  DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCERSEVGICLSTSGRWVLNNEHY  154\n",
      "              SII      +E  +    T   +  R    +       L   G   +    Y\n",
      "Sbjct  197  YHSII---GGEVETFLARFLTGRPQPFRAAQGKARFHATELVFEGGRPVAISPY  247\n",
      "\n",
      "\n",
      ">4B2O_A Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      " 4B2O_B Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      " 4B2O_C Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      " 4B2O_D Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      "Length=286\n",
      "\n",
      " Score = 59.3 bits (142),  Expect = 1e-10, Method: Composition-based stats.\n",
      " Identities = 28/155 (18%), Positives = 43/155 (28%), Gaps = 32/155 (21%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTVLRAI--NGDFL---HFLPRVF-SAVGNICYTPSKLIEYSDFAT-  53\n",
      "            + R   F   G PG  +  +  NG  L   +   R F   + +      +LI  +   T \n",
      "Sbjct  85   LVRPANFPE-GTPGKGITYVKANGKELAVINLQGRTFLPPLDDPFLKADELIAEAAKRTP  143\n",
      "\n",
      "Query  54   -SACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYV  98\n",
      "                   AE T  K A+G                V    D  +L    +Y     D    \n",
      "Sbjct  144  YIFIDFHAEATSEKLALGWYTDGRASAVVGTHTHVQTA-DNRILPKGTAYI---TDVGMT  199\n",
      "\n",
      "Query  99   LMDGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCE  133\n",
      "                 I+      +   ++   T      R    E\n",
      "Sbjct  200  GPYDGILGMDRETI---IKRFKTNL--PVRFTVAE  229\n",
      "\n",
      "\n",
      "\n",
      "Lambda      K        H        a         alpha\n",
      "   0.318    0.131    0.357    0.792     4.96 \n",
      "\n",
      "Gapped\n",
      "Lambda      K        H        a         alpha    sigma\n",
      "   0.267   0.0401    0.140     1.90     42.6     43.6 \n",
      "\n",
      "Effective search space used: 1206105420\n",
      "Results from round 3\n",
      "\n",
      "\n",
      "Query= P1;NSP4\n",
      "\n",
      "Length=156\n",
      "                                                                      Score     E\n",
      "Sequences producing significant alignments:                          (Bits)  Value\n",
      "Sequences used in model and found again:\n",
      "\n",
      "1T70_A Crystal structure of a novel phosphatase from Deinococcus ...  175     2e-55\n",
      "2Z06_A Crystal structure of uncharacterized conserved protein fro...  171     6e-54\n",
      "2CV9_A Crystal structure of a hypothetical protein from Thermus t...  165     2e-51\n",
      "4B2O_A Crystal structure of Bacillus subtilis YmdB, a global regu...  132     3e-38\n",
      "\n",
      "Sequences not found previously or not previously below threshold:\n",
      "\n",
      "1T71_A Crystal structure of a novel phosphatase Mycoplasma pneumo...  57.4    5e-10\n",
      "\n",
      "\n",
      ">1T70_A Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_B Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_C Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_D Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_E Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_F Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_G Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_H Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      "Length=255\n",
      "\n",
      " Score = 175 bits (444),  Expect = 2e-55, Method: Composition-based stats.\n",
      " Identities = 54/177 (31%), Positives = 74/177 (42%), Gaps = 31/177 (18%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTVLRA--INGDFL---HFLPRVF-SAVGNICYTPSKLIEYSDFATS  54\n",
      "            I R + +  PG PG   R   +NG+ L   + L RVF  AV N   T   L+E  D  T \n",
      "Sbjct  84   IVRPLNYADPGTPGVGWRTFDVNGEKLTVVNLLGRVFMEAVDNPFRTMDALLERDDLGTV  143\n",
      "\n",
      "Query  55   ACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYVLM  100\n",
      "                 AE T  K+AMG                VP   DT +L+G  +Y   + D  +   \n",
      "Sbjct  144  FVDFHAEATSEKEAMGWHLAGRVAAVIGTHTHVPTA-DTRILKGGTAY---QTDAGFTGP  199\n",
      "\n",
      "Query  101  DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCE-RSEV-GICLSTSGRWVLNNEHYR  155\n",
      "              SII    + +EG ++   T      R+G  E R+E+ G+ L   G      E YR\n",
      "Sbjct  200  HDSII---GSAIEGPLQRFLT--ERPHRYGVAEGRAELNGVALHFEGGKATAAERYR  251\n",
      "\n",
      "\n",
      ">2Z06_A Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      " 2Z06_B Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      " 2Z06_C Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      " 2Z06_D Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      "Length=252\n",
      "\n",
      " Score = 171 bits (434),  Expect = 6e-54, Method: Composition-based stats.\n",
      " Identities = 32/174 (18%), Positives = 55/174 (32%), Gaps = 28/174 (16%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTV-LR-AINGD---FLHFLPRVF-SAVGNICYTPSKLIEYSDFATS  54\n",
      "            + R + +  PG PG    R  + G+   F+  + R+F   + +      +L+E       \n",
      "Sbjct  82   VVRPLNYP-PGTPGKGFWRLEVGGESLLFVQVMGRIFMDPLDDPFRALDRLLEEEKADYV  140\n",
      "\n",
      "Query  55   ACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYVLM  100\n",
      "               + AE T  K A+                 VP    T L +G++     + D      \n",
      "Sbjct  141  LVEVHAEATSEKMALAHYLDGRASAVLGTHTHVPTLDATRLPKGTL----YQTDVGMTGT  196\n",
      "\n",
      "Query  101  DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCERSEVGICLSTSGRWVLNNEHY  154\n",
      "              SII      +E  +    T   +  R    +       L   G   +    Y\n",
      "Sbjct  197  YHSII---GGEVETFLARFLTGRPQPFRAAQGKARFHATELVFEGGRPVAISPY  247\n",
      "\n",
      "\n",
      ">2CV9_A Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      " 2CV9_B Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      " 2CV9_C Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      " 2CV9_D Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      "Length=252\n",
      "\n",
      " Score = 165 bits (417),  Expect = 2e-51, Method: Composition-based stats.\n",
      " Identities = 32/174 (18%), Positives = 54/174 (31%), Gaps = 28/174 (16%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTV-LR-AINGD---FLHFLPRVF-SAVGNICYTPSKLIEYSDFATS  54\n",
      "            + R + +  PG PG    R  + G+   F+    R+F   + +      +L+E       \n",
      "Sbjct  82   VVRPLNYP-PGTPGKGFWRLEVGGESLLFVQVXGRIFXDPLDDPFRALDRLLEEEKADYV  140\n",
      "\n",
      "Query  55   ACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYVLM  100\n",
      "               + AE T  K A+                 VP    T L +G++     + D      \n",
      "Sbjct  141  LVEVHAEATSEKXALAHYLDGRASAVLGTHTHVPTLDATRLPKGTL----YQTDVGXTGT  196\n",
      "\n",
      "Query  101  DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCERSEVGICLSTSGRWVLNNEHY  154\n",
      "              SII      +E  +    T   +  R    +       L   G   +    Y\n",
      "Sbjct  197  YHSII---GGEVETFLARFLTGRPQPFRAAQGKARFHATELVFEGGRPVAISPY  247\n",
      "\n",
      "\n",
      ">4B2O_A Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      " 4B2O_B Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      " 4B2O_C Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      " 4B2O_D Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      "Length=286\n",
      "\n",
      " Score = 132 bits (332),  Expect = 3e-38, Method: Composition-based stats.\n",
      " Identities = 28/155 (18%), Positives = 43/155 (28%), Gaps = 32/155 (21%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTVLRAI--NGDFL---HFLPRVF-SAVGNICYTPSKLIEYSDFAT-  53\n",
      "            + R   F   G PG  +  +  NG  L   +   R F   + +      +LI  +   T \n",
      "Sbjct  85   LVRPANFPE-GTPGKGITYVKANGKELAVINLQGRTFLPPLDDPFLKADELIAEAAKRTP  143\n",
      "\n",
      "Query  54   -SACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYV  98\n",
      "                   AE T  K A+G                V    D  +L    +Y     D    \n",
      "Sbjct  144  YIFIDFHAEATSEKLALGWYTDGRASAVVGTHTHVQTA-DNRILPKGTAYI---TDVGMT  199\n",
      "\n",
      "Query  99   LMDGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCE  133\n",
      "                 I+      +   ++   T      R    E\n",
      "Sbjct  200  GPYDGILGMDRETI---IKRFKTNL--PVRFTVAE  229\n",
      "\n",
      "\n",
      ">1T71_A Crystal structure of a novel phosphatase Mycoplasma pneumoniaefrom \n",
      "[Mycoplasma pneumoniae M129]\n",
      "Length=281\n",
      "\n",
      " Score = 57.4 bits (137),  Expect = 5e-10, Method: Composition-based stats.\n",
      " Identities = 23/118 (19%), Positives = 32/118 (27%), Gaps = 22/118 (19%)\n",
      "\n",
      "Query  37   NICYTPSKLIEYSDFATSACVLAAECTIFKDAMG--------------KPVPYCYDTNLL  82\n",
      "            N      +LI   D         AE T  K+A                  VP   D  + \n",
      "Sbjct  136  NPFKVLKELILKRDCDLHIVDFHAETTSEKNAFCMAFDGYVTTIFGTHTHVPSA-DLRIT  194\n",
      "\n",
      "Query  83   EGSISYSELRPDTRYVLM-DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCERSEVGI  139\n",
      "                +Y     D        GS+I       E S+R+      E+     C     G+\n",
      "Sbjct  195  PKGSAYI---TDVGMCGPGFGSVI---GANPEQSIRLFCAGSREHFEVSKCGAQLNGV  246\n",
      "\n",
      "\n",
      "\n",
      "Lambda      K        H        a         alpha\n",
      "   0.317    0.129    0.330    0.792     4.96 \n",
      "\n",
      "Gapped\n",
      "Lambda      K        H        a         alpha    sigma\n",
      "   0.267   0.0395    0.140     1.90     42.6     43.6 \n",
      "\n",
      "Effective search space used: 1206105420\n",
      "Results from round 4\n",
      "\n",
      "\n",
      "Query= P1;NSP4\n",
      "\n",
      "Length=156\n",
      "                                                                      Score     E\n",
      "Sequences producing significant alignments:                          (Bits)  Value\n",
      "Sequences used in model and found again:\n",
      "\n",
      "1T70_A Crystal structure of a novel phosphatase from Deinococcus ...  174     8e-55\n",
      "2Z06_A Crystal structure of uncharacterized conserved protein fro...  168     1e-52\n",
      "2CV9_A Crystal structure of a hypothetical protein from Thermus t...  161     5e-50\n",
      "4B2O_A Crystal structure of Bacillus subtilis YmdB, a global regu...  131     8e-38\n",
      "1T71_A Crystal structure of a novel phosphatase Mycoplasma pneumo...  113     7e-31\n",
      "\n",
      "Sequences not found previously or not previously below threshold:\n",
      "\n",
      "\n",
      "\n",
      ">1T70_A Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_B Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_C Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_D Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_E Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_F Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_G Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      " 1T70_H Crystal structure of a novel phosphatase from Deinococcus radiodurans \n",
      "[Deinococcus radiodurans R1]\n",
      "Length=255\n",
      "\n",
      " Score = 174 bits (440),  Expect = 8e-55, Method: Composition-based stats.\n",
      " Identities = 52/177 (29%), Positives = 70/177 (40%), Gaps = 31/177 (18%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTVLRA--INGDFL---HFLPRVF-SAVGNICYTPSKLIEYSDFATS  54\n",
      "            I R + +  PG PG   R   +NG+ L   + L RVF  AV N   T   L+E  D  T \n",
      "Sbjct  84   IVRPLNYADPGTPGVGWRTFDVNGEKLTVVNLLGRVFMEAVDNPFRTMDALLERDDLGTV  143\n",
      "\n",
      "Query  55   ACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYVLM  100\n",
      "                 AE T  K+AMG                VP   DT +L+G  +Y   + D  +   \n",
      "Sbjct  144  FVDFHAEATSEKEAMGWHLAGRVAAVIGTHTHVPTA-DTRILKGGTAY---QTDAGFTGP  199\n",
      "\n",
      "Query  101  DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCE--RSEVGICLSTSGRWVLNNEHYR  155\n",
      "              SII    + +EG ++   T      R+G  E      G+ L   G      E YR\n",
      "Sbjct  200  HDSII---GSAIEGPLQRFLT--ERPHRYGVAEGRAELNGVALHFEGGKATAAERYR  251\n",
      "\n",
      "\n",
      ">2Z06_A Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      " 2Z06_B Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      " 2Z06_C Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      " 2Z06_D Crystal structure of uncharacterized conserved protein from Thermus \n",
      "thermophilus [Thermus thermophilus HB8]\n",
      "Length=252\n",
      "\n",
      " Score = 168 bits (425),  Expect = 1e-52, Method: Composition-based stats.\n",
      " Identities = 32/174 (18%), Positives = 55/174 (32%), Gaps = 28/174 (16%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTV-LR-AINGD---FLHFLPRVF-SAVGNICYTPSKLIEYSDFATS  54\n",
      "            + R + +  PG PG    R  + G+   F+  + R+F   + +      +L+E       \n",
      "Sbjct  82   VVRPLNYP-PGTPGKGFWRLEVGGESLLFVQVMGRIFMDPLDDPFRALDRLLEEEKADYV  140\n",
      "\n",
      "Query  55   ACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYVLM  100\n",
      "               + AE T  K A+                 VP    T L +G++     + D      \n",
      "Sbjct  141  LVEVHAEATSEKMALAHYLDGRASAVLGTHTHVPTLDATRLPKGTL----YQTDVGMTGT  196\n",
      "\n",
      "Query  101  DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCERSEVGICLSTSGRWVLNNEHY  154\n",
      "              SII      +E  +    T   +  R    +       L   G   +    Y\n",
      "Sbjct  197  YHSII---GGEVETFLARFLTGRPQPFRAAQGKARFHATELVFEGGRPVAISPY  247\n",
      "\n",
      "\n",
      ">2CV9_A Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      " 2CV9_B Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      " 2CV9_C Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      " 2CV9_D Crystal structure of a hypothetical protein from Thermus thermophilus \n",
      "HB8 [Thermus thermophilus HB8]\n",
      "Length=252\n",
      "\n",
      " Score = 161 bits (408),  Expect = 5e-50, Method: Composition-based stats.\n",
      " Identities = 32/174 (18%), Positives = 54/174 (31%), Gaps = 28/174 (16%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTV-LR-AINGD---FLHFLPRVF-SAVGNICYTPSKLIEYSDFATS  54\n",
      "            + R + +  PG PG    R  + G+   F+    R+F   + +      +L+E       \n",
      "Sbjct  82   VVRPLNYP-PGTPGKGFWRLEVGGESLLFVQVXGRIFXDPLDDPFRALDRLLEEEKADYV  140\n",
      "\n",
      "Query  55   ACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYVLM  100\n",
      "               + AE T  K A+                 VP    T L +G++     + D      \n",
      "Sbjct  141  LVEVHAEATSEKXALAHYLDGRASAVLGTHTHVPTLDATRLPKGTL----YQTDVGXTGT  196\n",
      "\n",
      "Query  101  DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCERSEVGICLSTSGRWVLNNEHY  154\n",
      "              SII      +E  +    T   +  R    +       L   G   +    Y\n",
      "Sbjct  197  YHSII---GGEVETFLARFLTGRPQPFRAAQGKARFHATELVFEGGRPVAISPY  247\n",
      "\n",
      "\n",
      ">4B2O_A Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      " 4B2O_B Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      " 4B2O_C Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      " 4B2O_D Crystal structure of Bacillus subtilis YmdB, a global regulator \n",
      "of late adaptive responses. [Bacillus subtilis subsp. subtilis \n",
      "str. 168]\n",
      "Length=286\n",
      "\n",
      " Score = 131 bits (329),  Expect = 8e-38, Method: Composition-based stats.\n",
      " Identities = 28/177 (16%), Positives = 49/177 (28%), Gaps = 32/177 (18%)\n",
      "\n",
      "Query  1    ITREIGFIVPGLPGTVLRAI--NGDFL---HFLPRVF-SAVGNICYTPSKLIEYSDFAT-  53\n",
      "            + R   F   G PG  +  +  NG  L   +   R F   + +      +LI  +   T \n",
      "Sbjct  85   LVRPANFPE-GTPGKGITYVKANGKELAVINLQGRTFLPPLDDPFLKADELIAEAAKRTP  143\n",
      "\n",
      "Query  54   -SACVLAAECTIFKDAMG--------------KPVPYCYDTNLLEGSISYSELRPDTRYV  98\n",
      "                   AE T  K A+G                V    D  +L    +Y     D    \n",
      "Sbjct  144  YIFIDFHAEATSEKLALGWYTDGRASAVVGTHTHVQTA-DNRILPKGTAYI---TDVGMT  199\n",
      "\n",
      "Query  99   LMDGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCERSEVGICLST--SGRWVLNNEH  153\n",
      "                 I+      +   ++   T           + +  G+ +      +  +  E \n",
      "Sbjct  200  GPYDGILGMDRETI---IKRFKTNLPVRFTVAEGKTTLSGVVIDIDDQTKKAVKIER  253\n",
      "\n",
      "\n",
      ">1T71_A Crystal structure of a novel phosphatase Mycoplasma pneumoniaefrom \n",
      "[Mycoplasma pneumoniae M129]\n",
      "Length=281\n",
      "\n",
      " Score = 113 bits (282),  Expect = 7e-31, Method: Composition-based stats.\n",
      " Identities = 23/118 (19%), Positives = 32/118 (27%), Gaps = 22/118 (19%)\n",
      "\n",
      "Query  37   NICYTPSKLIEYSDFATSACVLAAECTIFKDAMG--------------KPVPYCYDTNLL  82\n",
      "            N      +LI   D         AE T  K+A                  VP   D  + \n",
      "Sbjct  136  NPFKVLKELILKRDCDLHIVDFHAETTSEKNAFCMAFDGYVTTIFGTHTHVPSA-DLRIT  194\n",
      "\n",
      "Query  83   EGSISYSELRPDTRYVLM-DGSIIQFPNTYLEGSVRVVTTFDAEYCRHGTCERSEVGI  139\n",
      "                +Y     D        GS+I       E S+R+      E+     C     G+\n",
      "Sbjct  195  PKGSAYI---TDVGMCGPGFGSVI---GANPEQSIRLFCAGSREHFEVSKCGAQLNGV  246\n",
      "\n",
      "\n",
      "\n",
      "Lambda      K        H        a         alpha\n",
      "   0.318    0.130    0.335    0.792     4.96 \n",
      "\n",
      "Gapped\n",
      "Lambda      K        H        a         alpha    sigma\n",
      "   0.267   0.0398    0.140     1.90     42.6     43.6 \n",
      "\n",
      "Effective search space used: 1206105420\n",
      "\n",
      "Search has CONVERGED!\n",
      "\n",
      "\n",
      "  Database: PDB protein database\n",
      "    Posted date:  Dec 10, 2020  3:46 AM\n",
      "  Number of letters in database: 31,663,421\n",
      "  Number of sequences in database:  120,434\n",
      "\n",
      "\n",
      "\n",
      "Matrix: BLOSUM62\n",
      "Gap Penalties: Existence: 11, Extension: 1\n",
      "Neighboring words threshold: 11\n",
      "Window for multiple hits: 40\n"
     ]
    }
   ],
   "source": [
    "#!update_blastdb.pl --decompress pdbaa #to download the pdbaa database locally\n",
    "!psiblast -db pdbaa -query \"NSP4.faa\" -inclusion_ethresh 1e-3 -evalue 1e-3 -num_iterations 0"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Template Selection\n",
    "\n",
    "Protein **1T70 (chain 'A')** which is a ***Deinococcus* phosphatase** is the hit with lowest E-value: 8e-55 and the highest sequence identity: ~30% (in round 4). The rest of the hits have sequence identity <20%. \n",
    "We will use 1T70A as a template for our model."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2. Refining Target-Template Alignment\n",
    "Get the `.pdb` file for the protein 1T70."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false,
    "jupyter": {
     "outputs_hidden": false
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Structure exists: '/Users/shivamchitnis/Documents/GitHub/Modeller/t7/pdb1t70.ent' \n"
     ]
    }
   ],
   "source": [
    "pdb_file = PDBList().retrieve_pdb_file('1T70',file_format='pdb')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "jupyter": {
     "outputs_hidden": false
    }
   },
   "source": [
    "Align the template and the target sequence precisely."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkapsa__637W> No residue topology library is in memory.\n",
      "              Better radii would be used if topology.read() is called first.\n",
      "iup2crm_280W> No topology library in memory or assigning a BLK residue.\n",
      "              Default CHARMM atom type assigned:  N -->  N\n",
      "              This message is written only for the first such atom.\n",
      "\n",
      "Pairwise dynamic programming alignment (ALIGN2D):\n",
      "  Residue-residue metric   : $(LIB)/as1.sim.mat\n",
      "  Diagonal                 :          100\n",
      "  Overhang                 :            0\n",
      "  Maximal gap length       :       999999\n",
      "  Local alignment          :            F\n",
      "  MATRIX_OFFSET (local aln):       0.0000\n",
      "  FIX_OFFSETS              :      0.0    -1.0    -2.0    -3.0    -4.0\n",
      "  N_SUBOPT                 :            0\n",
      "  SUBOPT_OFFSET            :       0.0000\n",
      "  Alignment block          :            1\n",
      "  Gap introduction penalty :    -100.0000\n",
      "  Gap extension penalty    :       0.0000\n",
      "  Gap diagonal penalty     :       0.0000\n",
      "  Structure gap penalties  :    3.500   3.500   3.500   0.200   4.000   6.500   2.000   0.000\n",
      "  Break-break bonus        :   10000.0000\n",
      "  Length of alignment      :          259\n",
      "  Score                    :   84183.5000\n"
     ]
    }
   ],
   "source": [
    "hg.precise_align(env,targ='NSP4',tmp='1t70',tmp_pdb=pdb_file,model_segment=('FIRST:A','LAST:A'))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Observe the alignment of the template 17T0A and the target NSP4."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": false,
    "jupyter": {
     "outputs_hidden": false
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " _aln.pos         10        20        30        40        50        60\n",
      "1t70A     MRVLFIGDVFGQPGRRVLQNHLPTIRPQFDFVIVNMENSAGGFGMHRDAARGALEAGAGCLTLGNHAW \n",
      "NSP4      -------------------------------------------------------------------- \n",
      " _consrvd\n",
      "\n",
      " _aln.p   70        80        90       100       110       120       130\n",
      "1t70A     HHKDIYPMLSEDTYPIVRPLNYADPGTPGVGWRTFDVNGEKLTVVNLLGRVFMEAVDNPFRTMDALLE \n",
      "NSP4      ---------------ITREIGFIVPGLPGTVLRAIN-----GDFLHFLPRVFS-AVGNICYTPSKLIE \n",
      " _consrvd                * *      ** **   *              * ***  ** *   *   * *\n",
      "\n",
      " _aln.pos  140       150       160       170       180       190       200\n",
      "1t70A     RDDLGTVFVDFHAEATSEKEAMGWHLAG--RVAAVIGTHTHVPT-ADTRILK-GGTAYQTDAGFTGPH \n",
      "NSP4      YSDFATSACVLAAECTIFKDAMGKPVPYCYDTNLLEGSISYSELRPDTRYVLMDGSIIQFPNTYLE-- \n",
      " _consrvd   *  *      ** *  * ***             *         ***     *   *\n",
      "\n",
      " _aln.pos    210       220       230       240       250\n",
      "1t70A     DSIIGSAIEGPLQRFLTERPHRYGVAEGRAELNGVALHFEGGKATAAERYRFIED \n",
      "NSP4      ----GSV--RVVTTFDAE-YCRHGTCE-RSEV-GICLSTSGRWVLNNEHYR---- \n",
      " _consrvd     **        *  *   * *  * * *  *  *   *      * **\n"
     ]
    }
   ],
   "source": [
    "!cat NSP4-1t70A.pap"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3. Constructing Model \n",
    "### by Satisying Spatial Restraints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": false,
    "jupyter": {
     "outputs_hidden": false
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "The following 1 residues contain 6-membered rings with poor geometries\n",
      "after transfer from templates. Rebuilding rings from internal coordinates:\n",
      "   <Residue 88 (type TYR)>\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    12801    11716\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ARG:OXT  ARG\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      155\n",
      "Number of all, selected real atoms                :     1215    1215\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    11716   11716\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   192249\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -12802.9893\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -12802.989258\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : /usr/local/Cellar/modeller/9.25_1/modlib/surf5.de\n",
      "Pair library             : /usr/local/Cellar/modeller/9.25_1/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    30.323000\n",
      "Sequence length          :          155\n",
      "Compactness              :     0.254398\n",
      "Native energy (pair)     :    30.361987\n",
      "Native energy (surface)  :    13.738875\n",
      "Native energy (combined) :     4.539245\n",
      "Z score (pair)           :    -2.069588\n",
      "Z score (surface)        :     0.080161\n",
      "Z score (combined)       :    -1.389705\n",
      "GA341 score              :     0.099237\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      155\n",
      "Number of all, selected real atoms                :     1215    1215\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    11716   11716\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     2493\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1182.2100\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    1244       0      0   0.005   0.005      9.4306       1.000\n",
      " 2 Bond angle potential               :    1693       0      2   2.035   2.035      132.79       1.000\n",
      " 3 Stereochemical cosine torsion poten:     835       0     29  48.352  48.352      306.55       1.000\n",
      " 4 Stereochemical improper torsion pot:     532       0      0   1.257   1.257      17.867       1.000\n",
      " 5 Soft-sphere overlap restraints     :    2493       0      0   0.005   0.005      6.4783       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    2568       0      3   0.647   0.647      140.66       1.000\n",
      "10 Distance restraints 2 (N-O)        :    2785       0      0   0.698   0.698      168.65       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     154       0      0   4.102   4.102      30.560       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     135       0      2  87.438  87.438      49.628       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      92       0      1  85.198  85.198      45.026       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      34       0      0  61.536  61.536      24.906       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      13       0      0  96.272  96.272      8.7994       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    1284       0      0   0.691   0.691      37.931       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     153      24     15  37.287  67.819      179.62       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     194       0      1   1.391   1.391      23.305       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: NSP4.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   18545.1836\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4311   7F   8I C   N      57   59  -90.94 -120.60   67.73    2.89  -63.40  116.32   18.21\n",
      "    1          8I   8I N   CA     59   60   69.41  130.30                  -43.60\n",
      "    2   4314  10P  11G C   N      79   81  167.48   82.20   98.71    7.70  -62.40  163.75   23.96\n",
      "    2         11G  11G N   CA     81   82   58.21    8.50                  -41.20\n",
      "    3   4325  21N  22G C   N     156  158   96.30   78.70   43.66    0.66   82.20  146.12    7.80\n",
      "    3         22G  22G N   CA    158  159  153.94 -166.10                    8.50\n",
      "    4   4335  31V  32F C   N     241  243   58.77   58.10    3.52    0.23  -63.20  142.53   25.68\n",
      "    4         32F  32F N   CA    243  244   29.45   32.90                  -44.30\n",
      "    5   4337  33S  34A C   N     258  260 -143.13 -134.00   26.90    1.65  -68.20   78.57    7.36\n",
      "    5         34A  34A N   CA    260  261  121.69  147.00                  145.30\n",
      "    6   4353  49S  50D C   N     380  382 -100.81  -96.50   81.71    3.43  -63.30   81.72    8.67\n",
      "    6         50D  50D N   CA    382  383   32.61  114.20                  -40.00\n",
      "    7   4355  51F  52A C   N     399  401   70.14  -68.20  151.40   14.52  -62.50  173.79   27.39\n",
      "    7         52A  52A N   CA    401  402 -153.19  145.30                  -40.90\n",
      "    8   4378  74P  75Y C   N     558  560 -149.99 -124.30   37.50    1.62  -63.50  176.53   32.17\n",
      "    8         75Y  75Y N   CA    560  561  162.71  135.40                  -43.40\n",
      "    9   4379  75Y  76C C   N     570  572  -61.68  -63.00    1.53    0.20 -117.90 -174.25    8.00\n",
      "    9         76C  76C N   CA    572  573  -41.86  -41.10                  141.10\n",
      "   10   4380  76C  77Y C   N     576  578 -110.24  -98.40   41.41    4.19  -63.50  140.15   20.10\n",
      "   10         77Y  77Y N   CA    578  579   88.72  128.40                  -43.40\n",
      "   11   4381  77Y  78D C   N     588  590 -155.10  -63.30  103.09   17.60  -63.30  103.09   17.60\n",
      "   11         78D  78D N   CA    590  591  -86.91  -40.00                  -40.00\n",
      "   12   4389  85S  86I C   N     646  648  129.11  -63.40  167.50   29.89 -120.60 -154.84   15.53\n",
      "   12         86I  86I N   CA    648  649  -42.69  -43.60                  130.30\n",
      "   13   4391  87S  88Y C   N     660  662   61.54   55.90   26.66    1.48 -124.30 -147.38   17.59\n",
      "   13         88Y  88Y N   CA    662  663   13.44   39.50                  135.40\n",
      "   14   4403  99L 100M C   N     766  768   77.45  -63.40  141.13   25.57  -63.40  141.13   25.57\n",
      "   14        100M 100M N   CA    768  769  -49.40  -40.50                  -40.50\n",
      "   15   4404 100M 101D C   N     774  776  -60.97  -63.30    2.54    0.43   54.50  140.41   18.10\n",
      "   15        101D 101D N   CA    776  777  -38.99  -40.00                   40.90\n",
      "   16   4411 107F 108P C   N     828  830 -124.76  -64.50   60.86    6.50  -64.50   60.86    6.50\n",
      "   16        108P 108P N   CA    830  831  138.72  147.20                  147.20\n",
      "   17   4415 111Y 112L C   N     862  864  -71.98  -63.50   11.27    1.48 -108.50  170.24    8.77\n",
      "   17        112L 112L N   CA    864  865  -33.78  -41.20                  132.50\n",
      "   18   4416 112L 113E C   N     870  872  -57.56  -63.60   25.18    3.07 -117.80  169.52    9.25\n",
      "   18        113E 113E N   CA    872  873  -64.75  -40.30                  136.80\n",
      "   19   4418 114G 115S C   N     883  885  -92.18  -72.40   23.25    1.24  -64.10  162.82   13.66\n",
      "   19        115S 115S N   CA    885  886  164.62  152.40                  -35.00\n",
      "   20   4420 116V 117R C   N     896  898 -111.76 -125.20   24.01    0.81  -63.00  168.99   20.03\n",
      "   20        117R 117R N   CA    898  899  120.71  140.60                  -41.10\n",
      "   21   4421 117R 118V C   N     907  909  -79.92  -73.50   25.22    2.05  -62.40  155.00   20.67\n",
      "   21        118V 118V N   CA    909  910  163.59  139.20                  -42.40\n",
      "   22   4426 122F 123D C   N     946  948  -65.65  -96.50   79.76    3.31  -63.30  132.27   16.47\n",
      "   22        123D 123D N   CA    948  949 -172.24  114.20                  -40.00\n",
      "   23   4428 124A 125E C   N     959  961 -119.33 -117.80   18.44    0.90  -63.60  173.70   26.31\n",
      "   23        125E 125E N   CA    961  962  155.18  136.80                  -40.30\n",
      "   24   4441 137V 138G C   N    1066 1068  -69.12  -62.40    7.92    1.50 -167.20  170.95    6.78\n",
      "   24        138G 138G N   CA   1068 1069  -45.39  -41.20                  174.60\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    8   19   78   89  122  133  131  161  179  187\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ARG:OXT  ARG\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      155\n",
      "Number of all, selected real atoms                :     1215    1215\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    11716   11716\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   187888\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -12878.1895\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -12878.189453\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : /usr/local/Cellar/modeller/9.25_1/modlib/surf5.de\n",
      "Pair library             : /usr/local/Cellar/modeller/9.25_1/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    30.323000\n",
      "Sequence length          :          155\n",
      "Compactness              :     0.136425\n",
      "Native energy (pair)     :    34.800232\n",
      "Native energy (surface)  :    10.799020\n",
      "Native energy (combined) :     3.895143\n",
      "Z score (pair)           :    -1.381103\n",
      "Z score (surface)        :    -0.820283\n",
      "Z score (combined)       :    -1.367832\n",
      "GA341 score              :     0.077455\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      155\n",
      "Number of all, selected real atoms                :     1215    1215\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    11716   11716\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     2456\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1124.6552\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    1244       0      0   0.005   0.005      8.6199       1.000\n",
      " 2 Bond angle potential               :    1693       0      2   2.133   2.133      146.60       1.000\n",
      " 3 Stereochemical cosine torsion poten:     835       0     25  49.066  49.066      321.04       1.000\n",
      " 4 Stereochemical improper torsion pot:     532       0      0   1.317   1.317      19.398       1.000\n",
      " 5 Soft-sphere overlap restraints     :    2456       0      0   0.004   0.004      4.5783       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    2568       0      3   0.585   0.585      111.82       1.000\n",
      "10 Distance restraints 2 (N-O)        :    2785       0      1   0.679   0.679      157.44       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     154       0      1   4.371   4.371      34.698       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     135       0      0  86.755  86.755      45.301       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      92       0      2  73.695  73.695      43.312       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      34       0      0  61.481  61.481      19.576       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      13       0      0 102.038 102.038      6.5032       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    1284       0      0   0.568   0.568      30.227       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     153      19     18  36.563  64.448      157.88       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     194       0      0   1.521   1.521      17.661       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: NSP4.V99990002\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   17383.8242\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4325  21N  22G C   N     156  158   96.11   78.70   41.81    0.63   82.20  148.04    7.88\n",
      "    1         22G  22G N   CA    158  159  155.89 -166.10                    8.50\n",
      "    2   4335  31V  32F C   N     241  243   58.82   58.10    0.86    0.13  -63.20  144.64   26.09\n",
      "    2         32F  32F N   CA    243  244   33.38   32.90                  -44.30\n",
      "    3   4337  33S  34A C   N     258  260 -145.03 -134.00   22.90    1.41  -68.20   79.00    7.26\n",
      "    3         34A  34A N   CA    260  261  126.94  147.00                  145.30\n",
      "    4   4353  49S  50D C   N     380  382 -104.51  -96.50   71.66    3.00  -63.30   92.65    9.84\n",
      "    4         50D  50D N   CA    382  383   42.99  114.20                  -40.00\n",
      "    5   4355  51F  52A C   N     399  401   71.95  -68.20  153.40   14.72  -62.50  174.62   27.59\n",
      "    5         52A  52A N   CA    401  402 -152.32  145.30                  -40.90\n",
      "    6   4378  74P  75Y C   N     558  560  173.71 -124.30   62.57    4.42  -63.50 -148.14   39.28\n",
      "    6         75Y  75Y N   CA    560  561  143.94  135.40                  -43.40\n",
      "    7   4379  75Y  76C C   N     570  572  -60.46  -63.00    5.84    0.61 -117.90 -178.15    7.86\n",
      "    7         76C  76C N   CA    572  573  -46.36  -41.10                  141.10\n",
      "    8   4389  85S  86I C   N     646  648  129.92  -63.40  166.83   29.57 -120.60 -160.44   15.16\n",
      "    8         86I  86I N   CA    648  649  -36.55  -43.60                  130.30\n",
      "    9   4391  87S  88Y C   N     660  662   74.18   55.90   47.16    2.28 -124.30 -146.66   17.43\n",
      "    9         88Y  88Y N   CA    662  663   -3.97   39.50                  135.40\n",
      "   10   4403  99L 100M C   N     766  768  -17.75  -73.00   84.55    5.31  -63.40  127.92   22.72\n",
      "   10        100M 100M N   CA    768  769   78.99  143.00                  -40.50\n",
      "   11   4404 100M 101D C   N     774  776 -158.26  -63.30   96.75   14.36   54.50  159.92   14.59\n",
      "   11        101D 101D N   CA    776  777  -21.51  -40.00                   40.90\n",
      "   12   4418 114G 115S C   N     883  885 -122.18 -136.60   23.55    0.82  -64.10  177.36   10.92\n",
      "   12        115S 115S N   CA    885  886  132.58  151.20                  -35.00\n",
      "   13   4421 117R 118V C   N     907  909 -113.26 -125.40   17.68    0.50  -62.40 -179.82   20.37\n",
      "   13        118V 118V N   CA    909  910  130.45  143.30                  -42.40\n",
      "   14   4422 118V 119V C   N     914  916 -150.75  -62.40   91.18   15.29  -62.40   91.18   15.29\n",
      "   14        119V 119V N   CA    916  917  -64.91  -42.40                  -42.40\n",
      "   15   4426 122F 123D C   N     946  948  -58.66  -96.50   61.98    2.55  -63.30  156.78   19.08\n",
      "   15        123D 123D N   CA    948  949  163.29  114.20                  -40.00\n",
      "   16   4428 124A 125E C   N     959  961  -74.77  -69.30   14.81    0.93  -63.60  163.81   22.46\n",
      "   16        125E 125E N   CA    961  962  156.27  142.50                  -40.30\n",
      "   17   4438 134R 135S C   N    1044 1046  -73.35  -64.10   14.54    1.09 -136.60 -173.94    7.90\n",
      "   17        135S 135S N   CA   1046 1047  -23.78  -35.00                  151.20\n",
      "   18   4448 144S 145G C   N    1111 1113   63.40   78.70   22.60    0.47   82.20  159.09    8.66\n",
      "   18        145G 145G N   CA   1113 1114 -149.47 -166.10                    8.50\n",
      "   19   4449 145G 146R C   N    1115 1117 -107.70 -125.20   96.05    4.23   57.30  165.20   18.86\n",
      "   19        146R 146R N   CA   1117 1118   46.16  140.60                   38.00\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    7   23  101   85  119  119  147  126  178  184\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ARG:OXT  ARG\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      155\n",
      "Number of all, selected real atoms                :     1215    1215\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    11716   11716\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   191934\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -12980.0156\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -12980.015625\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : /usr/local/Cellar/modeller/9.25_1/modlib/surf5.de\n",
      "Pair library             : /usr/local/Cellar/modeller/9.25_1/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    30.323000\n",
      "Sequence length          :          155\n",
      "Compactness              :     0.228891\n",
      "Native energy (pair)     :    24.688493\n",
      "Native energy (surface)  :     8.726888\n",
      "Native energy (combined) :     3.167280\n",
      "Z score (pair)           :    -1.968432\n",
      "Z score (surface)        :    -1.105710\n",
      "Z score (combined)       :    -1.899608\n",
      "GA341 score              :     0.153011\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      155\n",
      "Number of all, selected real atoms                :     1215    1215\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    11716   11716\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     2514\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1194.4651\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    1244       0      0   0.006   0.006      11.566       1.000\n",
      " 2 Bond angle potential               :    1693       0      2   2.227   2.227      157.80       1.000\n",
      " 3 Stereochemical cosine torsion poten:     835       0     36  49.353  49.353      319.32       1.000\n",
      " 4 Stereochemical improper torsion pot:     532       0      0   1.365   1.365      21.209       1.000\n",
      " 5 Soft-sphere overlap restraints     :    2514       0      0   0.005   0.005      7.6494       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    2568       0      3   0.541   0.541      116.24       1.000\n",
      "10 Distance restraints 2 (N-O)        :    2785       0      4   0.661   0.661      181.80       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     154       0      0   4.391   4.391      35.018       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     135       0      1  80.584  80.584      42.442       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      92       0      1  75.833  75.833      45.619       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      34       0      0  80.354  80.354      29.953       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      13       0      0 125.387 125.387      8.8319       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    1284       0      1   0.617   0.617      37.665       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     153      21     23  36.065  67.347      158.18       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     194       0      2   1.111   1.111      21.180       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: NSP4.V99990003\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   18077.3750\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4316  12L  13P C   N      91   93  -56.93  -58.70   27.73    2.15  -64.50  154.82   11.96\n",
      "    1         13P  13P N   CA     93   94  -58.17  -30.50                  147.20\n",
      "    2   4317  13P  14G C   N      98  100  -66.58  -80.20   18.83    1.34   78.70  145.44    7.50\n",
      "    2         14G  14G N   CA    100  101 -172.90  174.10                 -166.10\n",
      "    3   4325  21N  22G C   N     156  158   91.09   78.70   31.81    0.49   82.20  156.35    8.04\n",
      "    3         22G  22G N   CA    158  159  164.60 -166.10                    8.50\n",
      "    4   4335  31V  32F C   N     241  243   60.11   58.10    5.29    0.28  -63.20  142.94   25.74\n",
      "    4         32F  32F N   CA    243  244   28.00   32.90                  -44.30\n",
      "    5   4337  33S  34A C   N     258  260 -150.93 -134.00   21.03    1.16  -68.20   83.43    7.42\n",
      "    5         34A  34A N   CA    260  261  134.53  147.00                  145.30\n",
      "    6   4353  49S  50D C   N     380  382 -110.63  -96.50   65.87    2.75  -63.30  101.56   10.77\n",
      "    6         50D  50D N   CA    382  383   49.87  114.20                  -40.00\n",
      "    7   4355  51F  52A C   N     399  401   77.82  -68.20  160.95   15.48  -62.50  175.91   28.13\n",
      "    7         52A  52A N   CA    401  402 -146.99  145.30                  -40.90\n",
      "    8   4378  74P  75Y C   N     558  560  168.37 -124.30   67.34    5.03  -63.50 -139.15   30.47\n",
      "    8         75Y  75Y N   CA    560  561  136.48  135.40                  -43.40\n",
      "    9   4379  75Y  76C C   N     570  572  -61.62  -63.00    2.44    0.26 -117.90 -175.43    7.96\n",
      "    9         76C  76C N   CA    572  573  -43.11  -41.10                  141.10\n",
      "   10   4389  85S  86I C   N     646  648  118.49  -63.40  179.24   31.34 -120.60 -164.33   15.18\n",
      "   10         86I  86I N   CA    648  649  -23.54  -43.60                  130.30\n",
      "   11   4391  87S  88Y C   N     660  662   72.12   55.90   34.57    1.65 -124.30 -153.25   17.02\n",
      "   11         88Y  88Y N   CA    662  663    8.97   39.50                  135.40\n",
      "   12   4402  98V  99L C   N     758  760  -61.00  -63.50    2.51    0.43  -70.70  177.34   13.81\n",
      "   12         99L  99L N   CA    760  761  -41.33  -41.20                  141.60\n",
      "   13   4403  99L 100M C   N     766  768  -66.39  -73.00   37.86    2.46  -63.40  146.25   21.60\n",
      "   13        100M 100M N   CA    768  769  105.72  143.00                  -40.50\n",
      "   14   4411 107F 108P C   N     828  830 -125.10  -64.50   61.48    6.62  -64.50   61.48    6.62\n",
      "   14        108P 108P N   CA    830  831  136.89  147.20                  147.20\n",
      "   15   4415 111Y 112L C   N     862  864  -65.10  -63.50   12.46    1.83 -108.50  179.27   10.06\n",
      "   15        112L 112L N   CA    864  865  -53.56  -41.20                  132.50\n",
      "   16   4417 113E 114G C   N     879  881  152.31 -167.20   41.07    0.64   82.20 -173.34   12.33\n",
      "   16        114G 114G N   CA    881  882 -178.51  174.60                    8.50\n",
      "   17   4418 114G 115S C   N     883  885  -86.93  -72.40   53.76    2.70  -64.10  122.98   10.43\n",
      "   17        115S 115S N   CA    885  886 -155.84  152.40                  -35.00\n",
      "   18   4420 116V 117R C   N     896  898  -71.10 -125.20   60.73    1.84  -63.00  154.32   20.31\n",
      "   18        117R 117R N   CA    898  899  113.01  140.60                  -41.10\n",
      "   19   4428 124A 125E C   N     959  961   73.08   54.60   41.02    2.09  -63.60  144.24   24.48\n",
      "   19        125E 125E N   CA    961  962    5.78   42.40                  -40.30\n",
      "   20   4448 144S 145G C   N    1111 1113   62.05   78.70   30.23    0.49   82.20  150.73    8.32\n",
      "   20        145G 145G N   CA   1113 1114 -140.87 -166.10                    8.50\n",
      "   21   4449 145G 146R C   N    1115 1117 -112.90  -63.00   89.91   10.19   57.30  170.26   19.97\n",
      "   21        146R 146R N   CA   1117 1118   33.69  -41.10                   38.00\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    8   19   76   91  121  135  123  163  170  203\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ARG:OXT  ARG\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      155\n",
      "Number of all, selected real atoms                :     1215    1215\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    11716   11716\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   188014\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -12621.9756\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -12621.975586\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : /usr/local/Cellar/modeller/9.25_1/modlib/surf5.de\n",
      "Pair library             : /usr/local/Cellar/modeller/9.25_1/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    30.323000\n",
      "Sequence length          :          155\n",
      "Compactness              :     0.159532\n",
      "Native energy (pair)     :    28.502182\n",
      "Native energy (surface)  :    11.255480\n",
      "Native energy (combined) :     3.956695\n",
      "Z score (pair)           :    -1.540629\n",
      "Z score (surface)        :    -0.576348\n",
      "Z score (combined)       :    -1.316537\n",
      "GA341 score              :     0.077450\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      155\n",
      "Number of all, selected real atoms                :     1215    1215\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    11716   11716\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     2527\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1176.8257\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    1244       0      0   0.005   0.005      10.082       1.000\n",
      " 2 Bond angle potential               :    1693       0      0   2.054   2.054      134.26       1.000\n",
      " 3 Stereochemical cosine torsion poten:     835       0     32  48.563  48.563      308.20       1.000\n",
      " 4 Stereochemical improper torsion pot:     532       0      0   1.250   1.250      17.995       1.000\n",
      " 5 Soft-sphere overlap restraints     :    2527       0      0   0.005   0.005      7.4069       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    2568       0      4   0.538   0.538      115.25       1.000\n",
      "10 Distance restraints 2 (N-O)        :    2785       0      4   0.694   0.694      181.57       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     154       0      1   4.203   4.203      32.079       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     135       0      3  88.193  88.193      49.516       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      92       0      0  81.953  81.953      38.392       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      34       0      1  78.828  78.828      31.274       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      13       0      0 108.103 108.103      8.2430       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    1284       0      1   0.754   0.754      41.469       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     153      25     22  38.133  71.171      180.84       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     194       0      0   1.445   1.445      20.247       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: NSP4.V99990004\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   18897.1172\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4311   7F   8I C   N      57   59  -97.14 -120.60   75.73    3.48  -63.40  107.34   16.60\n",
      "    1          8I   8I N   CA     59   60   58.30  130.30                  -43.60\n",
      "    2   4325  21N  22G C   N     156  158   97.84   78.70   43.38    0.64   82.20  147.30    7.93\n",
      "    2         22G  22G N   CA    158  159  154.97 -166.10                    8.50\n",
      "    3   4335  31V  32F C   N     241  243   62.35   58.10    8.64    0.44  -63.20  143.59   25.83\n",
      "    3         32F  32F N   CA    243  244   25.38   32.90                  -44.30\n",
      "    4   4337  33S  34A C   N     258  260 -141.11 -134.00   26.97    1.64  -68.20   76.86    7.23\n",
      "    4         34A  34A N   CA    260  261  120.98  147.00                  145.30\n",
      "    5   4353  49S  50D C   N     380  382 -104.13  -96.50   79.65    3.34  -63.30   85.32    9.05\n",
      "    5         50D  50D N   CA    382  383   34.92  114.20                  -40.00\n",
      "    6   4355  51F  52A C   N     399  401   73.96  -68.20  155.63   14.93  -62.50  175.58   27.83\n",
      "    6         52A  52A N   CA    401  402 -151.38  145.30                  -40.90\n",
      "    7   4378  74P  75Y C   N     558  560 -163.87 -124.30   49.59    2.45  -63.50 -178.42   33.51\n",
      "    7         75Y  75Y N   CA    560  561  165.28  135.40                  -43.40\n",
      "    8   4379  75Y  76C C   N     570  572  -59.28  -63.00   11.54    1.22 -117.90  176.88    7.67\n",
      "    8         76C  76C N   CA    572  573  -52.02  -41.10                  141.10\n",
      "    9   4380  76C  77Y C   N     576  578 -113.09  -98.40   52.62    5.34  -63.50  131.03   18.59\n",
      "    9         77Y  77Y N   CA    578  579   77.88  128.40                  -43.40\n",
      "   10   4381  77Y  78D C   N     588  590 -134.31  -63.30   79.31   13.54  -63.30   79.31   13.54\n",
      "   10         78D  78D N   CA    590  591  -75.31  -40.00                  -40.00\n",
      "   11   4389  85S  86I C   N     646  648  120.72  -63.40  176.64   32.05 -120.60 -152.83    8.52\n",
      "   11         86I  86I N   CA    648  649  -59.89  -43.60                  130.30\n",
      "   12   4390  86I  87S C   N     654  656  -62.02  -72.40   11.11    0.66  -64.10  176.60   12.67\n",
      "   12         87S  87S N   CA    656  657  148.41  152.40                  -35.00\n",
      "   13   4391  87S  88Y C   N     660  662  -72.81  -63.50    9.39    1.56 -124.30 -175.07    7.86\n",
      "   13         88Y  88Y N   CA    662  663  -42.22  -43.40                  135.40\n",
      "   14   4403  99L 100M C   N     766  768  -30.50  -73.00   71.44    4.46  -63.40  130.30   22.08\n",
      "   14        100M 100M N   CA    768  769   85.58  143.00                  -40.50\n",
      "   15   4404 100M 101D C   N     774  776 -141.51  -63.30   80.85   11.68   54.50  174.77   16.58\n",
      "   15        101D 101D N   CA    776  777  -19.52  -40.00                   40.90\n",
      "   16   4411 107F 108P C   N     828  830 -126.50  -64.50   63.09    6.82  -64.50   63.09    6.82\n",
      "   16        108P 108P N   CA    830  831  135.51  147.20                  147.20\n",
      "   17   4417 113E 114G C   N     879  881 -173.51 -167.20   19.17    0.79   82.20 -178.95   13.28\n",
      "   17        114G 114G N   CA    881  882  156.50  174.60                    8.50\n",
      "   18   4418 114G 115S C   N     883  885  -81.22  -72.40    9.21    0.72  -64.10  176.08   13.85\n",
      "   18        115S 115S N   CA    885  886  149.75  152.40                  -35.00\n",
      "   19   4419 115S 116V C   N     889  891  -70.40  -62.40    8.04    1.24 -125.40 -176.46   10.20\n",
      "   19        116V 116V N   CA    891  892  -41.59  -42.40                  143.30\n",
      "   20   4420 116V 117R C   N     896  898  -57.04 -125.20   71.06    2.36  -63.00  161.70   22.25\n",
      "   20        117R 117R N   CA    898  899  120.49  140.60                  -41.10\n",
      "   21   4421 117R 118V C   N     907  909 -148.52 -125.40   78.52    3.24  -62.40  131.42   21.67\n",
      "   21        118V 118V N   CA    909  910 -141.66  143.30                  -42.40\n",
      "   22   4424 120T 121T C   N     928  930  -72.41  -78.10   10.80    0.65  -63.20  159.18   21.02\n",
      "   22        121T 121T N   CA    930  931  158.98  149.80                  -42.10\n",
      "   23   4425 121T 122F C   N     935  937   63.80   58.10   15.26    0.80  -63.20  141.79   25.41\n",
      "   23        122F 122F N   CA    937  938   18.75   32.90                  -44.30\n",
      "   24   4426 122F 123D C   N     946  948 -134.21  -96.50  107.54    4.53  -63.30  126.78   20.15\n",
      "   24        123D 123D N   CA    948  949 -145.09  114.20                  -40.00\n",
      "   25   4428 124A 125E C   N     959  961  -74.88  -69.30   21.67    1.45  -63.60  156.66   21.51\n",
      "   25        125E 125E N   CA    961  962  163.44  142.50                  -40.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   13   17   80   95  138  144  145  143  166  201\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ARG:OXT  ARG\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      155\n",
      "Number of all, selected real atoms                :     1215    1215\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    11716   11716\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   188772\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -12692.2764\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -12692.276367\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : /usr/local/Cellar/modeller/9.25_1/modlib/surf5.de\n",
      "Pair library             : /usr/local/Cellar/modeller/9.25_1/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    30.323000\n",
      "Sequence length          :          155\n",
      "Compactness              :     0.259114\n",
      "Native energy (pair)     :    36.159579\n",
      "Native energy (surface)  :     9.469758\n",
      "Native energy (combined) :     3.726784\n",
      "Z score (pair)           :    -2.079117\n",
      "Z score (surface)        :    -0.910826\n",
      "Z score (combined)       :    -1.842813\n",
      "GA341 score              :     0.152793\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      155\n",
      "Number of all, selected real atoms                :     1215    1215\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    11716   11716\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     2461\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1093.8330\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    1244       0      0   0.005   0.005      8.4293       1.000\n",
      " 2 Bond angle potential               :    1693       0      3   2.028   2.028      132.68       1.000\n",
      " 3 Stereochemical cosine torsion poten:     835       0     35  49.304  49.304      319.15       1.000\n",
      " 4 Stereochemical improper torsion pot:     532       0      0   1.220   1.220      16.707       1.000\n",
      " 5 Soft-sphere overlap restraints     :    2461       0      0   0.004   0.004      5.2847       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    2568       0      3   0.582   0.582      109.42       1.000\n",
      "10 Distance restraints 2 (N-O)        :    2785       0      0   0.632   0.632      136.07       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     154       0      0   4.208   4.208      32.164       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     135       0      1  81.603  81.603      44.399       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      92       0      2  72.132  72.132      36.215       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      34       0      0  67.653  67.653      24.098       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      13       0      0 117.872 117.872      9.0883       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    1284       0      0   0.659   0.659      30.081       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     153      20     19  38.291  64.889      163.32       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     194       0      1   2.002   2.002      26.742       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: NSP4.V99990005\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   16698.1777\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4325  21N  22G C   N     156  158   93.73   78.70   39.04    0.60   82.20  149.81    7.85\n",
      "    1         22G  22G N   CA    158  159  157.87 -166.10                    8.50\n",
      "    2   4335  31V  32F C   N     241  243   64.29   58.10   12.05    0.62  -63.20  143.95   25.85\n",
      "    2         32F  32F N   CA    243  244   22.55   32.90                  -44.30\n",
      "    3   4337  33S  34A C   N     258  260 -145.49 -134.00   27.47    1.70  -68.20   80.72    7.53\n",
      "    3         34A  34A N   CA    260  261  122.04  147.00                  145.30\n",
      "    4   4353  49S  50D C   N     380  382 -103.62  -96.50   76.53    3.21  -63.30   87.81    9.32\n",
      "    4         50D  50D N   CA    382  383   38.00  114.20                  -40.00\n",
      "    5   4355  51F  52A C   N     399  401   73.48  -68.20  155.26   14.90  -62.50  175.09   27.75\n",
      "    5         52A  52A N   CA    401  402 -151.20  145.30                  -40.90\n",
      "    6   4378  74P  75Y C   N     558  560  176.37 -124.30   59.44    4.36  -63.50 -145.49   39.64\n",
      "    6         75Y  75Y N   CA    560  561  138.88  135.40                  -43.40\n",
      "    7   4379  75Y  76C C   N     570  572  -57.79  -63.00    7.67    0.86 -117.90 -177.64    7.90\n",
      "    7         76C  76C N   CA    572  573  -46.73  -41.10                  141.10\n",
      "    8   4389  85S  86I C   N     646  648  129.69  -63.40  167.04   30.04 -120.60 -149.42    8.71\n",
      "    8         86I  86I N   CA    648  649  -49.95  -43.60                  130.30\n",
      "    9   4391  87S  88Y C   N     660  662   72.56   55.90   36.84    1.76 -124.30 -152.17   17.09\n",
      "    9         88Y  88Y N   CA    662  663    6.65   39.50                  135.40\n",
      "   10   4396  92R  93P C   N     706  708  -62.99  -64.50   35.55    2.57  -58.70  142.24   11.41\n",
      "   10         93P  93P N   CA    708  709  111.68  147.20                  -30.50\n",
      "   11   4397  93P  94D C   N     713  715  156.52  -63.30  142.20   21.30  -63.30  142.20   21.30\n",
      "   11         94D  94D N   CA    715  716  -16.12  -40.00                  -40.00\n",
      "   12   4403  99L 100M C   N     766  768    4.37  -73.00  112.18    7.09  -63.40  122.69   23.48\n",
      "   12        100M 100M N   CA    768  769   61.77  143.00                  -40.50\n",
      "   13   4404 100M 101D C   N     774  776 -151.56  -63.30   88.66   13.64   54.50  170.16   14.99\n",
      "   13        101D 101D N   CA    776  777  -31.62  -40.00                   40.90\n",
      "   14   4411 107F 108P C   N     828  830 -126.81  -64.50   62.74    6.66  -64.50   62.74    6.66\n",
      "   14        108P 108P N   CA    830  831  139.86  147.20                  147.20\n",
      "   15   4418 114G 115S C   N     883  885  -74.40  -72.40    4.04    0.19  -64.10  169.40   12.91\n",
      "   15        115S 115S N   CA    885  886  155.91  152.40                  -35.00\n",
      "   16   4420 116V 117R C   N     896  898 -133.63 -125.20   89.80    4.51  -63.00  116.22   13.28\n",
      "   16        117R 117R N   CA    898  899   51.20  140.60                  -41.10\n",
      "   17   4426 122F 123D C   N     946  948  -72.14  -96.50   49.77    2.06  -63.30  162.63   20.62\n",
      "   17        123D 123D N   CA    948  949  157.61  114.20                  -40.00\n",
      "   18   4428 124A 125E C   N     959  961  -68.71  -69.30    9.54    0.75  -63.60  167.75   22.65\n",
      "   18        125E 125E N   CA    961  962  152.02  142.50                  -40.30\n",
      "   19   4434 130G 131T C   N    1011 1013  -60.88  -63.20    3.75    0.40 -124.80 -177.01    9.67\n",
      "   19        131T 131T N   CA   1013 1014  -45.04  -42.10                  143.50\n",
      "   20   4436 132C 133E C   N    1024 1026  -68.52  -63.60    8.86    1.45 -117.80 -177.68    9.83\n",
      "   20        133E 133E N   CA   1026 1027  -47.67  -40.30                  136.80\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    9   25  102   85  117  140  133  147  171  205\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf     DOPE score    GA341 score\n",
      "----------------------------------------------------------------------\n",
      "NSP4.B99990001.pdb            1182.20996   -12802.98926        0.09924\n",
      "NSP4.B99990002.pdb            1124.65515   -12878.18945        0.07746\n",
      "NSP4.B99990003.pdb            1194.46509   -12980.01562        0.15301\n",
      "NSP4.B99990004.pdb            1176.82568   -12621.97559        0.07745\n",
      "NSP4.B99990005.pdb            1093.83301   -12692.27637        0.15279\n",
      "\n"
     ]
    }
   ],
   "source": [
    "hg.model_maker(env,tmp='1t70A',targ='NSP4')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 4. Evaluating Model and Template\n",
    "\n",
    "Evaluate the model and the template. To do this, construct the DOPE profile. \n",
    "\n",
    "### Model Selection\n",
    "We select `NSP4.B9999003.pdb` since it has the lowest overall DOPE score."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": false,
    "jupyter": {
     "outputs_hidden": false
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "read_to_681_> topology.submodel read from topology file:        3\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ARG:OXT  ARG\n",
      "preppdf_453W> No fixed restraints selected; there may be some dynamic ones.\n",
      "preppdf_454W> Restraints file was probably not read; use restraints.append().\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      155\n",
      "Number of all, selected real atoms                :     1215    1215\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :        0       0\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   191935\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -12980.2217\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 2 Bond angle potential               :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 3 Stereochemical cosine torsion poten:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 4 Stereochemical improper torsion pot:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "13 Mainchain Omega dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "25 Phi/Psi pair of dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "31 Non-bonded restraints              :  191689       0    352   6.710   6.710     -12980.       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "do_prof_492W> Are you sure you want to calculate a normalized energy profile!\n",
      "\n",
      "\n",
      "\n",
      "# Energy of each residue is written to: NSP4.profile\n",
      "# The profile IS normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:   13\n",
      "# The sum of all numbers in the file:      -5.3855\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -12980.221680\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  ASP:OXT  ASP\n",
      "preppdf_453W> No fixed restraints selected; there may be some dynamic ones.\n",
      "preppdf_454W> Restraints file was probably not read; use restraints.append().\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      255\n",
      "Number of all, selected real atoms                :     1972    1972\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :        0       0\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   397191\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -30381.8125\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 2 Bond angle potential               :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 3 Stereochemical cosine torsion poten:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 4 Stereochemical improper torsion pot:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "13 Mainchain Omega dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "25 Phi/Psi pair of dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "31 Non-bonded restraints              :  397025       0    205   6.843   6.843     -30382.       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "do_prof_492W> Are you sure you want to calculate a normalized energy profile!\n",
      "\n",
      "\n",
      "\n",
      "# Energy of each residue is written to: 1t70A.profile\n",
      "# The profile IS normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:   13\n",
      "# The sum of all numbers in the file:      -9.6286\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -30381.812500\n"
     ]
    }
   ],
   "source": [
    "hg.dope_evaluate(env,'NSP4.B99990003.pdb', 'NSP4.profile')\n",
    "hg.dope_evaluate(env, pdb_file, '1t70A.profile', model_segment=('FIRST:A', 'LAST:A'))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 5. Visualizing Protein Structure\n",
    "### Model - `NSP4.B99990003.pdb`"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "0b82cda31ef149fb8db989e569fa80bd",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "NGLWidget()"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "mdl_v = ngv.show_structure_file('NSP4.B99990003.pdb')\n",
    "mdl_v.display()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "mdl_v.render_image();"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "\n",
      "text/plain": [
       "<IPython.core.display.Image object>"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "mdl_v._display_image()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Template - 1T70 Chain A"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/usr/local/lib/python3.9/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain A is discontinuous at line 16535.\n",
      "  warnings.warn(\n",
      "/usr/local/lib/python3.9/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain B is discontinuous at line 16593.\n",
      "  warnings.warn(\n",
      "/usr/local/lib/python3.9/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain C is discontinuous at line 16649.\n",
      "  warnings.warn(\n",
      "/usr/local/lib/python3.9/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain D is discontinuous at line 16704.\n",
      "  warnings.warn(\n",
      "/usr/local/lib/python3.9/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain E is discontinuous at line 16743.\n",
      "  warnings.warn(\n",
      "/usr/local/lib/python3.9/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain F is discontinuous at line 16769.\n",
      "  warnings.warn(\n",
      "/usr/local/lib/python3.9/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain G is discontinuous at line 16811.\n",
      "  warnings.warn(\n",
      "/usr/local/lib/python3.9/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain H is discontinuous at line 16827.\n",
      "  warnings.warn(\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "a9410b02940642f8be494563082ea42d",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "NGLWidget()"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "parser = PDBParser()\n",
    "structure = parser.get_structure(\"1t70\", pdb_file)\n",
    "tmp_v = ngv.show_biopython(structure[0][\"A\"])\n",
    "tmp_v"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "tmp_v.render_image();"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "\n",
      "text/plain": [
       "<IPython.core.display.Image object>"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "tmp_v._display_image()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 6. Visualizing DOPE Profile"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAcgAAAEJCAYAAAAQFbf7AAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjMuMywgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/Il7ecAAAACXBIWXMAAAsTAAALEwEAmpwYAAB4BUlEQVR4nO2dd3gc5bX/PzOzVb1LtmzL3QZXjMF0B0IzxhRfQg2EQG4a4QZ+BGIgoYRODASSwCX3kkAMmJJLbEhsExNwaHaMce9dliWrd+1q28zvj9lZaa1dbVV/P8/Dg7U7O/Puq9Gc95z3nO+RNE3TEAgEAoFAEITc1wMQCAQCgaA/IgykQCAQCAQhEAZSIBAIBIIQCAMpEAgEAkEIhIEUCAQCgSAEwkAKBAKBQBACYSAFAoFAIAiBqbcvWFFRwd13301dXR1jxoxh8eLFpKamBh3jdru5//772b59OzabjcWLFzNu3Dja2tq47777OHjwIAA//OEPmT9/fkzXb2hoQ1XjK/3MzU2jrq41rs8OVsScdEXMSVfEnAQj5qMrfTEnsiyRnZ0a9v1eN5APP/ww119/PfPnz+f3v/89L774InfffXfQMUuWLMFut7Ny5Uq++uorFi1axLvvvssf/vAHhg8fzvPPP09dXR2XX345c+bMIS8vL+rrq6oWt4E0Pi8IRsxJV8ScdEXMSTBiPrrS3+akV0OsHo+Hr776iosuugiAhQsXsmrVqi7HrVmzhssuuwyAU045hYaGBioqKjj11FO58cYbAcjNzSUrK4va2tre+wICgUAgGDL0qgfZ0NBAWloaJpN+2fz8fKqqqrocV11dTX5+fuDn/Px8KisrOfPMMwOvrVixArfbzfjx43t+4AKBQCAYcvSYgVy5ciVPPPFE0GujR4/ucpwkSVGdT5Y7nN2VK1fy+OOP87//+78BYxstublpMR1/PPn56Ql9fjAi5qQrYk66IuYkGDEfXelvc9JjBnLevHnMmzcv6DWPx8OcOXPw+XwoikJNTQ0FBQVdPltQUEBNTQ0lJSUAQcctWbKEV155hVdeeYVJkybFPK66uta449z5+enU1LTE9dnBipiTrog56YqYk2DEfHSlL+ZElqVunaZe3YM0m83Mnj2bFStWALBs2TLOOeecLsfNnTuX5cuXA7BhwwasVivDhw/no48+4tVXX2Xp0qVxGUeBQCAQCKJF6u12V+Xl5SxatIi6ujqGDRvGs88+S2ZmJkuXLqW6upqf/vSnuFwuHnjgAbZv347FYuHRRx9lypQpXHbZZdTX15Obmxs436OPPsq0adOivr7wIJOLmJOuiDnpipiTYMR8dCXcnDS0uPD5VPKy7Em/ZiQPstcNZF8jDGRyEXPSFTEnXRFzEoyYj66EmhOvT+Xn/70Wt8fHb+/oGm1MlH4VYhUIBAKBIFr2H22iocVFeoqlT64vDKRAIBAI+iVbDuh17jPG50Y4smcQBlIgEAgE/ZLN++sAmDk+erW0ZNLrUnMCgUAgEIRC0zTe+/QgZdV6rkhVvYNUm4nxIzL7ZDzCQAoEAoGgX1Ba1cLf15YGvXbSxHwUuW+CncJACgQCgaBfsMUfUj1pQh7nzBiOIktMGJHVZ+MRBlIgEAgE/YLN+/WknHNmDGdGH+07dkYk6QgEAoGgz6lrclJa2YLFJHNCSXZfDwcQBlIgEAgE/YANu/TOTieOzsFiVvp4NDrCQAoEAoGgz1m/QzeQfVXzGAphIAUCgUDQp7g9PjbvqwFg+ri+33s0EAZSIBAIBH3KztIG3B4fJUXpZKdb+3o4AUQWq0AgGFL4VJWPN5bT4vAwuiidWRPze/yaLo+PjzcexenyAZCaYqHN4QagpDCdkyf1/Bj6M1v92aszxvWf8CoIAykQCIYY63dVs/SjfQBIwK9/fAY5GbYeveYnG8t595MDId8zKTIv/r9zMClDN6C3p6wR6F/hVRAGUiAQDDE27dO9FUWW8KkaW/bXcu6sET18TX1/7cxpReRn2UlNtdLW5gJgVGH6kDaOAPUt+lwU5SS/52MiCAMpEAiGDF6fyo5DulrLJaeV8MGXh9lyoK5HDWSr08P+8iYUWeL68ydit5pEP8hOOF1eXG4fFrOC3dq/TNLQXrYIBIIhxd6yRpwuH8V5qZx38ggkYOfhBlxuX49dc9uBOjQNJo7M6ncGoD/Q1KbvxeZkWJEkqY9HE4z4bQkA8FUfpP2zV7Gd/R2UgnF9PRyBIGkcOtbMS8u24/L4cHtUAGaMzyMz1cKY4RkcrGjmZy9+gSxLFGTZ+dm1J2G1JK9Q3ZBPi7dlU4vDzfN/2coZU4s4r4dDwX1Boz+82tP7wPEgPEgBAO6tK1HrjuDZ+2VfD0UgSCqfbCqntqmdFocHl8eH2SRz2pRCAM6ePgyAtnYvLQ4PByqaOVDRlLRre30q2/0h3XgL4Jvb3BysaOZvXx5G07Skja2/0Ojfi83uhwZSeJACNJ8Xb9l2ANSmyj4ejUCQPFRNY+sB3UAtumEWRTkpWC0KVr+U2dyZxZwyuQCvT+Ptj/exdkcVlfUOThydk5Tr7/OHdIflplCQnRLXOYbnpZKVZqGx1c2RqlZKitKTMrb+QmOLHmLN7YcGUniQAnyVe8HjBISBFAwuDh9robnNTW6GlQkjMslItQSMo0GKzUxGqoUR+WkAVNY7knb9zf72TfGGVwEkSQp0tjDCtYOJpn7sQQoDKcBbuinwb621Ds3r7sPRCATJ4XBlM//8+iig7zlGSgApytE9vKp6Z1Kur2l6CYlx/UQYzAaysdVI0ul/BlKEWIc4mqbhLd2s/yAroPpQm6pQckf26bgEgkQor2nlkVc3YOzYRWOgCv0GsrK+LSljqGlqp7rRSarNxLjijITOdWJJNhaTTGllCw0trn4lx5YoRpKOCLEK+h1q4zG0lhokWzpK8RT9NRFmFQxwvt5bg4buFV506kimRLGnmJ9lR5Kgtqkdj1dNeAwVNbqhHV2UjiIn9qi1mJVAj8StBwaXF9noL/PIzuh/Rl8YyCGO4T0qI6cjZw8HhIEUDHy2+Pf+vnXuOK45bwKyHLm+zmySycu0oWlQ3Zh4mNXYyyzKSU34XAAzJuhesPHdBguNrf4yj8z+paIDwkAOeXxHNgNgKpmJnFkEgNooDKRg4NLU6uLQsWbMJjnmbFTDmFUlIVGnqkE/R2GS5NNm+HVKdx6ux+3pOWGD3sRQ0TGbZFJt/W/Hr/+NSNBraO2t+Kr2gaxgGjEVX+1hQHiQgoGHo93D6//YS4vTQ6vDA8AJJdldMlYjUZhjZ9vBJBnIgAcZX3nH8WSnWykpTKe0qoVdpQ0JJ/70BwwVnaw0S0wqOmprPa6v/oJl5qUo/shXTyAM5BDGV70fNA2laDySxY6cpRdNCwMpGGh8vq2SdTurgl47ZXJBzOcpztM9yAMVzQmP6ViSDSTAnBMLKa1qweHyJu2cfUlZdSsAeTGGV91bVuDd9yVy9ghhIAU9g9qoP1Dk7GIAJHsmmG3gakNtb0G2Da6CZMHgxSinuOzM0YwvzsRmMTE2jszRqWN0tZsdh+rxeFXMpvh2oZwuL02tbkyKnNTyhQtPGcmJo7MZWZCWtHP2JcbvberY6EPhmqbhPbIFANOwiT0yLgOxBzmEMTxFY+9RkiTkTF2CSxP7kIIBgqPdy96yRmRJ4vzZI5k6NpfxIzKR4xC+zs20MSI/DZfHx54jDXGPqbpBT/IpzLZHlSAULbIsMaowvd+JeseDqnaoHM2IoQ+k2lARyLyX88f21PAAYSCHNAEDmVUUeE3OFGFWwcBi+6E6fKrG+BGZpNnNCZ9v5gTdi0ykKL8yTHhV87pRW2rwNFajttR0+U9TYy8vcQ3QhJ2DFc20Oj0UZNkZlht9GNp7RBc2UUZNR0qwfCYSIsQ6hDneg9T/XRj0nkDQ3zHKHuIVAz+eGePz+NuXpWzZX8cNF2hxeWvH6vQayMJOBlLzeWl79z60llrCSREoo2aQcvGdUV/nQHkTT76xkW+dO54LTxlY4h5bD3aoDMUyx75Sf3h11MyeGFYQwoMcomiedrS2BpBNSGkd4Q3DmxSlHoKBgKpqbDuYuN5pZ8YMyyAjxUxdczvlNfGp6uw4XB84l4Hv2G60llpQzJgy85HScoP+Q5LwHdmK6ow+Qai+xYVP1dhX1hjXOPuSsio9QWfCiMyoP6O2t+jJhf7M+55GeJBDFLXJn6CTWRAUpugIsVaF/JxA0J/YX96kh+my7UnLFpUlienj8vh82zG2HKhlRIwJMc0ONwfLmzEpElPGZAdeNxJLLNMuYsT871JT0xL0OcfKZ/CVbcN3ZAvypLOjulZmqgXoKLYfSATC0DGEV31HtuqZ98MnI1l6XlggKg+ysrKSf/3rX/h8Po4dO9bTYxL0AqHCq/rP/hBrc2Vc+yECQW+yxS+7NmNcbGG6SCQiDr7tQB0aMHlUNjaL7oN01jw2lcwM+TkjZBjQRo6CLL8mqyH4PVDw+lRqGtuRgIKs6A2dt5OwSW8Q0YNcs2YNDz30ELIs89Zbb3HJJZfw61//mvPPP783xifoIcIZSMliR7Jnojmb0NrqkNLz+2J4qM3VeCt2YZ50NpIkdgLCoXnduLevBrcDpWgSplHT+3pIvcqWQDup2PcfveU78ZXvALMNy5TzgzySKWOyMSkSB8ubaW5zk+H31KJh8/5a0iQn81K24lq/DwDN44qYeWkaNQPXF0vwlu9A83mQlMgJR1n+cTW1udC0+PZL+4LapnZUTSMv0wr7/oWrtY76FAsuF5hPPBfZ3rVER+9buw3onf1HiMJA/u53v+Odd97h+9//PgUFBbz55pv8/Oc/FwZygKM2VQMg+T3GzshZRficTahN1ch9ZCDb//UKvmN7kMx2zONO7ZMxDAQ8u9bgXv+u/oPyD9K++xKSPDR2TqobnVTUtmG3KkwYmRXTZzWvG+eHz4PXH5rUNKyzLgu8b7OYmDwqm+2H6tl2sI4zpw2L7ryaxu7SBi6xb2HEsb24jwu4mUpmhs28lNPzkHNGotaX4Tu2J6o9NotZIcVqwuHy0tbuTUoWb29QWaeHV2dl1OH6/D0ADB9YczZjO+vGLp/R+9a2I2cXI2f0znMp4tJcVVUKCjoUKU444YSEVikVFRXccMMNXHzxxfzoRz+ira3rJrjb7ebuu+9m3rx5XHnllRw4cACAtrY2br/9dhYsWMAVV1zBl19+Gfc4hjqaoxEAOTW7y3tSSnbQMb2N2t6i/zEA3tKNfTKGgULnXp74PGjNNX03mF4mUGQ+JheTEluUwVexUzeOsi5FFzSPfuIJs7Y4PLS1e5hm0ftQmqdfjOWUq7CcchXW067Fcuq3uv28EToMNZ5wZKb59yFbBs4+pLH/eIJcCujNEjJP0xco3tJNaJrW5TPGnPRWeBWiMJB2u52KioqAUdywYQNWa/xtSR5++GGuv/56Vq1axdSpU3nxxRe7HLNkyRLsdjsrV67kvvvuY9GiRQD86U9/oqSkhA8++IBnnnmGe+65J+5xDHU0h14EbRjDzkipWQCojvgLpRPB2IgH8JZtQ1MHZp1XT6O52vAd2wuSHAjbDaXyHMNAxpO96i3tSJhBMaPWHEI9bkFolI1s96vqRENlvYNipZ5M2YGUkoV1zjVYT7oU60mXYpl+ccjQYWdMo2b4x7c5pJEIRVaasQ85cAykLuSuMdylOz/WWZeRc96N+vZOWz1qfVnQ8Z33cJVeCq9CFAbyrrvu4pZbbuHIkSNcc8013HbbbfzsZz+L62Iej4evvvqKiy66CICFCxeyatWqLsetWbOGyy7TVxOnnHIKDQ0NVFRU8JOf/IQ77rgDgKNHj5KZGX16sCAY1dEEgJTSdQ7llCwANP8xvY2xEQ+Aqw1f1f4+GUd/x3t0O2g+lKIJKAVDy0A6XV72HGlEkmDauNj2H4OkysaeilJ8ItCRZWqQl2lnRH4qLrePvVGWUVTVO5hq1r1H06iZMUfb5IKxSPYMtNY61IbyqD7TYSAHTqJOZZ2DYUojNnejvi9bMBZJkjGVdCwQOhPUt7ZgXK+NM+JmxahRo3jnnXfYtGkTqqoyY8YMcnJiayFj0NDQQFpaGiaTftn8/HyqqrqWE1RXV5Of3xFjzs/Pp7KykuHDh2Mymbj11ltZu3Ytv/rVr+IaR1/gayin/aOXsJ76H5hKTurTsWheN7jaQFaQbF1T2CXDQLb1vgepb8RvB8A0ehbewxtxrngGZfhk7BffIRJ20H9/jr8/jVqrh6dMo2aCP6FjqNSv7jhUj0/VmBCjeo57979wr/8LWnsLUkoWct4oTKNm4juyRY9cTJ4bdPyM8XkcrWlj8/5apoyJ/NyrbHAw1R9eNR72sSBJMsrIGXj3foa3dDNKzoiIn8lKG3ilHmc1/51JGf7w6qgZgb9rZdRMPLs/xb1xOe5tH3Z8wKeLsysje149pzMRDeS3v/1tVq1axdy5cyMdGsTKlSt54okngl4bPXp0l+OiXWHJnSbllVdeoby8nGuvvZaTTjqJceOiX1Hk5iYm8pufH5+Ad+3Xn6E2HEXd8SH5s89JaAyJ4mmopBVQ0rIpKOjqQTodwzkGKJ7WqL5vvHMSCuehrbR6nJjzRlBw/g1UvLodzevGV7aVDE8VtuKeFSdOFsmck+Np2/1vWv1etWxLpfCUc3HXlVP5JSiO2h69diIkc1y7P9KzQ8+cURzTectXfInWrtcfZs66gJyCTHypczm6aTkpufnkHXeu02cU8/e1pZTXtkV1HWdTPaNMdWiymcIZc5DN4bejwp2v5YRZ1Oz9DFNzWVTXHFGkh23dPq3f/u47U7ZvP9MVPbSKbCL/1Aux+8ddNPM0jq5/C29jtb6I74xsIv/UCwLH9gYRDWRxcTEbN25k5syZQUYqEvPmzWPevHlBr3k8HubMmYPP50NRFGpqaoISgAwKCgqoqamhpKQEIHDc+vXrGT16NAUFBRQXF3PSSSexb9++mAxkXV0rqhpdbP948vPTuxT3RoOmabTt+QoAV/keqsoq+rRThreyQh+XLSPk91Hd+orU3VQX8fvGOyfhaN+6FgCpeDrNci6pN/0O19o38exaQ+2WL7Faossm7EuSPSfH075dnyPLrMuxnHQpDW4zKvpCx1VztEevHS/JnBNV1Vi/Q/eUxw2L7bzuWv3eT736CXxZw/yflbFf/xwqdDmXXdEX8EerW6O6jq16BwCe/EnUNbrpyM0Mprv58Ml6XkB7dXlU11TQn2fHaqIbY1+z9/OPGQOU2iYx5fr/R6vJSmtNC/n56dQ1urFd9QS4nV0/aDIHjk0Wsix16zRFtHgHDhzg+uuvZ/r06cyaNYuTTjqJWbNmxTUYs9nM7NmzWbFiBQDLli3jnHO6elNz585l+fLlQEdS0PDhw1mzZg1/+MMfAD0Mu337dqZNmxbXWHoTtf4oWqter4Wm6aGcPsRI0DH2Go/H2JfUHI1RJwokA31vaDMAij8MLZksmMbMBmIroB6saJraaf9sdqBWTkrLAcWk16+GergMIgyR6/wsG8NjUGHR2lt179FkQTq+/leSQkazstIsWM0KrU4PrU5Pt+dXVY2R7oMA2MbGv40Sq1hHYA+ybWCEWC1V+haKXDILydTVw5b8Wz9d/gtxbE8T0YN84403knrBBx98kEWLFvHSSy8xbNgwnn32WQCWLl1KdXU1P/3pT7nxxht54IEHmD9/PhaLhaeffhqAH//4x9x///0sWLAARVG47777KC4uTur4eoJA0onZBp52PHs/R0rNRhmeWMlMvGiBBJ2skO9LFntgrLgdYE3tlXGpTcfQmquRrGlBG/HKsElgsqLWl6G21iGnJUeUeiCi1hxCczYjpeUiZ3fsT0mSjJxRpIfxm6pQ8kf33SB7kIMVzXz0tZ7hGKvIdWdxjHCf0zQNX9U+lLzRSCa9y31hjp0jVa1U1jsYX9x1S0JtrgZZoa7dxASTXvhoHxufEwGxi3UE9iBbkp+kozZW4qs/YowMZfjkhKJfbc1NFHkr8CExauac5AyyB4kqxLpixQo+++wzPB4PZ511FldccUXcFywuLmbJkiVdXr/uuusC/7ZarTz11FNdjklLS+P555+P+9p9he+ovmKynnwlrnVL8VXswlmxC9uFt2MefXKvj8dIvglnII33tKZKVEcjSi8ZSJ8/Oef4NjaSYsY0Yirew1/jPbody+TY9sMHEwG5shAZknJmoW4g68sGpYGsrHfw6J83BH6eEWN5R4f+cFHYY7yHNtD+0e+xzJyP1V+zWJSTwpGqVqpCGEjN1Ubb/z2IZLZSP+4KhkteauR80kPUF8dCh1hHVUSxjsxOZR6qpsXVBzMUmtdN2/JHgvYCleIppMy/O+5zHt26geGSRrlczOTMrCSMsmeJGGJ95ZVXePnll5k0aRJTpkzhT3/6Ey+99FJvjG3QoDbo+x6mcadiPf26QM2a9+CG7j7Wc+MxRAK6MZCBUo+2xh4fj4ExT0oIKS7Zn80XCFUPUbrTolSGTdKP8ctxDTZKK/W9p9wMK5ecVsIJJbEZoVD9T4/HWDR6DqwPbC8YIuhGcXtnvGXbwONEczSSuvvvALTknBDTuEJhGHG1MbL2tdWskJlmwadq1De3J3xtA1/FLnC1Idkz9W0OScFXsQutvTXuc7pq9QxfV+aoZA2zR4loIJctW8Ybb7zBzTffzHe/+11ef/113n///d4Y26CgY9/DipSShWXaRdjP/U8AvGVb+6QIPlKItfN7vamm090DrPO+6FBFba1DrSsDkzVgDDsTUGE5ug1N9fby6Hoew0CdemIhV31jXMyekmFsuvMglYJxSLZ0tJaawPGF3RnITrWTmR5dxSjnhMSlEQNt56Ksay3KDj/GeDG+m/mEb2C/4CcowyaCpuI9msACrEX34k3dLFL6E1GlpaaldWT5pKenB+oYBZEJte8hZw3TNVD7qAg+oKLjV8wJRYeaTmPPD8hPx1yF0If1K/6ovejR9jcC4dURU5FMXcWz5YwC5Ozh4HbqCjuDjCqjPVKcba2iCbFKsowyUhd8N+bbuF7VccZHU314y4IT7pq1FIZPSqIHGWXbucLAGJOToBWq+0iHDN7muM9rb9cjQCkFkes7+wMRDWRxcTGvvfYaHo8Hj8fDq6++yvDhw3tjbIOCcF5RPK1tkoUahQcp97IHqbn1MBWKSW8eexwdHmTi6j6Odi9/XLGLnf6mtgOFQPZqN1qUfXlfhUJtqsL54fMce+Mh3Dv+mdC5KhMwkJqmdjKQXRdgnTEK/D3bVuH4+9PkbXiZyeZyqhqc+FQf7V+8jmf/On1x62pDyijAo9gAqEubgOLXd02EWEKs0H0YOB7U+jK0tnq/mEIJbo+PDw7r9ZZ65Cv2CIWqqmSq+uI8p7gkKePsaSIayIcffpiPPvqImTNnMnPmTP7xj3/wwAMP9MbYBgWGssnxq1bTSL08xVe1r1fHo/m8+qa7JIdU0TGQUnXVkN7a81Ob/Q+vjKKQajlSqiGgnri6T0VtG59vPcY7nwwsCTu1Wi8hMKTRQqEYq/wj0Wt59iTuravwlm7CeXgbrrVL0Tzx7ZFpmhZ4+BfGYyDbGsDnRrJnIEVIOjONmAYWO5qzGV/5Tji2gytSN+HxqjQe2o1nx0e4d3ykZ3kD5vGns1MbD4Blwmkxjy0UUkY+KBZdci4KRatwXm68+Cr155IyYgqSJFPX3M7fdzip1zLA7QzkC8RCa0MDdslNu2YmPU41tt4mYqy0sLCQxx9/nLy8PHw+H0eOHBkQpRX9hXBhQzlbn0MtyhBKstBc+ga7ZE3tVrYtUIvVS9qeHQuJ0Kt7yZ4BSGjOFjTVh5TAKr2kKA2LWeZIVSsNLS6y03u/vipWNJ9H/91JcmCxEAqlYDxYU9Gaq1GbjqFk9V20p7PmqWxPR3W24D26A/OY2DO3m9vctLt9pNpMpMfR0incQjUUksVO6lWP+j+j4fzHbxnmrSdbbqWl0kMuIKflYho1g5RvPUarks2fP/qCPPMEHpyZnNZskmxCKT4B35EteMu2RszcLszRe1kmzYP0/90r/udUYU4KGSlmyj0Z5FiaUZsqUXJjS7SpO1pKHtAoZ5Pfi3JxiRBxlEuWLOHHP/4xdrudhoYGbr/9dt59993eGNugIGxj4pQsMFnR2lsSygqLFeNa3XmP0NlAVkdVrJwokTIMJVlBsqcDGpqzOaFrmU0KU0brK9gtcXSM7wu0TuLy3S1sJFnG5N9D8/VxmFWtOxII02XOWaCPqbMQfQx09h7jqR0O93cYDjktF9OIKZhGTA30ZTzRXI6rrjzoPEp2MVsPNeJFoWDUaKyWxMOrBoZmczS/x/wsO7IkUdfUjsebeOKfMV+GoIIsSUwfl0e1T9/qiEfz11GtZ7C2WwdOHXNEA/n222+zdOlSAEaOHMmyZcv485//3OMDGwwE7Xsc9+CXJKljn6G597xIQ4dSsnZvICWzTfdUVC9aa88bkWgeYMnMrDVq6L7eU01TW//vgmB85+72jQ2MB+vx3Sl6m0BJyqgZpE7wqyEd2YKmxb7gqmrQk08Ks+NN0Al+4MeC0YJqqvkoNHf9ezb6RcZalxntdb3lO/QGA90dq8jkZ9nQgOqGxBN1DAOopRcGDO6M8XlUq/o+pKOmnKoGR0yynd5O5xwoRDSQPp+vSxZrX6i/DET0fQ+Pvu9h6fqHHfDSerEDQ7QeJHTOpOv58RkLie4eYFISazOn+1sk7TjcwJ2//bzfe5LR1K4amEZO1WvWKveiHS/43It4/ZKKppKZmPNHIaXlojmbUWsOx3wuowN9kT+UGCvR1ECGQxk1Aw2JCeZjpLYHL+Q8Xh87/MleM2JsuxUJOTUbOa8EvG58x3ZHPL67cpRY0HwefVEsSTzyf4d55LWvAZgyJlvfgwSOHjjAvS+v4/m/RC+bqbRWA2DJ6f96ygYRDeTYsWNZvHgxZWVllJWV8fzzz4fsyiHoitqq/+FIaaFXlrHWOiUD44EZnYE0wqw97+EayUByevhVuJzE0pOsNCsXnjIy0Cpp95G+aQ4dLcaiIBoPUrKkIOeOAE2LOguyJzD21+X8MUiS1KlMYFPM5wpksObGp+oUyx7k8cgpmag5JZgllUy1Keg8u4804vaojCpIIyfDFtfYukMp0rvX+OqORjw2N1O/fn1LYpqsanM1aBpeey7l9S7cfg/SZjExY5Yebi5UminIsjF+RHQ9eb0+Fbm9EYC8EQOjxAOizGItLS3liiuu4KqrruLw4cM89NBDvTC0gU9AFDxMvWFvemgGgf3OCCFWiD3VPF401YfmbAEkfzJOaJItXnDtNydw00V6wX2y6sd6ikCItZva1c7Imfoqva8aKAclFfl/p4ESlDj2IfUO9FCYHbsH2dkjkjO6l20Lh3V0J/Fxe6auV0xHeHV6ksOrBsbfoBbF7zGrk+RcIhiLiQYpC4AZ4zq+28Vzp4LZjl1y8/jN01hwxuiozrm3rJF0SV+c5xQODJEAiCKLNS8vj9/+9reAPyvN68Vsjj2LbCgSSbGmwwD1pgcZQ4g14OH2rAepJ91oSPbMbrNTOwxk8rw9Iz3+WBIVSHoCNYY9SOib8H1nQiUVKcMng9mGWheb6LxPVQP7avHsQRoekZRREOh+EiuWMSfh3fhXALwpupHVNI2tfgM5s4cNZDQLHUO0vKk1sT1141qlbfpczxjf8XuSJAk5qwi15hBqYyVyUXTC5Vv21TBP0stiJHt0Xmd/IKIHuWHDBl588UXcbjcLFy4Malcl6J5IouCdDVA8iQtxjakf7kFGI57e+X01CWIBBgXZdiSgttGJ19c7v4N40GLYg4S+Cd93JlRSkaSYMRVPAWJLIKptasenauRkWOPKEo01gzUUcs5IWiXdGLRacjl0rJnXV++lrtlFRqqF0cN6pr9rLL/HZHmQhrd6sM2O3aowcWRW8JiO82o9XpXVG8qobQwdhdE0jf0HjqJIGqolFUkZOEpsEQ3kr3/9a2bOnMlHH31EXl4ef//73/njH//YG2Mb8BgP8rB9Fy0pevjJ5w4YiZ4mYCCjCLFK6Xkgm/Ri5QRLK7pDjTJ82BMC6hazQk6GDZ+qUduUPKHnZBNLFiv0fYg1XFJRPHJlRvF73BmsEWpso0GSJGoyJgNwTM3hDx/s5JONesnHzPF5Seug0eW6qdm6YICzOWLClWEgE/UgfbWlAFT5Mpk6JheTEmwmjl847z7SwNKP9vHmR6FFT47VOfC06PkYSoJdTnqbqLJYzzjjDL788kvOP/98RowYgdoLdXGDgWgear0dZo0lxCrJJj0sBvh6sENEh3fUfeilpwTUi5JcZN0TxJKkA8EJVr0VnehMuHtfGTUDkPBV7ETzROfpdGSwxmcgtUAGa2LZk9JJV/Ln1rN4+0gRVfUOUm0mrjt/Av8xt2v3mWQhSXLUyXKZRl/IBDxIta0Bte4IHkwc9uYHhVcNjg/fjyzQnyU7Dtfj8nStwdxyoJYMWfcuw+Vj9FciGkhVVdm6dStr1qzhzDPPZO/evXg83XfWFuhEk1jR24k6sYRYobO2Z+yZh9HS8TDtfnUppRhqOs1J7YJSmGSZrmQTnPASXSjPaLqLz4PW2vuas+EMumzPQC4YCz4v3vIdUZ2r0r//2JMi5dEweWwh27TxNLTp9970cblcMHsk6SldheOTSbRh1jS7GUWWaGv3xi0WYHj2uz3D8EkK08aGMJDHjScrzcroonQ8XpVdpV0jYVv21ZIp639bkf7G+xsRDeQPf/hD7rrrLq666ipGjBjBD3/4Q+64445eGNrAJ5rEil7fK4ohxAodws3eo9t1HdceINrwoSSbkqam05lk1Y/1FNGq6BxPX+5Ddhc2Nwrgo1X6qUpAgxU6t7lKrEC9swITJF8YIBzRLqJlSerkRcYXZjUyjLe7RzCuODOk8e8scGKobBlJSluPqydudXrYV95ElqxvX0SKEvU3Iu6WXnjhhVx44YWBn1evXo2iJE9OabCied1+UXCle1HwXvQgNU1Fcxt1kNHVk8np+cjZI1AbjuI7tgfTiClJH5fRwiqaBBQpJUv3IB2NkKT9jEAnhLr4DaR71xpc698F1UerJKFpGpLZhu382zAVTUhofLHuPxrImUX4ju3RQ2F+ubRYcX70e1ylW2n2mPhT+4WcfOoMLo0itd8Y83//o4zdK/6FJEmcMrmAm+dNxlRyEu4N7/lVdbSwwiNVDQ6eeWszdf694XhEAjRXm78fq6VbDdtomTE+j037alFkialjekcyLZZtmKw0K/XNLhpbXeRnxTZfmteti7MDO93FXBhmASCZbfrfoaMRrbUOKSOfGePzWPb5If61pYJ1OztCwaqmoWkwMsMLnujLlPoLMSvGCuMYHdGu+nt1D9LlAE0Dsx1Jjj6TTBmmFyurDZGLleMhlhq/ntiHNB4kdQl0Y3dvXaUviDztaG6n/n9HI56dHyc8vlhUdDqTaPhea2/Fe/ArFJ+LbLmNcZTy4foj+KLIQTB+PzUuK+1uH06Xl8+2VNDc5kbOGaE3JXY2dft7/GLbMWqb2tHQ97nyMmM3kJ1bXMXifYdj1sR88jJtnDG1iBRb72RjSv7aTTWKzjqZqfGXeqgNFeDzUEs2zVoKJ5SEX1Acf2+NKkxj7PAMNA3a3b7Af26PiixJjErXQ76xLvL6moGTbzvAiHbVL2cUgCShtdai+Txx12lFNaYYEnQ609FqKnnlFZ2JxUOSU7LwkdzGycZDpbHV3a1HEw61sVJPBLGmknbt0+TlZ1B98CCO9x70985LrPtI3B5kgiHW4z83ytbKx41eDpQ3d0n9Px5fWyMS4DKl8bvbz+GPK3ezcU81Ww7Ucvb04bp3296C2ngMOYxnt2W/bhBuXziNGRPiyxTtCK8mpzg9zW7m6R+dkZRzRUssvVmz/J1pGuJI1FGb9Lmq8OjCDt3t+cpZRfiO7dbvkZHTkCSJ+288mXZ3171PRZbw/O1TVGJf5PU1A6PnyAAksOqP4BVJigkpPV+XBWuu7tExxZqgY2Dc1NH0pYuVaFV0DAwvM5kepN1qwmZR8PpUHK7Y91kDotwjpyFZU5GtKSh5JUiZheBq0xvrJkCsGawGiXqQhvfVouoSZiUperLM5gi6tZrPi+RqxadJpGflkGIzMWeqPhbD6EUy3nVN7ZRVt2I1K0wdmxt3GUUyaiD7ms5Rk0g9PrMS8SD9UawqbzqZaRbs1vD+U6h7S5Ik7FZTl/8sZiXqWuf+RlQGctWqVTz33HM4nU7+9re/9fSYBgUxeUW9VZAfp4EM/IE6k+9BdqjoZETlZSU7xKp5XXgOfc3paWVkSA4a49CxNDL/jIxfg44M4M0JjTHaOtHjkdLzQZLRWuoidoMIeV2/97XDo2tnGt3gIwm7G/dJi2anMFe/12afoCfI7DhUj8erdtp7D126sPWAfo0pY3Iwm+Jfxycrg7UvkUwWsKSA6gtEgcKRiFiAMVfVvgyKItScxrI1pKlqIKlOGmBJOhHvvD/84Q8sXbqUVatW0d7ezu9+9zt+//vf98bYBjSxrJg66op6WNLNFVsGq0HAa0tiWNNAa/MLuke5suxQ00nOWNxfL6d99W+5go/4XvonNMbY+kpzteGr3AuSjGnktKD3jKJ4X4Jtp6KtEz0eSTH596801OaamK/r9RvIfZ4iNMWC4m4lx+rjWJ0joI0acrz+spJm1R7IPC3ITmFkQRouj4/95U0RdX63H0pOhwwjbBhPF4/+hBxl5MQIsTYlEGKtVjMiZgzHEr7XnE2gqUjWtB7dQuoJIhrIv//97/zP//wPdrud7Oxs3nnnHeFFRkFg5ZpREPFYo4BZa+phUfC4Q6z6HlGyjFJnOuYpOhHpWPZjIqFpGp5DGwI/l5jqaKuNbZHiLdsGmopSNBHJGpwZrBRNAIsdtbEiofB5tHWioeiITsR+bzlrK/znGIaSpS/i5ozU3zNCpaEwvmutLz1oH2u0X7ezorYtos5vRa2ebT1mWOSweziC+rEOYA8Som/11nk/PRY0TQv2ICMYSCk9DyQFrbU+YnTCuB+kKJ6F/Y2IBtJkMmGxdNTCZGRkYDKJ3J5IxPKHKUcINyWLWGTmgrCmgmwCtwPNm5jO4/HE+gBLZk9ItekYWnM1kjWNylQ9U9dcuT2mcwT2H/31op2RZBOmEbpXmYjQQueM6FiJN3yvaSpKm+51Fo8bE5Cum5qri4R0F2Y1rlWtBj9oi3I7BBn0haOE1lLTpb7W61OpaWxHQtfKjRetrRG8biRbepfFy0Aj2q0Fw4OMNcSqOZvA0067ZMOh2SIbSLlzdKL751bHPvDAaZRsENFADhs2jDVr1iBJEm63m5deeoni4uLeGNuARVNV1Obob4oOKake3oOMN4tVkgIP52RnsqoxyoAF1HTaE1fTMQrVlVHTacnR6zvTGyI3pjXQVJ/uQQKmUSeFPCagPRpnmFXzefU6PklGssXuTXWEMmNbfKmt9Zg0L82qjamTRgTu0RG2NmRJYm9ZI4720AlNPn/YtNqXQWGn2kVjX6uywYGkmHUvRFNRW4K965pGJ6qmkZNhw2KOP/t3MCToGMhRbi3Eq6Zj7CXWqvo9VhhFzWm0+5CBTOIEpf76gogG8pe//CV/+tOf2LNnDzNnzuTTTz/ll7/8ZW+MbcCitdWBz4uUkhXoG9cdUmo2mKITJE5oXHGGWCH5e38GsQpJB9R0tMTVdDon1/iGTUHVIM95CMeKxfiqD0b8vK9qP7jakDKLwu5xmUZOR0PCU74bt6Ml5DHltW385t0tPPXGRp56YyP/88EO3H5NSyPhRUrJRJJjT1bpCGXGFmKtOnwIgHqyKClKDzwMTW01jB+RiU/V2H4odJjVU69fy2nNxWbpiDZ19iA7j007zngbvTmN4+PFMJDSIDCQ0UZO4lXTMeaqwp2OLElRiQxE2w5PG8Bh7oh/cYWFhbz22mt8/fXXrF+/nqVLlwoPMgKdi5OjIRZB4kSIO8RKz3TS0Pc9Yl/lB+oyoyicDnvt9lZ8VftAUjCNnEpGTi77vEUoqPiObse96YOI5+gwsF3Dq4HrWFMpVQuRUWk9ENqL3He0ka0H6thT1sieskbW7qhi0z49hJloeryxB67FmKRz7JC+QNDSC5AlCTlHz2T1Ve1nxjhdbi3UPqSmqeD3CK05wR5DfpYdSdLbV3m8atjwryH5FymTMhKBxdcAT9CB2LK3M1Nj7+qhtej3R40vnfwsW5cOHqGIdn97IIdYI24m/ulPfwr5+ne/+92kD2awEE9xspxZpDeSbapEKeiZ7gDxhlihZ+oPNUcjeF16dlsMY5IzClFrS1GbqlAKx8d1bW/ZVtA0lOJJSJYUMtPgNy3nMiurnmvlD/GW70DzuvUU+zD4AvuPM8MeU1rZwpb24YxOqcResxM4q8sxc2cMp6QwHZfbx6Z9tazeUMaW/bXMObEwYsu0SARC4+3NaKoatRfaVqO3ckov0rNy5NyRSPZMtLZ6ZhW4eRfYdrAOVdWQ5Y4aRa2tAVn10KLamDRueNA5TYpMfqad6kYn1Y1O8gOLwuAHbGVAezX+/Uf9vIMnxGr8/UUTwcmKo6uHIbzR1CnzOBLR5E5oqi+QpDMQfw8R/1r27t0b+G/79u289tpr7N4d/T7NUKRjXy02Awnh096TQTJCrMk0kIEQWIwrfGMvI5G5Ot77y0y14MLMhtZhyHkl4HXjqwh/n6tNVfr1LSl6tmoYtuyvZbtbNzK+sq0BcefOSJLEmGEZTC7J5txZenRm28E6fKqK5jA8yPjqx/SQdIYekm6PLiTd1OrC5tQ92KKSMf4xyoG5ymrcTUGWnVanhwMVwXvS3gY987XKl8GMCV21PIMSdQI9K48PsTqCjo2XwWQgY1LTiaMW0jhvs5YSddeUjhB5+D1IraUWVB9Sag6S2Rr1ePoLET3IJ554Iujn+vp67rnnnh4b0GAgntTyns5k1TQt7jpIiD5JIBbiTcFPJKlJ01S05mq8R/3JNSV6co3dasJqUXC5fajDp0NtKd4jmzGNmh7yPEHqOd3o2m7eX0u1moHHnovZWYev+kC34uVFOSkU5qRQVe/g6z01TG0xDGT8QttSSqa+v93WCFF4olsO1DFS0Y2pJa/DC1RKZuDZ8yneI5uZMf46Vm8oY/P+WiaM6DhnVelhsoBWc07IBsf6a3UcKG9i+PgMUum60KlsSDzEqvm8/rChhJw58MoLjqdzklwkOcSsOPYgDQPZpNo5JUoDKdkzwWxDc7WitbeGXHgP5PAqxCE1l5OTQ3l5eU+MZdAQV4i1p1sTed3g84Jihm7ChuHoSBJIntyc2mDU2cXpQcYxV661S2l7exG4nchZw4PqVI2Vd2vuCYDuZYaT9gp4oN2EV+ub2zlS1YrFrGAbOwsAXxTlHjP9TWr/e/kONm7VZeoSUSAxjGu03v+O/VXkyK1oSMjpHfNjKp4Cigm1+hCzRulzdfw+ZGO53o3elhscXjUwOnKs/PcR7n9jL25N0Y23WzeKjnYPTa1uTIpMToYt+i95HGpLNWgaUnregCtOD4VksoLFDqpXF8Xvhsy02MUC1ICBTIk6xCpJUsQyoliz1PsbMe1BaprG9u3byc3tnTYvAxHN69aVRCQ5oMIfDZ1vNE1Tk9J5IGhcnfYfYxXjBr3tFYDaErsiSzh85XrNYax7rp0TmmKZK01V8e5fp58jaxiWWZcFvZ+VaqGq3kG9qYD0lCy0tnrU+jKU3FHB53E78B3zq+eMCFbP6czWA7rxmDI6B8voApw7VuM9shnrnKu7Hec3Tipmb1kjR2vaSPU0gLlj/uNBTsnUBd6jNJCO2gpkCXypuUhKxyNCMtuQc0tQqw8wOqUNu1WhoraN6kYnBf6sx/SmfQDkjTsh5LlnTshn7Y4qWpwe2t1eanwZFJsa9P3k/DHsOKwvwMYMSw/a24yVWLOjBwKSPQPN7URtb0bpZpsk1hCr0ZrPp0k4NGtMjanlzCLU2sN67kSIfICB/nuIaQ9y3759DBs2jMWLF/fG2AYk+oa0hpSRH1NLKcmaimRLB6+7ZyTd2lv814k9vAogZeQlpOt5PGpzte5BWuyBdlpRj8WSou+r+TwxebS+6gNo7S1I6fmkfOtxzONPC3o/INPV5gmEVkPpqHrLtoPmQyma0O1+riHqPWN8HkrRRDDbURsiq+oUZqfwy++cwjnTh1PgD3UmkokZi/fv9amY/AIB5hCrfmMhJ7VUBfohGqIB1aWHyKUJh2Zh5NSZIc+fnW7lvhtP5onvn8ai62dR7a+7M/YujXPNTLAZsTbAPZdQGH+7Wnv3HmQgxBqlbKJR29ys2rGYTYHPR0OHTGYEDzJzYP4eYt6DFHRPIu11Ai2AmiqR03IifyAGEknQgQ7lDK2pCrW5GsWf9h8vgRDliO738MIhZw3D52zW2yWlRRfR6Jx1GsqL7izTZRp1Ep7dn+It3Yz1OE/TUMU5Xpy8My6Pj12lukGaMU73xEwjp+E9uB7vkS1Ypl4QcbwzR6eQcbgdD6aEmv12JFhFFnmoaXSSJ+tGWckOYSA7bQXMGD+Jr3ZXs3V/LRfMHknVtnWMBqrt4yg0RQ5rFuaksMmSA5RSV1ZK0fgzAl73jAQN5EDf+wpF4G+3PUrB8iiF9wP7j1oKhTn2mCJMkbY7AnkGA7TUJuyT6cYbb+x2ov785z/3yIAGOoloP0qZRVC1T7/Zik9M6rg6aiDjl9ySM4vwNVXp4ZREDaRfWaa7Pbzux1KI79gefa5GTO32WM3twL31Qzz71vqvGVr1pnNoShlxIihm1JqDqI6mgFC4rp6zFdCTVsKx63ADHq/KmGHpgT0h06gZuoEs3RyVgRyf7sINVHvT0Zrd1DY5aWx1M+fE2B76HSUCkT3IynoHBYq/tCTEQ62zesr0GXlIEuw+0sjSj/YxrUJPfOpuXo4nrWAEVG+ipeoojopmWp0e8rNsDEs0gzUQ2huYD+ZQdHiQoQUnDNJSOqvpqBG7oRih92bVHlN4FbqXMtQ8Lr0ZgawgRbmI7W+ENZDf/va3AVi9ejWtra38x3/8B4qisHz5cjIy4hcQHuwkkloeWJ1H0UImVhKpgTSQM4vwsSXh8WmqD9+xPYCuNBPvWCC6rF/31g9xb1yu/2BN1cOdIchK7/AgJZMVZfgJ+Mq24juyBXnyOfr16o/q6jnp+d2GjTqHVw2MMgnfsT1RNVGWW/3i0WoGZXtreP+LQ7S1exmRn0pxfgx1ozF4kFX1Tgpk/V4JJbTf+YGYZjczaWQWu4808tGGI1ycXQkSjJp5WpfPhWP4mLFQDVJrTdCcxbNP3pmBnhwSCuNvN1LLK1mSyEi10NDioqnVRV4EVRwtIQMZPh8g8DvIKEyoYXhfEnZpcdFFF3HRRRdx5MgRXnzxRS644ALOO+88nnvuuYTqICsqKrjhhhu4+OKL+dGPfkRbW9d4utvt5u6772bevHlceeWVHDhwIOh9r9fLNddcw3vvvRf3OHqKeGogDXqyL6SxbyHZ0uM+R7LGp9dGefXaqDgNdqCGLopaSCOca55yPimX/Cwo8aQzWanB2X8dOqqbA8cY11NyR4V9iGuaxhZ/P8MZ4zoMpGRLQ0rNAdUblQqQYfxrfBn8fe1h2vzap5EaFh9PLDWslfUO0iT9+4dqYG08ELXmGjTVyy3zT+C6b07ghrOHYZJUVHMqaZlZUY9t5Bg9EmFXW/l8q74PmfD+o9uhyxAq5oRC0/2NgIGMEGKFztGQyPuQWhwZrIExWez6/eXzBNqcGcSqKNYfiZik09DQgMvVEctua2ujqSl+weqHH36Y66+/nlWrVjF16lRefPHFLscsWbIEu93OypUrue+++1i0aFHQ+7///e85fPhw3GPoSbQEQjs9aiATqIE0SFYpSiKLCAMpKzppPrWtAbWuFEwWrHOuRskfE/bY4zshGB6f9+iOQGJSNH/0B4420dTqJjvdyqjC4PmOlNQQNHa/Ma7xZdDs8ARe767VVCgkuz887GwKKVTQmcp6Bymy30CGuFckk0UPl2k+tJZa8jLtXHDKSOZO0kP3pvTYDJLJb8DSpXZaHC7sVoWJI7NiOsfxdM6cTHY2eF8SCLFG8CAhNjUdQ0UnHg8SwtclGwpJA1kLN+Ldc+mll3L11Vfzwgsv8Pzzz3P11Vdz9dXdp6mHw+Px8NVXX3HRRRcBsHDhQlatWtXluDVr1nDZZXpixCmnnEJDQwMVFfrq8uuvv2bPnj2ce+65cY2hJ9HaW/Wb12SNSztTL2gO3QIoGWODxEOs0L1yRjQkQ+FETi/wZ9XWdptVG0gGKp7SrWwcdCTpNLW60TQNOS0XOXcUeF34ju32jz1yEtb6nfr3CxUqjKU/o2GMzcdpmh4ob6LZEX0msaSY9MhBFGo6VfVtpEnt+ufC3Cuhujh09KzMinpcxti8plQUSSNNcjFlTG5UOqDdMZgUdDrT4UFGbmgQS6mH2jlJJw5xhnBdPQaDFm7E9MGf/vSnTJ06lbVr9QSHRYsWMXfu3Lgu1tDQQFpaWqCfZH5+PlVVXT2A6upq8vM76r7y8/OprKwkIyODJ598kpdeeqlflpp0/sOMZw/FaAGktdSgNlehZCdPFD5Q5mGLP0lHSskCk7Vb5YxoSEboRVJMSOn5aM1VqM01KDmh58oIjypRJAN1VtNxuryk2MyYSmbirjuCt3QzppHTA2OXutnb+sowkOO6JiZE3QGhk5B78dixUFlJqs3EqMJ0dpU28LPff4Hsv8dMiszN8ybjUzX++ulBfvqt6QzLDf49SylZaO0tuiELY8Qc7V7a29pQrBqYLGEXFHJmEb7yHUEegxqngQS/19nQRobsCIgkJMKgN5BReJBGR4+mKEo9vC0NyIDPmkGKLZ6M8tCRpcHwewg7GwcOHGDcuHHs2LGDoqIirrzyysB7O3bsYMqUKd2eeOXKlV1KREaPHt3luGgNiSzLPPzww/zwhz8kLy/+PYrc3Pg9KID8/PB7eK3VbTgAW15Rt8d1h2/4WBx7akhpOUzGxMlxjrIr5T4nPiC7qBBbnGMDcOcOx111iAy5BVu+biRi/a4Vjho8QPaoMaQkMBZvfjHO5irStSZS80PPVWn9EQAKps7GnB35WrkZNr3jvcVMfn467dPPoGLj+2hHt5KXl0arvzlswdhxKKldz+f1qRyq0L20s2aNxGYN/hNzjBpD5VowOWq6nTefo5lWrwvJmsJF507jo+2NXDinhNHDMtj7+ga8Pg3QVX68qobZaqamtpXqRidf7a3l1suCH0rerDyc9WWkm1ykhrnuv7cfI9UfXlVSMsKOr2lECXU7weKqDxzTsNuBC0jLKyAnzOfCnc+bnYez4ShjszW+edoY0uyJKd9U1OhiBVljJ5GWwP3V08T6d+NSCygHZE9bxM+OHKaH1Z0eNeKxTS79fs3Iy4vrudU2cgxV68DkrA18Xm1vo6XuCEgyBRMmo9ije+7G+9zsKcIayKeffpqXX36Z22+/vct7kiTxz3/+s9sTz5s3j3nz5gW95vF4mDNnDj6fD0VRqKmpoaCga6ZcQUEBNTU1lJSUAFBTU0N+fj5r165l7969vPDCCxw7dox169ZhMpkC4dhoqKtrRVVDy4dFIj8/nZqa8CnW7kp9xeQxpXV7XHeohVNgz79p2PFvXKPOjOscIcfWqv8RNDklWuIcG4CaVgBVh6g/fACzZVjEOQmFy98pokXKoi2BsXjt+kKpoewgjtyuZTGa24mvtQEUEw1uO1IU10r3P5wPHqnHJoNmLkCyZ+BtrqVy8zo0lwMsKdS1SUgh+jtW1TvwqRq5GVZamp0cf4SK/uBqrynvdt589UcBff9Q8vlY/OMzAu+9dNdcfJ3uYVmSMJtkbP5EwbXbjnHZ6SVB5/Oa9QdU47FjOLJCX/fTjWWk+hN0NHNK2PF5lSwAHJVHAse01+jiB05Cf667+8Rj0h+KN5xViLO1HWdre8jjokFrb6X96G6QFBwZ43AmcH/1JPH83ahOPfTsbWuO+FlF0/eaq+rauj1W01Qkv3RdSnpmXM8tVTLu6aOBz3sOrAfVhzJsEvWtGrRGPm88c5Iosix16zSFNZAvv/wyAB9//HHSBmM2m5k9ezYrVqxgwYIFLFu2jHPOOafLcXPnzmX58uXMnj2bDRs2YLVaKS4u5vPPPw8cs2jRIk499dSYjGNPowY6L8SfOacYpQAVO9E8rqQp4HfsQSa2Qou2i3jYcSSxNqqjLCZ0uDIQys0ojLrNUyA05c/+M7pYePZ8hnvrqsB1w0U+jgVaNYXey5HS80BW0Nrqu/39GmpKoUKWJkXGFCJrfvyITFJtJqrqHVTWO4ISLiJlsqqaxpYDdQwzEnS6CZ93dOEIsQeZ2nW8kQjUmCZBQer4NmaDic57kJEFy6Pcg3Q7kdBo18wU5sX3bJAy8oNUtiSTJSoxjYFAxKfGgQMHePfdd9E0jdtuu41vfvObrFu3Lu4LPvjgg7zzzjtccsklbNiwgTvuuAOApUuX8vzzzwO6SIHb7Wb+/Pk89thjPP3003Ffrzcx6szkOB4SBnJKJnL+WPB58ZbvCHucr+4I6nFp1WHHpXrB4wRJ0gWPEyCRThqdPydnFCRcGxVIGooklBzDHkioB4uxf+k7uj3i+QKtmsIZSFkJ1Beqzd300YtjT0+RZaaN1RcdK9aV8uX2Y4H/DjXqD9NwRqi0soWmVjcFdt3z6C7bWUrLBdmE5mhE8+jenrEHGU/fymT2Go2mifVARVLMYLKC5gNP9152VkCwvPs9SGPh3KZa4+6eYqhsgYbaXKNrHkchpjEQiLgj++CDD3L11VfzySefUF9fz+OPP86zzz7L22+/HdcFi4uLWbJkSZfXr7vuusC/rVYrTz31VLfnefLJJ+O6fk8Sbybf8ZhKZuKuOYjvyGbMo2d1eV9trsbx14eRMwtJueqxiPu4gRpIa1rCae+JlqKoDeVB50loLP5EGV9DechmwPGUkxgPloZOMl2m4qmgmPRuKJ2uG4qqCB4k+AWeG4+h1nUVQg+MPc57acb4PNbtrOLzrcf4fGtHpux0czO3pkNLfQ2hemQYGqjj803Q0L0HKckycmaBrivbVImSNzqhez9ZvUY11duljdlgQ7KlobW69CS5bha7hppOq9PTrZqOkfDTplkpiKPEw6BDZesYmrtNF9PIKBywGqwGEZ+WLpeLyy67jC+++IJ58+YxZ84cPB5PpI8NSZJpIAG8pVvQtK51a97Dm0D16Q+oxorI4wrUQMafwWrQOQsz1Ngi4T3iX1kOm5TwWKSULKT0fHA78FUf6PJ+PHJjBdn6Q6eqwdlxHbMV29zvYRp/OuYTz8M8qeu2gEFlBA8SOr67t2xb2GOMeynWaMTJk/K5eM4oTp9SGPSfPUv3LNubQtdQGrWVo/yXi5Sh3CHUUImmqoHoSTxtuZLVa9RXuS9kG7PBRLRyc4aaDnTf9kp16udp0yzkZsS/ndN54ezr1AouUUWkviaiB+l2u6mtrWXNmjW8/PLL1NbWBgkHCDowCm7jCTN1Rs4ZiZSao7dbqjncpR1UZ2UXb+mWiOUgAeWNBGogDYxOGnoD3gYg+gdiZx3TeDVYg8YiSZhKZuLZvhrfkc1dGhHHE2I1PD/DEzQwjz+tS/ePUERjIE0lM3Gtewtv2dawknPd7UF2h0mRufrcrm2H9u+xw7/+D8XVtQ6yocVFaVULFrNMns2Hj8iCEkESY+0toKl6hCKO3ovJ8iCN8KoyCMOrBrGUemSl6XJzjW3usHJzLn/yXjt2bJbYSzwMOnITqlBr9MVqMv7G+5qIHuQ111zDueeey8knn8z48eO56qqr+M53vtMbYxtQaB6Xvs8nmyBBT8148EOHqHfgOq42vRehH18nYxl2bO2Jq+h0puOPIXKxe2f0Fb4DObMoabVRRhLA8W2pOtcRSjGEWAuy7EhATZMTry82D9np8uqdQBSZ3G6a/cqZRbq6iNuhz0kIVKfhkWXFNIZwjB6rS7qlak4q64IfrkZ49cSSHCS3IUkYyUB2CB4kGjkxvE7N0RRXVMLA26lby2AlLrm5brp6OJv1+8xnTiyhyYgs+cq2drSxO27BOhCJuGS4/vrrufbaa5H9+zt//etfyc4ePPqGyaJzFl8ywgqmUTPx7PxYb7c0W69BdW9fjWf3p6D5kPNGo9aX4avaF7Fov0OoPDk1RnJWEb7KvTHvQ8ZStB8tyrBJ/j6L5ajNNcj+JtWas0lPZLCmxrQwMJtkcjNt1Da1U93gZHhe9Iudan9YdlheasRmv6aSmXi2rsJ7ZDOm4V1rOI3ejYlGIwLXM1tol+zYcPLK/60PCoUa4545IQ/taHSKS4ZQgrd0C2qdvyQlzuQ0STEjWdN0AQpnS0xh2vZ1byGnZmMaOR2tqUoXow/RuHew0CE3F1lNx+gi051YgKu1CRugWRJb1AcS5vwLu3jb2PU3InqQbW1tPProo3znO9+hsbGR5557LqTA+FAnESWRUCjDJ4PJglpXitpaj+Zpx7XubdT6MgDMk89BGTYZNC0QtgxHR4lH4nuQAHK27o34qrru+3VHYG8iianfep9Fvd1V59Czr2o/AEp2ccwLlqLc0GHWSOwv1x8OJUWRFyLGHPhCNGTWNK3Tnl5WTGPoDsOAuZrq2H+0KfBfc5sbi1lmxvi8qDV7lZxiMNvA40Rt0A1kuISjqMbmX9iotaVRf0ZTVTzbVuNa9zaefV8CeneYgdo5Ihpi8SCz/drCdU3hM169jsQaqQfGlZIVVLZlGj8nofP1FyKa+EcffZSCggLq6uqwWq20trbywAMP8Mwzz/TG+AYMWgJp7qGQTBZMxVPwlm7Ce2SL3llB9SLnlmCbewty7khQffjKd+At3Yx5whlhz5UMHdbOmEZOw7WWwB5aNKhNlbrHaUlJeujFNGom3oNfBTUi9pbqoWkljnZaRdkpbKc+sJ8YLUao8uTJkSX0lKLxYEkJzEtQyNnVBqoXLPak1cECpGTl4ms9xq3nDcdVEKyElZdpIzPVQmuU94pkSSH16idQW2r0n2UTcl5Jt5/pDtPIabhrDuke9ajofmeSLKMUTcB3bHegRnUwh1chNgNp7IN3dx+r/vMoKYlFlyRJInXhw/gaK5As9sAieqAT0YPctWsXd955JyaTCbvdzuLFi9m1a1dvjG1A0bEPE3sWXziUTu2WAvVdY2ej5JX4C9j975dt617cPAmdPDojZw1DyiwEVxvtR/dE9ZnA+Htgha+Mmg5I+Cp2o7mdaJqKryz+hsyFUTxYjqfd7WX3kQYkYPYJkQ2kJJsCvTANY26QSE1ht9f0C1gMS/EwcWRW0H85/j1TI3QXzWJKTs3GVDQRU9FElIKxCf1eO+8la1r0SleB36/PA5KCKULz7IGOkYkeKYsVoDCQkd3Nfex/NlhSE+/xK9nS9HshZ+SAz141iGgg5eNqy3w+X5fXBB0ZrImo6BxPoMFu+U68pRv11zo98OWMfOTsYvA48VXuDXUKIPkepD42fRyOfRuiOt5INuqJFb5sS0cuHAeqF8++L/Ae+hrN2YyUlqvPT4zEE2LdcagBr09j7PCMQNusSJj8RdTe4xKtklUudDxyp2SYUGg+j75vKylgTkxQIlbk/NFI9ky0tnp8R7aE7dCiuR268IWfzveTMmxiUkqZ+jNGHkE0e5DGQq+6wYkvTJszxavf47aMxA3kYCSipTvllFP49a9/TXt7O5999hm33347c+YMjvhyMukIsSbPg5RTspDzx+irY1eb/4EfHLoI9CwMsZcVGFuSs1ih48EUjYEMZN5KMqaR05I2hlDjcX3xOu0f/T7wWjwrWUNRJBYPcttBf4PkGJr9mkZOB0nGd2wvmruj7tJoPJtsA9mhWNMQ8v3Oe9W97QEYkn4Azg9/g+P9x7p4kmpLDa2v30n7p68GXuucET3QZc2iIZYQq9WskJNhxadq1IbZhzT79PsuLUMkXoYiooH82c9+RkpKCunp6Tz33HNMmjSJe+65pzfGNqBQm3WxZik9P8KRsWGddRly7kjknBFYZy/s8uBS/Ioh3iPhQ1PJzmIF9H1Eix1PXXnE1k3esm2g+VCKJvTYCt888SyUYZOQs4fr/xWMxTzlm3GdKzvdiiRBs8MTdanH0Rp9RT9hRPQLJMmaipw9HDRfUMmMt2InAEoCe3ohr+ePboTzIAP7iQlq5MaLeeoFusyiYkKtLe3SM9N7cAN4XXDcvrfl1KtQRs3EPDF54v79lViaJgOB/o7hoiE21W8gc4SBDEXEJJ0XXniBu+66i9tuu603xjNgiUfWLBpMJSd1K5ulFIzTU+Sbq1GbjqFkDe9yTLKzWKFjD8174N94j2zGMu2isMf2ZHjVQE7JImXBvck5lyyRmWqhsdVNU6ub3MzwNY2gZ51W1kUWCAh5rcwi1PqjumRbwVi/mIIhlzYzrvGHvZY/uhFOsUaLQ3komSi5I0m98gGcH7+Md/9afKWbg+7ncHWO5jGzMY+Z3Ysj7Tti8SBB3y7YVdpAZZ2D6eOC31M97ZglH15NJiOjf7WZ6i9E9CDXrFnTC8MY2KjtLXrmodmGZE9eiDUaJFn2J6l0UzLg6tBiTSZRhXc7q+eMGjj6mJkxdGRvdXpwuLzYLEpA3itajte29VXtB1dbUsUUDKRUvwfZFjrE2l8a3BoLws73ldbeqosqSEqPhekHBGabvkfsdYXdp+1MYLugk3SigSES4NCspNgS68E5WInoQY4YMYJbbrmFWbNmkZra4YF897vf7dGBDSQ6r7z7InvLVDIT774v9TKHGZcEv+l2gKbqxltJbuFu8B6aI2R7IeOBL2UWJt277kmy06yU0hKVgewsLxfr798QPjd0Y402QckUUzAwFm+aszmMuLu/PVgf/55MI6eCpOCr2kf72qWYJ89FrSsFTUUZfsKga2MVC5IkIdlS9d+hqw3J1P2CrLuEs+aGBtKAdrl3E7IGEhGfmFlZWQCUl5f39FgGLH298jaN8D9QKruq6gQky5K4/2gg2dKwjZxM+5GdeMu2Yx53apdjvD0gDtAbZPn7QjZGaBcE0emvhqOzpin0jJiCgaSYkGzpaO0taO3NXZKAjD2/vvYgJUsKSvEJ+I5ux7PtQ9T6o4H7d6DdRz2BZEvXDWR7K6R2v3fYXclSQ20daYBXEQYyHBEN5BNPPBH4944dO5gyZUo3Rw9Nemr/MVokSwrK8El6OUjZ1iDRAM3wCjIj1+bFQ8r4k3UDWboppIH0DVB9zFhCrFX1eviquxZX4egcYg0WU+gZuTQpJVM3kI4m6GQgNVVFbar2j6ln7pVYsJ1zC97DX+Na+ya+Y7vB7ykNtPuoJ4hFsDwvw4ZJkWhoceFy+7BaOmpVj5ZVMRIwJ6EGcrASU0HjL37xi54ax4AmnrZKySZQaH2cuHlPe7cpE/TkCF1VJzjjU22q0rMzB6BwseFBRmo4C5GbJHeHZEvTvSOvC/euNYAhptAzOpYdnTOC9yG11jpQvUip2Ujm7pOSegM5LQfL1AtQCifoWatuJ3LWsH5hvPuajpZXkQ2kLEsUGJmsnQQDVE2jplovTcoUGaxhERX/SSCwd9OXBtJQ3SnbGlRI3dPG25xbjJShq+r4qvcHvRfIOuzBB35PEYsHmUiIFdBViQDP1g+BDgGBnkAK9F4MLvXoL+HV41E6hVQHcxurWDCy0aPNZDUUdTqHWQ8da0bx6D+nZgkDGY6YDGQsElBDBU1VUZsNI9R3q1s5owA5azi4nUEtlHo6/CtJUofij3//THU20/rWPbjWvgUMzLBYdsBAdu9BqqoWaK5cmBPfXk5H13XNL6YQu35s1NcyPEi/8pOB2sOh+HjpfO90V+40lIi1FtJYuG3cW8PdL37B+l1VbDtQR6qsL/7kJCpsDTYiGsjOouTLli3rybEMSHzVB8DnRcooQLL07Wa3MvwEANTaw4HXesO7DXivfgPpPfgVWnM1oCGl5wUM6ECiI0mnew/yQEUTXp9KQXb8DWdNo2fqfUQB04TTe1QuLVxz4o5IwzD6E3LWMJRRM5ALxw/qNlaxEHMtpN9Art9VTV2zi4raNvIy7WSa9MVfssu/BhMR/6LXrFnDXXfd1RtjGZAEklD6gREwvES1UTeKmtupPwgVE1JaTo9dVxk2Ue/J2FiB2lwdCK1az/ku5klnI0kDL5KfnmJBkvQaR69PxaSE/g6b/R08ZoyLXmLueMyjT8Z0y8ugqUhKz9ajdcjNNQa93hFp6F8epCRJpFx8Z18Po18R0GONNsR6XOh/6thcxhdn4ihPxVeWXI3mwYaog0yQQBlDPwj/BGrq/A+7gPeYUdSjRkpX1ZmG9+B6PAf+ja98FyBhGjVzQBpHiF5NZ+v+OgBmjk9Mnk3vhNHzfQzlwB5kY9DrfV2qJIieeEOsAOkpZsYO07NWe0KjebAh6iBjQPW48Bz4N3jdejhT01AbysFsRyma2NfD61RTVxn0/94oPzGVzMR7cD3uTR/ofSsLxiZVuL0vyEyz0tjqprHNFdJA1jQ6Ka9tw25VmDAyq/cHGAehQqya161nsUpK0rWEBT1AjCHW9BQzKVYTDpeX6WNzkWVdzKJDo1kYyHBEXQfZ3NxMxhBvidK88R+0//NVAOSckZgnnwPoyh/JVqmJByk1FxQTmqMRze1Erdc7vfeGV2Co6uCXvxoMBd2Gmk5Dswu6StwGwqtTx+SGDcH2N/R+pRKaownN40IyW1GbjUhDftJ7dQqSjxyjgZQkieF5qewvbwrqNmP0lOwJEZHBQsS/6kOHDjF//nzmz59PVVUV8+bN48CBA70xtn5H6uQ5mE/8JphtqPVleHZ+DPQfYyDJMnKG34tsrsLrbxqsDJvU89e2pWGbeyumiWdhnnoBljg7afQn8rP0pKvqxq46lgBb/QZyZgwtrvoaSTHrLdQ0FV+53jXESNCRRHh1QCDZ/SFSZ+iuLKG47vwJfOvcccyapEcINH9tKUgwhKX7IhHRQD7yyCPcd9995ObmUlhYyLe//W0eeOCB3hhbv8OcWYDtrBsDCTlq4zGQpIBYeH/A8Ba9R3ei1pWByYoyfHKvXNs88Uzs3/getjNuGBSNaw0dy1AyXU6Xl91HGpEkmDaub9pDxUsg6/iIrvva10pQghixpIBiAU97UB/R7hgzLIN5c0qQJSO86m+4bE3poskr6CDizDQ2NnLmmR191m644QZaW6Nz7QcrnTNWlYLxyP0oRGE85DzbVgG6TmtPZ0YOVopCFFgb7DhUj0/VGF+cSZp9YM1vR1nOFjRNFQk6AwxJkvyh8vC9PSPR0QJP7D92R1RLB5fLFehSUFNTg6pG10R2sBLYb6Nnui4kgvGQ05zNwMAs0u8vGOnxoTohbDHKOwZQeNVAzhmJlJqD5mzyNybueyUoQWzIfpFy1RG6dVkkAgk6IoO1WyIayOuvv55bb72Vuro6nnnmGa655hquu+663hhbv0WypemGR7FgHntKXw8nCLlwHOBvuWS2CXmuBMhKt2Ixy7Q4PLS1e4Le21+ur9ynjO65+tKeQpKkwMLJc2A9am0pAHJ2iEwkQb+kIxtZeJA9ScTUy6uuuoqSkhLWrFmD1+vlV7/6FWeddVZvjK1fYzvvh2huZ78rZVCyhpN63a/RHI1IabnI9qGdeZwIsiRRlJ3CkepWKusdjBuu/669PpWaxnYkYFjuwExwMI2agWfnx3h2rAafFzl/jLhXBhAdIdb4PEiEgYyKqGoTxo8fT2NjI7IsM2OG8EgAJJMlYrPSvkJOz4P0gRf6648U5ugGsqqTgaxpdKJqGrkZNizmgVkWoQw/QW8hZZTliFD8gEJKMUKs8XmQqhAJiIqIIdbVq1dz4YUX8tprr/G///u/XHDBBaxbt643xiYQ9DmhGs4a/R+LBqj3CPoCz1Tc0du1v5QqCaLDiFwdLzofNUIkICoiepDPPfccr7/+OpMm6bV0O3bs4Be/+AV//etfe3xwAkFfM8xvICtqOwxkoL1V9sA1kKAnmHlLNyGlZiPnjurr4QhiQPIn6cQbYhUyc9ER0UDabLaAcQSYMmVKIKNVIBjsjC3W9+V2lzYERMuNxrPxtrfqL5jHnorv6HZMo2eJv+kBhrEHGW+IVcjMRUfEEOucOXP4wx/+gMPhwOVy8fbbbzNhwgSamppobGzshSEKBH1HYXYKRTkpOFxe9h/VH0aVdX4PcgCHWAEkix37+bdhHn96Xw9FECNymLZl0SKyWKMjogf55z//GZ/Px7PPPhv0+vLly5EkiV27dvXY4ASC/sDM8XmsWn+ELQdqmVySTWXD4AixCgYwlhRQzAE1nVh70QoPMjoiGsgdO3b0xjgEgn7LjPG5rFp/hM3767jszDE0tboxKTI5YVpgCQQ9ja6mk4XWUoPmaIrdQIo9yKgQInwCQQTGj8gk1Waiqt7BV7urAb3Hniz27QR9SKC3Zwyi5QCapqG161qswoPsnl7v0VRRUcHdd99NXV0dY8aMYfHixUGNmAHcbjf3338/27dvx2azsXjxYsaNG4fH42HOnDmMHDkycOx7772HogzMWjTBwECRZaaNzWXdzir+skbvZDN1zMBT0BEMLqQY214F8DhB84HJKnSaI9DrHuTDDz/M9ddfz6pVq5g6dSovvvhil2OWLFmC3W5n5cqV3HfffSxatAiAPXv2cNJJJ7F8+fLAf8I4CnoDQ3O11enx/zywOngIBiFWw0C2xPQxkaATPWENpNfrDfuh3bt3x3Uxj8fDV199xUUXXQTAwoULWbVqVZfj1qxZw2WXXQbAKaecQkNDAxUVFWzbto36+nquvvpqrr76atavXx/XOASCWJk6NicQUk2xmhg/on9JDAqGHgEP0hWbByn2H6MnrIH81re+Ffj3I488EvTevffeG9fFGhoaSEtLw2TSI7v5+flUVVV1Oa66upr8/PzAz/n5+VRWViJJEt/85jd5++23eeihh7jzzjupr6+PaywCQSyk2sxMHKkbxWnjclFEDz1BHxNviFVksEZP2D1ITdMC/964cWPY98KxcuVKnnjiiaDXRo8e3eW4aAuUZVnm2muvDfx84oknMn36dDZu3Mj5558f1TkAcnMTuyny8/tP78f+wlCZk//45kR+9+5mrjx3QsTvPFTmJBbEnAST6Hw05+VRC1hxx3SuliofTsCWmdXvfif9bTxhDWR3hisaozZv3jzmzZsX9JqRZOPz+VAUhZqaGgoKCrp8tqCggJqaGkpKSgACxy1btoxZs2YxapQui6VpGmZzbJvMdXWtqGpkAx+K/Px0ampii/cPdobSnIwrTOO5n+idbLr7zkNpTqJFzEkwyZgPj0d/fLc3NcR0Lnd1jf5/bP3qd9IX94gsS906TWHjRNF4ibFiNpuZPXs2K1asAGDZsmWcc845XY6bO3cuy5cvB2DDhg1YrVaGDx/Onj17+OMf/wjAwYMH2bVrFyeffHLSxykQCAT9HRFi7XnCGsie0mZ88MEHeeedd7jkkkvYsGEDd9xxBwBLly7l+eefB+DGG2/E7XYzf/58HnvsMZ5++mkAbrvtNurr67n00kv56U9/ylNPPUVamvglCwSCoYckslh7nLAh1j179jBr1iwA2tvbA//WNA232x33BYuLi1myZEmX16+77rrAv61WK0899VSXY9LS0njhhRfivrZAIBAMFiSbXj+uudpi+lxHFmtqhCMFYQ3k6tWre3McAoFAIIiBzmUemqYiSdFlVhsGVbL1r4SY/khYA1lcXMzevXs5fPgwM2bMoLCwsDfHJRAIBIJukGQTmO26Mo7bCVF6hCLEGj1hlxz/93//x7e//W3+53/+h8suu4zPP/+8N8clEAgEggjEk6hj7FkKoYDIhPUglyxZwgcffEBhYSGbNm3iueee46yzzurNsQkEAoGgGyRbmt7Rw9UKRBflE1ms0dNt0NoIq5500kk0NDT0yoAEAoFAEB1Gok20HqTmdYPXDZICZtGuLRJRl3kIUXCBQCDoX8QaYu1I0EntsVK+wUTUgpJiMgUCgaB/YWSiRm0gAwk6IoM1GqKqg4SOWkhN05AkqYs+q0AgEAh6l4BYQJQdPcT+Y2yIOkiBQCAYoATEAmL1IEUGa1R0WwcJsH37dr7++mskSWLWrFlMnTq11wYnEAgEgvDE7EEaJR42oaITDWENpKqq3H333axbt46TTz4Zt9vNyy+/zKmnnsozzzyDLPrhCQQCQZ8Sc5KO8CBjIqyBfPXVV/H5fHzyySdYLBYA2trauPfee/njH//I9773vV4bpEAgEAi6En8WqzCQ0RDWDfzb3/7GI488EjCOAKmpqTzyyCO8//77vTI4gUAgEIQn9hCryGKNhbAGUlVV0tO7TmJmZmaP9IoUCAQCQWzEXObhEiHWWAhrIB0OB6qqdnldVVU8Hk+PDkogEAgEUWCygGICnwfN64p4eMCQihBrVIQ1kHPmzOG1117r8vorr7zCGWec0aODEggEAkFkJEnq1Dg5shfZEWIVWazREDZJ58477+S6665j27ZtzJ49G6/Xy7///W8OHjzI22+/3ZtjFAgEAkEYJFsamqNRN35pud0eK0KssRHWg8zJyeEvf/kLkydP5tNPP+XLL79kxowZvPvuu2RkZPTmGAUCgUAQhmg9SE1VweXwf0Z4kNEQ1oMESE9P5/vf/35vjUUgEAgEMRIo9YiQyaq52wANrKlIsmg+EQ3dGsjVq1fzyiuvsGfPHux2OxMnTuSWW27hnHPO6a3xCQQCgaAbot6DFCIBMRPWQC5btowXX3yR//qv/2Ly5MlIksTWrVt59NFH+dnPfsaFF17Ym+MUCAQCQQii9iBFgk7MhDWQS5Ys4dVXX2X48OGB18aNG8fMmTO57777hIEUCASCfkC0ajoiQSd2wibpeDyeIONoMGbMGFyuyPU2AoFAIOh5ojaQAQ9SGMhoCWsgFSX8Jq5Q0hEIBIL+QbRyc0KoPHZESw6BQCAYwMQcYhUeZNSE3YPcs2cPs2bN6vK6pmm43e4eHZRAIBAIoiPqOkgRYo2ZsAZy9erVvTkOgUAgEMRB7FmswkBGS1gDWVxc3JvjEAgEAkE8WFMACdxONFVFCtPMXmSxxo7YgxQIBIIBjCTJYLbpP3icYY8THmTsdKukMxTQNI2Ghhrc7nag++zc6mo5ZAuwoUznOVEUE2lpWdjtohBZIOhNJGsKmseJ5naE1VkVWayxM+QNZGtrE5IkUVg4Ql+JdYPJJOP1CgPZGWNONE3D43HT2FgDIIykQNCLSBY7GqC5HNC1zz2apoks1jgY8iFWp7OV9PSsiMZR0D2SJGGxWMnKyqe1tbGvhyMQDCkkSwoAmjtMiNXTDqoPTBYkk6UXRzawGfJWQVV9KMqQd6SThtlswefz9vUwBIKhhcWu/z+MgRQJOvEx5A0k6N6PIDmIuRQIep8OD9IR8n2tvU0/ToRXY0IYyH7GsWMVnHXWbJ5++rGg1/ft28NZZ81mxYoPojrPxo0b+MlPuu/l+dhjD0V9PoFA0H+R/B5kWAMp9h/jQhjIfkhmZib//vdafD5f4LV//nM1WVnZfTgqgUDQX4m0B6m1t+jHiRBrTIjNt36I3Z7ChAkT2bJlE7NmzQZg/fp1zJ59KgBffPEZ//M/L6FpKsOHF3P33feRk5PL+vXreOGFZ7FYLJSUjA6c7+jRMhYvfoLm5iasVht33nk3EydO7ouvJhAIeoKIIVbhQcZDrxvIiooK7r77burq6hgzZgyLFy8mNTW4JMDtdnP//fezfft2bDYbixcvZty4cWiaxosvvsjq1atxOp386Ec/4oorrkjq+H7z7ha2HqhL6jkNpo/L5Y5vzYjq2HPPvYBPPvkns2bNZteuHYwfP8Ffs1nPH/7wIi+99ArDhg3nzTf/zLPPPs0DDzzCY489yPPP/zejR4/hyScfCZzrscce5M4772HixMkcOnSQ++77GUuXvtcj31EgEPQ+klU3kAgDmVR6PcT68MMPc/3117Nq1SqmTp3Kiy++2OWYJUuWYLfbWblyJffddx+LFi0C4P333+fLL7/knXfe4fXXX+fpp5+mubm5t79Cr3DWWWezbt2XqKrKP/+5mvPOuwAAm83GCSdMYdgwvVfnZZct5Ouvv+Lgwf3k5uYzevQYAObNuxQAh8PBrl07efzxX3Hzzdfz8MO/wOl00tTU2CffSyAQJJ+OPUiRxZpMetWD9Hg8fPXVV/z+978HYOHChXz729/m7rvvDjpuzZo1/PSnPwXglFNOoaGhgYqKClauXMktt9yCxWIhPz+fN998E5vNltQxdufh9aZQQEpKKuPHT2Dr1s1s3PgVP/zhT/jnP/+Bqgar/Wia5t+rlNC0jrEZ/TxVVcVisfLqq28G3quuriIjI7NXvodAIOh5AgbSJbJYk0mvepANDQ2kpaVhMul2OT8/n6qqqi7HVVdXk5+fH/g5Pz+fyspKSktLOXDgANdccw1XXnklO3fuxGIZvEWv5513Pv/9379j0qQTA3PmcrWzc+c2jh2rAOD9999j1qyTGT9+Ag0NDezbtxeAjz76EIC0tDRGjBjJhx+uAOCrr9Zx223dZ7cKBIKBRcQkHeFBxkWPeZArV67kiSeeCHpt9OjRXY6Ltm5OlmV8Ph979uzh9ddfp7a2luuuu44TTzwx5HnDkZsbfINUV8uYTNGvE2I5Nh4URQ5cZ+7cb/Dkk4/ygx/8CJNJRpIk8vLyWLToF9x//914PB6KioZx//0PYLNZeOSRx3n00QdQFBOTJk1GkiRMJplf/eoxnnrqcd5888+YzWYee+xJzGYFSZKQZSnh73T852VZJj8/hN7VEGKof/9QiDkJJpnz4ZbycQCK2h7yvEfdrfiAnOFFWPvx76G/3SM9ZiDnzZvHvHnzgl7zeDzMmTMHn8+HoijU1NRQUFDQ5bMFBQXU1NRQUlICEDguLy+Piy++GLPZzLBhw5gxYwY7d+6MyUDW1bUGhSlVVY06bNobIdaCgiL+8pcP8HpVLBYb//zn5wB4vSr33fdg4LjTTz876HNer8q0aSfx2mtvdXl9xIgSfvvbl7u8bpwvke8Uak5UVaWmpiXucw508vPTh/T3D4WYk2CSPR9qm/436HW0hjyvp1lPPGx0W5D76e+hL+4RWZa6OE1B7/fiWDCbzcyePZsVK/Rw37JlyzjnnHO6HDd37lyWL18OwIYNG7BarQwfPpxzzz2XlStX+rM5G9i6dSsnnHBCb34FgUAg6HcYWayhQqya6tXrICUJyZbR20Mb0PR6FuuDDz7IO++8wyWXXMKGDRu44447AFi6dCnPP/88ADfeeCNut5v58+fz2GOP8fTTTwNw8803k5eXx6WXXsp1113Hj3/8Y8aMGdPbX0EgEAj6F4oFJAV8HjSfJ+gtzdEEgGTPDNtMWRCaXq+DLC4uZsmSJV1ev+666wL/tlqtPPXUU12OMZvN3HfffT06PoFAIBhoSJKkt7xytaK5nUh2c+A9zdGoH5OS1TeDG8CI5YRAIBAMBsKIBagBAylKu2JFGEiBQCAYBIQTC9DaGgGQU4SWc6wIAykQCASDgEAt5HFiAZrwIONGGEiBQCAYBITrCRkwkKnCg4wV0c2jn/HMM0+xbdsWvF4PR4+WMXr0WAC+9a1rmT//sqRdZ+PGDfzxj3/gd7/7Q9hjPv/8U44ePcK11347adcVCAQ9g5SWA4BafxTGnhJ4XfVnscoiSSdmhIHsZ9x1188BvXHy7bf/IEhDtbfZs2dXn11bIBDEhmnENDzbV+Mt3Yx19pWB1zVHAyCyWONBGMgBQLh+jo899hA2m52tWzfT2trCf/3XXXz44Qr279/L2Wd/g9tvv5MVKz7gX//6mObmZhoa6jnzzLP5yU/uDDr/pk1f84c/vIjL1U5LSws/+tF/MWbMWJYv11tiFRUN49xzz+fZZ5/i4MEDqKrKDTfcxAUXXNwX0yEQCEKgDJ8MJgtqXSlqWwOyP6RqJOlIqVl9N7gBijCQx+FY+Sy+sq09cm5l5HRS5v2/mD/XXT/H2toaXnttKStX/o0nnniYpUvfw2q1csUVl/Dd7/4nALt37+RPf3qT9PQMbr/9B3z66Sekp3coavzf/73NokW/pKRkNF9//RXPP7+YP//5bS6/fCEA8+dfxksv/ZZJk07gF794mLa2Vn74w1s48cSplJSMSsLMCASCRJFMFkzFU/CWbsJbuhnLiecKFZ0EEQayn9O5n6NB536Op512BgCFhUWMGTOO7Gx9HyIjI4OWFr1X5plnnkNOTi4A3/zmhXz99Vd84xvfDJzvl798hC+//IxPPvmIHTu24XR2lavasGE9Llc7f//7+wC0t7dz6NBBYSAFgn6EUjJTN5BH/Aay1R9etWUIFZ04EAbyOLrz8HqzH6RBpH6OZnOHYobRA/J4FKXj16xpatDPALfd9p/MmnUyJ510MieffAoPP/yLEOPw8ctfPsKkSZMBqK+vEz0lBYJ+hmnUDFyAr3wnmteFt3wHAHK+kOSMB7Gk6Ocko5/junVf0traisvlYvXqDwNeJ0BzcxNlZaXceusPOf30s1i/fh2qqi8CFEXxN2OGWbNOYdmyvwBQW1vLd75zHVVVlcn4igKBIEnIKVm6MfR58JXvxFu6GQBTycw+HddARXiQA4AHH3yUX/9a7+doMpn51a8ej7qPJkB2djZ33/1fNDY2ctFFlzBnzuls3LgBgIyMTC699ApuvPFqUlNTmTJlOu3t7TidTmbOnMVjjz1ETk4Ot9zynzzzzFPceOPVqKrKj3/8XxQXj+iprywQCOLEVDITd80hPAe/wle+U39t1Iw+HtXARNI0TYt82ODh+H6QlZWlFBWVRPXZvgixJsqKFR+wadPX3H//Qz1y/lBzEsucDkZE78OuiDkJpifnw1dbiuO9jt6xct5oUhc+1CPXSiZDvh+kQCAQCHoWOXcUUnp+4GfTmNl9OJqBjQixDnIuuWQBl1yyoK+HIRAIeglJkkiZfze+yr1gsor9xwQQBlIgEAgGGXJGAXJGQV8PY8AjQqzAENuG7VE0TQWiTyASCASC/sqQN5Amk4W2tmZhJBNE0zS8Xg+NjbVYLLa+Ho5AIBAkzJAPsWZn59PQUENra2PEY2VZDtQICnQ6z4ksK9jtaaSlCQEBgUAw8BnyBlJRTOTlDYvqWJGq3hUxJwKBYLAy5EOsAoFAIBCEQhhIgUAgEAhCMORCrLKcWIZlop8fjIg56YqYk66IOQlGzEdXentOIl1vyEnNCQQCgUAQDSLEKhAIBAJBCISBFAgEAoEgBMJACgQCgUAQAmEgBQKBQCAIgTCQAoFAIBCEQBhIgUAgEAhCIAykQCAQCAQhEAZSIBAIBIIQCAMpEAgEAkEIhIGMkg8++IBLLrmECy64gDfeeKOvh9Mn3HTTTcyfP5/LL7+cyy+/nC1btgzZeWltbeXSSy/l6NGjAHz55ZcsWLCACy+8kOeeey5w3K5du/iP//gPLrroIu6//368Xm9fDbnHOX5O7r33Xi688MLA/bJ69Wog/FwNJn73u98xf/585s+fz9NPPw2IeyTUnPT7e0QTRKSyslI799xztYaGBq2trU1bsGCBtm/fvr4eVq+iqqp25plnah6PJ/DaUJ2XzZs3a5deeqk2ZcoUraysTHM6ndrcuXO1I0eOaB6PR7vlllu0NWvWaJqmafPnz9c2bdqkaZqm3Xvvvdobb7zRhyPvOY6fE03TtEsvvVSrqqoKOq67uRosfPHFF9o111yjuVwuze12azfddJP2wQcfDOl7JNSc/OMf/+j394jwIKPgyy+/5LTTTiMrK4uUlBQuuugiVq1a1dfD6lUOHjyIJEn853/+J5dddhmvv/76kJ2Xd955hwcffJCCggIAtm7dSklJCSNHjsRkMrFgwQJWrVpFeXk57e3tzJw5E4CFCxcO2vk5fk4cDgcVFRX88pe/ZMGCBbzwwguoqhp2rgYT+fn5LFq0CIvFgtlsZty4cRw+fHhI3yOh5qSioqLf3yNDrptHPFRXV5Ofnx/4uaCggK1bt/bhiHqf5uZmTj/9dB566CHa29u56aabmDdv3pCcl8ceeyzo51D3R1VVVZfX8/Pzqaqq6rVx9ibHz0ldXR2nnXYav/rVr0hJSeEHP/gBf/nLX0hJSQk5V4OJCRMmBP59+PBhVqxYwY033jik75FQc/Lmm2+yfv36fn2PCA8yCrQQDU8kaWi1qjnppJN4+umnSUlJIScnh6uuuooXXnihy3FDbV4g/P0xlO+bkSNH8vvf/57c3Fzsdjs33ngj//rXv4bUnOzbt49bbrmFn//854waNarL+0PxHuk8J2PHju3394gwkFFQWFhIbW1t4Ofq6upAKGmosGHDBtauXRv4WdM0iouLh/y8QPj74/jXa2pqhsz87Nmzhw8//DDws6ZpmEymIfO39PXXX3PzzTdz1113ceWVV4p7hK5zMhDuEWEgo+CMM85g7dq11NfX43Q6+cc//sE555zT18PqVVpaWnj66adxuVy0trby17/+lV//+tdDfl4AZsyYwaFDhygtLcXn8/G3v/2Nc845h+LiYqxWK19//TUAy5YtGzLzo2kajz/+OE1NTXg8Ht5++20uuOCCsHM1mDh27Bi33XYbixcvZv78+YC4R0LNyUC4R8QeZBQUFhZy5513ctNNN+HxeLjqqquYPn16Xw+rVzn33HPZsmULV1xxBaqqcv3113PyyScP+XkBsFqtPPnkk9x+++24XC7mzp3LxRdfDMDixYv5xS9+QVtbGyeeeCI33XRTH4+2d5g8eTLf//73ue666/B6vVx44YVceumlAGHnarDwyiuv4HK5ePLJJwOvXXvttUP6Hgk3J/39HpG0UAFfgUAgEAiGOCLEKhAIBAJBCISBFAgEAoEgBMJACgQCgUAQAmEgBQKBQCAIgTCQAoFAIBCEQBhIgSBOPB4PZ511FrfeemvgtX//+9+BVPXnn3+eZcuW9dHouqelpSVsOcFvf/tbTjvtNC6//HKuuOIKFixYwM0338yhQ4fiulZVVRXXXnstAGVlZdx+++1dXhcI+iOiDlIgiJPVq1czadIkduzYwYEDBxg3blzQ+z/96U/7aGSRaWpqYtu2bWHfv+SSS3jggQcCPy9ZsoS77rqL9957L+ZrFRYW8tZbbwFQUVERMLSdXxcI+iPCQAoEcbJ06VIuueQSSkpKeO211/jVr34V9P6iRYuYMGECt956K//6179YvHgxsixzwgkn8OWXXwbEmlevXo0sy5SWlmI2m3nqqaeYOHEiN954I1OmTGHdunXU1dVx0003UVdXx/r163E6nfzmN79h0qRJtLS08Nhjj7F37148Hg+nn34699xzDyaTiWnTpvH973+fL774gurqam666SZuvvlm7r33Xtrb27n88st57733UBSl2+96+umn8+yzzwJQWVnJQw89RHl5OZqmccUVV/C9730Pr9fLI488wsaNGzGbzYwYMYInnniChoYGFixYwIYNG/jFL35BVVUVt956Kw8//DALFixg06ZNeDwennzySdauXYuiKEyfPp17772XtLQ0zjvvPK688krWrl3LsWPHmDdvHvfcc0+P/V4FAgMRYhUI4mD//v1s3ryZefPmccUVV7B8+XIaGhpCHtvQ0MA999zDr3/9a5YvX86cOXOCuhN89dVX/PKXv+Rvf/sbs2bN4pVXXgm8V15ezrJly/jd737H4sWLOfXUU3nvvfc4++yzef311wF4/PHHmTJlCu+99x7Lli2joaGBP/3pTwC43W6ys7N56623eOGFF3jmmWdwuVw88cQT2Gw2li9fHtE4er1e/vKXvzBnzhwAfvaznzFnzhw++OADli5dyvvvv8/f//53Nm/ezPr163n//fd57733GDlyJHv27AmcR1EUHn30UUaNGhX0HQFeeuklqqurWb58OcuXL0dV1UBTXdDbZ7355pu89dZbvP7665SVlUXzaxIIEkIYSIEgDpYuXco3vvENsrKymD59OiNGjODtt98OeeyGDRsYN24ckydPBuDKK68kLS0t8P6UKVMoKioC4MQTT6SpqSnw3gUXXADo3TEAzj77bABGjRoVOG7NmjW8/fbbXH755SxcuJCtW7eyd+/ewDm++c1vBq7jdrtxOBwRv9+KFSsCXd4vu+wy6urqeOSRR3A4HGzcuJEbbrgBgPT0dBYuXMinn37KxIkTURSFb33rW/zmN7/hoosuYtasWVHMJnz66adce+21mM1mZFnmxhtv5LPPPuvyHQoLC8nNzQ2aI4GgpxAhVoEgRhwOB8uWLcNqtXLeeecB0NrayhtvvMG0adO6HK8oSpcWPrLcsTa12WyBfx/fAslisQR9zmw2dzm/qqo8//zzgT3Q5ubmoPZAVqs1cG4I3Z7reI7fgzRobW3t8nlVVfF6vWRkZLB8+XI2btzIunXruOOOO7jppps4//zzI15PVdUuP3s8ni7fwfgeQiFT0BsID1IgiJEPPviA7OxsPvvsMz7++GM+/vhjPvroIxwOB3V1dV2OnzVrFocPH2b37t0AfPjhh12MWCKcddZZvPrqq2iahtvt5kc/+lEg/BoOk8mEz+eL2dCkpaUxY8YM3njjDUDPhl22bBlnnHEGn3zyCTfffDMnnXQSt99+O1dccUXgOxsoihJk+AzOPvts3nrrLTweD6qq8sYbb3DmmWfGNDaBINkIAykQxMjSpUv57ne/G7R3l5GRwY033shrr73W5fisrCyeffZZfv7zn3PllVfy+eefYzKZsNvtSRnP/fffj8PhYMGCBSxYsICJEyfyve99r9vP5Ofnc+KJJzJv3rywe6fhWLx4MWvXrmXBggVcddVVXHjhhSxcuJBzzjmH8ePHc+mll7Jw4UI2bdrET37yk6DPTpgwAUVRuOqqq4KM849+9CPy8vK44oormDdvHl6vl/vvvz+mcQkEyUZ08xAIepjW1lZefPFFbr/9dux2Ozt27OAHP/gBn3322aDuHi8QDHTEHqRA0MOkpaVhNpu56qqrMJlMmEwmfvOb3wjjKBD0c4QHKRAIBAJBCMQepEAgEAgEIRAGUiAQCASCEAgDKRAIBAJBCISBFAgEAoEgBMJACgQCgUAQAmEgBQKBQCAIwf8HWyhtMWTZd1IAAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 504x288 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "hg.dope_plot(env,\"1t70A\",\"NSP4\",save=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Comments\n",
    "The DOPE profile of the model follows that of the template quite well except in a small region at the C-terminus. This is despite the low sequence similarity of the template with the target sequence. Retrospectively, the sequence similarity is low since we are only modelling a small region of the NSP4 protein."
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.1"
  },
  "widgets": {
   "application/vnd.jupyter.widget-state+json": {
    "state": {},
    "version_major": 2,
    "version_minor": 0
   }
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
